<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Philipp Bayer" />

<meta name="date" content="2021-01-10" />

<title>GO enrichment</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Amphibolis Posidonia Comparison</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">GO enrichment</h1>
<h4 class="author">Philipp Bayer</h4>
<h4 class="date">01/10/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-20
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Amphibolis_Posidonia_Comparison/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210414code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210414)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210414code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210414)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree2691cad4eb460fddb2fe90aace812b24d4b9dc24targetblank2691cada"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/2691cad4eb460fddb2fe90aace812b24d4b9dc24" target="_blank">2691cad</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree2691cad4eb460fddb2fe90aace812b24d4b9dc24targetblank2691cada" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/2691cad4eb460fddb2fe90aace812b24d4b9dc24" target="_blank">2691cad</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/OTT.nb.html
    Ignored:    analysis/plotGenes.nb.html
    Ignored:    analysis/plotRgenes.nb.html

Untracked files:
    Untracked:  data/Chloroplast_assemblies/
    Untracked:  output/lost_in_amphi_GO.txt.png
    Untracked:  output/lost_in_posi_GO.txt.png
    Untracked:  output/lost_in_zmar_GO.txt.png
    Untracked:  output/lost_in_zmuel_GO.txt.png
    Untracked:  output/missing_amphi_vs_all_GO.txt.png
    Untracked:  output/missing_aquatics_GO.txt.png
    Untracked:  output/missing_arabidopsis_vs_all_GO.txt.png
    Untracked:  output/missing_posi_vs_all_GO.txt.png
    Untracked:  output/missing_seagrasses_GO.txt.png
    Untracked:  output/missing_zmar_vs_all_GO.txt.png
    Untracked:  output/missing_zmuel_vs_all_GO.txt.png
    Untracked:  output/only_in_posi_GO.txt.png
    Untracked:  output/only_in_zmar_GO.txt.png
    Untracked:  output/only_in_zmuel_GO.txt.png
    Untracked:  output/only_seagrasses_GO.txt.png

Unstaged changes:
    Modified:   analysis/Orthofinder.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/GOenrichment.Rmd</code>) and HTML (<code>docs/GOenrichment.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/2691cad4eb460fddb2fe90aace812b24d4b9dc24/analysis/GOenrichment.Rmd" target="_blank">2691cad</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-12-20
</td>
<td>
add tables
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/292db59bce9791ed43dcb422518e0aedecd602de/docs/GOenrichment.html" target="_blank">292db59</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/35aa72f17934fa027bc006e34a1da721a1fdfef1/analysis/GOenrichment.Rmd" target="_blank">35aa72f</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-18
</td>
<td>
wflow_publish(c(“analysis/GOenrichment.Rmd”, “analysis/plotGenes.Rmd”,
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/7e370e9730c8ff0d2f639461b7c6195b588b6624/analysis/GOenrichment.Rmd" target="_blank">7e370e9</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Updated GOenrichment
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/95586f9a994c41619ed3eb0c6f948debc4926f94/analysis/GOenrichment.Rmd" target="_blank">95586f9</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Add missing data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/95586f9a994c41619ed3eb0c6f948debc4926f94/docs/GOenrichment.html" target="_blank">95586f9</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Add missing data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/c0db4a55cee717b3c1b0b2b02c5e9d1bfddac065/docs/GOenrichment.html" target="_blank">c0db4a5</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/08b28eacde8985a9d65405b6bc555c4c9d934f92/analysis/GOenrichment.Rmd" target="_blank">08b28ea</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
wflow_publish(files = c("analysis/*"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="lost-in-seagrasses" class="section level1">
<h1>Lost in seagrasses</h1>
<p>The GO enrichment does not work well on my laptop so I’m setting this to eval=FALSE and run it on a remote server. The script which writes the input files is a Python script in the scripts/ folder: findClustersUniqueToAquatic.py. This script parses Orthofinder output to pull out genome-specific groups and genes.</p>
<pre class="r"><code># give properly formatted background in format: GO:0005838  GSBRNA2T00088508001;GSBRNA2T00088313001;GSBRNA2T00035842001 
#annAT &lt;- readMappings(&#39;BACKGROUND.txt.gz&#39;, sep=&quot;\t&quot;, IDsep=&quot;;&quot;)
#save(annAT, file=&#39;annAtObject.RData&#39;)

load(&#39;annAtObject.RData&#39;)
allgenes &lt;- unique(unlist(annAT))

compare &lt;- function(genelistfile, outname, allgenes, annAT) {
  # give file with your genes of interest, one gene_id per line
  
  mygenes &lt;-scan(genelistfile ,what=&quot;&quot;)
  geneList &lt;- factor(as.integer(allgenes %in% mygenes))
  names(geneList) &lt;- allgenes
  
  GOdata &lt;-new (&quot;topGOdata&quot;, ontology = &#39;BP&#39;, allGenes = geneList, nodeSize = 5, annot=annFUN.GO2genes, GO2genes=annAT)
  # using ClassicCount:
  #test.stat &lt;-new (&quot;classicCount&quot;, testStatistic = GOFisherTest, name = &quot;Fisher Test&quot;)
  #resultsFisherC &lt;-getSigGroups (GOdata, test.stat)
  # using weight01:
  weight01.fisher &lt;- runTest(GOdata, statistic = &quot;fisher&quot;)
  # using ClassicCount:
  # allRes &lt;- GenTable(GOdata, classicFisher= resultsFisherC, topNodes = 30)
  # using weight01:
  allRes &lt;- GenTable(GOdata, classicFisher=weight01.fisher,topNodes=30)#,topNodes=100)
  names(allRes)[length(allRes)] &lt;- &quot;p.value&quot;
  p_values &lt;- score(weight01.fisher)
  adjusted_p &lt;- p.adjust(p_values)
  adjusted_p[adjusted_p &lt; 0.05] %&gt;% enframe() %&gt;% write_csv(&#39;data/&#39; + outname)

}</code></pre>
</div>
<div id="lost-in-seagrasses-each-one-uniquely-vs-all-terrestrials" class="section level1">
<h1>lost in seagrasses, each one uniquely vs all terrestrials</h1>
<pre class="r"><code>compare(&#39;lost_in_amphi_vs_all.txt&#39;, &#39;missing_amphi_vs_all_GO.txt&#39;, allgenes, annAT)
compare(&#39;lost_in_posi_vs_all.txt&#39;, &#39;missing_posi_vs_all_GO.txt&#39;, allgenes, annAT)
compare(&#39;lost_in_zmar_vs_all.txt&#39;, &#39;missing_zmar_vs_all_GO.txt&#39;, allgenes, annAT)
compare(&#39;lost_in_zmuel_vs_all.txt&#39;, &#39;missing_zmuel_vs_all_GO.txt&#39;, allgenes, annAT)</code></pre>
</div>
<div id="lost-in-seagrasses-vs-all-terrestrials" class="section level1">
<h1>lost in seagrasses vs all terrestrials</h1>
<p>Here we compare GO terms for seagrasses and aquatics (seagrasses+duckweeds) vs all terrestrials</p>
<pre class="r"><code>compare(&#39;lost_in_seagrasses.txt&#39;, &#39;missing_seagrasses_GO.txt&#39;, allgenes, annAT)
compare(&#39;lost_in_aquatics.txt&#39;, &#39;missing_aquatics_GO.txt&#39;, allgenes, annAT)
compare(&#39;only_in_seagrasses.txt&#39;, &#39;only_seagrasses_GO.txt&#39;, allgenes, annAT)</code></pre>
</div>
<div id="seagrass-only-comparisons" class="section level1">
<h1>Seagrass only comparisons</h1>
<p>Now we compare seagrasses within each other.</p>
<p>For the seagrass-only comparisons, I’m using a Seagrass-only background as that makes more biological sense to me.s</p>
<pre class="r"><code># give properly formatted background in format: GO:0005838  GSBRNA2T00088508001;GSBRNA2T00088313001;GSBRNA2T00035842001 
#sannAT &lt;- readMappings(&#39;SEAGRASSBACKGROUND.txt&#39;, sep=&quot;\t&quot;, IDsep=&quot;;&quot;)
#save(sannAT, file=&#39;sannAtObject.RData&#39;)

load(&#39;annAtObject.RData&#39;)
sallgenes &lt;- unique(unlist(sannAT))

compare(&#39;lost_in_amphi.txt&#39;, &#39;lost_in_amphi_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;lost_in_posi.txt&#39;, &#39;lost_in_posi_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;lost_in_zmar.txt&#39;, &#39;lost_in_zmar_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;lost_in_zmuel.txt&#39;, &#39;lost_in_zmuel_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)</code></pre>
</div>
<div id="present-in-seagrasses" class="section level1">
<h1>Present in seagrasses</h1>
<p>The opposite - which GO-terms are present only in one of the four species?</p>
<pre class="r"><code>compare(&#39;only_in_amphi.txt&#39;, &#39;only_in_amphi_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;only_in_posi.txt&#39;, &#39;only_in_posi_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;only_in_zmar.txt&#39;, &#39;only_in_zmar_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;only_in_zmuel.txt&#39;, &#39;only_in_zmuel_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)</code></pre>
<p>Alright now we have all these different GO terms in all these files - we can send them to revigo for visualiation and some deduplication!</p>
</div>
<div id="revigo" class="section level1">
<h1>Revigo</h1>
<p>This code is based on <a href="http://revigo.irb.hr/CodeExamples/revigo.R.txt" class="uri">http://revigo.irb.hr/CodeExamples/revigo.R.txt</a></p>
<pre class="r"><code>results_list &lt;- list()

for (f in list.files(&#39;./data/&#39;, pattern=&#39;GO.txt&#39;)){
  filename &lt;- paste(&#39;./data/&#39;, f, sep=&#39;&#39;)
  go_and_pvalues &lt;- readChar(filename, file.info(filename)$size)
  go_and_pvalues &lt;- gsub(&#39;,&#39;, &#39; &#39;, go_and_pvalues)
  
  httr::POST(
    url = &quot;http://revigo.irb.hr/StartJob.aspx&quot;,
    body = list(
      cutoff = &quot;0.7&quot;,
      valueType = &quot;pvalue&quot;,
     # speciesTaxon = &quot;4577&quot;, # zea mays
     #speciesTaxon = &#39;39947&#39;, # japonica
     speciesTaxon = &#39;3702&#39;, # arabidopsis
      measure = &quot;SIMREL&quot;,
      goList = go_and_pvalues
    ),
    # application/x-www-form-urlencoded
    encode = &quot;form&quot;
  ) -&gt; res
  
  dat &lt;- httr::content(res, encoding = &quot;UTF-8&quot;)
  
  jobid &lt;- jsonlite::fromJSON(dat,bigint_as_char=TRUE)$jobid
  
  # Check job status
  running &lt;- &quot;1&quot;
  while (running != &quot;0&quot; ) {
      httr::POST(
        url = &quot;http://revigo.irb.hr/QueryJobStatus.aspx&quot;,
        query = list( jobid = jobid )
      ) -&gt; res2
      dat2 &lt;- httr::content(res2, encoding = &quot;UTF-8&quot;)
      running &lt;- jsonlite::fromJSON(dat2)$running
      Sys.sleep(1)
  }
  
  # Fetch results
  httr::POST(
    url = &quot;http://revigo.irb.hr/ExportJob.aspx&quot;,
    query = list(
      jobid = jobid, 
      namespace = &quot;1&quot;,
      type = &quot;CSVTable&quot;
    )
  ) -&gt; res3
  
  dat3 &lt;- httr::content(res3, encoding = &quot;UTF-8&quot;)
  
  dat3 &lt;- stri_replace_all_fixed(dat3, &quot;\r&quot;, &quot;&quot;)
  # Now we have a csv table in a string!

  # read_csv does not like the &#39;, &#39;, it wants &#39;,&#39;
  dat &lt;- read_csv(gsub(&#39;, &#39;, &#39;,&#39;, dat3), show_col_types = FALSE) 
  
  # do we even have results?
  if(nrow(dat) == 0){next}
  results_list[[f]] &lt;- dat
  
}</code></pre>
<p>OK we have a list with all results in a big list. Now we can plot!</p>
</div>
<div id="making-supplementary-tables-for-the-revigo-results" class="section level1">
<h1>Making Supplementary Tables for the REVIGO results</h1>
<p>Let’s also pull these terms out as tables</p>
<pre class="r"><code>for( i in names(results_list)) {
  print(i)
  knitr::kable(results_list[[i]] %&gt;% filter(Eliminated == FALSE), label=i)
}</code></pre>
<pre><code>[1] &quot;lost_in_amphi_GO.txt&quot;
[1] &quot;lost_in_posi_GO.txt&quot;
[1] &quot;lost_in_zmar_GO.txt&quot;
[1] &quot;lost_in_zmuel_GO.txt&quot;
[1] &quot;missing_amphi_vs_all_GO.txt&quot;
[1] &quot;missing_aquatics_GO.txt&quot;
[1] &quot;missing_arabidopsis_vs_all_GO.txt&quot;
[1] &quot;missing_posi_vs_all_GO.txt&quot;
[1] &quot;missing_seagrasses_GO.txt&quot;
[1] &quot;missing_zmar_vs_all_GO.txt&quot;
[1] &quot;missing_zmuel_vs_all_GO.txt&quot;
[1] &quot;only_in_posi_GO.txt&quot;
[1] &quot;only_in_zmar_GO.txt&quot;
[1] &quot;only_in_zmuel_GO.txt&quot;
[1] &quot;only_seagrasses_GO.txt&quot;</code></pre>
</div>
<div id="venn-diagram-go-overlaps" class="section level1">
<h1>Venn diagram GO overlaps</h1>
<p>Let’s also check which GO terms overlap between the 4 seagrasses!</p>
<p>Let’s have a look at all of these plots. Manually zooming in leads to ggrepel reloading labels, so on the small scale a lot of these plots don’t have labels.</p>
<pre class="r"><code>for(i in names(plot_list)) {
  file_name = paste(&#39;output/&#39;,  i, &#39;.png&#39;, sep=&#39;&#39;)
  
  cowplot::save_plot(file_name, plot_list[[i]], base_height=8)
  
}</code></pre>
<p>How many shared lost GO-terms are there? Hopefully, all four species will have lost the most GO-terms.</p>
<pre class="r"><code>a &lt;- list(`P. australis` = results_list$missing_posi_vs_all_GO.txt$Name,
          `A. antarctica` = results_list$missing_amphi_vs_all_GO.txt$Name,
          `Z. marina` = results_list$missing_zmar_vs_all_GO.txt$Name,
          `Z. muelleri` = results_list$missing_zmuel_vs_all_GO.txt$Name,
          `A. thaliana` = results_list$missing_arabidopsis_vs_all_GO.txt$Name)

a_go_ids &lt;- list(`P. australis` = results_list$missing_posi_vs_all_GO.txt$TermID,
          `A. antarctica` = results_list$missing_amphi_vs_all_GO.txt$TermID,
          `Z. marina` = results_list$missing_zmar_vs_all_GO.txt$TermID,
          `Z. muelleri` = results_list$missing_zmuel_vs_all_GO.txt$TermID,
          `A. thaliana` = results_list$missing_arabidopsis_vs_all_GO.txt$TermID)</code></pre>
<pre class="r"><code>plot(euler(a), 
     quantities = TRUE,
     fill = rev(wes_palette(&quot;Zissou1&quot;, 15, type = &#39;continuous&#39;)),
    alpha = 0.5,
     labels = list(font = 4))</code></pre>
<p><img src="figure/GOenrichment.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>upset(fromList(a), order.by=&#39;freq&#39;, )</code></pre>
<p><img src="figure/GOenrichment.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>a_no_ara &lt;- list(`P. australis` = results_list$missing_posi_vs_all_GO.txt$Name,
          `A. antarctica` = results_list$missing_amphi_vs_all_GO.txt$Name,
          `Z. marina` = results_list$missing_zmar_vs_all_GO.txt$Name,
          `Z. muelleri` = results_list$missing_zmuel_vs_all_GO.txt$Name)</code></pre>
<p>What are the shared GO-terms in seagrasses, WITHOUT the Ara loss??</p>
<pre class="r"><code>setdiff(Reduce(intersect, a_no_ara), Reduce(intersect, a)) %&gt;% enframe() %&gt;% writexl::write_xlsx(&#39;./data/shared_lost_genes.xlsx&#39;)</code></pre>
<p>What if we remove Posidonia?</p>
<pre class="r"><code>b &lt;- list(`A. antarctica` = results_list$missing_amphi_vs_all_GO.txt$Name,
          `Z. marina` = results_list$missing_zmar_vs_all_GO.txt$Name,
          `Z. muelleri` = results_list$missing_zmuel_vs_all_GO.txt$Name)
intersections &lt;- Reduce(intersect, b)
intersections[grepl(&#39;ethylene&#39;, intersections)]</code></pre>
<pre><code>[1] &quot;jasmonic acid and ethylene-dependent systemic resistance,ethylene mediated signaling pathway&quot;
[2] &quot;regulation of ethylene-activated signaling pathway&quot;                                          </code></pre>
<p>OK we need a big list of all GO-terms here - which GO-term is lost in which species. That will be a supplementary table.</p>
<pre class="r"><code>all_species &lt;- c(&quot;P. australis&quot;,&quot;A. antarctica&quot;,&quot;Z. marina&quot;,&quot;Z. muelleri&quot;, &#39;A. thaliana&#39;)  
all_go_terms &lt;- Reduce(union, a)
all_go_ids &lt;- Reduce(union, a_go_ids)
results_d &lt;- data.frame(&#39;GOID&#39; = character(),
                        &#39;GO&#39; = character(),
                        &#39;P. australis&#39; = character(),
                        &#39;A. antarctica&#39; = character(),
                        &#39;Z. marina&#39; = character(),
                        &#39;Z. muelleri&#39; = character(),
                        &#39;A. thaliana&#39; = character())


for (index in seq_along(all_go_terms)){
  go &lt;- all_go_terms[index]
  go_id &lt;- all_go_ids[index]
  specs &lt;- c()
  for (species in names(a)) {
    if ( length(a[[species]][grep(paste(&#39;^&#39;, go, &#39;$&#39;, sep=&#39;&#39;), a[[species]])]) &gt; 0 ) {
      specs &lt;- c(specs, species)
    }
  }
  
  results_d[index,] &lt;- c(go_id, go, gsub(&#39;FALSE&#39;, &#39;Present&#39;, gsub(&#39;TRUE&#39;, &#39;Lost&#39;, all_species %in% specs)))
}</code></pre>
<pre class="r"><code>writexl::write_xlsx(results_d, &#39;data/Lost_GO_terms_in_five_species.xlsx&#39;)</code></pre>
</div>
<div id="after-filtering-for-plant-specific-go-terms" class="section level1">
<h1>after filtering for plant-specific GO-terms</h1>
<p>We will use the GO-terms that are plant-specific as identified by the GOMAP paper. See <a href="https://github.com/wkpalan/GOMAP-maize-analysis/blob/main/6.plantSpecific/1.getSppSpecific.R" class="uri">https://github.com/wkpalan/GOMAP-maize-analysis/blob/main/6.plantSpecific/1.getSppSpecific.R</a> or <a href="https://plantmethods.biomedcentral.com/articles/10.1186/s13007-021-00754-1" class="uri">https://plantmethods.biomedcentral.com/articles/10.1186/s13007-021-00754-1</a></p>
<pre class="r"><code>go_plant &lt;- read_tsv(&#39;https://raw.githubusercontent.com/wkpalan/GOMAP-maize-analysis/main/data/go/speciesSpecificGOTerms.txt&#39;)</code></pre>
<pre><code>Rows: 45031 Columns: 5</code></pre>
<pre><code>-- Column specification --------------------------------------------------------
Delimiter: &quot;\t&quot;
chr (1): GOterm
dbl (4): NCBITaxon:10090, NCBITaxon:33090, NCBITaxon:3702, NCBITaxon:40674</code></pre>
<pre><code>
i Use `spec()` to retrieve the full column specification for this data.
i Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
<pre class="r"><code># taxon 33090 is Viridiplantae
plantSpecificGO &lt;- go_plant %&gt;% dplyr::filter(`NCBITaxon:33090`==1) %&gt;% pull(GOterm)
plantSpecificGO &lt;- c(plantSpecificGO,c(&quot;GO:0005575&quot;,&quot;GO:0008150&quot;,&quot;GO:0003674&quot;))</code></pre>
<pre class="r"><code>results_d %&gt;% filter(GOID %in% plantSpecificGO) %&gt;% writexl::write_xlsx(&#39;data/Lost_GO_terms_in_five_species.PlantSpecific.xlsx&#39;)</code></pre>
<p>Now let’s redo the Venn diagram with those filtered GO-terms</p>
<pre class="r"><code>filters &lt;- lapply(a_go_ids, function(ch) ch %in% plantSpecificGO)

newa &lt;- list()
for (species in names(filters)) {
  before &lt;- a[[species]]
  after &lt;- before[filters[[species]]]
  newa[[species]] &lt;- after
}</code></pre>
<pre class="r"><code>plot(euler(newa), 
     quantities = TRUE,
     fill = rev(wes_palette(&quot;Zissou1&quot;, 15, type = &#39;continuous&#39;)),
    alpha = 0.5,
     labels = list(font = 4))</code></pre>
<p><img src="figure/GOenrichment.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Not much difference?</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] rrvgo_1.4.4          UpSetR_1.4.0         eulerr_6.1.1        
 [4] ggrepel_0.9.1        stringi_1.7.5        httr_1.4.2          
 [7] wesanderson_0.3.6    rvest_1.0.2          forcats_0.5.1       
[10] stringr_1.4.0        dplyr_1.0.7          purrr_0.3.4         
[13] readr_2.0.2          tidyr_1.1.4          tibble_3.1.5        
[16] ggplot2_3.3.5        tidyverse_1.3.1      topGO_2.44.0        
[19] SparseM_1.81         GO.db_3.13.0         AnnotationDbi_1.54.1
[22] IRanges_2.26.0       S4Vectors_0.30.2     Biobase_2.52.0      
[25] graph_1.70.0         BiocGenerics_0.38.0  workflowr_1.6.2     

loaded via a namespace (and not attached):
  [1] colorspace_2.0-2       ellipsis_0.3.2         rprojroot_2.0.2       
  [4] XVector_0.32.0         fs_1.5.0               rstudioapi_0.13       
  [7] farver_2.1.0           bit64_4.0.5            fansi_0.5.0           
 [10] lubridate_1.8.0        xml2_1.3.2             cachem_1.0.6          
 [13] GOSemSim_2.18.1        knitr_1.36             polyclip_1.10-0       
 [16] jsonlite_1.7.2         broom_0.7.9            gridBase_0.4-7        
 [19] dbplyr_2.1.1           png_0.1-7              pheatmap_1.0.12       
 [22] shiny_1.7.1            compiler_4.1.0         backports_1.2.1       
 [25] assertthat_0.2.1       fastmap_1.1.0          cli_3.0.1             
 [28] later_1.3.0            htmltools_0.5.2        tools_4.1.0           
 [31] igraph_1.2.7           NLP_0.2-1              gtable_0.3.0          
 [34] glue_1.4.2             GenomeInfoDbData_1.2.6 Rcpp_1.0.7            
 [37] slam_0.1-48            cellranger_1.1.0       jquerylib_0.1.4       
 [40] vctrs_0.3.8            Biostrings_2.60.2      writexl_1.4.0         
 [43] polylabelr_0.2.0       xfun_0.27              mime_0.12             
 [46] lifecycle_1.0.1        zlibbioc_1.38.0        scales_1.1.1          
 [49] treemap_2.4-3          vroom_1.5.5            hms_1.1.1             
 [52] promises_1.2.0.1       RColorBrewer_1.1-2     yaml_2.2.1            
 [55] curl_4.3.2             memoise_2.0.0          gridExtra_2.3         
 [58] sass_0.4.0             RSQLite_2.2.8          highr_0.9             
 [61] GenomeInfoDb_1.28.4    rlang_0.4.12           pkgconfig_2.0.3       
 [64] matrixStats_0.61.0     bitops_1.0-7           evaluate_0.14         
 [67] lattice_0.20-44        labeling_0.4.2         cowplot_1.1.1         
 [70] bit_4.0.4              tidyselect_1.1.1       plyr_1.8.6            
 [73] magrittr_2.0.1         R6_2.5.1               generics_0.1.1        
 [76] DBI_1.1.1              pillar_1.6.4           haven_2.4.3           
 [79] whisker_0.4            withr_2.4.2            KEGGREST_1.32.0       
 [82] RCurl_1.98-1.5         modelr_0.1.8           crayon_1.4.1          
 [85] wordcloud_2.6          utf8_1.2.2             tzdb_0.1.2            
 [88] rmarkdown_2.11         grid_4.1.0             readxl_1.3.1          
 [91] data.table_1.14.2      blob_1.2.2             git2r_0.28.0          
 [94] reprex_2.0.1           digest_0.6.28          xtable_1.8-4          
 [97] tm_0.7-8               httpuv_1.6.3           munsell_0.5.0         
[100] bslib_0.3.1           </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
