<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Philipp Bayer" />

<meta name="date" content="2021-01-10" />

<title>GO enrichment</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Amphibolis Posidonia Comparison</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">GO enrichment</h1>
<h4 class="author">Philipp Bayer</h4>
<h4 class="date">01/10/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-20
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Amphibolis_Posidonia_Comparison/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210414code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210414)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210414code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210414)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree73bffa46db8c16fe7ec5f8be2cb868e1822deab8targetblank73bffa4a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/73bffa46db8c16fe7ec5f8be2cb868e1822deab8" target="_blank">73bffa4</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree73bffa46db8c16fe7ec5f8be2cb868e1822deab8targetblank73bffa4a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/73bffa46db8c16fe7ec5f8be2cb868e1822deab8" target="_blank">73bffa4</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/OTT.nb.html
    Ignored:    analysis/plotGenes.nb.html
    Ignored:    analysis/plotRgenes.nb.html

Untracked files:
    Untracked:  data/Chloroplast_assemblies/
    Untracked:  output/lost_in_amphi_GO.txt.png
    Untracked:  output/lost_in_posi_GO.txt.png
    Untracked:  output/lost_in_zmar_GO.txt.png
    Untracked:  output/lost_in_zmuel_GO.txt.png
    Untracked:  output/missing_amphi_vs_all_GO.txt.png
    Untracked:  output/missing_aquatics_GO.txt.png
    Untracked:  output/missing_arabidopsis_vs_all_GO.txt.png
    Untracked:  output/missing_posi_vs_all_GO.txt.png
    Untracked:  output/missing_seagrasses_GO.txt.png
    Untracked:  output/missing_zmar_vs_all_GO.txt.png
    Untracked:  output/missing_zmuel_vs_all_GO.txt.png
    Untracked:  output/only_in_posi_GO.txt.png
    Untracked:  output/only_in_zmar_GO.txt.png
    Untracked:  output/only_in_zmuel_GO.txt.png
    Untracked:  output/only_seagrasses_GO.txt.png

Unstaged changes:
    Modified:   analysis/Orthofinder.Rmd
    Modified:   data/Lost_GO_terms_in_five_species.PlantSpecific.xlsx
    Modified:   data/Lost_GO_terms_in_five_species.xlsx
    Modified:   data/shared_lost_genes.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/GOenrichment.Rmd</code>) and HTML (<code>docs/GOenrichment.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/73bffa46db8c16fe7ec5f8be2cb868e1822deab8/analysis/GOenrichment.Rmd" target="_blank">73bffa4</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-12-20
</td>
<td>
add tables
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/c1288a408b91a20d358e23035d98f223fa704335/docs/GOenrichment.html" target="_blank">c1288a4</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-12-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/2691cad4eb460fddb2fe90aace812b24d4b9dc24/analysis/GOenrichment.Rmd" target="_blank">2691cad</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-12-20
</td>
<td>
add tables
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/292db59bce9791ed43dcb422518e0aedecd602de/docs/GOenrichment.html" target="_blank">292db59</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/35aa72f17934fa027bc006e34a1da721a1fdfef1/analysis/GOenrichment.Rmd" target="_blank">35aa72f</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-18
</td>
<td>
wflow_publish(c(“analysis/GOenrichment.Rmd”, “analysis/plotGenes.Rmd”,
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/7e370e9730c8ff0d2f639461b7c6195b588b6624/analysis/GOenrichment.Rmd" target="_blank">7e370e9</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Updated GOenrichment
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/95586f9a994c41619ed3eb0c6f948debc4926f94/analysis/GOenrichment.Rmd" target="_blank">95586f9</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Add missing data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/95586f9a994c41619ed3eb0c6f948debc4926f94/docs/GOenrichment.html" target="_blank">95586f9</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Add missing data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/c0db4a55cee717b3c1b0b2b02c5e9d1bfddac065/docs/GOenrichment.html" target="_blank">c0db4a5</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/08b28eacde8985a9d65405b6bc555c4c9d934f92/analysis/GOenrichment.Rmd" target="_blank">08b28ea</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
wflow_publish(files = c("analysis/*"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="lost-in-seagrasses" class="section level1">
<h1>Lost in seagrasses</h1>
<p>The GO enrichment does not work well on my laptop so I’m setting this to eval=FALSE and run it on a remote server. The script which writes the input files is a Python script in the scripts/ folder: findClustersUniqueToAquatic.py. This script parses Orthofinder output to pull out genome-specific groups and genes.</p>
<pre class="r"><code># give properly formatted background in format: GO:0005838  GSBRNA2T00088508001;GSBRNA2T00088313001;GSBRNA2T00035842001 
#annAT &lt;- readMappings(&#39;BACKGROUND.txt.gz&#39;, sep=&quot;\t&quot;, IDsep=&quot;;&quot;)
#save(annAT, file=&#39;annAtObject.RData&#39;)

load(&#39;annAtObject.RData&#39;)
allgenes &lt;- unique(unlist(annAT))

compare &lt;- function(genelistfile, outname, allgenes, annAT) {
  # give file with your genes of interest, one gene_id per line
  
  mygenes &lt;-scan(genelistfile ,what=&quot;&quot;)
  geneList &lt;- factor(as.integer(allgenes %in% mygenes))
  names(geneList) &lt;- allgenes
  
  GOdata &lt;-new (&quot;topGOdata&quot;, ontology = &#39;BP&#39;, allGenes = geneList, nodeSize = 5, annot=annFUN.GO2genes, GO2genes=annAT)
  # using ClassicCount:
  #test.stat &lt;-new (&quot;classicCount&quot;, testStatistic = GOFisherTest, name = &quot;Fisher Test&quot;)
  #resultsFisherC &lt;-getSigGroups (GOdata, test.stat)
  # using weight01:
  weight01.fisher &lt;- runTest(GOdata, statistic = &quot;fisher&quot;)
  # using ClassicCount:
  # allRes &lt;- GenTable(GOdata, classicFisher= resultsFisherC, topNodes = 30)
  # using weight01:
  allRes &lt;- GenTable(GOdata, classicFisher=weight01.fisher,topNodes=30)#,topNodes=100)
  names(allRes)[length(allRes)] &lt;- &quot;p.value&quot;
  p_values &lt;- score(weight01.fisher)
  adjusted_p &lt;- p.adjust(p_values)
  adjusted_p[adjusted_p &lt; 0.05] %&gt;% enframe() %&gt;% write_csv(&#39;data/&#39; + outname)

}</code></pre>
</div>
<div id="lost-in-seagrasses-each-one-uniquely-vs-all-terrestrials" class="section level1">
<h1>lost in seagrasses, each one uniquely vs all terrestrials</h1>
<pre class="r"><code>compare(&#39;lost_in_amphi_vs_all.txt&#39;, &#39;missing_amphi_vs_all_GO.txt&#39;, allgenes, annAT)
compare(&#39;lost_in_posi_vs_all.txt&#39;, &#39;missing_posi_vs_all_GO.txt&#39;, allgenes, annAT)
compare(&#39;lost_in_zmar_vs_all.txt&#39;, &#39;missing_zmar_vs_all_GO.txt&#39;, allgenes, annAT)
compare(&#39;lost_in_zmuel_vs_all.txt&#39;, &#39;missing_zmuel_vs_all_GO.txt&#39;, allgenes, annAT)</code></pre>
</div>
<div id="lost-in-seagrasses-vs-all-terrestrials" class="section level1">
<h1>lost in seagrasses vs all terrestrials</h1>
<p>Here we compare GO terms for seagrasses and aquatics (seagrasses+duckweeds) vs all terrestrials</p>
<pre class="r"><code>compare(&#39;lost_in_seagrasses.txt&#39;, &#39;missing_seagrasses_GO.txt&#39;, allgenes, annAT)
compare(&#39;lost_in_aquatics.txt&#39;, &#39;missing_aquatics_GO.txt&#39;, allgenes, annAT)
compare(&#39;only_in_seagrasses.txt&#39;, &#39;only_seagrasses_GO.txt&#39;, allgenes, annAT)</code></pre>
</div>
<div id="seagrass-only-comparisons" class="section level1">
<h1>Seagrass only comparisons</h1>
<p>Now we compare seagrasses within each other.</p>
<p>For the seagrass-only comparisons, I’m using a Seagrass-only background as that makes more biological sense to me.s</p>
<pre class="r"><code># give properly formatted background in format: GO:0005838  GSBRNA2T00088508001;GSBRNA2T00088313001;GSBRNA2T00035842001 
#sannAT &lt;- readMappings(&#39;SEAGRASSBACKGROUND.txt&#39;, sep=&quot;\t&quot;, IDsep=&quot;;&quot;)
#save(sannAT, file=&#39;sannAtObject.RData&#39;)

load(&#39;annAtObject.RData&#39;)
sallgenes &lt;- unique(unlist(sannAT))

compare(&#39;lost_in_amphi.txt&#39;, &#39;lost_in_amphi_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;lost_in_posi.txt&#39;, &#39;lost_in_posi_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;lost_in_zmar.txt&#39;, &#39;lost_in_zmar_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;lost_in_zmuel.txt&#39;, &#39;lost_in_zmuel_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)</code></pre>
</div>
<div id="present-in-seagrasses" class="section level1">
<h1>Present in seagrasses</h1>
<p>The opposite - which GO-terms are present only in one of the four species?</p>
<pre class="r"><code>compare(&#39;only_in_amphi.txt&#39;, &#39;only_in_amphi_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;only_in_posi.txt&#39;, &#39;only_in_posi_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;only_in_zmar.txt&#39;, &#39;only_in_zmar_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)
compare(&#39;only_in_zmuel.txt&#39;, &#39;only_in_zmuel_vs_seagrasses_GO.txt&#39;, sallgenes, sannAT)</code></pre>
<p>Alright now we have all these different GO terms in all these files - we can send them to revigo for visualiation and some deduplication!</p>
</div>
<div id="revigo" class="section level1">
<h1>Revigo</h1>
<p>This code is based on <a href="http://revigo.irb.hr/CodeExamples/revigo.R.txt" class="uri">http://revigo.irb.hr/CodeExamples/revigo.R.txt</a></p>
<pre class="r"><code>results_list &lt;- list()

for (f in list.files(&#39;./data/&#39;, pattern=&#39;GO.txt&#39;)){
  filename &lt;- paste(&#39;./data/&#39;, f, sep=&#39;&#39;)
  go_and_pvalues &lt;- readChar(filename, file.info(filename)$size)
  go_and_pvalues &lt;- gsub(&#39;,&#39;, &#39; &#39;, go_and_pvalues)
  
  httr::POST(
    url = &quot;http://revigo.irb.hr/StartJob.aspx&quot;,
    body = list(
      cutoff = &quot;0.7&quot;,
      valueType = &quot;pvalue&quot;,
     # speciesTaxon = &quot;4577&quot;, # zea mays
     #speciesTaxon = &#39;39947&#39;, # japonica
     speciesTaxon = &#39;3702&#39;, # arabidopsis
      measure = &quot;SIMREL&quot;,
      goList = go_and_pvalues
    ),
    # application/x-www-form-urlencoded
    encode = &quot;form&quot;
  ) -&gt; res
  
  dat &lt;- httr::content(res, encoding = &quot;UTF-8&quot;)
  
  jobid &lt;- jsonlite::fromJSON(dat,bigint_as_char=TRUE)$jobid
  
  # Check job status
  running &lt;- &quot;1&quot;
  while (running != &quot;0&quot; ) {
      httr::POST(
        url = &quot;http://revigo.irb.hr/QueryJobStatus.aspx&quot;,
        query = list( jobid = jobid )
      ) -&gt; res2
      dat2 &lt;- httr::content(res2, encoding = &quot;UTF-8&quot;)
      running &lt;- jsonlite::fromJSON(dat2)$running
      Sys.sleep(1)
  }
  
  # Fetch results
  httr::POST(
    url = &quot;http://revigo.irb.hr/ExportJob.aspx&quot;,
    query = list(
      jobid = jobid, 
      namespace = &quot;1&quot;,
      type = &quot;CSVTable&quot;
    )
  ) -&gt; res3
  
  dat3 &lt;- httr::content(res3, encoding = &quot;UTF-8&quot;)
  
  dat3 &lt;- stri_replace_all_fixed(dat3, &quot;\r&quot;, &quot;&quot;)
  # Now we have a csv table in a string!

  # read_csv does not like the &#39;, &#39;, it wants &#39;,&#39;
  dat &lt;- read_csv(gsub(&#39;, &#39;, &#39;,&#39;, dat3), show_col_types = FALSE) 
  
  # do we even have results?
  if(nrow(dat) == 0){next}
  results_list[[f]] &lt;- dat
  
}</code></pre>
<p>OK we have a list with all results in a big list. Now we can plot!</p>
</div>
<div id="making-supplementary-tables-for-the-revigo-results" class="section level1">
<h1>Making Supplementary Tables for the REVIGO results</h1>
<p>Let’s also pull these terms out as tables</p>
<pre class="r"><code>for( i in names(results_list)) {
  print(i)
  print(knitr::kable(results_list[[i]] %&gt;% filter(Eliminated == FALSE), label=i))
  cat(&quot;\n&quot;)
}</code></pre>
<pre><code>[1] &quot;lost_in_amphi_GO.txt&quot;


|TermID     |Name                                                                    |  Frequency|PlotX             |PlotY               |   LogSize|       Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:-----------------------------------------------------------------------|----------:|:-----------------|:-------------------|---------:|-----------:|----------:|--------------:|:--------------|:----------|
|GO:0002474 |antigen processing and presentation of peptide antigen via MHC class I  |  1.0871030|-1.92538582927823 |8.30126103161696    | 2.3856063|   -2.689686|  1.0000000|      0.0000000|null           |FALSE      |
|GO:0006278 |RNA-dependent DNA biosynthetic process                                  |  0.2605453|-5.43393361856214 |3.40135449852066    | 1.7708520| -154.570365|  0.8150443|      0.0000000|null           |FALSE      |
|GO:0010726 |positive regulation of hydrogen peroxide metabolic process              |  0.0179686|3.02920501263946  |5.43797723136727    | 0.6989700|  -19.887919|  0.8675983|      0.0000000|null           |FALSE      |
|GO:0015709 |thiosulfate transport                                                   |  0.0134765|-3.66374857484653 |-5.93176901665721   | 0.6020600|   -7.854354|  0.9228281|      0.0000000|null           |FALSE      |
|GO:0080181 |lateral root branching                                                  |  0.0134765|5.73582425403849  |-1.79863357019473   | 0.6020600|  -17.324362|  0.9541347|      0.0000000|null           |FALSE      |
|GO:0032197 |transposition,RNA-mediated                                              |  0.0044922|2.3622469465905   |-1.00542434488715   | 0.3010300| -153.541147|  0.9813284|      0.0208561|null           |FALSE      |
|GO:0071395 |cellular response to jasmonic acid stimulus                             |  0.4222632|1.19696389659204  |-5.53608747182279   | 1.9777236|  -23.603685|  0.8904716|      0.0291553|null           |FALSE      |
|GO:0006012 |galactose metabolic process                                             |  0.1212884|-7.68320229408839 |-1.62420046026126   | 1.4471580|   -1.606369|  0.8947722|      0.0910746|null           |FALSE      |
|GO:0016101 |diterpenoid metabolic process                                           |  0.2740218|-6.68740410423279 |-2.6943093389161    | 1.7923917|   -3.540981|  0.8589484|      0.0973387|null           |FALSE      |
|GO:1904580 |regulation of intracellular mRNA localization                           |  0.0044922|6.0808787618578   |2.19295763062448    | 0.3010300|   -3.743490|  0.9250105|      0.1175562|null           |FALSE      |
|GO:0045962 |positive regulation of development,heterochronic                        |  0.0733720|0.819723958330985 |6.86150563118112    | 1.2304489|   -9.318720|  0.9096345|      0.1222856|null           |FALSE      |
|GO:0006032 |chitin catabolic process                                                |  0.1078119|-3.72160019267578 |0.0382030595411626  | 1.3979400|   -6.683057|  0.9064345|      0.1338994|null           |FALSE      |
|GO:0070562 |regulation of vitamin D receptor signaling pathway                      |  0.1497720|3.04749146349184  |2.98600396694869    | 1.5314789|   -6.913688|  0.8866193|      0.1448064|null           |FALSE      |
|GO:0002229 |defense response to oomycetes                                           |  0.3234356|3.30530721478547  |-5.61482564448417   | 1.8633229|   -3.722154|  0.9556362|      0.1680904|null           |FALSE      |
|GO:0009648 |photoperiodism                                                          |  0.3414042|2.90421719253273  |-4.46323826054106   | 1.8864907|   -3.590301|  0.9495265|      0.1689057|null           |FALSE      |
|GO:0010337 |regulation of salicylic acid metabolic process                          |  0.0943354|4.50065502002873  |3.97820742862482    | 1.3424227|  -10.566727|  0.9132621|      0.2045798|null           |FALSE      |
|GO:0090059 |protoxylem development                                                  |  0.0134765|5.73443834787798  |-3.18267739824047   | 0.6020600|   -3.236933|  0.9816131|      0.2096160|null           |FALSE      |
|GO:0006508 |proteolysis                                                             |  5.0132519|-6.20178954635686 |4.41639774359818    | 3.0480532|  -79.607924|  0.8973448|      0.2178377|null           |FALSE      |
|GO:0090110 |COPII-coated vesicle cargo loading                                      |  0.0539059|-3.13261906030803 |-3.37325004612047   | 1.1139434|   -3.236933|  0.8847802|      0.2267775|null           |FALSE      |
|GO:0003094 |glomerular filtration                                                   |  0.0943354|6.49041586796582  |-0.811261062702283  | 1.3424227|   -3.506441|  0.9676868|      0.2354660|null           |FALSE      |
|GO:0006275 |regulation of DNA replication                                           |  0.2515610|3.93421116795531  |4.40552948753057    | 1.7558749|   -3.730017|  0.9069957|      0.2462666|null           |FALSE      |
|GO:0009554 |megasporogenesis                                                        |  0.0539059|6.31245053062388  |-1.79381978905471   | 1.1139434|   -1.321762|  0.9365606|      0.2579909|null           |FALSE      |
|GO:0010951 |negative regulation of endopeptidase activity                           |  0.2066394|4.53334946496296  |4.75350031543935    | 1.6720979|   -3.564044|  0.9005586|      0.2621373|null           |FALSE      |
|GO:0000350 |generation of catalytic spliceosome for second transesterification step |  0.0134765|-3.40678120452542 |3.88611949696471    | 0.6020600|   -4.812729|  0.8336208|      0.2873025|null           |FALSE      |
|GO:0000495 |box H/ACA RNA 3&#39;-end processing                                         |  0.0044922|-3.95618347404845 |5.73955377496304    | 0.3010300|   -1.315281|  0.8874169|      0.3189026|null           |FALSE      |
|GO:0080021 |response to benzoic acid                                                |  0.0044922|0.612463516276164 |-6.43635748256932   | 0.3010300|   -6.390517|  0.9425977|      0.3221125|null           |FALSE      |
|GO:0042178 |xenobiotic catabolic process                                            |  0.0179686|0.111425985851438 |-4.6748501083436    | 0.6989700|   -3.362176|  0.8807235|      0.3435777|null           |FALSE      |
|GO:2000636 |positive regulation of primary miRNA processing                         |  0.0134765|3.42124832963329  |5.76859970640441    | 0.6020600|   -1.315281|  0.8825285|      0.3606938|null           |FALSE      |
|GO:0072344 |rescue of stalled ribosome                                              |  0.0314451|-5.84160122651776 |3.1662246892737     | 0.9030900|   -1.315281|  0.8437476|      0.3612349|null           |FALSE      |
|GO:0072756 |cellular response to paraquat                                           |  0.0044922|0.343580098633531 |-5.87052405691359   | 0.3010300|   -2.168010|  0.9249762|      0.3674946|null           |FALSE      |
|GO:0072488 |ammonium transmembrane transport                                        |  0.0269530|-4.0667650118002  |-5.47235482759066   | 0.8450980|   -6.112218|  0.9101838|      0.3749833|null           |FALSE      |
|GO:0010942 |positive regulation of cell death                                       |  0.1527335|2.52416967174724  |5.53443814888718    | 1.5440680|  -14.685584|  0.8783753|      0.3819289|null           |FALSE      |
|GO:0010555 |response to mannitol                                                    |  0.0673824|1.21931473500362  |-6.06972630452485   | 1.2041200|   -3.407172|  0.9327828|      0.3904216|null           |FALSE      |
|GO:2000630 |positive regulation of miRNA metabolic process                          |  0.0134765|3.16514419610942  |5.89017798280165    | 0.6020600|   -1.315281|  0.8825285|      0.3973892|null           |FALSE      |
|GO:0009629 |response to gravity                                                     |  0.4626926|3.04755134248039  |-4.80389818140857   | 2.0170333|   -3.039850|  0.9485452|      0.3974675|null           |FALSE      |
|GO:0090501 |RNA phosphodiester bond hydrolysis                                      |  0.8804636|-4.65433754863761 |4.99651056862295    | 2.2944662|  -99.312749|  0.8499513|      0.3997076|null           |FALSE      |
|GO:1901333 |positive regulation of lateral root development                         |  0.0179686|1.67041752582499  |6.69197645698812    | 0.6989700|   -2.689686|  0.8738294|      0.4072265|null           |FALSE      |
|GO:0044245 |polysaccharide digestion                                                | 11.9626252|6.24417253863704  |-1.12621708083176   | 3.4255342|  -13.713775|  0.9582622|      0.4113275|null           |FALSE      |
|GO:1990481 |mRNA pseudouridine synthesis                                            |  0.0224608|-4.38994957874008 |5.6843496586618     | 0.7781513|   -3.677092|  0.8790236|      0.4142760|null           |FALSE      |
|GO:0070126 |mitochondrial translational termination                                 |  0.0179686|-4.49092814237451 |2.27903180743564    | 0.6989700|   -1.315281|  0.8076625|      0.4155310|null           |FALSE      |
|GO:0034626 |fatty acid elongation,polyunsaturated fatty acid                        |  0.0179686|-6.97619181497637 |-0.419742670050834  | 0.6989700|   -3.464066|  0.7987183|      0.4495755|null           |FALSE      |
|GO:0009089 |lysine biosynthetic process via diaminopimelate                         |  0.0673824|-6.71687699653787 |0.773955514289539   | 1.2041200|   -2.699278|  0.8263017|      0.4703089|null           |FALSE      |
|GO:0033259 |plastid DNA replication                                                 |  0.0089843|-4.66499363030025 |2.76360657677431    | 0.4771213|   -7.854354|  0.8190097|      0.4730520|null           |FALSE      |
|GO:0006182 |cGMP biosynthetic process                                               |  0.0224608|-5.78554157872798 |2.09215360573125    | 0.7781513|   -8.073348|  0.8202206|      0.4778367|null           |FALSE      |
|GO:0015695 |organic cation transport                                                |  0.0718746|-4.27030961833203 |-5.59166961629841   | 1.2304489|   -5.943629|  0.9274628|      0.4798653|null           |FALSE      |
|GO:0090305 |nucleic acid phosphodiester bond hydrolysis                             |  1.6171780|-4.91894473821623 |4.71116711770831    | 2.5575072|  -16.906247|  0.8489085|      0.4820982|null           |FALSE      |
|GO:0071422 |succinate transmembrane transport                                       |  0.0179686|-3.65665872926601 |-5.65722919607031   | 0.6989700|   -5.214207|  0.8974115|      0.4841726|null           |FALSE      |
|GO:2000032 |regulation of secondary shoot formation                                 |  0.0718746|1.00706383347584  |7.06432317321209    | 1.2304489|   -1.698816|  0.9053898|      0.4855638|null           |FALSE      |
|GO:0006334 |nucleosome assembly                                                     |  0.1841786|-1.29092861435095 |-0.0489670831771986 | 1.6232493|   -4.587444|  0.9074488|      0.4888051|null           |FALSE      |
|GO:0001561 |fatty acid alpha-oxidation                                              |  0.0089843|-6.34268898346823 |-1.97441233251259   | 0.4771213|   -1.315281|  0.8394512|      0.4943296|null           |FALSE      |
|GO:0080153 |negative regulation of reductive pentose-phosphate cycle                |  0.0134765|5.19414607070645  |4.52207122621196    | 0.6020600|   -3.464066|  0.9139349|      0.4953921|null           |FALSE      |
|GO:0071465 |cellular response to desiccation                                        |  0.0763667|1.91555640208608  |-5.38996211187142   | 1.2552725|   -6.390517|  0.8897283|      0.5346686|null           |FALSE      |
|GO:0009451 |RNA modification                                                        |  1.4105386|-4.61556926429093 |4.72982607644471    | 2.4983106|  -13.803692|  0.8446371|      0.5481052|null           |FALSE      |
|GO:1902609 |(R)-2-hydroxy-alpha-linolenic acid biosynthetic process                 |  0.0089843|-6.94015795553534 |-0.64387581845247   | 0.4771213|   -1.315281|  0.8193718|      0.5551354|null           |FALSE      |
|GO:0051258 |protein polymerization                                                  |  0.1347648|-1.0980834644771  |0.242326257073962   | 1.4913617|   -1.810823|  0.9128928|      0.5760110|null           |FALSE      |
|GO:0035445 |borate transmembrane transport                                          |  0.0269530|-3.89756238816197 |-5.793966955405     | 0.8450980|   -2.035237|  0.8991229|      0.5766489|null           |FALSE      |
|GO:0048657 |anther wall tapetum cell differentiation                                |  0.0134765|5.96951378152395  |-1.91528321752652   | 0.6020600|   -2.917278|  0.9311978|      0.5928954|null           |FALSE      |
|GO:1903335 |regulation of vacuolar transport                                        |  0.0089843|5.81780283726708  |2.35822108767338    | 0.4771213|   -2.864523|  0.9226310|      0.5958785|null           |FALSE      |
|GO:0006264 |mitochondrial DNA replication                                           |  0.0404295|-6.85938544389333 |2.17622789039239    | 1.0000000|   -4.812729|  0.8650425|      0.6158060|null           |FALSE      |
|GO:0043693 |monoterpene biosynthetic process                                        |  0.0179686|-6.46282262735183 |-0.585674371195019  | 0.6989700|   -1.315281|  0.8484448|      0.6402458|null           |FALSE      |
|GO:0048385 |regulation of retinoic acid receptor signaling pathway                  |  0.1668089|3.31445111598832  |2.92965050505224    | 1.5797836|   -6.913688|  0.8858973|      0.6457510|null           |FALSE      |
|GO:0046622 |positive regulation of organ growth                                     |  0.0269530|1.7467695764253   |6.51811233144361    | 0.8450980|   -1.315281|  0.8711268|      0.6477545|null           |FALSE      |
|GO:0009737 |response to abscisic acid                                               |  2.3538925|1.47753585231756  |-5.63457937946029   | 2.7201593|   -3.781098|  0.9118466|      0.6696275|null           |FALSE      |

[1] &quot;lost_in_posi_GO.txt&quot;


|TermID     |Name                                                        | Frequency|PlotX              |PlotY             |   LogSize|      Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:-----------------------------------------------------------|---------:|:------------------|:-----------------|---------:|----------:|----------:|--------------:|:--------------|:----------|
|GO:0009451 |RNA modification                                            | 1.4105386|5.31713158677838   |0.750209946309509 | 2.4983106| -42.299053|  0.8657301|      0.0000000|null           |FALSE      |
|GO:0010064 |embryonic shoot morphogenesis                               | 0.0044922|4.75377371475025   |-4.41630101076762 | 0.3010300|  -6.780537|  0.9500264|      0.0000000|null           |FALSE      |
|GO:0052472 |modulation by host of symbiont transcription                | 0.0044922|-4.22482265160858  |-4.63322175957405 | 0.3010300|  -5.743965|  0.8690038|      0.0000000|null           |FALSE      |
|GO:1902184 |negative regulation of shoot apical meristem development    | 0.0044922|-2.32682339102221  |5.89503071201695  | 0.3010300|  -8.322884|  0.8604211|      0.0000000|null           |FALSE      |
|GO:1904158 |axonemal central apparatus assembly                         | 0.2632906|0.11620421824961   |-5.73643586564387 | 1.7708520|  -6.383832|  0.9134237|      0.0326324|null           |FALSE      |
|GO:0033485 |cyanidin 3-O-glucoside biosynthetic process                 | 0.0847272|5.91990204800458   |4.41394579850124  | 1.2787536|  -9.116936|  0.7863560|      0.0895787|null           |FALSE      |
|GO:0050434 |positive regulation of viral transcription                  | 0.0089843|-6.87663285920282  |1.82396976309249  | 0.4771213|  -5.148961|  0.8855099|      0.1002280|null           |FALSE      |
|GO:0031098 |stress-activated protein kinase signaling cascade           | 0.3252325|-4.84169535347222  |0.670098275179084 | 1.8633229|  -4.952222|  0.8477464|      0.1287540|null           |FALSE      |
|GO:0048660 |regulation of smooth muscle cell proliferation              | 0.1549796|-2.79719547570035  |2.99859312486821  | 1.5440680|  -3.111010|  0.9071052|      0.1795904|null           |FALSE      |
|GO:0070848 |response to growth factor                                   | 0.9496038|-5.92614054472538  |-2.12799370913948 | 2.3263359|  -3.109409|  0.9399029|      0.1812971|null           |FALSE      |
|GO:0030244 |cellulose biosynthetic process                              | 0.2515610|5.77235083292305   |2.89549527331299  | 1.7558749|  -2.313130|  0.8397400|      0.2639733|null           |FALSE      |
|GO:0010727 |negative regulation of hydrogen peroxide metabolic process  | 0.0044922|-2.55763084278906  |6.97814084834928  | 0.3010300|  -1.940740|  0.8764862|      0.2761077|null           |FALSE      |
|GO:0009653 |anatomical structure morphogenesis                          | 3.8587665|5.3624086082923    |-3.85543712868776 | 2.9344985|  -1.681340|  0.9593916|      0.2816487|null           |FALSE      |
|GO:0043406 |positive regulation of MAP kinase activity                  | 0.0202147|-5.52799547843428  |3.79862639578361  | 0.6989700|  -4.619586|  0.8409126|      0.2978380|null           |FALSE      |
|GO:0042407 |cristae formation                                           | 0.0089843|-0.646214545539496 |-5.9276054614972  | 0.4771213|  -5.459308|  0.9463818|      0.3096290|null           |FALSE      |
|GO:0034307 |regulation of ascospore formation                           | 0.0044922|-2.39710488486283  |1.24739283674545  | 0.3010300|  -2.411002|  0.8502347|      0.3222164|null           |FALSE      |
|GO:0071602 |phytosphingosine biosynthetic process                       | 0.4459718|4.51661306021937   |3.6819775076701   | 2.0000000| -10.064067|  0.7340214|      0.3281840|null           |FALSE      |
|GO:0018008 |N-terminal peptidyl-glycine N-myristoylation                | 0.0089843|5.99781313814251   |2.19383147035782  | 0.4771213|  -6.780537|  0.8485922|      0.3409569|null           |FALSE      |
|GO:0048658 |anther wall tapetum development                             | 0.0853511|4.04189872616588   |-3.75943541620103 | 1.3010300|  -2.860563|  0.9326866|      0.3741144|null           |FALSE      |
|GO:2000117 |negative regulation of cysteine-type endopeptidase activity | 0.0044922|-3.31015678283516  |6.70877888218987  | 0.3010300|  -4.611786|  0.8684102|      0.3754329|null           |FALSE      |
|GO:0010030 |positive regulation of seed germination                     | 0.0898432|-5.160900816373    |2.62995106089086  | 1.3222193|  -1.592823|  0.8298786|      0.3789123|null           |FALSE      |
|GO:0031333 |negative regulation of protein-containing complex assembly  | 0.0853511|-3.34242834310024  |5.77886117250202  | 1.3010300|  -3.096418|  0.8607925|      0.3803810|null           |FALSE      |
|GO:0002238 |response to molecule of fungal origin                       | 0.0853511|-4.84311711021576  |-3.18437132326707 | 1.3010300|  -2.175661|  0.8956274|      0.3822097|null           |FALSE      |
|GO:0019471 |4-hydroxyproline metabolic process                          | 0.1326792|3.25364043662855   |5.9206901806479   | 1.4771213|  -4.570910|  0.8587831|      0.3936324|null           |FALSE      |
|GO:0050728 |negative regulation of inflammatory response                | 0.0346538|-4.28634492687529  |5.57763900824942  | 0.9030900|  -2.411002|  0.8494763|      0.3948101|null           |FALSE      |
|GO:0009833 |plant-type primary cell wall biogenesis                     | 0.1257805|0.760902774807325  |-5.82381691069585 | 1.4623980|  -4.679683|  0.9269866|      0.4045949|null           |FALSE      |
|GO:0002237 |response to molecule of bacterial origin                    | 0.2111316|-5.18680525771552  |-3.08062294082792 | 1.6812412|  -1.494860|  0.8899101|      0.4142112|null           |FALSE      |
|GO:0033542 |fatty acid beta-oxidation,unsaturated,even number           | 0.0044922|3.04247894785448   |6.84106045104841  | 0.3010300|  -4.842284|  0.8756371|      0.4233632|null           |FALSE      |
|GO:0051782 |negative regulation of cell division                        | 0.0404295|-3.05881062680884  |6.18938276830495  | 1.0000000|  -1.358334|  0.8663148|      0.4365510|null           |FALSE      |
|GO:0018401 |peptidyl-proline hydroxylation to 4-hydroxy-L-proline       | 0.0628903|6.550301536265     |0.38902046946302  | 1.1760913|  -4.570910|  0.8971229|      0.4457143|null           |FALSE      |
|GO:0042351 |&#39;de novo&#39; GDP-L-fucose biosynthetic process                 | 0.0179686|4.42351522007205   |4.1254088625343   | 0.6989700|  -4.842284|  0.7998257|      0.4814266|null           |FALSE      |
|GO:0090305 |nucleic acid phosphodiester bond hydrolysis                 | 1.6171780|4.29885275258937   |0.510985491784601 | 2.5575072| -40.107639|  0.8796109|      0.5091922|null           |FALSE      |
|GO:0006488 |dolichol-linked oligosaccharide biosynthetic process        | 0.0449216|5.12112902656404   |3.50939031087055  | 1.0413927|  -1.695773|  0.7826602|      0.5112697|null           |FALSE      |
|GO:0060776 |simple leaf morphogenesis                                   | 0.0224608|4.80807244609888   |-4.11531776627645 | 0.7781513|  -2.475052|  0.9436756|      0.5336678|null           |FALSE      |
|GO:0042353 |fucose biosynthetic process                                 | 0.0089843|4.32168143825897   |5.72142669402314  | 0.4771213|  -1.940740|  0.8434152|      0.5389857|null           |FALSE      |
|GO:0042450 |arginine biosynthetic process via ornithine                 | 0.0134765|4.16293184065081   |5.15261720837209  | 0.6020600|  -2.239011|  0.8230194|      0.5451423|null           |FALSE      |
|GO:1904837 |beta-catenin-TCF complex assembly                           | 0.5076142|0.374349705604716  |-5.48586778912037 | 2.0569049|  -4.842284|  0.9099486|      0.5451883|null           |FALSE      |
|GO:0051131 |chaperone-mediated protein complex assembly                 | 0.0494138|0.34449322914455   |-5.12015562417554 | 1.0791812|  -2.989676|  0.9206363|      0.5532566|null           |FALSE      |
|GO:0046740 |transport of virus in host,cell to cell                     | 0.0089843|-4.49470415848224  |-4.58869031406119 | 0.4771213|  -4.842284|  0.9030650|      0.5611286|null           |FALSE      |
|GO:1902290 |positive regulation of defense response to oomycetes        | 0.0449216|-6.13544953927906  |3.08199408222255  | 1.0413927|  -2.284022|  0.8475028|      0.5951678|null           |FALSE      |
|GO:0071507 |pheromone response MAPK cascade                             | 0.5744351|-4.58879993551974  |0.927679802571203 | 2.1072100|  -2.411002|  0.7010977|      0.6404046|null           |FALSE      |

[1] &quot;lost_in_zmar_GO.txt&quot;


|TermID     |Name                                                                                                   |  Frequency|PlotX              |PlotY              |   LogSize|      Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:------------------------------------------------------------------------------------------------------|----------:|:------------------|:------------------|---------:|----------:|----------:|--------------:|:--------------|:----------|
|GO:0009083 |branched-chain amino acid catabolic process                                                            |  0.1167962|-6.93826998154665  |4.35599107891625   | 1.4313638| -23.825644|  0.8187945|      0.0000000|null           |FALSE      |
|GO:0009846 |pollen germination                                                                                     |  0.2605453|-0.319526231430999 |-6.05321049026938  | 1.7708520|  -7.057244|  0.9413712|      0.0000000|null           |FALSE      |
|GO:0010199 |organ boundary specification between lateral organs and the meristem                                   |  0.0404295|2.26471947596313   |-6.20829673434534  | 1.0000000|  -2.173048|  0.9566396|      0.0000000|null           |FALSE      |
|GO:0044829 |positive regulation by host of viral genome replication                                                |  0.0044922|4.70051066294047   |-5.05553524994427  | 0.3010300|  -5.002696|  1.0000000|      0.0000000|null           |FALSE      |
|GO:1900153 |positive regulation of nuclear-transcribed mRNA catabolic process,deadenylation-dependent decay        |  0.0748694|4.41200206580293   |3.18343224394622   | 1.2304489|  -6.908359|  0.8022199|      0.0000000|null           |FALSE      |
|GO:1902418 |(+)-abscisic acid D-glucopyranosyl ester transmembrane transport                                       |  0.0089843|-4.1610318515477   |-4.93911162194645  | 0.4771213| -16.818712|  0.9211253|      0.0207144|null           |FALSE      |
|GO:0043617 |cellular response to sucrose starvation                                                                |  0.0179686|-6.44428156546529  |0.687517920226477  | 0.6989700| -11.063985|  0.9087316|      0.0216481|null           |FALSE      |
|GO:0034333 |adherens junction assembly                                                                             |  0.0269530|-0.881394693029845 |-1.47671645350495  | 0.8450980|  -2.627603|  0.8978029|      0.0222344|null           |FALSE      |
|GO:0007059 |chromosome segregation                                                                                 |  0.5480437|6.33006963143281   |-2.96030262707983  | 2.0899051|  -5.880062|  0.9655979|      0.0278349|null           |FALSE      |
|GO:0007018 |microtubule-based movement                                                                             |  0.2919905|7.15380226164213   |-1.7589110310574   | 1.8195439|  -4.860293|  0.9446473|      0.0301445|null           |FALSE      |
|GO:0097502 |mannosylation                                                                                          |  0.1796864|5.3251052822447    |-3.82091872310717  | 1.6127839|  -2.726690|  0.9605456|      0.0642401|null           |FALSE      |
|GO:0046685 |response to arsenic-containing substance                                                               |  0.0718746|-7.02459521217403  |-1.41066405301197  | 1.2304489|  -6.914746|  0.9571229|      0.1186977|null           |FALSE      |
|GO:0080037 |negative regulation of cytokinin-activated signaling pathway                                           |  0.0224608|4.79136909778065   |5.49672793911495   | 0.7781513|  -6.440151|  0.8737143|      0.1404410|null           |FALSE      |
|GO:0043966 |histone H3 acetylation                                                                                 |  0.0853511|-3.24560867869498  |7.10279738613501   | 1.3010300|  -5.141144|  0.8492390|      0.1548915|null           |FALSE      |
|GO:0006075 |(1-&gt;3)-beta-D-glucan biosynthetic process                                                              |  0.0539059|-2.29557999429268  |7.66424981278343   | 1.1139434|  -3.716869|  0.9005991|      0.1568095|null           |FALSE      |
|GO:0106004 |tRNA (guanine-N7)-methylation                                                                          |  0.0134765|-3.32475879275095  |5.44112117891204   | 0.6020600|  -5.114010|  0.8486656|      0.2043834|null           |FALSE      |
|GO:0015700 |arsenite transport                                                                                     |  0.0224608|-5.07990951976141  |-4.34597535388894  | 0.7781513| -13.260514|  0.9314680|      0.2103830|null           |FALSE      |
|GO:0040029 |regulation of gene expression,epigenetic                                                               |  0.4941377|2.52257613323135   |1.07109217726358   | 2.0453230|  -5.265680|  0.7601406|      0.2756117|null           |FALSE      |
|GO:0061326 |renal tubule development                                                                               |  0.0628903|0.459928294187426  |-6.32054589885356  | 1.1760913|  -2.627603|  0.9171585|      0.2806508|null           |FALSE      |
|GO:1902198 |3-methylbut-2-enoyl-CoA(4-) metabolic process                                                          |  0.0202147|-5.20034273094633  |4.81296969540711   | 0.6989700|  -8.703391|  0.7880841|      0.3105322|null           |FALSE      |
|GO:1903902 |positive regulation of viral life cycle                                                                |  0.0089843|1.6317012296222    |7.04472432474427   | 0.4771213|  -2.883008|  0.8794685|      0.3214516|null           |FALSE      |
|GO:0006368 |transcription elongation from RNA polymerase II promoter                                               |  0.0763667|-3.78957648491673  |5.63958303258478   | 1.2552725|  -2.824435|  0.8306791|      0.3425941|null           |FALSE      |
|GO:0043524 |negative regulation of neuron apoptotic process                                                        |  0.0044922|6.54511740820781   |3.59281407819087   | 0.3010300|  -2.053021|  0.9028342|      0.3528003|null           |FALSE      |
|GO:0006293 |nucleotide-excision repair,preincision complex stabilization                                           |  0.0119791|-1.42702562329947  |4.09216166446965   | 0.4771213|  -3.979599|  0.7106876|      0.3556051|null           |FALSE      |
|GO:0009556 |microsporogenesis                                                                                      |  0.1123040|1.20878616182785   |-5.29964465478766  | 1.4149733|  -6.737358|  0.8486051|      0.3691219|null           |FALSE      |
|GO:0042048 |olfactory behavior                                                                                     | 12.9958223|-6.67650072682039  |-0.629032309007162 | 3.4614985|  -2.627603|  0.9387128|      0.3826800|null           |FALSE      |
|GO:0033234 |negative regulation of protein sumoylation                                                             |  0.0044922|5.68110798706138   |2.69624319405183   | 0.3010300|  -1.637393|  0.8725010|      0.3941790|null           |FALSE      |
|GO:1900150 |regulation of defense response to fungus                                                               |  0.1841786|3.66924991096079   |6.8153002008953    | 1.6232493|  -2.374111|  0.9067682|      0.4425117|null           |FALSE      |
|GO:1900400 |regulation of iron ion import into cell by regulation of transcription from RNA polymerase II promoter |  0.2241268|5.02469597425656   |3.88546300742578   | 1.6989700|  -6.440151|  0.8669133|      0.4546553|null           |FALSE      |
|GO:0000454 |snoRNA guided rRNA pseudouridine synthesis                                                             |  0.0089843|-2.53114062862441  |3.10229468577127   | 0.4771213|  -3.979599|  0.8019349|      0.4609134|null           |FALSE      |
|GO:0051729 |germline cell cycle switching,mitotic to meiotic cell cycle                                            |  0.0044922|3.39270392257064   |3.43726051422035   | 0.3010300|  -5.002696|  0.8151355|      0.4685567|null           |FALSE      |
|GO:0006396 |RNA processing                                                                                         |  4.2765374|-4.03191805642209  |4.83344202713665   | 2.9790929|  -2.281902|  0.7985447|      0.4772279|null           |FALSE      |
|GO:1901684 |arsenate ion transmembrane transport                                                                   |  0.0224608|-4.75439706207443  |-4.67732413001445  | 0.7781513| -13.048999|  0.9039704|      0.5053225|null           |FALSE      |
|GO:0045948 |positive regulation of translational initiation                                                        |  0.0134765|4.13806633883544   |2.89054465331184   | 0.6020600|  -1.889705|  0.8559779|      0.5078317|null           |FALSE      |
|GO:0009226 |nucleotide-sugar biosynthetic process                                                                  |  0.1886708|-4.65685217983191  |5.23446625809102   | 1.6334685|  -1.665611|  0.7959752|      0.5295428|null           |FALSE      |
|GO:0006337 |nucleosome disassembly                                                                                 |  0.0044922|0.2778398398817    |-1.33349710455843  | 0.3010300|  -2.257923|  0.8716732|      0.5351491|null           |FALSE      |
|GO:0080092 |regulation of pollen tube growth                                                                       |  0.1662100|2.58687110640179   |5.32174466147864   | 1.5797836|  -1.916242|  0.8989017|      0.5428390|null           |FALSE      |
|GO:1900000 |regulation of anthocyanin catabolic process                                                            |  0.0044922|6.17013948111656   |0.767603533269832  | 0.3010300|  -5.114010|  0.8940841|      0.5482978|null           |FALSE      |
|GO:1904211 |membrane protein proteolysis involved in retrograde protein transport,ER to cytosol                    |  0.9478460|-5.42338534441794  |0.698959960477185  | 2.3263359|  -8.703391|  0.6907598|      0.5495590|null           |FALSE      |
|GO:0010104 |regulation of ethylene-activated signaling pathway                                                     |  0.1257805|3.97639507395384   |6.38435357682811   | 1.4623980|  -4.578960|  0.9003057|      0.5524117|null           |FALSE      |
|GO:0098660 |inorganic ion transmembrane transport                                                                  |  1.9810431|-4.67382384570729  |-4.41931898260442  | 2.6454223|  -2.721593|  0.8754988|      0.5543701|null           |FALSE      |
|GO:0048599 |oocyte development                                                                                     |  0.0044922|0.92017299228498   |-5.76013022101947  | 0.3010300|  -2.568839|  0.8462735|      0.5577273|null           |FALSE      |
|GO:0035194 |post-transcriptional gene silencing by RNA                                                             |  0.3144513|5.12221082928765   |3.29531038286037   | 1.8512583|  -2.797707|  0.8444017|      0.5954572|null           |FALSE      |
|GO:0042631 |cellular response to water deprivation                                                                 |  0.1707021|-6.2978480628151   |0.107089408135958  | 1.5910646|  -1.383875|  0.8870843|      0.6083709|null           |FALSE      |
|GO:0055047 |generative cell mitosis                                                                                |  0.0134765|0.700730601192429  |-3.40780386102219  | 0.6020600|  -2.265927|  0.8158557|      0.6102124|null           |FALSE      |
|GO:0032508 |DNA duplex unwinding                                                                                   |  0.2919905|0.637264155912155  |-1.57180943882823  | 1.8195439|  -3.700286|  0.8604762|      0.6507685|null           |FALSE      |
|GO:0070911 |global genome nucleotide-excision repair                                                               |  0.0119791|-4.84252092734838  |2.87432142839768   | 0.4771213|  -2.554750|  0.8101733|      0.6716500|null           |FALSE      |
|GO:0006294 |nucleotide-excision repair,preincision complex assembly                                                |  0.0134765|-3.42368086767935  |1.79503447041482   | 0.6020600|  -1.668643|  0.7308664|      0.6760592|null           |FALSE      |

[1] &quot;lost_in_zmuel_GO.txt&quot;


|TermID     |Name                                                    | Frequency|PlotX             |PlotY              |   LogSize|     Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:-------------------------------------------------------|---------:|:-----------------|:------------------|---------:|---------:|----------:|--------------:|:--------------|:----------|
|GO:0001841 |neural tube formation                                   | 0.0134765|-0.36201361027221 |5.41376252646244   | 0.6020600| -2.894824|  0.9694855|      0.0000000|null           |FALSE      |
|GO:0006597 |spermine biosynthetic process                           | 0.0269530|-4.37347702942198 |-5.4409860988995   | 0.8450980| -7.719845|  0.7979714|      0.0000000|null           |FALSE      |
|GO:0071461 |cellular response to redox state                        | 0.0089843|6.04466258392492  |2.80350802152024   | 0.4771213| -1.547943|  1.0000000|      0.0000000|null           |FALSE      |
|GO:0090071 |negative regulation of ribosome biogenesis              | 0.0089843|3.52027575522102  |5.16482513758913   | 0.4771213| -1.345457|  1.0000000|      0.0000000|null           |FALSE      |
|GO:0089709 |L-histidine transmembrane transport                     | 0.0089843|4.85596290160271  |-2.83300943402113  | 0.4771213| -3.309566|  0.6163984|      0.0189823|null           |FALSE      |
|GO:0031037 |myosin II filament disassembly                          | 0.2021473|-4.19041790517318 |4.57023876367769   | 1.6627578| -4.228771|  0.9311904|      0.0230801|null           |FALSE      |
|GO:0097502 |mannosylation                                           | 0.1796864|-6.66818324027485 |0.0353994449046674 | 1.6127839| -5.271569|  0.9067343|      0.0577619|null           |FALSE      |
|GO:0006490 |oligosaccharide-lipid intermediate biosynthetic process | 0.0628903|-3.95359262705638 |-3.95923153990229  | 1.1760913| -7.691661|  0.8502187|      0.2211080|null           |FALSE      |
|GO:0042372 |phylloquinone biosynthetic process                      | 0.0449216|-2.7670652197161  |-6.20857811215572  | 1.0413927| -1.854951|  0.8765778|      0.2285982|null           |FALSE      |
|GO:0015697 |quaternary ammonium group transport                     | 0.0269530|4.05753894093073  |-4.88549398643119  | 0.8450980| -2.047144|  0.7817432|      0.3173486|null           |FALSE      |
|GO:0010197 |polar nucleus fusion                                    | 0.1302727|-2.46285188025498 |4.94610897264595   | 1.4771213| -1.891385|  0.9024962|      0.3677093|null           |FALSE      |
|GO:0015847 |putrescine transport                                    | 0.0123534|5.00358558526413  |-3.83535851944143  | 0.4771213| -1.345457|  0.7355369|      0.4049477|null           |FALSE      |
|GO:0072488 |ammonium transmembrane transport                        | 0.0269530|4.17604952125999  |-3.51230923542749  | 0.8450980| -2.416930|  0.7067819|      0.4348824|null           |FALSE      |
|GO:0033577 |protein glycosylation in endoplasmic reticulum          | 0.0044922|-5.21710907900686 |-2.62618133970948  | 0.3010300| -2.894824|  0.7895839|      0.5737773|null           |FALSE      |

[1] &quot;missing_amphi_vs_all_GO.txt&quot;


|TermID     |Name                                                                                         |  Frequency|PlotX              |PlotY              |   LogSize|       Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:--------------------------------------------------------------------------------------------|----------:|:------------------|:------------------|---------:|-----------:|----------:|--------------:|:--------------|:----------|
|GO:0010082 |regulation of root meristem growth                                                           |  0.1527335|-1.57278604803594  |-5.05387432733019  | 1.5440680|  -62.316817|  0.8146377|      0.0000000|null           |FALSE      |
|GO:0048512 |circadian behavior                                                                           |  0.0224608|-2.70489930198424  |-2.71247628589482  | 0.7781513|   -6.692248|  0.9967821|      0.0000000|null           |FALSE      |
|GO:0051762 |sesquiterpene biosynthetic process                                                           |  0.1123040|-5.94606976195204  |2.84134969721899   | 1.4149733| -155.412480|  0.8463748|      0.0000000|null           |FALSE      |
|GO:1902025 |nitrate import                                                                               |  0.0808589|-6.25408760466433  |-2.1694290446808   | 1.2787536|  -20.184886|  0.9234934|      0.0000000|null           |FALSE      |
|GO:0080171 |lytic vacuole organization                                                                   |  0.0179686|1.64337729422586   |7.28079713603681   | 0.6989700|   -4.451511|  0.9370750|      0.0215931|null           |FALSE      |
|GO:0009626 |plant-type hypersensitive response                                                           |  0.3279278|5.92193701289709   |1.24872445155554   | 1.8692317|  -89.199181|  0.8410891|      0.0266051|null           |FALSE      |
|GO:0009780 |photosynthetic NADP+ reduction                                                               |  0.0044922|0.113800815856986  |0.900878662390277  | 0.3010300|   -7.317334|  0.9495474|      0.0658471|null           |FALSE      |
|GO:0051554 |flavonol metabolic process                                                                   |  0.0853511|3.70888276497169   |7.45792467425636   | 1.3010300|  -25.072818|  0.9372024|      0.0732686|null           |FALSE      |
|GO:0090305 |nucleic acid phosphodiester bond hydrolysis                                                  |  1.6171780|-3.28800592169992  |5.5133293531747    | 2.5575072| -151.260631|  0.8693669|      0.1056641|null           |FALSE      |
|GO:0080184 |response to phenylpropanoid                                                                  |  0.0044922|5.31468404516893   |5.070935274823     | 0.3010300|   -8.019366|  0.9227587|      0.1196905|null           |FALSE      |
|GO:0015979 |photosynthesis                                                                               |  1.1230403|3.29680165508143   |7.83790421762169   | 2.3996737|   -3.877351|  0.9361109|      0.1284458|null           |FALSE      |
|GO:0002230 |positive regulation of defense response to virus by host                                     |  0.0224608|0.394451950093978  |-6.63559245373321  | 0.7781513|  -16.511798|  0.8834373|      0.1301418|null           |FALSE      |
|GO:0010951 |negative regulation of endopeptidase activity                                                |  0.2066394|2.86521991513111   |-4.82976722014909  | 1.6720979|  -49.045011|  0.8490284|      0.1529435|null           |FALSE      |
|GO:0080155 |regulation of double fertilization forming a zygote and endosperm                            |  0.2335924|0.503469432442411  |-4.43174467810526  | 1.7242759|   -5.581798|  0.9086829|      0.1582520|null           |FALSE      |
|GO:0009786 |regulation of asymmetric cell division                                                       |  0.0718746|3.39252636965926   |-3.4285203115264   | 1.2304489|   -1.856958|  0.9150600|      0.1632380|null           |FALSE      |
|GO:0032886 |regulation of microtubule-based process                                                      |  0.1347648|2.95340003835585   |-3.80248180954085  | 1.4913617|   -4.732038|  0.9112813|      0.1712600|null           |FALSE      |
|GO:0010942 |positive regulation of cell death                                                            |  0.1527335|2.90077939191461   |-5.76642254168226  | 1.5440680|  -12.967396|  0.8738844|      0.1729524|null           |FALSE      |
|GO:0031146 |SCF-dependent proteasomal ubiquitin-dependent protein catabolic process                      |  0.3773415|-1.98174689070169  |6.13215315557179   | 1.9294189|  -73.404016|  0.8907318|      0.1763754|null           |FALSE      |
|GO:0018316 |peptide cross-linking via L-cystine                                                          |  0.0044922|-1.30863449706719  |7.75586789726149   | 0.3010300|   -1.778525|  0.9325242|      0.1861765|null           |FALSE      |
|GO:0043457 |regulation of cellular respiration                                                           |  0.0044922|5.51902290130016   |-4.48043854990886  | 0.3010300|   -2.586543|  0.9212950|      0.1969718|null           |FALSE      |
|GO:0006359 |regulation of transcription by RNA polymerase III                                            |  0.0044922|5.22257393675367   |-4.78155381108087  | 0.3010300|   -4.435722|  0.9187440|      0.2065919|null           |FALSE      |
|GO:0018364 |peptidyl-glutamine methylation                                                               |  0.0224608|-1.71547224684658  |7.4695535749479    | 0.7781513|   -5.201144|  0.9247028|      0.2076085|null           |FALSE      |
|GO:0090143 |nucleoid organization                                                                        |  0.0179686|2.18059543924179   |7.08117560033804   | 0.6989700|   -3.317884|  0.9423272|      0.2119540|null           |FALSE      |
|GO:0080164 |regulation of nitric oxide metabolic process                                                 |  0.0089843|5.51887679278792   |-4.12336562538826  | 0.4771213|   -2.851155|  0.9213570|      0.2158309|null           |FALSE      |
|GO:2000232 |regulation of rRNA processing                                                                |  0.0134765|4.7736534252585    |-4.74699695610487  | 0.6020600|   -2.371507|  0.9141265|      0.2216287|null           |FALSE      |
|GO:0009819 |drought recovery                                                                             |  0.0494138|5.95865245837433   |3.21696273867247   | 1.0791812|  -33.921860|  0.8901025|      0.2221654|null           |FALSE      |
|GO:0009410 |response to xenobiotic stimulus                                                              |  0.0359373|5.95228201933188   |4.47295724331113   | 0.9542425|   -1.585466|  0.9224121|      0.2284321|null           |FALSE      |
|GO:0019218 |regulation of steroid metabolic process                                                      |  0.0269530|4.68651877058436   |-4.06342693295084  | 0.8450980|   -3.561907|  0.9107783|      0.2286552|null           |FALSE      |
|GO:0032350 |regulation of hormone metabolic process                                                      |  0.1257805|1.2383179541745    |-3.10496857496148  | 1.4623980|  -16.977277|  0.8593137|      0.2320730|null           |FALSE      |
|GO:0010310 |regulation of hydrogen peroxide metabolic process                                            |  0.0539059|4.15302420199593   |-4.33189361453068  | 1.1139434|  -24.429451|  0.9084541|      0.2326787|null           |FALSE      |
|GO:0036371 |protein localization to T-tubule                                                             |  0.0314451|-6.29411842625139  |-1.17335551453245  | 0.9030900|  -13.799813|  0.9197518|      0.2340388|null           |FALSE      |
|GO:0071466 |cellular response to xenobiotic stimulus                                                     |  0.0224608|4.67004068873439   |3.30723347205778   | 0.7781513|  -20.553402|  0.8923160|      0.2373516|null           |FALSE      |
|GO:0033387 |putrescine biosynthetic process from ornithine                                               |  0.0123534|-3.86336572184278  |3.85060578962735   | 0.4771213|   -8.213157|  0.8713230|      0.2419183|null           |FALSE      |
|GO:0009969 |xyloglucan biosynthetic process                                                              |  0.0808589|-0.743420205458083 |5.32291494808535   | 1.2787536|  -25.974199|  0.8386727|      0.2530424|null           |FALSE      |
|GO:0009962 |regulation of flavonoid biosynthetic process                                                 |  0.1257805|3.8882372401849    |-4.194541677584    | 1.4623980|  -13.359591|  0.9056118|      0.2560442|null           |FALSE      |
|GO:0072756 |cellular response to paraquat                                                                |  0.0044922|4.75905278285483   |4.11306702130424   | 0.3010300|   -5.896024|  0.8970816|      0.2629602|null           |FALSE      |
|GO:1902075 |cellular response to salt                                                                    |  0.0044922|4.50802564277272   |3.83617171839393   | 0.3010300|   -7.118648|  0.9024747|      0.2629602|null           |FALSE      |
|GO:0035998 |7,8-dihydroneopterin 3&#39;-triphosphate biosynthetic process                                    |  0.0044922|-4.13896151747253  |4.2611430625435    | 0.3010300|   -7.547472|  0.8742499|      0.2663207|null           |FALSE      |
|GO:0010233 |phloem transport                                                                             |  0.0943354|-5.10443035685714  |-3.23406295260336  | 1.3424227|  -15.280791|  0.8876925|      0.2779671|null           |FALSE      |
|GO:0045168 |cell-cell signaling involved in cell fate commitment                                         |  0.1662100|-2.5992002458779   |-4.59922083139835  | 1.5797836|  -40.098376|  0.8830619|      0.2805344|null           |FALSE      |
|GO:0044550 |secondary metabolite biosynthetic process                                                    |  0.6244104|-4.24032933672445  |2.06880285757212   | 2.1461280| -109.732139|  0.9025627|      0.2815131|null           |FALSE      |
|GO:0009268 |response to pH                                                                               |  0.0359373|3.64719769350485   |1.41428243251654   | 0.9542425|   -4.047383|  0.9420042|      0.2827029|null           |FALSE      |
|GO:0007263 |nitric oxide mediated signal transduction                                                    |  0.0089843|3.81587278576452   |-0.584487163269622 | 0.4771213|   -3.561389|  0.8407722|      0.2860549|null           |FALSE      |
|GO:0009611 |response to wounding                                                                         |  1.0062441|6.40676004782227   |1.63939841538101   | 2.3521825|   -2.692392|  0.9204062|      0.2870502|null           |FALSE      |
|GO:0010375 |stomatal complex patterning                                                                  |  0.0494138|-4.14516853298291  |-5.40555080526035  | 1.0791812|  -16.637979|  0.9329263|      0.2884235|null           |FALSE      |
|GO:0000495 |box H/ACA RNA 3&#39;-end processing                                                              |  0.0044922|-4.33708393803314  |6.01237784080176   | 0.3010300|  -10.241991|  0.9046619|      0.3020801|null           |FALSE      |
|GO:0097366 |response to bronchodilator                                                                   |  2.0370454|5.92493806424637   |2.91786187279273   | 2.6570559|  -29.267556|  0.8928191|      0.3085869|null           |FALSE      |
|GO:0033259 |plastid DNA replication                                                                      |  0.0089843|-1.88041197719015  |4.93081172926406   | 0.4771213|   -7.843935|  0.8511383|      0.3176572|null           |FALSE      |
|GO:0009631 |cold acclimation                                                                             |  0.2560532|6.90087963469631   |1.65796772996969   | 1.7634280|   -4.906945|  0.9237845|      0.3235798|null           |FALSE      |
|GO:0043153 |entrainment of circadian clock by photoperiod                                                |  0.0224608|5.80462510042335   |-0.732120623342921 | 0.7781513|   -9.756528|  0.8554710|      0.3241684|null           |FALSE      |
|GO:0016567 |protein ubiquitination                                                                       |  3.7105251|-2.26056540026256  |6.41670751072134   | 2.9175055|  -31.401111|  0.8923024|      0.3270943|null           |FALSE      |
|GO:0010970 |transport along microtubule                                                                  |  0.0044922|-5.5038150963137   |-1.20680178441898  | 0.3010300|  -10.948753|  0.9059482|      0.3293628|null           |FALSE      |
|GO:1904962 |plastid to vacuole vesicle-mediated transport                                                |  0.0044922|-6.5359379912021   |-0.982682425166701 | 0.3010300|   -8.533431|  0.9256220|      0.3293628|null           |FALSE      |
|GO:0072344 |rescue of stalled ribosome                                                                   |  0.0314451|-3.108168622905    |4.69827594918458   | 0.9030900|   -4.047250|  0.8755780|      0.3317231|null           |FALSE      |
|GO:0010555 |response to mannitol                                                                         |  0.0673824|5.78394081709105   |3.9700535896922    | 1.2041200|   -7.292125|  0.9065467|      0.3342636|null           |FALSE      |
|GO:0009609 |response to symbiotic bacterium                                                              |  0.0089843|7.1936573262768    |-0.414138867701959 | 0.4771213|  -10.155539|  0.9193677|      0.3473421|null           |FALSE      |
|GO:0015853 |adenine transport                                                                            |  0.0179686|-6.77928940190925  |-1.68422261554423  | 0.6989700|   -3.130204|  0.9440687|      0.3534636|null           |FALSE      |
|GO:0001172 |transcription,RNA-templated                                                                  |  0.0359373|-3.66715027137268  |4.45742508126108   | 0.9542425|   -6.630651|  0.8629532|      0.3541850|null           |FALSE      |
|GO:0048544 |recognition of pollen                                                                        |  0.5570280|-2.61890559506909  |-4.05690399183027  | 2.0969100|   -6.623676|  0.8750689|      0.3590221|null           |FALSE      |
|GO:0042430 |indole-containing compound metabolic process                                                 |  0.3369121|-3.68763785879389  |5.77772088699245   | 1.8808136|  -11.640271|  0.8888277|      0.3665151|null           |FALSE      |
|GO:0006778 |porphyrin-containing compound metabolic process                                              |  0.3458964|-3.59926395529471  |6.05558882412774   | 1.8920946|   -5.415218|  0.8906283|      0.3675002|null           |FALSE      |
|GO:0045056 |transcytosis                                                                                 |  0.0044922|-5.17903571506906  |-3.41703909119299  | 0.3010300|   -4.682696|  0.8948569|      0.3677463|null           |FALSE      |
|GO:2000630 |positive regulation of miRNA metabolic process                                               |  0.0134765|3.82841366870144   |-5.46896038386575  | 0.6020600|   -6.661836|  0.8899836|      0.3747924|null           |FALSE      |
|GO:0090559 |regulation of membrane permeability                                                          |  0.0089843|-0.653060330738766 |-1.45013126974393  | 0.4771213|   -2.849913|  0.9024839|      0.3779708|null           |FALSE      |
|GO:0000966 |RNA 5&#39;-end processing                                                                        |  0.0808589|-3.90700201160144  |5.81046812728176   | 1.2787536|  -35.554319|  0.8901889|      0.3797277|null           |FALSE      |
|GO:0006334 |nucleosome assembly                                                                          |  0.1841786|1.42523959228921   |6.63861970965236   | 1.6232493|  -22.223748|  0.9063329|      0.3804956|null           |FALSE      |
|GO:0048211 |Golgi vesicle docking                                                                        |  0.0089843|-6.51275406358739  |-1.1598062369268   | 0.4771213|   -4.682696|  0.9215993|      0.3809345|null           |FALSE      |
|GO:0009871 |jasmonic acid and ethylene-dependent systemic resistance,ethylene mediated signaling pathway |  0.0089843|5.10181759362254   |0.413020136322447  | 0.4771213|  -27.782072|  0.7887465|      0.3823772|null           |FALSE      |
|GO:0007267 |cell-cell signaling                                                                          |  0.3414042|6.69582926871778   |-2.43931284701507  | 1.8864907|   -5.337881|  0.9432407|      0.3827963|null           |FALSE      |
|GO:0031120 |snRNA pseudouridine synthesis                                                                |  0.0134765|-3.44398885624408  |6.38177791364419   | 0.6020600|   -3.083481|  0.8958199|      0.3882252|null           |FALSE      |
|GO:0009608 |response to symbiont                                                                         |  0.0539059|7.03128637191391   |0.138921830435819  | 1.1139434|   -4.565462|  0.9137146|      0.3943270|null           |FALSE      |
|GO:0010618 |aerenchyma formation                                                                         |  0.0089843|-3.48926375609406  |-6.0748874857661   | 0.4771213|  -17.526209|  0.9449728|      0.3960626|null           |FALSE      |
|GO:1904541 |fungal-type cell wall disassembly involved in conjugation with cellular fusion               |  0.0044922|1.17037507552819   |5.27582814863375   | 0.3010300|   -7.425102|  0.9096390|      0.3968848|null           |FALSE      |
|GO:2000636 |positive regulation of primary miRNA processing                                              |  0.0134765|3.73963032357785   |-5.54823179644277  | 0.6020600|   -4.418909|  0.8885481|      0.3973892|null           |FALSE      |
|GO:0010438 |cellular response to sulfur starvation                                                       |  0.0314451|5.50520042169721   |0.925803091452775  | 0.9030900|   -7.761069|  0.8715320|      0.3994265|null           |FALSE      |
|GO:0015851 |nucleobase transport                                                                         |  0.1212884|-6.54869059807225  |-1.75567471162127  | 1.4471580|   -5.208378|  0.9398440|      0.4022496|null           |FALSE      |
|GO:0006403 |RNA localization                                                                             |  0.4222632|-6.08993635408378  |-1.58366542832704  | 1.9777236|   -4.389670|  0.9368869|      0.4054138|null           |FALSE      |
|GO:0080027 |response to herbivore                                                                        |  0.0808589|6.8771394967923    |0.208569395343619  | 1.2787536|  -29.746406|  0.9111889|      0.4067789|null           |FALSE      |
|GO:0034394 |protein localization to cell surface                                                         |  0.0134765|-6.63972642521676  |-1.08454192318227  | 0.6020600|   -2.573966|  0.9237485|      0.4074874|null           |FALSE      |
|GO:0002238 |response to molecule of fungal origin                                                        |  0.0853511|6.18935287894594   |2.05918854133376   | 1.3010300|   -8.989515|  0.8812072|      0.4084989|null           |FALSE      |
|GO:0006952 |defense response                                                                             |  7.2503481|5.81664428576227   |1.70016565999497   | 3.2081725|  -56.366823|  0.9032811|      0.4100881|null           |FALSE      |
|GO:0071732 |cellular response to nitric oxide                                                            |  0.0179686|5.2968648940498    |3.76376006588685   | 0.6989700|  -24.560403|  0.8715674|      0.4112191|null           |FALSE      |
|GO:0090615 |mitochondrial mRNA processing                                                                |  0.0539059|-3.92121025948905  |5.95375137809278   | 1.1139434|  -34.637510|  0.8870104|      0.4120236|null           |FALSE      |
|GO:0006124 |ferredoxin metabolic process                                                                 |  2.3735457|-2.14688588826814  |6.57534212814239   | 2.7234557|   -1.836460|  0.9015294|      0.4126013|null           |FALSE      |
|GO:0006260 |DNA replication                                                                              |  0.6423790|-2.60308098400636  |3.93568569187245   | 2.1583625|   -8.131429|  0.8780833|      0.4139708|null           |FALSE      |
|GO:0043100 |pyrimidine nucleobase salvage                                                                |  0.0179686|-4.24950271532957  |4.38381965005087   | 0.6989700|   -2.341970|  0.8758662|      0.4142189|null           |FALSE      |
|GO:0016045 |detection of bacterium                                                                       |  0.0134765|7.17939479132513   |-0.261003312222472 | 0.6020600|   -6.767682|  0.9172987|      0.4143200|null           |FALSE      |
|GO:0007166 |cell surface receptor signaling pathway                                                      |  0.5794888|4.11451119720333   |-1.07865349042768  | 2.1139434|   -1.416541|  0.7926556|      0.4151148|null           |FALSE      |
|GO:1904526 |regulation of microtubule binding                                                            |  0.0044922|6.90320240412177   |-3.64012760484658  | 0.3010300|  -36.680089|  0.9235025|      0.4216804|null           |FALSE      |
|GO:1905255 |regulation of RNA binding transcription factor activity                                      |  0.0044922|6.86080915813178   |-3.74809870295359  | 0.3010300|   -5.323769|  0.9235025|      0.4216804|null           |FALSE      |
|GO:0035864 |response to potassium ion                                                                    |  0.0269530|6.01858079262587   |4.15131332515944   | 0.8450980|   -4.534107|  0.9113408|      0.4217509|null           |FALSE      |
|GO:0080021 |response to benzoic acid                                                                     |  0.0044922|5.40346502834963   |4.92305936825371   | 0.3010300|   -5.896024|  0.9183643|      0.4230449|null           |FALSE      |
|GO:1902348 |cellular response to strigolactone                                                           |  0.0044922|5.0442324911475    |4.19122881122183   | 0.3010300|   -2.194635|  0.8861429|      0.4230449|null           |FALSE      |
|GO:0009625 |response to insect                                                                           |  0.1482413|6.8391258043134    |0.404192190458651  | 1.5314789|  -57.851440|  0.9071220|      0.4269324|null           |FALSE      |
|GO:0010221 |negative regulation of vernalization response                                                |  0.0044922|0.404750033484902  |-6.48661606959516  | 0.3010300|   -7.774115|  0.8795791|      0.4281669|null           |FALSE      |
|GO:1901021 |positive regulation of calcium ion transmembrane transporter activity                        |  0.0065885|3.93549626092911   |-5.95279429383104  | 0.3010300|  -13.799813|  0.8550853|      0.4318958|null           |FALSE      |
|GO:0032527 |protein exit from endoplasmic reticulum                                                      |  0.0359373|-6.38084591486064  |-1.43423838044017  | 0.9542425|   -7.057949|  0.9118738|      0.4324736|null           |FALSE      |
|GO:0006432 |phenylalanyl-tRNA aminoacylation                                                             |  0.0134765|-3.7570667063467   |4.67345846737831   | 0.6020600|   -1.736491|  0.8563147|      0.4324757|null           |FALSE      |
|GO:1900706 |positive regulation of siderophore biosynthetic process                                      |  0.1379093|3.24266091877854   |-5.25752367310936  | 1.4913617|   -9.429721|  0.8650472|      0.4414794|null           |FALSE      |
|GO:0000304 |response to singlet oxygen                                                                   |  0.0583981|5.84470472146954   |3.46891005864262   | 1.1461280|  -31.541887|  0.8941146|      0.4434063|null           |FALSE      |
|GO:0035725 |sodium ion transmembrane transport                                                           |  0.0673824|-6.00815767084078  |-2.25045044864326  | 1.2041200|   -9.283878|  0.9116537|      0.4453910|null           |FALSE      |
|GO:0045007 |depurination                                                                                 |  0.1069134|1.3262612223389    |3.47294236493061   | 1.3802112|   -3.935188|  0.8068557|      0.4469741|null           |FALSE      |
|GO:0001101 |response to acid chemical                                                                    |  1.7923723|6.01747824330915   |2.97907457105181   | 2.6020600|   -7.032843|  0.8941018|      0.4485887|null           |FALSE      |
|GO:0009311 |oligosaccharide metabolic process                                                            |  0.4267553|0.176271972124688  |8.18205446182384   | 1.9822712|   -9.223514|  0.9428999|      0.4499218|null           |FALSE      |
|GO:0002011 |morphogenesis of an epithelial sheet                                                         |  0.0134765|-3.35298091616761  |-6.11372616513526  | 0.6020600|   -2.355975|  0.9469313|      0.4499787|null           |FALSE      |
|GO:0034626 |fatty acid elongation,polyunsaturated fatty acid                                             |  0.0179686|-5.63053581494034  |2.98966916336314   | 0.6989700|  -12.617325|  0.8562360|      0.4523009|null           |FALSE      |
|GO:0015843 |methylammonium transport                                                                     |  0.0134765|-6.50357321960934  |-2.13922284911754  | 0.6020600|   -8.488784|  0.9367108|      0.4574417|null           |FALSE      |
|GO:2000083 |negative regulation of L-ascorbic acid biosynthetic process                                  |  0.0044922|3.33651146424447   |-4.81256329236427  | 0.3010300|  -12.868980|  0.8886893|      0.4617814|null           |FALSE      |
|GO:0048240 |sperm capacitation                                                                           |  0.0044922|-3.58034481603477  |-4.66668830120507  | 0.3010300|   -6.220075|  0.8990019|      0.4680595|null           |FALSE      |
|GO:0009972 |cytidine deamination                                                                         |  0.0404295|-4.0759236875963   |4.97998506296254   | 1.0000000|  -14.887132|  0.8485017|      0.4682354|null           |FALSE      |
|GO:0045962 |positive regulation of development,heterochronic                                             |  0.0733720|1.85998667228618   |-3.1131555994635   | 1.2304489|  -26.146973|  0.9050035|      0.4708316|null           |FALSE      |
|GO:0051245 |negative regulation of cellular defense response                                             |  0.0044922|0.319449335291154  |-6.44753765310627  | 0.3010300|  -11.104116|  0.8786469|      0.4718007|null           |FALSE      |
|GO:0052544 |defense response by callose deposition in cell wall                                          |  0.0718746|1.80955768213532   |2.35460652362911   | 1.2304489|  -22.469007|  0.7919110|      0.4735237|null           |FALSE      |
|GO:0006314 |intron homing                                                                                |  0.0224608|-3.13108674229621  |6.07722139434809   | 0.7781513|   -1.438530|  0.8997380|      0.4738444|null           |FALSE      |
|GO:0009644 |response to high light intensity                                                             |  0.2785140|7.31953748891365   |1.92294742036282   | 1.7993405|   -8.889919|  0.9306229|      0.4743964|null           |FALSE      |
|GO:0006885 |regulation of pH                                                                             |  0.2515610|0.264133368298824  |-2.49009426118918  | 1.7558749|  -15.718231|  0.8759538|      0.4774270|null           |FALSE      |
|GO:0050821 |protein stabilization                                                                        |  0.1437492|0.0631288378557764 |-2.27610205306704  | 1.5185139|   -3.583269|  0.8835571|      0.4825149|null           |FALSE      |
|GO:0006672 |ceramide metabolic process                                                                   |  0.1212884|-5.29926120278496  |4.61263064652466   | 1.4471580|   -5.977702|  0.8580154|      0.4834454|null           |FALSE      |
|GO:0034090 |maintenance of meiotic sister chromatid cohesion                                             |  0.0179686|1.301294145471     |5.4737681177984    | 0.6989700|  -11.006435|  0.9020542|      0.4861281|null           |FALSE      |
|GO:0051607 |defense response to virus                                                                    |  0.2246081|6.47965463663451   |0.735020736002424  | 1.7075702|   -2.417605|  0.8968545|      0.4865194|null           |FALSE      |
|GO:2000023 |regulation of lateral root development                                                       |  0.1257805|1.90953922254334   |-3.39575948821504  | 1.4623980|   -2.770101|  0.8880428|      0.4901117|null           |FALSE      |
|GO:0033292 |T-tubule organization                                                                        |  0.0716381|-1.25096489319321  |0.501462882982961  | 1.2041200|  -13.799813|  0.8768482|      0.4928611|null           |FALSE      |
|GO:0051090 |regulation of DNA-binding transcription factor activity                                      |  0.0539059|4.32719230234837   |-4.70588613851849  | 1.1139434|   -3.268476|  0.8894667|      0.4981222|null           |FALSE      |
|GO:0036159 |inner dynein arm assembly                                                                    |  0.0179686|1.17203768507298   |6.2384843834557    | 0.6989700|   -4.478424|  0.9071048|      0.4982292|null           |FALSE      |
|GO:0010150 |leaf senescence                                                                              |  0.5076142|-3.80472371918732  |-5.31573232462745  | 2.0569049|  -22.322993|  0.9137213|      0.5023722|null           |FALSE      |
|GO:0032870 |cellular response to hormone stimulus                                                        |  4.2316158|5.51098150525491   |2.63759161705027   | 2.9745117|  -25.204027|  0.8260952|      0.5032008|null           |FALSE      |
|GO:0010324 |membrane invagination                                                                        |  0.0044922|2.05420717433361   |7.34812899331886   | 0.3010300|   -5.425451|  0.9437924|      0.5057439|null           |FALSE      |
|GO:0009451 |RNA modification                                                                             |  1.4105386|-3.1259234464312   |5.74386125874865   | 2.4983106| -129.609917|  0.8636467|      0.5091922|null           |FALSE      |
|GO:0044245 |polysaccharide digestion                                                                     | 11.9626252|-4.32760364095971  |-5.09877794048353  | 3.4255342|   -8.477229|  0.9150699|      0.5111975|null           |FALSE      |
|GO:0036309 |protein localization to M-band                                                               |  0.2713265|-6.15819412567899  |-1.37338435790419  | 1.7853298|  -13.799813|  0.9057202|      0.5168809|null           |FALSE      |
|GO:0009089 |lysine biosynthetic process via diaminopimelate                                              |  0.0673824|-4.21474499416949  |3.18953565064066   | 1.2041200|   -5.170693|  0.8679299|      0.5195745|null           |FALSE      |
|GO:0070667 |negative regulation of mast cell proliferation                                               |  0.1886708|1.8907059934822    |-4.85526855902306  | 1.6334685|   -5.323769|  0.8816263|      0.5215364|null           |FALSE      |
|GO:0010258 |NADH dehydrogenase complex (plastoquinone) assembly                                          |  0.0359373|1.45872215051145   |6.98769977906038   | 0.9542425|   -7.397841|  0.9251752|      0.5225012|null           |FALSE      |
|GO:0007095 |mitotic G2 DNA damage checkpoint signaling                                                   |  0.0269530|3.65281188028655   |-0.913204900815795 | 0.8450980|   -3.532261|  0.7809686|      0.5255739|null           |FALSE      |
|GO:0009093 |cysteine catabolic process                                                                   |  0.0134765|-4.93480984313561  |3.086497651849     | 0.6020600|   -7.249705|  0.8844616|      0.5312752|null           |FALSE      |
|GO:2000280 |regulation of root development                                                               |  0.3593729|2.02087056639354   |-3.80518902181371  | 1.9084850|  -57.962462|  0.8936830|      0.5325899|null           |FALSE      |
|GO:0051070 |galactomannan biosynthetic process                                                           |  0.0044922|-1.92405541513709  |3.29789303007599   | 0.3010300|   -5.647766|  0.9124074|      0.5326031|null           |FALSE      |
|GO:0010275 |NAD(P)H dehydrogenase complex assembly                                                       |  0.0494138|1.57247413576099   |6.92655891769059   | 1.0791812|  -16.151049|  0.9236632|      0.5344635|null           |FALSE      |
|GO:0009961 |response to 1-aminocyclopropane-1-carboxylic acid                                            |  0.0179686|5.5390944664434    |4.44202986749729   | 0.6989700|   -8.733801|  0.9064142|      0.5385267|null           |FALSE      |
|GO:0071422 |succinate transmembrane transport                                                            |  0.0179686|-6.35378934631128  |-2.31918003762232  | 0.6989700|   -9.471159|  0.9126433|      0.5392725|null           |FALSE      |
|GO:0001561 |fatty acid alpha-oxidation                                                                   |  0.0089843|-6.12118840564167  |2.99418019855899   | 0.4771213|   -2.661712|  0.8657356|      0.5404419|null           |FALSE      |
|GO:0048462 |carpel formation                                                                             |  0.0179686|-3.56530125582914  |-5.21726391017996  | 0.6989700|  -20.294388|  0.9062722|      0.5433997|null           |FALSE      |
|GO:0009617 |response to bacterium                                                                        |  2.1113158|6.42105825561637   |0.950499076186188  | 2.6730209|  -16.475633|  0.8837821|      0.5453319|null           |FALSE      |
|GO:1901684 |arsenate ion transmembrane transport                                                         |  0.0224608|-6.07773307459821  |-2.47666300556564  | 0.7781513|   -1.535372|  0.9151066|      0.5470253|null           |FALSE      |
|GO:2000013 |regulation of arginine biosynthetic process via ornithine                                    |  0.0044922|5.08346573883771   |-4.35859477027144  | 0.3010300|   -2.586543|  0.9127823|      0.5494059|null           |FALSE      |
|GO:0071731 |response to nitric oxide                                                                     |  0.0269530|5.7002505663832    |4.26425683535965   | 0.8450980|   -1.633238|  0.9044707|      0.5514906|null           |FALSE      |
|GO:0000914 |phragmoplast assembly                                                                        |  0.0179686|1.81020176402268   |6.45998754541047   | 0.6989700|   -2.811388|  0.9215273|      0.5551364|null           |FALSE      |
|GO:0035874 |cellular response to copper ion starvation                                                   |  0.0089843|5.52680127522607   |0.708497681922917  | 0.4771213|   -1.331841|  0.8812321|      0.5580156|null           |FALSE      |
|GO:0044403 |biological process involved in symbiotic interaction                                         |  0.4312475|7.59347179462246   |-1.12738441353963  | 1.9867717|  -12.489256|  0.9544023|      0.5582824|null           |FALSE      |
|GO:1990169 |stress response to copper ion                                                                |  0.0224608|6.0713100294263    |3.60503604132747   | 0.7781513|   -3.561907|  0.9064163|      0.5582896|null           |FALSE      |
|GO:0002239 |response to oomycetes                                                                        |  0.4177710|6.69079659160806   |0.64894908797104   | 1.9731279|  -14.752078|  0.8992269|      0.5594759|null           |FALSE      |
|GO:0006468 |protein phosphorylation                                                                      |  5.0671578|-2.42525462752271  |6.09268883907468   | 3.0526939|  -19.955290|  0.8790911|      0.5616227|null           |FALSE      |
|GO:0061723 |glycophagy                                                                                   |  0.0112304|-0.700182353905508 |4.10243969454783   | 0.4771213|   -9.144910|  0.9003162|      0.5621264|null           |FALSE      |
|GO:0000375 |RNA splicing,via transesterification reactions                                               |  1.1140560|-3.39553839420606  |5.58858126268879   | 2.3961993|   -5.862753|  0.8660784|      0.5628313|null           |FALSE      |
|GO:0048509 |regulation of meristem development                                                           |  0.2964826|2.00253017785876   |-3.68518554639384  | 1.8260748|  -24.496572|  0.8951924|      0.5634440|null           |FALSE      |
|GO:1902265 |abscisic acid homeostasis                                                                    |  0.0089843|-0.801132473111927 |-1.61800671909771  | 0.4771213|   -5.647414|  0.9007760|      0.5661740|null           |FALSE      |
|GO:0010469 |regulation of signaling receptor activity                                                    |  0.3059115|1.78663171430724   |-5.66841618016911  | 1.8260748|  -19.981450|  0.8293042|      0.5703573|null           |FALSE      |
|GO:0016099 |monoterpenoid biosynthetic process                                                           |  0.0044922|-6.23886260921351  |2.6799338631602    | 0.3010300|  -37.842515|  0.8749184|      0.5751028|null           |FALSE      |
|GO:0010114 |response to red light                                                                        |  0.2785140|7.30177748659021   |2.02983400748889   | 1.7993405|   -4.245214|  0.9306229|      0.5759002|null           |FALSE      |
|GO:0051258 |protein polymerization                                                                       |  0.1347648|1.55432596292142   |6.80549486587576   | 1.4913617|   -5.247348|  0.9184843|      0.5760110|null           |FALSE      |
|GO:0080003 |thalianol metabolic process                                                                  |  0.0089843|-6.10147129854874  |3.85158542791498   | 0.4771213|  -18.906081|  0.8726602|      0.5772455|null           |FALSE      |
|GO:0046330 |positive regulation of JNK cascade                                                           |  0.0179686|1.60503146251519   |-6.38644547293195  | 0.6989700|   -7.242124|  0.8477208|      0.5779213|null           |FALSE      |
|GO:0097167 |circadian regulation of translation                                                          |  0.0044922|-1.22986207506815  |-4.8187172506083   | 0.3010300|   -2.194635|  0.8691245|      0.5780757|null           |FALSE      |
|GO:0007584 |response to nutrient                                                                         |  0.0583981|6.22386884781849   |2.19141113853406   | 1.1461280|   -4.526773|  0.8948132|      0.5817840|null           |FALSE      |
|GO:1902290 |positive regulation of defense response to oomycetes                                         |  0.0449216|1.45552668953856   |-6.43336765267984  | 1.0413927|   -6.031032|  0.8577687|      0.5823779|null           |FALSE      |
|GO:0061370 |testosterone biosynthetic process                                                            |  0.0867445|-4.83235059439964  |3.90112998245814   | 1.3010300|  -11.851829|  0.8192482|      0.5838530|null           |FALSE      |
|GO:1901599 |(-)-pinoresinol biosynthetic process                                                         |  0.0089843|-4.36337080953627  |3.75681592515697   | 0.4771213|   -1.703570|  0.8599142|      0.5842097|null           |FALSE      |
|GO:0071229 |cellular response to acid chemical                                                           |  0.2740218|5.08750191753486   |2.89439369254601   | 1.7923917|   -7.395724|  0.8669923|      0.5845806|null           |FALSE      |
|GO:0015886 |heme transport                                                                               |  0.0718746|-6.31229773445324  |-2.02186680860245  | 1.2304489|   -2.007792|  0.9289409|      0.5868171|null           |FALSE      |
|GO:0080037 |negative regulation of cytokinin-activated signaling pathway                                 |  0.0224608|1.04610004472128   |-5.97832845084443  | 0.7781513|   -7.201643|  0.8501800|      0.5870088|null           |FALSE      |
|GO:0006812 |cation transport                                                                             |  2.2505728|-6.1092570023931   |-1.99989170036067  | 2.7007037|  -11.834559|  0.9162810|      0.5885695|null           |FALSE      |
|GO:0061760 |antifungal innate immune response                                                            |  0.0044922|6.91422839762305   |-0.171195135840574 | 0.3010300|  -14.684490|  0.9157478|      0.5885794|null           |FALSE      |
|GO:0033007 |negative regulation of mast cell activation involved in immune response                      |  0.0898432|1.81563501548017   |-5.79781471223215  | 1.3222193|   -5.323769|  0.8340324|      0.5888528|null           |FALSE      |
|GO:0003341 |cilium movement                                                                              |  0.0059895|-2.74422820333344  |-0.899239043295222 | 0.3010300|  -10.740017|  0.9554976|      0.5897750|null           |FALSE      |
|GO:0048587 |regulation of short-day photoperiodism,flowering                                             |  0.0494138|0.485295849739703  |-6.22425634724235  | 1.0791812|   -4.157373|  0.8805683|      0.5918906|null           |FALSE      |
|GO:0007165 |signal transduction                                                                          |  8.3913571|4.10212724793466   |-1.2279098156215   | 3.2716093| -124.739426|  0.7370332|      0.5942163|null           |FALSE      |
|GO:0009664 |plant-type cell wall organization                                                            |  0.6693320|1.6001334738349    |6.4189983563857    | 2.1760913|   -9.877092|  0.9183848|      0.5956427|null           |FALSE      |
|GO:0060294 |cilium movement involved in cell motility                                                    |  0.0059895|-5.16169185703137  |-1.52020617688361  | 0.4771213|   -8.400755|  0.9160881|      0.5985002|null           |FALSE      |
|GO:0006164 |purine nucleotide biosynthetic process                                                       |  0.5390593|-4.05327889527339  |4.4931991612705    | 2.0827854|   -9.283372|  0.8166479|      0.6003424|null           |FALSE      |
|GO:0098869 |cellular oxidant detoxification                                                              |  0.7591752|5.32626237077978   |2.71667340010043   | 2.2304489|   -3.378602|  0.8585189|      0.6019382|null           |FALSE      |
|GO:0009684 |indoleacetic acid biosynthetic process                                                       |  0.0494138|-3.09890515590602  |2.82073634183668   | 1.0791812|   -7.549382|  0.7387055|      0.6020967|null           |FALSE      |
|GO:0080181 |lateral root branching                                                                       |  0.0134765|-3.71551777728719  |-5.58722825975627  | 0.6020600|   -7.129021|  0.9239176|      0.6026219|null           |FALSE      |
|GO:0071465 |cellular response to desiccation                                                             |  0.0763667|5.55911906292601   |2.01380973247686   | 1.2552725|   -4.454452|  0.8246449|      0.6040405|null           |FALSE      |
|GO:0009877 |nodulation                                                                                   |  0.0988275|-3.49666403458119  |-5.9529410460003   | 1.3617278|  -19.544215|  0.9358429|      0.6076887|null           |FALSE      |
|GO:0010423 |negative regulation of brassinosteroid biosynthetic process                                  |  0.0044922|1.12013555071045   |-3.79940524742264  | 0.3010300|   -2.632486|  0.8542806|      0.6098624|null           |FALSE      |
|GO:0071236 |cellular response to antibiotic                                                              |  0.8690604|5.26226874855774   |2.65455272109281   | 2.2878017|  -12.890584|  0.8587557|      0.6122276|null           |FALSE      |
|GO:0030639 |polyketide biosynthetic process                                                              |  0.0179686|-4.70166196175891  |1.8699753645666    | 0.6989700|   -6.693629|  0.9163210|      0.6137667|null           |FALSE      |
|GO:0000350 |generation of catalytic spliceosome for second transesterification step                      |  0.0134765|-1.03533443038387  |5.64120237693714   | 0.6020600|   -1.856958|  0.8543752|      0.6146354|null           |FALSE      |
|GO:0010930 |negative regulation of auxin mediated signaling pathway                                      |  0.0134765|0.947717623231213  |-6.10375165773611  | 0.6020600|   -2.569735|  0.8546739|      0.6147577|null           |FALSE      |
|GO:0080126 |ovary septum development                                                                     |  0.0044922|-3.76934798550313  |-5.03395746705032  | 0.3010300|  -15.121344|  0.9267345|      0.6149303|null           |FALSE      |
|GO:0006264 |mitochondrial DNA replication                                                                |  0.0404295|-2.56721872853072  |3.33278360388031   | 1.0000000|   -2.425188|  0.8979845|      0.6158060|null           |FALSE      |
|GO:0015709 |thiosulfate transport                                                                        |  0.0134765|-6.22630469157984  |-2.57621703971582  | 0.6020600|  -17.014164|  0.9349286|      0.6161640|null           |FALSE      |
|GO:0010376 |stomatal complex formation                                                                   |  0.0224608|-3.60581127100575  |-5.57998032916261  | 0.7781513|   -6.623515|  0.9162003|      0.6165396|null           |FALSE      |
|GO:0071211 |protein targeting to vacuole involved in autophagy                                           |  0.0089843|-5.91593117083588  |-0.549240532907634 | 0.4771213|   -8.533431|  0.8689251|      0.6223831|null           |FALSE      |
|GO:0009099 |valine biosynthetic process                                                                  |  0.0404295|-4.33183104993993  |3.10649489820053   | 1.0000000|   -2.075832|  0.8721197|      0.6234625|null           |FALSE      |
|GO:0071446 |cellular response to salicylic acid stimulus                                                 |  0.2605453|5.29192736898411   |3.21453969138425   | 1.7708520|  -19.892445|  0.8536174|      0.6241860|null           |FALSE      |
|GO:0045926 |negative regulation of growth                                                                |  0.1751943|1.22805069939994   |-4.84768563561718  | 1.6020600|  -49.287844|  0.8851154|      0.6297772|null           |FALSE      |
|GO:0048584 |positive regulation of response to stimulus                                                  |  1.1320246|1.882000993486     |-5.71263879017167  | 2.4031205|   -3.398922|  0.8315580|      0.6332632|null           |FALSE      |
|GO:0000028 |ribosomal small subunit assembly                                                             |  0.1527335|1.39788636708621   |6.74159430886883   | 1.5440680|   -1.314658|  0.9110900|      0.6348736|null           |FALSE      |
|GO:0090057 |root radial pattern formation                                                                |  0.0314451|-3.98174112940758  |-5.43479017004204  | 0.9030900|   -7.618934|  0.9266296|      0.6353165|null           |FALSE      |
|GO:0080110 |sporopollenin biosynthetic process                                                           |  0.0359373|-2.37580027783384  |2.27388929757335   | 0.9542425|   -4.321347|  0.7717040|      0.6360522|null           |FALSE      |
|GO:0015700 |arsenite transport                                                                           |  0.0224608|-6.27537462899896  |-2.48371928892091  | 0.7781513|   -7.398065|  0.9329468|      0.6364426|null           |FALSE      |
|GO:0005995 |melibiose catabolic process                                                                  |  0.0157226|-0.405242060361325 |7.65867409903928   | 0.6020600|   -2.194635|  0.9211093|      0.6387332|null           |FALSE      |
|GO:0042773 |ATP synthesis coupled electron transport                                                     |  0.2830062|0.0857590510320464 |1.32043595815285   | 1.8061800|   -6.661836|  0.9341820|      0.6395535|null           |FALSE      |
|GO:0035024 |negative regulation of Rho protein signal transduction                                       |  0.0044922|0.796387131021064  |-6.32981523507441  | 0.3010300|   -2.943564|  0.8613183|      0.6447047|null           |FALSE      |
|GO:0030994 |primary cell septum disassembly                                                              |  0.0817573|2.00450738046111   |6.75437600669182   | 1.2787536|   -5.157608|  0.9276838|      0.6452209|null           |FALSE      |
|GO:1905034 |regulation of antifungal innate immune response                                              |  0.0089843|0.126190311752538  |-6.73997897457728  | 0.4771213|   -3.767121|  0.8883255|      0.6459136|null           |FALSE      |
|GO:0071452 |cellular response to singlet oxygen                                                          |  0.0359373|5.42119091761929   |3.15626802483157   | 0.9542425|   -9.208437|  0.8623964|      0.6459430|null           |FALSE      |
|GO:0010500 |transmitting tissue development                                                              |  0.0179686|-3.68411293115031  |-4.98963023406633  | 0.6989700|   -8.013000|  0.9225241|      0.6473838|null           |FALSE      |
|GO:0010337 |regulation of salicylic acid metabolic process                                               |  0.0943354|3.77617418398996   |-4.52092791664207  | 1.3424227|  -10.612594|  0.8971729|      0.6479341|null           |FALSE      |
|GO:0006203 |dGTP catabolic process                                                                       |  0.0134765|-4.30495774811181  |4.88095913460574   | 0.6020600|   -1.438530|  0.8543665|      0.6484478|null           |FALSE      |
|GO:0045490 |pectin catabolic process                                                                     |  0.4447240|-1.20173301639332  |6.64272719947403   | 2.0000000|   -3.160321|  0.9098279|      0.6495072|null           |FALSE      |
|GO:0035246 |peptidyl-arginine N-methylation                                                              |  0.0539059|-1.81455114049211  |7.32988347530565   | 1.1139434|   -3.309595|  0.9192475|      0.6502278|null           |FALSE      |
|GO:0015708 |silicic acid import across plasma membrane                                                   |  0.0331729|-6.15179312238278  |-2.39677007332359  | 0.9030900|   -1.703570|  0.9104713|      0.6528448|null           |FALSE      |
|GO:0042742 |defense response to bacterium                                                                |  1.6216702|6.30164966202894   |1.0634535656645    | 2.5587086|   -8.042952|  0.8733747|      0.6545694|null           |FALSE      |
|GO:0012502 |induction of programmed cell death                                                           |  0.0044922|3.1777021842781    |-6.5458675920047   | 0.3010300|   -8.366183|  0.8981821|      0.6549617|null           |FALSE      |
|GO:1902042 |negative regulation of extrinsic apoptotic signaling pathway via death domain receptors      |  0.0044922|1.08539845075955   |-6.37014371249228  | 0.3010300|   -3.702997|  0.8570216|      0.6549617|null           |FALSE      |
|GO:0016127 |sterol catabolic process                                                                     |  0.0089843|-5.42030979984301  |4.26201602531894   | 0.4771213|   -3.230535|  0.8669013|      0.6550754|null           |FALSE      |
|GO:0034434 |sterol esterification                                                                        |  0.0089843|-6.00567714850397  |4.0318589740568    | 0.4771213|   -3.230535|  0.8773626|      0.6550754|null           |FALSE      |
|GO:0046622 |positive regulation of organ growth                                                          |  0.0269530|2.86418138409539   |-6.02232962796556  | 0.8450980|   -6.195458|  0.8717650|      0.6613690|null           |FALSE      |
|GO:0030150 |protein import into mitochondrial matrix                                                     |  0.1257805|-4.83999266938694  |-0.459874304247044 | 1.4623980|   -1.657532|  0.8584109|      0.6622637|null           |FALSE      |
|GO:0000226 |microtubule cytoskeleton organization                                                        |  0.7232379|1.19579564194398   |6.04276289788669   | 2.2095150|   -2.393661|  0.9041823|      0.6711268|null           |FALSE      |
|GO:0045338 |farnesyl diphosphate metabolic process                                                       |  0.0898432|-6.8437192858649   |2.74915994616036   | 1.3222193|  -20.990397|  0.8699702|      0.6735153|null           |FALSE      |
|GO:0046949 |fatty-acyl-CoA biosynthetic process                                                          |  0.0044922|-4.45998621939071  |4.31118869531821   | 0.3010300|   -1.706168|  0.8532950|      0.6744170|null           |FALSE      |
|GO:0070562 |regulation of vitamin D receptor signaling pathway                                           |  0.1497720|1.2222528102327    |-6.02676032763539  | 1.5314789|   -5.304743|  0.8499436|      0.6776065|null           |FALSE      |
|GO:0080143 |regulation of amino acid export                                                              |  0.0314451|4.47477568623369   |-5.72302780445155  | 0.9030900|  -13.226843|  0.8818763|      0.6811918|null           |FALSE      |
|GO:0071368 |cellular response to cytokinin stimulus                                                      |  0.3593729|5.20590857163239   |2.91923124267233   | 1.9084850|   -8.168346|  0.8598163|      0.6812564|null           |FALSE      |
|GO:0048385 |regulation of retinoic acid receptor signaling pathway                                       |  0.1668089|1.29450876414118   |-5.94731742077003  | 1.5797836|   -5.016143|  0.8528541|      0.6835973|null           |FALSE      |
|GO:0050829 |defense response to Gram-negative bacterium                                                  |  0.0988275|6.57160666460941   |0.510917153404676  | 1.3617278|   -2.811388|  0.8987720|      0.6840535|null           |FALSE      |
|GO:0051315 |attachment of mitotic spindle microtubules to kinetochore                                    |  0.0179686|-4.31471802148102  |-0.286028819132361 | 0.6989700|   -1.830805|  0.8685047|      0.6841486|null           |FALSE      |
|GO:0009852 |auxin catabolic process                                                                      |  0.0089843|-1.17787193654463  |-0.382540418791701 | 0.4771213|   -7.282761|  0.8347516|      0.6856031|null           |FALSE      |
|GO:1900449 |regulation of glutamate receptor signaling pathway                                           |  0.1735903|1.25423725231371   |-5.8874160156306   | 1.5682017|   -3.702997|  0.8524428|      0.6858401|null           |FALSE      |
|GO:2000028 |regulation of photoperiodism,flowering                                                       |  0.3324199|0.929419251637051  |-5.76303482209968  | 1.8750613|   -1.891045|  0.8637784|      0.6877818|null           |FALSE      |
|GO:0006722 |triterpenoid metabolic process                                                               |  0.0898432|-7.1392359497773   |2.27552404975817   | 1.3222193|   -3.946282|  0.8793921|      0.6931108|null           |FALSE      |
|GO:0006813 |potassium ion transport                                                                      |  0.4222632|-6.08391457587327  |-2.1121633988493   | 1.9777236|   -8.845589|  0.9242419|      0.6936154|null           |FALSE      |
|GO:0019371 |cyclooxygenase pathway                                                                       |  0.0044922|-5.7355788912889   |2.96204015060261   | 0.3010300|   -2.661712|  0.8661318|      0.6936856|null           |FALSE      |
|GO:0071395 |cellular response to jasmonic acid stimulus                                                  |  0.4222632|5.37836217037839   |3.0097712984399    | 1.9777236|  -25.690645|  0.8447806|      0.6939872|null           |FALSE      |
|GO:0001680 |tRNA 3&#39;-terminal CCA addition                                                                |  0.0224608|-4.15526636344438  |5.92561724076363   | 0.7781513|   -7.087420|  0.8948854|      0.6940175|null           |FALSE      |
|GO:0048767 |root hair elongation                                                                         |  0.3099591|-3.62041920799437  |-5.37435783070075  | 1.8450980|   -7.087678|  0.8687271|      0.6947824|null           |FALSE      |
|GO:0071281 |cellular response to iron ion                                                                |  0.0089843|5.13775773118928   |3.81006523883331   | 0.4771213|   -4.876646|  0.8837993|      0.6952859|null           |FALSE      |

[1] &quot;missing_aquatics_GO.txt&quot;


|TermID     |Name                                                                                         |  Frequency|PlotX              |PlotY              |   LogSize|       Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:--------------------------------------------------------------------------------------------|----------:|:------------------|:------------------|---------:|-----------:|----------:|--------------:|:--------------|:----------|
|GO:0009819 |drought recovery                                                                             |  0.0494138|6.41123789472825   |-3.50249647443537  | 1.0791812|  -34.412318|  0.8639859|      0.0000000|null           |FALSE      |
|GO:0010082 |regulation of root meristem growth                                                           |  0.1527335|-1.5838397396635   |0.615137880533818  | 1.5440680| -104.101084|  0.7320100|      0.0000000|null           |FALSE      |
|GO:0051762 |sesquiterpene biosynthetic process                                                           |  0.1123040|-5.31177804181081  |3.79569880459435   | 1.4149733| -140.122519|  0.8645610|      0.0000000|null           |FALSE      |
|GO:0006335 |DNA replication-dependent chromatin assembly                                                 |  0.0179686|6.10228152671637   |4.28456894140077   | 0.6989700|   -3.563722|  0.9488253|      0.0215931|null           |FALSE      |
|GO:0036371 |protein localization to T-tubule                                                             |  0.0314451|3.2108988620676    |5.56872992710472   | 0.9030900|  -20.442318|  0.9326830|      0.0224064|null           |FALSE      |
|GO:0051554 |flavonol metabolic process                                                                   |  0.0853511|2.75455006461683   |3.2985202871221    | 1.3010300|  -36.919497|  0.9565730|      0.0732686|null           |FALSE      |
|GO:0000966 |RNA 5&#39;-end processing                                                                        |  0.0808589|-6.57809005344708  |1.37970995529462   | 1.2787536|  -54.651110|  0.9320965|      0.0829109|null           |FALSE      |
|GO:0009311 |oligosaccharide metabolic process                                                            |  0.4267553|4.73792262116857   |3.72097938286709   | 1.9822712|   -4.513778|  0.9631681|      0.0941713|null           |FALSE      |
|GO:0097503 |sialylation                                                                                  |  0.0134765|-7.36399166771239  |-0.914300751021861 | 0.6020600|   -2.841938|  0.9705268|      0.1067597|null           |FALSE      |
|GO:0031146 |SCF-dependent proteasomal ubiquitin-dependent protein catabolic process                      |  0.3773415|-6.76034617700914  |0.0256685530855221 | 1.9294189|  -49.521214|  0.9251523|      0.1347712|null           |FALSE      |
|GO:0006885 |regulation of pH                                                                             |  0.2515610|-4.68942015490961  |-2.5638041492487   | 1.7558749|  -21.919135|  0.8488315|      0.1360520|null           |FALSE      |
|GO:0009962 |regulation of flavonoid biosynthetic process                                                 |  0.1257805|-4.16634948589822  |-5.19583615512023  | 1.4623980|  -25.385631|  0.8919468|      0.1471751|null           |FALSE      |
|GO:0010469 |regulation of signaling receptor activity                                                    |  0.3059115|-1.12470240212087  |-5.48861040892998  | 1.8260748|  -34.419769|  0.8068734|      0.1578322|null           |FALSE      |
|GO:0009617 |response to bacterium                                                                        |  2.1113158|6.36874374069356   |-1.15540275757272  | 2.6730209|  -28.320920|  0.8416671|      0.1641436|null           |FALSE      |
|GO:0010942 |positive regulation of cell death                                                            |  0.1527335|-0.123359015697467 |-6.04414729186773  | 1.5440680|  -10.877695|  0.8489993|      0.1784806|null           |FALSE      |
|GO:0060294 |cilium movement involved in cell motility                                                    |  0.0059895|4.11137591090519   |5.68275119702454   | 0.4771213|  -16.454330|  0.9246009|      0.1997941|null           |FALSE      |
|GO:0080184 |response to phenylpropanoid                                                                  |  0.0044922|5.60006531603148   |-5.68072254730973  | 0.3010300|  -12.881606|  0.8966471|      0.2014785|null           |FALSE      |
|GO:0010310 |regulation of hydrogen peroxide metabolic process                                            |  0.0539059|-0.982980493149573 |-3.95249688471709  | 1.1139434|   -5.241183|  0.8943056|      0.2101184|null           |FALSE      |
|GO:2000083 |negative regulation of L-ascorbic acid biosynthetic process                                  |  0.0044922|-2.92629651549028  |-6.84838973025583  | 0.3010300|  -19.161283|  0.8895867|      0.2131819|null           |FALSE      |
|GO:0071466 |cellular response to xenobiotic stimulus                                                     |  0.0224608|5.34236520420663   |-4.74764924972049  | 0.7781513|   -9.352212|  0.8664891|      0.2217277|null           |FALSE      |
|GO:0097167 |circadian regulation of translation                                                          |  0.0044922|-0.814594875172545 |0.0971000232338532 | 0.3010300|   -5.541363|  0.8421098|      0.2266615|null           |FALSE      |
|GO:0009410 |response to xenobiotic stimulus                                                              |  0.0359373|6.04067325350241   |-5.03916353990849  | 0.9542425|   -9.193698|  0.8985267|      0.2284321|null           |FALSE      |
|GO:1902025 |nitrate import                                                                               |  0.0808589|2.54739571697853   |6.26482657785928   | 1.2787536|  -16.147194|  0.9578419|      0.2340388|null           |FALSE      |
|GO:0009611 |response to wounding                                                                         |  1.0062441|6.7811573777506    |-2.17041321129501  | 2.3521825|   -7.598088|  0.8882125|      0.2425631|null           |FALSE      |
|GO:0009865 |pollen tube adhesion                                                                         |  0.0134765|-0.113772218642432 |2.84949109710103   | 0.6020600|   -5.970647|  0.8594145|      0.2428352|null           |FALSE      |
|GO:0019756 |cyanogenic glycoside biosynthetic process                                                    |  0.1069562|-6.28479821968428  |1.95207621246638   | 1.4313638|  -34.342172|  0.8892980|      0.2582345|null           |FALSE      |
|GO:0033292 |T-tubule organization                                                                        |  0.0716381|-0.165558993934286 |5.25374494488186   | 1.2041200|  -20.442318|  0.8637325|      0.2722071|null           |FALSE      |
|GO:0010233 |phloem transport                                                                             |  0.0943354|1.27882076751497   |4.91620584015071   | 1.3424227|   -4.279925|  0.8416486|      0.2779671|null           |FALSE      |
|GO:0006898 |receptor-mediated endocytosis                                                                |  0.1976551|2.86272795997995   |6.2704915001059    | 1.6532125|   -2.571880|  0.9422735|      0.2825621|null           |FALSE      |
|GO:0001101 |response to acid chemical                                                                    |  1.7923723|6.08231519170314   |-3.55157668412712  | 2.6020600|   -8.247666|  0.8598671|      0.3051929|null           |FALSE      |
|GO:0010305 |leaf vascular tissue pattern formation                                                       |  0.1347648|-0.906564569224176 |4.93061592506114   | 1.4913617|   -2.850372|  0.8854660|      0.3105240|null           |FALSE      |
|GO:0010114 |response to red light                                                                        |  0.2785140|4.98002393496127   |-0.398791997470481 | 1.7993405|  -10.626773|  0.9133953|      0.3255957|null           |FALSE      |
|GO:0016567 |protein ubiquitination                                                                       |  3.7105251|-6.7543892444314   |0.422967480880486  | 2.9175055|  -25.711249|  0.9233067|      0.3270943|null           |FALSE      |
|GO:0048312 |intracellular distribution of mitochondria                                                   |  0.0044922|4.00771856343661   |5.14118151603913   | 0.3010300|   -2.509467|  0.9258702|      0.3293628|null           |FALSE      |
|GO:0032504 |multicellular organism reproduction                                                          |  0.8040969|-0.232673573616295 |3.76272817820022   | 2.2552725|   -3.443456|  0.8974981|      0.3415835|null           |FALSE      |
|GO:1903553 |positive regulation of extracellular exosome assembly                                        |  0.0044922|1.13311308600194   |-7.53954720658183  | 0.3010300|   -2.509467|  0.8744927|      0.3498300|null           |FALSE      |
|GO:0031542 |positive regulation of anthocyanin biosynthetic process                                      |  0.0179686|0.606882207217044  |-6.96199553378086  | 0.6989700|   -3.567657|  0.8767168|      0.3513679|null           |FALSE      |
|GO:0035600 |tRNA methylthiolation                                                                        |  0.0089843|-6.69688653225285  |0.936739788143396  | 0.4771213|   -3.409318|  0.9360372|      0.3671116|null           |FALSE      |
|GO:0010275 |NAD(P)H dehydrogenase complex assembly                                                       |  0.0494138|6.29147468403264   |4.24402368511241   | 1.0791812|   -1.784576|  0.9450963|      0.3768295|null           |FALSE      |
|GO:0009867 |jasmonic acid mediated signaling pathway                                                     |  0.3998023|2.93523510104014   |-3.81009462063803  | 1.9542425|  -28.343365|  0.7074973|      0.3795777|null           |FALSE      |
|GO:0045314 |regulation of compound eye photoreceptor development                                         |  0.0044922|-4.82877273298959  |-4.60279364661322  | 0.3010300|   -2.388180|  0.8774899|      0.3911017|null           |FALSE      |
|GO:0090559 |regulation of membrane permeability                                                          |  0.0089843|-6.08201932329568  |-2.37492294406639  | 0.4771213|   -5.932708|  0.8819490|      0.3950913|null           |FALSE      |
|GO:0010618 |aerenchyma formation                                                                         |  0.0089843|-2.35846609024393  |5.47056793112917   | 0.4771213|  -20.647083|  0.9230621|      0.3960626|null           |FALSE      |
|GO:0006355 |regulation of transcription,DNA-templated                                                    | 10.5880239|-1.76436332475812  |-4.50013711734362  | 3.3725438|   -2.480747|  0.8328582|      0.3986838|null           |FALSE      |
|GO:0007267 |cell-cell signaling                                                                          |  0.3414042|3.70166741100928   |-7.27358093441348  | 1.8864907|   -4.559472|  0.9226436|      0.4013598|null           |FALSE      |
|GO:0007584 |response to nutrient                                                                         |  0.0583981|6.59952820121774   |-2.66924063777237  | 1.1461280|   -5.303508|  0.8636664|      0.4049784|null           |FALSE      |
|GO:0007369 |gastrulation                                                                                 |  0.0134765|-1.12459885186962  |5.31706205278226   | 0.6020600|   -2.330386|  0.8906580|      0.4064931|null           |FALSE      |
|GO:0034394 |protein localization to cell surface                                                         |  0.0134765|3.33581756636322   |5.79652724463989   | 0.6020600|   -8.422620|  0.9393170|      0.4074874|null           |FALSE      |
|GO:0016045 |detection of bacterium                                                                       |  0.0134765|7.07160393693865   |0.155176055816696  | 0.6020600|   -7.398400|  0.8830454|      0.4090250|null           |FALSE      |
|GO:0006952 |defense response                                                                             |  7.2503481|5.90708039135803   |-2.17348437523622  | 3.2081725| -112.187033|  0.8624958|      0.4100881|null           |FALSE      |
|GO:0010221 |negative regulation of vernalization response                                                |  0.0044922|-1.79418068439216  |-7.41276902181564  | 0.3010300|  -13.569914|  0.8696346|      0.4116470|null           |FALSE      |
|GO:0090615 |mitochondrial mRNA processing                                                                |  0.0539059|-6.70183309767378  |1.30278785729848   | 1.1139434|  -34.286111|  0.9298481|      0.4120236|null           |FALSE      |
|GO:0009871 |jasmonic acid and ethylene-dependent systemic resistance,ethylene mediated signaling pathway |  0.0089843|4.22110964763431   |-2.78586576861356  | 0.4771213|  -11.901183|  0.7436290|      0.4175401|null           |FALSE      |
|GO:0001666 |response to hypoxia                                                                          |  1.1499933|6.26363183159571   |-2.02531372071015  | 2.4099331|   -2.354401|  0.8821888|      0.4246154|null           |FALSE      |
|GO:0007166 |cell surface receptor signaling pathway                                                      |  0.5794888|2.20086094748107   |-3.56916025281425  | 2.1139434|   -9.350409|  0.7520413|      0.4253892|null           |FALSE      |
|GO:0044364 |disruption of cells of other organism                                                        |  0.0269530|7.49462338931481   |1.29128106046712   | 0.8450980|   -2.509467|  0.9414822|      0.4301109|null           |FALSE      |
|GO:1904526 |regulation of microtubule binding                                                            |  0.0044922|-3.37971246638296  |-0.741204757203643 | 0.3010300|   -9.815879|  0.9064320|      0.4321507|null           |FALSE      |
|GO:1905255 |regulation of RNA binding transcription factor activity                                      |  0.0044922|-3.63310350310737  |-0.862980084880104 | 0.3010300|   -9.134702|  0.9064320|      0.4321507|null           |FALSE      |
|GO:0003283 |atrial septum development                                                                    |  1.6401201|-1.31989529742254  |4.42307892569543   | 3.2564772|   -6.840108|  0.8381908|      0.4359177|null           |FALSE      |
|GO:1901021 |positive regulation of calcium ion transmembrane transporter activity                        |  0.0065885|0.402094965740273  |-6.04958024551012  | 0.3010300|  -20.442318|  0.8311122|      0.4428862|null           |FALSE      |
|GO:0000304 |response to singlet oxygen                                                                   |  0.0583981|6.42308258031725   |-3.75808418570044  | 1.1461280|  -17.925687|  0.8654510|      0.4434063|null           |FALSE      |
|GO:0009684 |indoleacetic acid biosynthetic process                                                       |  0.0494138|-5.22250041375247  |-0.190473481507155 | 1.0791812|  -11.837791|  0.7597786|      0.4446232|null           |FALSE      |
|GO:0097185 |cellular response to azide                                                                   |  0.0561520|5.56911288825511   |-4.37894205464189  | 1.1139434|  -14.134822|  0.8494849|      0.4472027|null           |FALSE      |
|GO:0002011 |morphogenesis of an epithelial sheet                                                         |  0.0134765|-1.82893119102762  |5.70064678321407   | 0.6020600|   -7.657648|  0.9052186|      0.4499787|null           |FALSE      |
|GO:0007492 |endoderm development                                                                         |  0.0134765|-2.02413327302933  |5.67679773475108   | 0.6020600|   -2.509467|  0.9235035|      0.4499787|null           |FALSE      |
|GO:0009745 |sucrose mediated signaling                                                                   |  0.0089843|3.71961396227966   |-4.4442618359608   | 0.4771213|   -3.552022|  0.7787014|      0.4555919|null           |FALSE      |
|GO:0071218 |cellular response to misfolded protein                                                       |  0.1033197|5.37516524378233   |-3.12145117758362  | 1.3802112|   -4.517344|  0.8249461|      0.4588220|null           |FALSE      |
|GO:0046482 |para-aminobenzoic acid metabolic process                                                     |  0.0179686|-5.78541565816408  |2.01734236121055   | 0.6989700|   -1.352411|  0.9166516|      0.4635037|null           |FALSE      |
|GO:0019722 |calcium-mediated signaling                                                                   |  0.3279278|2.34438629818429   |-3.64538990243955  | 1.8692317|   -2.188992|  0.7569337|      0.4636835|null           |FALSE      |
|GO:0071236 |cellular response to antibiotic                                                              |  0.8690604|5.463545834703     |-3.53755081917077  | 2.2878017|  -17.414451|  0.8232939|      0.4715434|null           |FALSE      |
|GO:0036092 |phosphatidylinositol-3-phosphate biosynthetic process                                        |  0.0359373|-5.3936539407807   |3.64155627028815   | 0.9542425|   -1.513457|  0.8897354|      0.4735951|null           |FALSE      |
|GO:0080027 |response to herbivore                                                                        |  0.0808589|6.85993448179526   |-0.265143524434894 | 1.2787536|  -13.062940|  0.8800541|      0.4757830|null           |FALSE      |
|GO:1902290 |positive regulation of defense response to oomycetes                                         |  0.0449216|-0.384588117546344 |-6.9803852169137   | 1.0413927|  -14.872143|  0.8393162|      0.4818706|null           |FALSE      |
|GO:0050821 |protein stabilization                                                                        |  0.1437492|-5.03968137153484  |-2.54726234545553  | 1.5185139|   -3.614097|  0.8589013|      0.4825149|null           |FALSE      |
|GO:0002213 |defense response to insect                                                                   |  0.0943354|6.50302263785761   |-0.605785709118971 | 1.3424227|   -8.013172|  0.8663284|      0.4825590|null           |FALSE      |
|GO:0045168 |cell-cell signaling involved in cell fate commitment                                         |  0.1662100|-0.637733139427883 |2.87825496839766   | 1.5797836|  -20.327221|  0.8348133|      0.4928611|null           |FALSE      |
|GO:0036309 |protein localization to M-band                                                               |  0.2713265|3.06278413609119   |5.68678859679335   | 1.7853298|  -20.442318|  0.9245556|      0.4950361|null           |FALSE      |
|GO:0036159 |inner dynein arm assembly                                                                    |  0.0179686|5.75660289533656   |4.67762062726481   | 0.6989700|  -10.806071|  0.9273301|      0.4957761|null           |FALSE      |
|GO:0032527 |protein exit from endoplasmic reticulum                                                      |  0.0359373|3.0882908417516    |5.87587627736175   | 0.9542425|   -2.917539|  0.9335864|      0.4998121|null           |FALSE      |
|GO:0010150 |leaf senescence                                                                              |  0.5076142|-1.15795798271104  |4.38413663246827   | 2.0569049|   -1.519786|  0.8693018|      0.5023722|null           |FALSE      |
|GO:0009625 |response to insect                                                                           |  0.1482413|6.90322499876334   |-0.531823355187328 | 1.5314789|  -27.178894|  0.8743942|      0.5035878|null           |FALSE      |
|GO:0061157 |mRNA destabilization                                                                         |  0.2156237|-2.05150821604518  |-5.01438244020879  | 1.6901961|   -4.106738|  0.7720469|      0.5065480|null           |FALSE      |
|GO:0097237 |cellular response to toxic substance                                                         |  0.8355420|5.34837391471004   |-3.49490306187107  | 2.2718416|   -6.813398|  0.8210195|      0.5080142|null           |FALSE      |
|GO:0052544 |defense response by callose deposition in cell wall                                          |  0.0718746|4.765533213343     |1.02068185570525   | 1.2304489|  -18.498763|  0.7757015|      0.5168809|null           |FALSE      |
|GO:0051607 |defense response to virus                                                                    |  0.2246081|6.42621216562704   |-0.868304397202835 | 1.7075702|   -5.648221|  0.8570249|      0.5246042|null           |FALSE      |
|GO:0035024 |negative regulation of Rho protein signal transduction                                       |  0.0044922|-1.60939977892779  |-7.1790622738816   | 0.3010300|   -8.593483|  0.8518081|      0.5272151|null           |FALSE      |
|GO:0009960 |endosperm development                                                                        |  0.1437492|-1.51922118856184  |4.71648943059576   | 1.5185139|   -2.180950|  0.8682090|      0.5317308|null           |FALSE      |
|GO:2000280 |regulation of root development                                                               |  0.3593729|-3.39772972733218  |-4.23620568408617  | 1.9084850|  -81.374590|  0.8551680|      0.5325899|null           |FALSE      |
|GO:0009627 |systemic acquired resistance                                                                 |  0.2919905|6.56997003848928   |-1.04262318855007  | 1.8195439|   -2.073394|  0.8539495|      0.5388023|null           |FALSE      |
|GO:0071368 |cellular response to cytokinin stimulus                                                      |  0.3593729|5.29735664023642   |-3.84407220941963  | 1.9084850|  -16.910416|  0.8196629|      0.5432496|null           |FALSE      |
|GO:0009626 |plant-type hypersensitive response                                                           |  0.3279278|5.77619192042941   |-1.54457282753411  | 1.8692317|  -23.662726|  0.7915493|      0.5453319|null           |FALSE      |
|GO:2000027 |regulation of animal organ morphogenesis                                                     |  0.2030457|-3.77669203070285  |-4.35347285231625  | 1.6627578|   -2.298305|  0.8593385|      0.5454109|null           |FALSE      |
|GO:0060465 |pharynx development                                                                          |  1.4053739|-1.24691968226032  |4.5827270758142    | 2.4927604|   -2.509467|  0.8577111|      0.5514316|null           |FALSE      |
|GO:0002239 |response to oomycetes                                                                        |  0.4177710|6.7666165829988    |-0.830803816900775 | 1.9731279|   -3.344113|  0.8633692|      0.5594759|null           |FALSE      |
|GO:0006468 |protein phosphorylation                                                                      |  5.0671578|-6.12790187506624  |1.25816373405905   | 3.0526939|  -25.501966|  0.9086131|      0.5616227|null           |FALSE      |
|GO:0048509 |regulation of meristem development                                                           |  0.2964826|-3.55226819637158  |-4.27463270008001  | 1.8260748|  -47.797196|  0.8572095|      0.5634440|null           |FALSE      |
|GO:1902265 |abscisic acid homeostasis                                                                    |  0.0089843|-5.92870047771504  |-2.18731876836311  | 0.4771213|   -7.025493|  0.8783736|      0.5661740|null           |FALSE      |
|GO:0010930 |negative regulation of auxin mediated signaling pathway                                      |  0.0134765|-1.65706651832865  |-6.86439951267889  | 0.6020600|   -8.423030|  0.8440504|      0.5666125|null           |FALSE      |
|GO:0009751 |response to salicylic acid                                                                   |  0.7232379|5.99448073956404   |-4.03255968848457  | 2.2095150|   -2.619993|  0.8445306|      0.5669335|null           |FALSE      |
|GO:0009826 |unidimensional cell growth                                                                   |  1.0736265|-1.66174063325968  |5.42148469835125   | 2.3802112|   -4.543084|  0.8720855|      0.5680023|null           |FALSE      |
|GO:0007498 |mesoderm development                                                                         |  0.3503886|-1.93246428814203  |5.28380358844385   | 1.8976271|   -3.610872|  0.9031958|      0.5707614|null           |FALSE      |
|GO:0080003 |thalianol metabolic process                                                                  |  0.0089843|-5.57991471218279  |3.38235762660592   | 0.4771213|  -30.502960|  0.8895440|      0.5772455|null           |FALSE      |
|GO:0071766 |Actinobacterium-type cell wall biogenesis                                                    |  1.0826108|6.22021239418573   |4.02792633893675   | 2.3838154|  -10.192160|  0.9170797|      0.5779561|null           |FALSE      |
|GO:0010438 |cellular response to sulfur starvation                                                       |  0.0314451|5.21682961102881   |-1.74524914918384  | 0.9030900|   -3.856586|  0.8397183|      0.5817840|null           |FALSE      |
|GO:0002230 |positive regulation of defense response to virus by host                                     |  0.0224608|-0.668052065751929 |-7.61290272720887  | 0.7781513|   -2.703275|  0.8701234|      0.5823779|null           |FALSE      |
|GO:0071229 |cellular response to acid chemical                                                           |  0.2740218|5.52211500113389   |-3.95448891868987  | 1.7923917|  -12.010943|  0.8370700|      0.5845806|null           |FALSE      |
|GO:1905663 |positive regulation of telomerase RNA reverse transcriptase activity                         |  0.0485153|-1.56518126086088  |-3.60089537865641  | 1.0413927|   -2.276294|  0.8610837|      0.5862896|null           |FALSE      |
|GO:0006812 |cation transport                                                                             |  2.2505728|2.70502634466561   |6.02815250740253   | 2.7007037|   -8.598854|  0.9510252|      0.5885695|null           |FALSE      |
|GO:0007165 |signal transduction                                                                          |  8.3913571|1.967643852622     |-3.620034334737    | 3.2716093|  -81.288547|  0.6838945|      0.5893442|null           |FALSE      |
|GO:0003341 |cilium movement                                                                              |  0.0059895|5.55415831706502   |5.47449870010919   | 0.3010300|  -16.666473|  0.9525226|      0.5985002|null           |FALSE      |
|GO:0045747 |positive regulation of Notch signaling pathway                                               |  0.0392043|-0.410210618612415 |-6.72475181853113  | 0.9542425|   -2.388180|  0.8235029|      0.6109914|null           |FALSE      |
|GO:0007398 |ectoderm development                                                                         |  0.3503886|-2.0096723605935   |5.1738789775631    | 1.8976271|   -2.388180|  0.9031958|      0.6126734|null           |FALSE      |
|GO:0046345 |abscisic acid catabolic process                                                              |  0.0269530|-5.15522584398676  |3.15088939350378   | 0.8450980|  -55.983799|  0.8550376|      0.6194937|null           |FALSE      |
|GO:0007422 |peripheral nervous system development                                                        |  3.6745878|-1.10860950249701  |4.58706022864553   | 2.9132839|   -3.610872|  0.8349779|      0.6229170|null           |FALSE      |
|GO:0006636 |unsaturated fatty acid biosynthetic process                                                  |  0.1033197|-5.42440698049982  |3.29313224576175   | 1.3802112|   -2.644815|  0.8721937|      0.6297662|null           |FALSE      |
|GO:0045926 |negative regulation of growth                                                                |  0.1751943|-3.02116679163972  |-5.97892048430296  | 1.6020600|  -15.727160|  0.8662875|      0.6297772|null           |FALSE      |
|GO:1904355 |positive regulation of telomere capping                                                      |  0.0044922|0.724366783591745  |-7.1543871118226   | 0.3010300|   -1.727692|  0.8616551|      0.6315629|null           |FALSE      |
|GO:1905614 |negative regulation of developmental vegetative growth                                       |  0.0134765|-3.83352603259     |-5.66179502187725  | 0.6020600|   -3.116242|  0.8602793|      0.6315937|null           |FALSE      |
|GO:0031640 |killing of cells of other organism                                                           |  1.3072189|7.48652716907798   |1.04534885947784   | 2.4653829|   -2.509567|  0.8954840|      0.6338299|null           |FALSE      |
|GO:0072498 |embryonic skeletal joint development                                                         |  1.8732312|-1.14781984465769  |4.75014362910628   | 2.6211763|   -2.509467|  0.8475362|      0.6342863|null           |FALSE      |
|GO:0048821 |erythrocyte development                                                                      |  0.0179686|-1.64046619589678  |2.14407411179743   | 0.6989700|   -2.384487|  0.7311547|      0.6349558|null           |FALSE      |
|GO:0042177 |negative regulation of protein catabolic process                                             |  0.0134765|-3.2414262659903   |-6.56835438770422  | 0.6020600|   -2.367953|  0.8835697|      0.6510962|null           |FALSE      |
|GO:0031347 |regulation of defense response                                                               |  1.2308522|-1.29295658825397  |-5.88418334213923  | 2.4393327|   -2.037689|  0.8328025|      0.6512180|null           |FALSE      |
|GO:0009828 |plant-type cell wall loosening                                                               |  0.0314451|6.47328999321998   |3.86988280029267   | 0.9030900|   -1.774122|  0.9357965|      0.6517822|null           |FALSE      |
|GO:0015708 |silicic acid import across plasma membrane                                                   |  0.0331729|2.43590645103525   |6.39085185076082   | 0.9030900|   -3.788987|  0.9433715|      0.6528448|null           |FALSE      |
|GO:0051513 |regulation of monopolar cell growth                                                          |  0.0224608|-3.56885648914363  |-4.85156896659284  | 0.7781513|   -2.998690|  0.8523124|      0.6532683|null           |FALSE      |
|GO:0042742 |defense response to bacterium                                                                |  1.6216702|6.29735758802211   |-1.31694635958714  | 2.5587086|  -15.100075|  0.8261203|      0.6545694|null           |FALSE      |
|GO:1902042 |negative regulation of extrinsic apoptotic signaling pathway via death domain receptors      |  0.0044922|-1.41338061284031  |-7.11631502737114  | 0.3010300|   -7.135037|  0.8446855|      0.6549617|null           |FALSE      |
|GO:0070301 |cellular response to hydrogen peroxide                                                       |  0.0449216|5.76423159963317   |-3.51863309288967  | 1.0413927|   -7.747259|  0.8274215|      0.6554547|null           |FALSE      |
|GO:2000026 |regulation of multicellular organismal development                                           |  1.5677643|-2.99115886928129  |-4.28669678401325  | 2.5440680|  -35.555401|  0.7996712|      0.6593517|null           |FALSE      |
|GO:0035204 |negative regulation of lamellocyte differentiation                                           |  0.0359373|-3.03315017521798  |-5.47442733606247  | 0.9542425|   -2.388180|  0.8083724|      0.6596159|null           |FALSE      |
|GO:0009914 |hormone transport                                                                            |  0.4447240|-1.5790776383672   |-0.722173614109183 | 2.0000000|   -8.520136|  0.8055263|      0.6604623|null           |FALSE      |
|GO:1900745 |positive regulation of p38MAPK cascade                                                       |  0.0179686|-0.265002057169368 |-6.98728791147825  | 0.6989700|   -1.752952|  0.8279057|      0.6611691|null           |FALSE      |
|GO:0043086 |negative regulation of catalytic activity                                                    |  1.1410089|-2.4197635541358   |-2.84978503070701  | 2.4065402|   -1.980503|  0.8650703|      0.6653174|null           |FALSE      |
|GO:0009866 |induced systemic resistance,ethylene mediated signaling pathway                              |  0.0179686|2.91529051069714   |-4.04415735581437  | 0.6989700|   -5.551154|  0.6576328|      0.6707259|null           |FALSE      |
|GO:0045338 |farnesyl diphosphate metabolic process                                                       |  0.0898432|-5.26778996318182  |3.99886663023903   | 1.3222193|  -40.331178|  0.8804222|      0.6735153|null           |FALSE      |
|GO:0009852 |auxin catabolic process                                                                      |  0.0089843|-5.75018319555722  |-1.60107398125491  | 0.4771213|  -11.473243|  0.8365254|      0.6856031|null           |FALSE      |
|GO:1900449 |regulation of glutamate receptor signaling pathway                                           |  0.1735903|-1.03304447648611  |-6.38054547161468  | 1.5682017|   -7.135037|  0.8356625|      0.6858401|null           |FALSE      |
|GO:0048265 |response to pain                                                                             | 11.9626252|3.75714264476187   |-0.339560762003704 | 3.4255342|   -2.509467|  0.7337153|      0.6889007|null           |FALSE      |
|GO:0010030 |positive regulation of seed germination                                                      |  0.0898432|-2.4094771676665   |-4.95259406106787  | 1.3222193|   -6.951280|  0.8070507|      0.6895163|null           |FALSE      |
|GO:0032870 |cellular response to hormone stimulus                                                        |  4.2316158|5.50703400918265   |-3.21115314060926  | 2.9745117|   -7.407748|  0.7786998|      0.6896194|null           |FALSE      |

[1] &quot;missing_arabidopsis_vs_all_GO.txt&quot;


|TermID     |Name                                                                                    |  Frequency|PlotX              |PlotY               |   LogSize|       Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:---------------------------------------------------------------------------------------|----------:|:------------------|:-------------------|---------:|-----------:|----------:|--------------:|:--------------|:----------|
|GO:0009819 |drought recovery                                                                        |  0.0494138|6.72881611287059   |0.431200832148061   | 1.0791812|  -31.057832|  0.9203897|      0.0000000|null           |FALSE      |
|GO:0009877 |nodulation                                                                              |  0.0988275|-3.13726787932495  |-6.16837116794157   | 1.3617278|  -11.377572|  0.8388537|      0.0000000|null           |FALSE      |
|GO:0010233 |phloem transport                                                                        |  0.0943354|-5.18727931413998  |-2.64619698936323   | 1.3424227|  -10.718896|  0.7607701|      0.0000000|null           |FALSE      |
|GO:0031146 |SCF-dependent proteasomal ubiquitin-dependent protein catabolic process                 |  0.3773415|0.65065562067172   |4.42861876762636    | 1.9294189|  -23.559069|  0.9133378|      0.0000000|null           |FALSE      |
|GO:0045926 |negative regulation of growth                                                           |  0.1751943|3.29142039538912   |-5.53486246602895   | 1.6020600|  -75.159828|  0.8952082|      0.0000000|null           |FALSE      |
|GO:0031589 |cell-substrate adhesion                                                                 |  0.0044922|4.76145596357251   |6.21072587509381    | 0.3010300|   -2.835492|  0.9749859|      0.0213516|null           |FALSE      |
|GO:0000226 |microtubule cytoskeleton organization                                                   |  0.7232379|-5.49575684567404  |2.81625241763343    | 2.2095150|  -21.958043|  0.8685834|      0.0316757|null           |FALSE      |
|GO:0045730 |respiratory burst                                                                       |  0.0224608|5.32499383714275   |5.7591410107341     | 0.7781513|   -1.825271|  0.9773775|      0.0600739|null           |FALSE      |
|GO:1904526 |regulation of microtubule binding                                                       |  0.0044922|0.855772697971772  |-0.0436017978381461 | 0.3010300|  -61.579741|  0.9393830|      0.1037129|null           |FALSE      |
|GO:0009311 |oligosaccharide metabolic process                                                       |  0.4267553|3.80784187126112   |6.02456370460019    | 1.9822712|   -7.895729|  0.9511658|      0.1045125|null           |FALSE      |
|GO:0097503 |sialylation                                                                             |  0.0134765|2.19220135378522   |4.37680981691861    | 0.6020600|   -3.416393|  0.9571898|      0.1181081|null           |FALSE      |
|GO:0031542 |positive regulation of anthocyanin biosynthetic process                                 |  0.0179686|6.14825667763596   |-4.59471289521933   | 0.6989700|   -3.627182|  0.9093678|      0.1293947|null           |FALSE      |
|GO:0000966 |RNA 5&#39;-end processing                                                                   |  0.0808589|-2.34762373080007  |5.66031083845377    | 1.2787536|  -16.663352|  0.9066769|      0.1347712|null           |FALSE      |
|GO:2000023 |regulation of lateral root development                                                  |  0.1257805|1.80881278099483   |-5.90647872367625   | 1.4623980|   -6.604582|  0.8705414|      0.1487254|null           |FALSE      |
|GO:0009617 |response to bacterium                                                                   |  2.1113158|5.91435222855867   |2.07741096271546    | 2.6730209|  -23.613172|  0.9003626|      0.1641436|null           |FALSE      |
|GO:0033387 |putrescine biosynthetic process from ornithine                                          |  0.0123534|-0.466387483138575 |6.4857891651487     | 0.4771213|  -13.401532|  0.8963321|      0.1969581|null           |FALSE      |
|GO:0007166 |cell surface receptor signaling pathway                                                 |  0.5794888|4.40907924159617   |-1.94286823470013   | 2.1139434|   -5.354689|  0.8273169|      0.2090014|null           |FALSE      |
|GO:0009410 |response to xenobiotic stimulus                                                         |  0.0359373|7.30611186952054   |0.0381353274679804  | 0.9542425|  -11.386409|  0.9412302|      0.2284321|null           |FALSE      |
|GO:0009962 |regulation of flavonoid biosynthetic process                                            |  0.1257805|3.30523364424126   |-6.4770394515763    | 1.4623980|   -2.600440|  0.9181712|      0.2324944|null           |FALSE      |
|GO:0080051 |cutin transport                                                                         |  0.0134765|-6.85286271695432  |-1.13593563645407   | 0.6020600|   -1.773667|  0.9591081|      0.2347702|null           |FALSE      |
|GO:0007492 |endoderm development                                                                    |  0.0134765|-2.36643817643627  |-6.622371440883     | 0.6020600|   -2.758626|  0.8830023|      0.2359966|null           |FALSE      |
|GO:1901949 |5alpha,9alpha,10beta-labda-8(20),13-dien-15-yl diphosphate biosynthetic process         |  0.1848275|-0.133453135554542 |6.66153358101333    | 1.6232493|  -14.605293|  0.8566769|      0.2545451|null           |FALSE      |
|GO:0009992 |cellular water homeostasis                                                              |  0.0089843|-2.62222262726544  |1.06989414277519    | 0.4771213|   -2.289354|  0.8684246|      0.2566157|null           |FALSE      |
|GO:1902025 |nitrate import                                                                          |  0.0808589|-6.83211510084484  |-0.325867249142163  | 1.2787536|   -5.828260|  0.9498478|      0.2646327|null           |FALSE      |
|GO:0010305 |leaf vascular tissue pattern formation                                                  |  0.1347648|-4.16726498218234  |-5.10750451117842   | 1.4913617|   -7.541784|  0.8290603|      0.2761625|null           |FALSE      |
|GO:0006898 |receptor-mediated endocytosis                                                           |  0.1976551|-6.33280082915706  |-0.84721242233106   | 1.6532125|   -6.047190|  0.9291035|      0.2825621|null           |FALSE      |
|GO:0006952 |defense response                                                                        |  7.2503481|5.67683048945332   |0.882318310210571   | 3.2081725|  -96.210331|  0.9192426|      0.2904696|null           |FALSE      |
|GO:0098657 |import into cell                                                                        |  0.1392570|-6.67881052544943  |-0.783314038251996  | 1.5051500|   -3.726525|  0.9537816|      0.2911576|null           |FALSE      |
|GO:0044550 |secondary metabolite biosynthetic process                                               |  0.6244104|2.19392016127029   |7.02486845877581    | 2.1461280|   -6.590408|  0.9236327|      0.2938489|null           |FALSE      |
|GO:0009733 |response to auxin                                                                       |  1.8597547|6.30601626005061   |0.173978152933545   | 2.6180481|   -4.547141|  0.9087672|      0.3061641|null           |FALSE      |
|GO:0048312 |intracellular distribution of mitochondria                                              |  0.0044922|-5.47390864239657  |1.01532371680651    | 0.3010300|   -2.755929|  0.8956797|      0.3162058|null           |FALSE      |
|GO:0010114 |response to red light                                                                   |  0.2785140|7.11293471441139   |1.34814083205374    | 1.7993405|   -2.973863|  0.9442679|      0.3255957|null           |FALSE      |
|GO:0032504 |multicellular organism reproduction                                                     |  0.8040969|-4.73519862847066  |-4.3243904345341    | 2.2552725|   -5.880605|  0.8375175|      0.3270246|null           |FALSE      |
|GO:0016567 |protein ubiquitination                                                                  |  3.7105251|0.115759812254014  |4.85019239189334    | 2.9175055|   -8.233559|  0.9012866|      0.3270943|null           |FALSE      |
|GO:0007339 |binding of sperm to zona pellucida                                                      |  0.0089843|-1.3300045412327   |-1.01922517738869   | 0.4771213|   -2.995739|  0.9668297|      0.3317891|null           |FALSE      |
|GO:1902042 |negative regulation of extrinsic apoptotic signaling pathway via death domain receptors |  0.0044922|4.38604716820672   |-6.01232299344844   | 0.3010300|   -7.626170|  0.8872515|      0.3379466|null           |FALSE      |
|GO:0015939 |pantothenate metabolic process                                                          |  0.0179686|-1.76941324893862  |6.52717936335594    | 0.6989700|   -6.369773|  0.9170995|      0.3411185|null           |FALSE      |
|GO:0071447 |cellular response to hydroperoxide                                                      |  0.1529288|5.71715634892504   |0.118545505986995   | 1.5440680|   -7.626170|  0.8854373|      0.3536993|null           |FALSE      |
|GO:0071766 |Actinobacterium-type cell wall biogenesis                                               |  1.0826108|-5.00347354693524  |3.19397916213478    | 2.3838154|  -10.738963|  0.8774736|      0.3567066|null           |FALSE      |
|GO:0036498 |IRE1-mediated unfolded protein response                                                 |  0.0134765|5.32435225961361   |-0.963865394814427  | 0.6020600|   -1.997416|  0.8469345|      0.3576515|null           |FALSE      |
|GO:0035600 |tRNA methylthiolation                                                                   |  0.0089843|-1.90645555542749  |5.27777621199884    | 0.4771213|   -3.993292|  0.9076597|      0.3671116|null           |FALSE      |
|GO:0002182 |cytoplasmic translational elongation                                                    |  0.0224608|-0.307824968563814 |5.99086723632767    | 0.7781513|   -1.428299|  0.8993510|      0.3680407|null           |FALSE      |
|GO:1903224 |regulation of endodermal cell differentiation                                           |  0.0044922|1.56270750175925   |-6.67993040617326   | 0.3010300|   -4.810111|  0.9089880|      0.3865517|null           |FALSE      |
|GO:0010359 |regulation of anion channel activity                                                    |  0.0494138|2.55275276009214   |-3.10885644179884   | 1.0791812|   -9.197787|  0.8883993|      0.3874463|null           |FALSE      |
|GO:0045184 |establishment of protein localization                                                   |  3.7823997|-6.35968676422123  |-0.313932678142344  | 2.9258276|   -4.787121|  0.9339116|      0.3880049|null           |FALSE      |
|GO:0006335 |DNA replication-dependent chromatin assembly                                            |  0.0179686|-4.88611458095363  |2.90962937102728    | 0.6989700|   -4.150067|  0.9062387|      0.3882255|null           |FALSE      |
|GO:0061820 |telomeric D-loop disassembly                                                            |  0.0044922|-3.28999469522099  |4.01699179698417    | 0.3010300|   -1.596827|  0.8737349|      0.3892522|null           |FALSE      |
|GO:0002011 |morphogenesis of an epithelial sheet                                                    |  0.0134765|-2.72876500809778  |-6.36983321152027   | 0.6020600|   -2.995739|  0.8538112|      0.3953158|null           |FALSE      |
|GO:0007369 |gastrulation                                                                            |  0.0134765|-3.87850241727127  |-5.6551105931267    | 0.6020600|   -5.952804|  0.8197372|      0.3953158|null           |FALSE      |
|GO:0010737 |protein kinase A signaling                                                              |  0.2467993|4.63170334416476   |-1.95750008613434   | 1.7403627|   -2.293020|  0.8386179|      0.4069935|null           |FALSE      |
|GO:0016045 |detection of bacterium                                                                  |  0.0134765|5.96801629133415   |3.12330392700911    | 0.6020600|  -12.125293|  0.9186815|      0.4090250|null           |FALSE      |
|GO:0007178 |transmembrane receptor protein serine/threonine kinase signaling pathway                |  0.2650375|4.30353450169462   |-1.73058932619583   | 1.7781513|   -3.010935|  0.8316591|      0.4096115|null           |FALSE      |
|GO:0090615 |mitochondrial mRNA processing                                                           |  0.0539059|-2.53880770685528  |5.77728057742403    | 1.1139434|   -8.152980|  0.8952376|      0.4120236|null           |FALSE      |
|GO:0048439 |flower morphogenesis                                                                    |  0.0269530|-3.43789210531599  |-4.78453615201797   | 0.8450980|   -4.558133|  0.8138287|      0.4134163|null           |FALSE      |
|GO:0035864 |response to potassium ion                                                               |  0.0269530|7.32188105814284   |-0.294311990246824  | 0.8450980|   -3.561205|  0.9403236|      0.4217509|null           |FALSE      |
|GO:0009745 |sucrose mediated signaling                                                              |  0.0089843|5.86308954221982   |-1.31821765458294   | 0.4771213|   -4.100581|  0.8526711|      0.4278823|null           |FALSE      |
|GO:0006486 |protein glycosylation                                                                   |  0.7322223|0.372359541342116  |5.82676916579518    | 2.2148438|   -2.967523|  0.8738888|      0.4293059|null           |FALSE      |
|GO:0044364 |disruption of cells of other organism                                                   |  0.0269530|5.88920013001877   |4.17172711055738    | 0.8450980|   -2.755929|  0.9672614|      0.4301109|null           |FALSE      |
|GO:0046777 |protein autophosphorylation                                                             |  0.7636674|-0.189665925046874 |4.95031435217934    | 2.2329961|   -7.305555|  0.9033473|      0.4315157|null           |FALSE      |
|GO:0046482 |para-aminobenzoic acid metabolic process                                                |  0.0179686|-1.54958517290339  |6.49631666646745    | 0.6989700|   -1.986689|  0.9087871|      0.4395354|null           |FALSE      |
|GO:0061157 |mRNA destabilization                                                                    |  0.2156237|3.93009258908499   |-4.55743229091249   | 1.6901961|   -4.943818|  0.8525187|      0.4427212|null           |FALSE      |
|GO:0045056 |transcytosis                                                                            |  0.0044922|-5.48983047665592  |-2.55634392582778   | 0.3010300|   -1.617946|  0.8433839|      0.4534531|null           |FALSE      |
|GO:0071921 |cohesin loading                                                                         |  0.0044922|-5.45703414208182  |1.24438875487963    | 0.3010300|   -1.496974|  0.8890119|      0.4563019|null           |FALSE      |
|GO:0009608 |response to symbiont                                                                    |  0.0539059|5.94953780414619   |2.77473366753934    | 1.1139434|   -7.017275|  0.9265259|      0.4588364|null           |FALSE      |
|GO:0098530 |positive regulation of strand invasion                                                  |  0.0152733|5.78712753592618   |-4.53129799331318   | 0.6020600|   -1.596827|  0.9046446|      0.4704375|null           |FALSE      |
|GO:0010150 |leaf senescence                                                                         |  0.5076142|-3.7759516921316   |-4.97639069360092   | 2.0569049|   -4.016755|  0.8004546|      0.4731624|null           |FALSE      |
|GO:0032526 |response to retinoic acid                                                               |  0.4215893|6.54993143131163   |-0.0166151702952586 | 1.9731279|   -1.471984|  0.9156223|      0.4739620|null           |FALSE      |
|GO:0015969 |guanosine tetraphosphate metabolic process                                              |  0.0179686|-1.15347724402741  |6.09653030725139    | 0.6989700|   -2.627528|  0.8980691|      0.4786646|null           |FALSE      |
|GO:0019878 |lysine biosynthetic process via aminoadipic acid                                        |  0.0179686|0.627479171818875  |6.71679476261812    | 0.6989700|   -4.260093|  0.9068749|      0.4806061|null           |FALSE      |
|GO:0031065 |positive regulation of histone deacetylation                                            |  0.0359373|5.43083943130965   |-4.52318306836351   | 0.9542425|   -1.496974|  0.8995954|      0.4820669|null           |FALSE      |
|GO:0060465 |pharynx development                                                                     |  1.4053739|-3.55277992108447  |-5.1107916816308    | 2.4927604|   -2.755929|  0.7850590|      0.4836985|null           |FALSE      |
|GO:0072699 |protein localization to cortical microtubule cytoskeleton                               |  0.0134765|-6.18628459341874  |0.00450042744570775 | 0.6020600|   -4.555906|  0.9316471|      0.4974365|null           |FALSE      |
|GO:1900449 |regulation of glutamate receptor signaling pathway                                      |  0.1735903|4.31740903675757   |-5.34562634578071   | 1.5682017|   -7.626170|  0.8748601|      0.5089636|null           |FALSE      |
|GO:0048211 |Golgi vesicle docking                                                                   |  0.0089843|-5.97618676069166  |-0.455908875621084  | 0.4771213|   -1.617946|  0.9310126|      0.5090434|null           |FALSE      |
|GO:0006627 |protein processing involved in protein targeting to mitochondrion                       |  0.0179686|-4.02573269292707  |2.26184143353039    | 0.6989700|   -1.720248|  0.8325597|      0.5141106|null           |FALSE      |
|GO:0090558 |plant epidermis development                                                             |  1.1679619|-2.61978551524698  |-6.12861302300884   | 2.4166405|   -2.738830|  0.8353260|      0.5184871|null           |FALSE      |
|GO:0009251 |glucan catabolic process                                                                |  0.2470689|2.98036649335184   |5.53921869536875    | 1.7481880|   -2.365504|  0.9305552|      0.5219667|null           |FALSE      |
|GO:2000280 |regulation of root development                                                          |  0.3593729|2.06234783568582   |-5.13052856790611   | 1.9084850|   -5.252086|  0.8889514|      0.5241877|null           |FALSE      |
|GO:0042761 |very long-chain fatty acid biosynthetic process                                         |  0.0988275|0.641079265209193  |7.15057530177619    | 1.3617278|   -2.053810|  0.8892492|      0.5277113|null           |FALSE      |
|GO:0051967 |negative regulation of synaptic transmission,glutamatergic                              |  0.0044922|4.38504282358557   |-6.1668231291041    | 0.3010300|   -4.712564|  0.8955352|      0.5278281|null           |FALSE      |
|GO:0035024 |negative regulation of Rho protein signal transduction                                  |  0.0044922|4.55949175773424   |-6.00049186648772   | 0.3010300|   -3.841743|  0.8837901|      0.5316060|null           |FALSE      |
|GO:0009626 |plant-type hypersensitive response                                                      |  0.3279278|5.26747959959152   |1.38247478474296    | 1.8692317|  -20.800214|  0.8545525|      0.5453319|null           |FALSE      |
|GO:0009826 |unidimensional cell growth                                                              |  1.0736265|-2.97875972291149  |-5.89619815745376   | 2.3802112|   -3.368105|  0.7929174|      0.5464143|null           |FALSE      |
|GO:0043588 |skin development                                                                        |  1.7580260|-3.32368646226899  |-5.14776278236428   | 3.2564772|   -2.755929|  0.7338234|      0.5560406|null           |FALSE      |
|GO:0044403 |biological process involved in symbiotic interaction                                    |  0.4312475|5.98768205904071   |4.00163296234076    | 1.9867717|   -9.395802|  0.9598398|      0.5582824|null           |FALSE      |
|GO:0009742 |brassinosteroid mediated signaling pathway                                              |  0.3638651|5.11589946128256   |-1.50743765101059   | 1.9138139|   -6.885721|  0.8072039|      0.5587782|null           |FALSE      |
|GO:0010083 |regulation of vegetative meristem growth                                                |  0.0329425|-0.709958614630112 |-5.58264541917539   | 0.9030900|  -25.517715|  0.7518998|      0.5625290|null           |FALSE      |
|GO:0048509 |regulation of meristem development                                                      |  0.2964826|2.01775236777001   |-5.33470829711292   | 1.8260748|   -5.238104|  0.8905056|      0.5634440|null           |FALSE      |
|GO:0052544 |defense response by callose deposition in cell wall                                     |  0.0718746|-1.82820583288502  |0.577236574552807   | 1.2304489|   -6.905833|  0.8005612|      0.5779561|null           |FALSE      |
|GO:0008610 |lipid biosynthetic process                                                              |  2.7716634|1.7607148214011    |6.73339859007492    | 2.7909885|   -1.447198|  0.9035673|      0.5789751|null           |FALSE      |
|GO:0009218 |pyrimidine ribonucleotide metabolic process                                             |  0.1257805|-1.03455073665915  |5.90702418567564    | 1.4623980|   -1.426628|  0.8845295|      0.5850997|null           |FALSE      |
|GO:0014013 |regulation of gliogenesis                                                               |  0.0044922|-0.795649147718899 |-5.37784706260994   | 0.3010300|   -2.217819|  0.7400230|      0.5911068|null           |FALSE      |
|GO:0007275 |multicellular organism development                                                      | 10.3544315|-3.39286508268708  |-5.32155954875234   | 3.3628593|   -4.193035|  0.7594223|      0.5926294|null           |FALSE      |
|GO:1904851 |positive regulation of establishment of protein localization to telomere                |  0.0404295|5.1024265716192    |-4.28969798488751   | 1.0000000|   -1.379635|  0.8802193|      0.5976487|null           |FALSE      |
|GO:0001701 |in utero embryonic development                                                          |  1.7833880|-3.6535894584803   |-5.33085031249649   | 2.6211763|   -4.688566|  0.7761709|      0.6033090|null           |FALSE      |
|GO:0009734 |auxin-activated signaling pathway                                                       |  0.9972598|4.93906320537422   |-1.69448730796898   | 2.3483049|   -5.072313|  0.7982169|      0.6060261|null           |FALSE      |
|GO:0003341 |cilium movement                                                                         |  0.0059895|-6.60234630949948  |3.94412684699268    | 0.3010300|  -17.515455|  0.9470345|      0.6208058|null           |FALSE      |
|GO:0060294 |cilium movement involved in cell motility                                               |  0.0059895|-6.74469366433474  |1.01317710761541    | 0.4771213|  -19.092525|  0.9036820|      0.6208058|null           |FALSE      |
|GO:0010470 |regulation of gastrulation                                                              |  0.1539688|1.87651013993092   |-5.72164680098519   | 1.5440680|   -4.810111|  0.8666181|      0.6218939|null           |FALSE      |
|GO:0101030 |tRNA-guanine transglycosylation                                                         |  0.0333169|-1.64401028800936  |5.18792039718572    | 0.9030900|   -2.835031|  0.9002309|      0.6260287|null           |FALSE      |
|GO:0006336 |DNA replication-independent chromatin assembly                                          |  0.0404295|-4.87135404186484  |2.77889123435678    | 1.0000000|   -3.787082|  0.9015843|      0.6310454|null           |FALSE      |
|GO:0031640 |killing of cells of other organism                                                      |  1.3072189|6.1191224608187    |3.95766855811662    | 2.4653829|   -2.756107|  0.9313587|      0.6338299|null           |FALSE      |
|GO:0007165 |signal transduction                                                                     |  8.3913571|4.45500954691604   |-2.15257211942826   | 3.2716093| -119.239761|  0.7796595|      0.6344952|null           |FALSE      |
|GO:0048821 |erythrocyte development                                                                 |  0.0179686|-2.13573468613292  |-4.34580240136757   | 0.6989700|   -2.676151|  0.7088042|      0.6384362|null           |FALSE      |
|GO:0051513 |regulation of monopolar cell growth                                                     |  0.0224608|2.04560149809659   |-6.06357755812973   | 0.7781513|   -3.381197|  0.8826127|      0.6393644|null           |FALSE      |
|GO:0007349 |cellularization                                                                         |  0.0404295|-3.54592180444974  |-5.68900499268661   | 1.0000000|   -1.998218|  0.8090526|      0.6411372|null           |FALSE      |
|GO:0006468 |protein phosphorylation                                                                 |  5.0671578|-0.102292403475927 |5.23316381356457    | 3.0526939|  -44.190986|  0.8862054|      0.6414721|null           |FALSE      |
|GO:1900745 |positive regulation of p38MAPK cascade                                                  |  0.0179686|5.09916788797614   |-5.21789108235489   | 0.6989700|   -1.997416|  0.8630862|      0.6447047|null           |FALSE      |
|GO:0044806 |G-quadruplex DNA unwinding                                                              |  0.1482413|-4.71348769733131  |2.69914924812957    | 1.5314789|   -1.596827|  0.9058033|      0.6448448|null           |FALSE      |
|GO:0060325 |face morphogenesis                                                                      |  0.5812857|-3.0206432986557   |-6.00712830962117   | 2.1139434|   -1.496974|  0.8176179|      0.6464135|null           |FALSE      |
|GO:0009828 |plant-type cell wall loosening                                                          |  0.0314451|-5.1086509901458   |3.61493922884023    | 0.9030900|   -2.793297|  0.9115500|      0.6517822|null           |FALSE      |
|GO:0015708 |silicic acid import across plasma membrane                                              |  0.0331729|-6.96766031762191  |-0.462052078816071  | 0.9030900|   -4.085583|  0.9369619|      0.6528448|null           |FALSE      |
|GO:0042742 |defense response to bacterium                                                           |  1.6216702|5.74780157060293   |1.9285695392519     | 2.5587086|   -5.879025|  0.8908942|      0.6545694|null           |FALSE      |
|GO:0060548 |negative regulation of cell death                                                       |  0.1662100|3.71954739681249   |-4.94834601160867   | 1.5797836|   -1.766217|  0.8984414|      0.6583366|null           |FALSE      |
|GO:0090708 |specification of plant organ axis polarity                                              |  0.0404295|-3.94492654819929  |-5.30493208674715   | 1.0000000|   -5.847839|  0.8086101|      0.6587402|null           |FALSE      |
|GO:0097306 |cellular response to alcohol                                                            |  1.1275325|5.94759099474173   |-0.02134251793076   | 2.4014005|   -2.674237|  0.8715329|      0.6692924|null           |FALSE      |
|GO:0036159 |inner dynein arm assembly                                                               |  0.0179686|-5.64756165276223  |2.96102600864485    | 0.6989700|  -13.046176|  0.8773282|      0.6711268|null           |FALSE      |
|GO:0009867 |jasmonic acid mediated signaling pathway                                                |  0.3998023|5.17724398002445   |-1.58865582572559   | 1.9542425|   -2.146426|  0.8057820|      0.6755918|null           |FALSE      |
|GO:0009690 |cytokinin metabolic process                                                             |  0.1751943|0.294399452886004  |2.22434859969294    | 1.6020600|   -8.510983|  0.8317378|      0.6792812|null           |FALSE      |
|GO:0030240 |skeletal muscle thin filament assembly                                                  |  0.0592965|-3.43987665077859  |-2.86128165468701   | 1.1461280|   -6.369436|  0.7314163|      0.6944233|null           |FALSE      |

[1] &quot;missing_posi_vs_all_GO.txt&quot;


|TermID     |Name                                                                                                    | Frequency|PlotX                |PlotY              |   LogSize|       Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:-------------------------------------------------------------------------------------------------------|---------:|:--------------------|:------------------|---------:|-----------:|----------:|--------------:|:--------------|:----------|
|GO:0009819 |drought recovery                                                                                        | 0.0494138|-1.68935718522876    |-6.49734292631286  | 1.0791812|  -39.762484|  0.9095380|      0.0000000|null           |FALSE      |
|GO:0010375 |stomatal complex patterning                                                                             | 0.0494138|-2.67268803401348    |6.4068906379754    | 1.0791812|  -33.754478|  0.9266859|      0.0000000|null           |FALSE      |
|GO:0040011 |locomotion                                                                                              | 0.1841786|-6.17795147226829    |-5.68200608191421  | 1.6232493|   -2.110800|  1.0000000|      0.0000000|null           |FALSE      |
|GO:0045926 |negative regulation of growth                                                                           | 0.1751943|-5.82569672908366    |2.31773461303048   | 1.6020600|  -56.870827|  0.8529387|      0.0000000|null           |FALSE      |
|GO:0090305 |nucleic acid phosphodiester bond hydrolysis                                                             | 1.6171780|5.98784515812599     |1.41495931403664   | 2.5575072| -202.077576|  0.8940943|      0.0000000|null           |FALSE      |
|GO:0003341 |cilium movement                                                                                         | 0.0059895|6.55885908794859     |-3.49972323684888  | 0.3010300|  -11.828706|  0.9577144|      0.0240391|null           |FALSE      |
|GO:0007023 |post-chaperonin tubulin folding pathway                                                                 | 0.0179686|-6.51024705952577    |-5.36276818302846  | 0.6989700|   -2.977004|  0.9808062|      0.0261106|null           |FALSE      |
|GO:0036371 |protein localization to T-tubule                                                                        | 0.0314451|4.43620469586134     |-4.25993297412743  | 0.9030900|  -15.023967|  0.9196698|      0.0273094|null           |FALSE      |
|GO:0051211 |anisotropic cell growth                                                                                 | 0.0359373|-0.186200241282128   |4.29278557923247   | 0.9542425|   -1.813416|  0.9767397|      0.0276119|null           |FALSE      |
|GO:0034724 |DNA replication-independent chromatin organization                                                      | 0.0494138|2.15434142650613     |5.92571290787772   | 1.0791812|  -12.343109|  0.9279718|      0.0283611|null           |FALSE      |
|GO:0009780 |photosynthetic NADP+ reduction                                                                          | 0.0044922|-5.72263080355836    |-6.06335361584344  | 0.3010300|   -8.261077|  0.9526537|      0.0782721|null           |FALSE      |
|GO:0051554 |flavonol metabolic process                                                                              | 0.0853511|1.6962063681239      |1.65953199116964   | 1.3010300|  -27.259447|  0.9328267|      0.0907318|null           |FALSE      |
|GO:1904526 |regulation of microtubule binding                                                                       | 0.0044922|-0.514916560777027   |1.64489151280658   | 0.3010300|  -40.641336|  0.9089796|      0.1037129|null           |FALSE      |
|GO:0051762 |sesquiterpene biosynthetic process                                                                      | 0.1123040|4.97731776214005     |3.80540792554849   | 1.4149733| -100.297555|  0.8771729|      0.1056641|null           |FALSE      |
|GO:0010881 |regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion        | 0.0089843|-2.67024326970673    |-2.44239679505855  | 0.4771213|  -28.972157|  0.7498298|      0.1236690|null           |FALSE      |
|GO:0050434 |positive regulation of viral transcription                                                              | 0.0089843|-7.34379155940284    |-2.45566194292707  | 0.4771213|  -11.444585|  0.8866056|      0.1236690|null           |FALSE      |
|GO:0015979 |photosynthesis                                                                                          | 1.1230403|6.96820756600755     |-2.93658870820728  | 2.3996737|   -1.786708|  0.9443915|      0.1284458|null           |FALSE      |
|GO:0009625 |response to insect                                                                                      | 0.1482413|-0.00490010358071543 |-7.09359494481751  | 1.5314789|  -15.643489|  0.9236552|      0.1332801|null           |FALSE      |
|GO:0009962 |regulation of flavonoid biosynthetic process                                                            | 0.1257805|-3.92607502895609    |0.393239963234801  | 1.4623980|  -14.549538|  0.8912145|      0.1487254|null           |FALSE      |
|GO:2000280 |regulation of root development                                                                          | 0.3593729|-5.10330105164505    |-1.19052973222531  | 1.9084850|  -53.716023|  0.8653202|      0.1617632|null           |FALSE      |
|GO:0080155 |regulation of double fertilization forming a zygote and endosperm                                       | 0.2335924|-5.62389682269815    |-2.53514322530743  | 1.7242759|   -7.626381|  0.8896629|      0.1657447|null           |FALSE      |
|GO:0007346 |regulation of mitotic cell cycle                                                                        | 0.3818337|-4.59048033639922    |-0.840547580702962 | 1.9344985|  -10.155337|  0.8834858|      0.1730192|null           |FALSE      |
|GO:0035902 |response to immobilization stress                                                                       | 0.0089843|1.18896745074831     |-6.7692027372331   | 0.4771213|  -10.850181|  0.9516355|      0.1749598|null           |FALSE      |
|GO:0031146 |SCF-dependent proteasomal ubiquitin-dependent protein catabolic process                                 | 0.3773415|5.96751029249196     |-0.337450689371356 | 1.9294189|  -62.059796|  0.9103382|      0.1763754|null           |FALSE      |
|GO:0043457 |regulation of cellular respiration                                                                      | 0.0044922|-6.50607644745599    |-3.29328987806612  | 0.3010300|   -3.301968|  0.9080881|      0.1788328|null           |FALSE      |
|GO:0036290 |protein trans-autophosphorylation                                                                       | 0.0044922|6.57004140453545     |-0.776411837188659 | 0.3010300|   -1.365237|  0.9345889|      0.1861765|null           |FALSE      |
|GO:0080164 |regulation of nitric oxide metabolic process                                                            | 0.0089843|-3.29713041812363    |2.53035358988617   | 0.4771213|   -3.416873|  0.9085331|      0.1865822|null           |FALSE      |
|GO:0080184 |response to phenylpropanoid                                                                             | 0.0044922|-3.32655121747911    |-6.72852217748094  | 0.3010300|   -8.913764|  0.9354852|      0.2014785|null           |FALSE      |
|GO:0006359 |regulation of transcription by RNA polymerase III                                                       | 0.0044922|-6.68095735010796    |-2.91844141354703  | 0.3010300|   -5.285495|  0.9059802|      0.2131819|null           |FALSE      |
|GO:0019218 |regulation of steroid metabolic process                                                                 | 0.0269530|-3.48221890696161    |1.70481046292596   | 0.8450980|   -4.076369|  0.8954141|      0.2210355|null           |FALSE      |
|GO:2000762 |regulation of phenylpropanoid metabolic process                                                         | 0.1212884|-4.50517042768095    |0.0427863519453638 | 1.4471580|   -9.682429|  0.8873431|      0.2228407|null           |FALSE      |
|GO:0010310 |regulation of hydrogen peroxide metabolic process                                                       | 0.0539059|-4.21050568921972    |1.07175200247867   | 1.1139434|   -5.432629|  0.8906538|      0.2239731|null           |FALSE      |
|GO:0090143 |nucleoid organization                                                                                   | 0.0179686|2.31609307574076     |6.50013104367368   | 0.6989700|   -4.046449|  0.9409583|      0.2251592|null           |FALSE      |
|GO:0009410 |response to xenobiotic stimulus                                                                         | 0.0359373|-2.25371023314117    |-7.10424856189371  | 0.9542425|   -2.970613|  0.9367607|      0.2284321|null           |FALSE      |
|GO:0030587 |sorocarp development                                                                                    | 0.0404295|-3.26932192494506    |6.8880258200913    | 1.0000000|   -2.979628|  0.9450099|      0.2421996|null           |FALSE      |
|GO:0048462 |carpel formation                                                                                        | 0.0179686|-1.7644410998639     |6.77022950210656   | 0.6989700|  -22.697279|  0.8994079|      0.2504087|null           |FALSE      |
|GO:0030198 |extracellular matrix organization                                                                       | 0.0988275|2.00642169970675     |5.71812907737073   | 1.3617278|   -6.950692|  0.9287994|      0.2515722|null           |FALSE      |
|GO:0009969 |xyloglucan biosynthetic process                                                                         | 0.0808589|3.9659655394716      |4.31410523795793   | 1.2787536|  -24.960744|  0.8455730|      0.2530424|null           |FALSE      |
|GO:0010233 |phloem transport                                                                                        | 0.0943354|2.89667831502319     |-2.43520410335296  | 1.3424227|  -23.811719|  0.8845906|      0.2564512|null           |FALSE      |
|GO:1902025 |nitrate import                                                                                          | 0.0808589|4.10242159172717     |-5.16261699931165  | 1.2787536|  -22.839119|  0.9261728|      0.2646327|null           |FALSE      |
|GO:0042176 |regulation of protein catabolic process                                                                 | 0.4267553|-4.4417489258401     |0.418818926470403  | 1.9822712|  -12.811444|  0.8661027|      0.2713463|null           |FALSE      |
|GO:0012502 |induction of programmed cell death                                                                      | 0.0044922|-7.39023015610802    |-1.90164964008202  | 0.3010300|   -9.431784|  0.8779370|      0.2746607|null           |FALSE      |
|GO:0009268 |response to pH                                                                                          | 0.0359373|-4.10386744092862    |-6.23194411847939  | 0.9542425|   -4.887249|  0.9506672|      0.2827029|null           |FALSE      |
|GO:0045168 |cell-cell signaling involved in cell fate commitment                                                    | 0.1662100|-2.79797528517352    |5.9673667831251    | 1.5797836|  -22.581085|  0.8800424|      0.2860549|null           |FALSE      |
|GO:0033206 |meiotic cytokinesis                                                                                     | 0.0404295|0.031033901131986    |7.49383806239513   | 1.0000000|  -12.323855|  0.9455294|      0.2861295|null           |FALSE      |
|GO:0071236 |cellular response to antibiotic                                                                         | 0.8690604|-1.93873117374797    |-5.89683961098909  | 2.2878017|  -15.558175|  0.8808219|      0.2873162|null           |FALSE      |
|GO:0019756 |cyanogenic glycoside biosynthetic process                                                               | 0.1069562|6.02927077918734     |2.6114237518247    | 1.4313638|  -25.518707|  0.8630848|      0.2895405|null           |FALSE      |
|GO:0042908 |xenobiotic transport                                                                                    | 0.3369121|4.62876758716835     |-4.85422115247353  | 1.8808136|   -5.906055|  0.9404357|      0.2944657|null           |FALSE      |
|GO:0050826 |response to freezing                                                                                    | 0.1167962|0.0786136642526453   |-5.92098254922509  | 1.4313638|   -7.571291|  0.9359131|      0.3058984|null           |FALSE      |
|GO:0043153 |entrainment of circadian clock by photoperiod                                                           | 0.0224608|-1.94883796180369    |-4.19932819540559  | 0.7781513|  -11.106471|  0.8578360|      0.3069126|null           |FALSE      |
|GO:0018008 |N-terminal peptidyl-glycine N-myristoylation                                                            | 0.0089843|6.27894785137675     |2.98818470568119   | 0.4771213|   -6.862429|  0.9092171|      0.3108274|null           |FALSE      |
|GO:0033387 |putrescine biosynthetic process from ornithine                                                          | 0.0123534|5.49693595556943     |2.8718126687907    | 0.4771213|   -9.050561|  0.9016999|      0.3196216|null           |FALSE      |
|GO:0090559 |regulation of membrane permeability                                                                     | 0.0089843|-0.236306404704201   |-0.887970103247391 | 0.4771213|   -3.415678|  0.8962437|      0.3244429|null           |FALSE      |
|GO:0016567 |protein ubiquitination                                                                                  | 3.7105251|6.62322888708831     |0.331621260184572  | 2.9175055|  -19.832729|  0.9044925|      0.3270943|null           |FALSE      |
|GO:0071347 |cellular response to interleukin-1                                                                      | 0.0044922|-2.98163749243927    |-5.95163686966833  | 0.3010300|   -9.101635|  0.9131985|      0.3281054|null           |FALSE      |
|GO:0010221 |negative regulation of vernalization response                                                           | 0.0044922|-6.96876229339692    |2.34984218375906   | 0.3010300|   -8.832352|  0.8584035|      0.3379466|null           |FALSE      |
|GO:2000083 |negative regulation of L-ascorbic acid biosynthetic process                                             | 0.0044922|-5.92193222383455    |2.72227505688499   | 0.3010300|  -14.028558|  0.8612217|      0.3379466|null           |FALSE      |
|GO:0016045 |detection of bacterium                                                                                  | 0.0134765|0.656494983571066    |-7.30153487854885  | 0.6020600|   -3.008507|  0.9346272|      0.3385947|null           |FALSE      |
|GO:0048211 |Golgi vesicle docking                                                                                   | 0.0089843|4.16643406243276     |-4.09683646869979  | 0.4771213|   -5.955050|  0.9225179|      0.3424968|null           |FALSE      |
|GO:0010344 |seed oilbody biogenesis                                                                                 | 0.0404295|0.00438320645109192  |6.10024360899976   | 1.0000000|  -16.161163|  0.8581097|      0.3457972|null           |FALSE      |
|GO:0010275 |NAD(P)H dehydrogenase complex assembly                                                                  | 0.0494138|2.78548943174385     |5.76906272491549   | 1.0791812|  -18.708367|  0.9193149|      0.3465167|null           |FALSE      |
|GO:0070846 |Hsp90 deacetylation                                                                                     | 0.1033197|6.92642757457311     |-0.189231129090755 | 1.3802112|   -3.744559|  0.9252017|      0.3466347|null           |FALSE      |
|GO:2000098 |negative regulation of smooth muscle cell-matrix adhesion                                               | 0.0089843|-6.34136211269916    |3.12538784603779   | 0.4771213|   -2.526602|  0.8710965|      0.3529006|null           |FALSE      |
|GO:0015853 |adenine transport                                                                                       | 0.0179686|4.5922101253032      |-5.22308098053422  | 0.6989700|   -4.452453|  0.9429150|      0.3534636|null           |FALSE      |
|GO:0001172 |transcription,RNA-templated                                                                             | 0.0359373|5.78987233070262     |2.77369314015332   | 0.9542425|   -5.524972|  0.8883773|      0.3541850|null           |FALSE      |
|GO:0002011 |morphogenesis of an epithelial sheet                                                                    | 0.0134765|-2.96333636543451    |6.90583780562942   | 0.6020600|   -3.289730|  0.9405610|      0.3568861|null           |FALSE      |
|GO:0048544 |recognition of pollen                                                                                   | 0.5570280|-1.61186197338535    |5.39439700274645   | 2.0969100|   -5.518977|  0.8810407|      0.3616901|null           |FALSE      |
|GO:0090480 |purine nucleotide-sugar transmembrane transport                                                         | 0.0269530|4.3282130477205      |-5.30800231548571  | 0.8450980|   -1.440548|  0.9239980|      0.3628069|null           |FALSE      |
|GO:0006778 |porphyrin-containing compound metabolic process                                                         | 0.3458964|6.16643916763119     |0.917784916972946  | 1.8920946|   -6.174433|  0.9112531|      0.3675002|null           |FALSE      |
|GO:0071466 |cellular response to xenobiotic stimulus                                                                | 0.0224608|-2.63040529258145    |-5.86085206582283  | 0.7781513|   -6.082287|  0.9096813|      0.3682726|null           |FALSE      |
|GO:1902457 |negative regulation of stomatal opening                                                                 | 0.0179686|-6.38208437878856    |2.75092348941678   | 0.6989700|  -16.296225|  0.8658271|      0.3692394|null           |FALSE      |
|GO:0019915 |lipid storage                                                                                           | 0.1302727|4.68413391614494     |-4.57164761110321  | 1.4771213|   -1.587403|  0.9393397|      0.3729983|null           |FALSE      |
|GO:0009653 |anatomical structure morphogenesis                                                                      | 3.8587665|-2.49868362161861    |7.04446373819639   | 2.9344985|  -11.417696|  0.9264078|      0.3743725|null           |FALSE      |
|GO:0070848 |response to growth factor                                                                               | 0.9496038|-1.95513191895406    |-6.58568018487963  | 2.3263359|   -3.742023|  0.9101486|      0.3776394|null           |FALSE      |
|GO:0000966 |RNA 5&#39;-end processing                                                                                   | 0.0808589|6.40401206608619     |1.73076077924999   | 1.2787536|  -40.450386|  0.9104844|      0.3797277|null           |FALSE      |
|GO:0045056 |transcytosis                                                                                            | 0.0044922|3.0591433770837      |-2.6417211771752   | 0.3010300|   -5.955050|  0.8947429|      0.3809345|null           |FALSE      |
|GO:0080027 |response to herbivore                                                                                   | 0.0808589|0.205538405807983    |-7.13881931911276  | 1.2787536|   -2.945089|  0.9270232|      0.3830913|null           |FALSE      |
|GO:0002238 |response to molecule of fungal origin                                                                   | 0.0853511|-1.31795181747925    |-6.85831970677594  | 1.3010300|   -6.476875|  0.9023050|      0.3846165|null           |FALSE      |
|GO:0002213 |defense response to insect                                                                              | 0.0943354|0.133128978029338    |-6.65662557660112  | 1.3424227|   -7.205055|  0.9173149|      0.3874721|null           |FALSE      |
|GO:0045007 |depurination                                                                                            | 0.1069134|1.68634359267088     |-2.34977729326164  | 1.3802112|   -5.052084|  0.8374342|      0.3894002|null           |FALSE      |
|GO:0080156 |mitochondrial mRNA modification                                                                         | 0.1347648|6.31396508365814     |1.29264867852319   | 1.4913617|  -30.657427|  0.8984874|      0.3977989|null           |FALSE      |
|GO:0015851 |nucleobase transport                                                                                    | 0.1212884|4.44253992195539     |-5.01657476889296  | 1.4471580|   -6.123632|  0.9383012|      0.4022496|null           |FALSE      |
|GO:0007166 |cell surface receptor signaling pathway                                                                 | 0.5794888|-3.48804959484294    |-2.92556994466654  | 2.1139434|   -3.979747|  0.7925644|      0.4023812|null           |FALSE      |
|GO:0034434 |sterol esterification                                                                                   | 0.0089843|4.16619792436862     |1.15351803798717   | 0.4771213|   -4.442857|  0.9112561|      0.4053371|null           |FALSE      |
|GO:0042273 |ribosomal large subunit biogenesis                                                                      | 0.4267553|2.59500307600988     |5.90574776137205   | 1.9822712|  -12.711372|  0.9158063|      0.4059178|null           |FALSE      |
|GO:0034394 |protein localization to cell surface                                                                    | 0.0134765|4.59894434936074     |-4.22592693847902  | 0.6020600|   -3.600617|  0.9237543|      0.4074874|null           |FALSE      |
|GO:1902325 |negative regulation of chlorophyll biosynthetic process                                                 | 0.0044922|-5.81212690551918    |2.83324185512915   | 0.3010300|   -3.756088|  0.8663889|      0.4086043|null           |FALSE      |
|GO:0048240 |sperm capacitation                                                                                      | 0.0044922|-1.17171039026929    |6.82995129296309   | 0.3010300|   -6.984336|  0.8960849|      0.4096889|null           |FALSE      |
|GO:0009914 |hormone transport                                                                                       | 0.4447240|0.368471168726447    |-2.11474333958303  | 2.0000000|  -20.141593|  0.8112191|      0.4103707|null           |FALSE      |
|GO:0009992 |cellular water homeostasis                                                                              | 0.0089843|0.852335435916256    |3.15093147319499   | 0.4771213|   -1.795237|  0.8427811|      0.4103707|null           |FALSE      |
|GO:1902184 |negative regulation of shoot apical meristem development                                                | 0.0044922|-6.45443651539495    |2.96228878220148   | 0.3010300|   -3.583883|  0.8646357|      0.4125001|null           |FALSE      |
|GO:0006970 |response to osmotic stress                                                                              | 2.3449081|-0.604525538247306   |-5.92942142162411  | 2.7185017|   -2.680020|  0.9189246|      0.4182879|null           |FALSE      |
|GO:0035864 |response to potassium ion                                                                               | 0.0269530|-2.89930363918495    |-6.72566479548436  | 0.8450980|   -5.334631|  0.9311235|      0.4217509|null           |FALSE      |
|GO:0034614 |cellular response to reactive oxygen species                                                            | 0.1482413|-1.57782571087564    |-5.92234026442434  | 1.5314789|  -14.569446|  0.8723508|      0.4299938|null           |FALSE      |
|GO:0010469 |regulation of signaling receptor activity                                                               | 0.3059115|-5.88330769607882    |0.381493264482398  | 1.8260748|  -34.558948|  0.8191877|      0.4321507|null           |FALSE      |
|GO:1905255 |regulation of RNA binding transcription factor activity                                                 | 0.0044922|-0.599840038476302   |1.88674314777247   | 0.3010300|   -6.024302|  0.9124346|      0.4321507|null           |FALSE      |
|GO:0097167 |circadian regulation of translation                                                                     | 0.0044922|-3.53495997653747    |3.84014080093407   | 0.3010300|   -2.803332|  0.8558297|      0.4338639|null           |FALSE      |
|GO:2000117 |negative regulation of cysteine-type endopeptidase activity                                             | 0.0044922|-5.35074049162947    |2.41831401320342   | 0.3010300|   -9.726094|  0.8426812|      0.4338639|null           |FALSE      |
|GO:0070667 |negative regulation of mast cell proliferation                                                          | 0.1886708|-5.91430265568821    |1.84843843826118   | 1.6334685|   -6.024302|  0.8418425|      0.4380368|null           |FALSE      |
|GO:0006470 |protein dephosphorylation                                                                               | 0.8624949|6.39663523137634     |0.204874781594742  | 2.2855573|   -5.193447|  0.9073305|      0.4380416|null           |FALSE      |
|GO:0035725 |sodium ion transmembrane transport                                                                      | 0.0673824|3.83703493143889     |-5.11645024637494  | 1.2041200|  -14.994073|  0.9106587|      0.4453910|null           |FALSE      |
|GO:0018107 |peptidyl-threonine phosphorylation                                                                      | 0.0628903|6.49951898196116     |-0.165939674226638 | 1.1760913|   -1.722416|  0.9202151|      0.4457143|null           |FALSE      |
|GO:0046949 |fatty-acyl-CoA biosynthetic process                                                                     | 0.0044922|5.46003621061712     |2.07881480850629   | 0.3010300|   -2.400073|  0.8794172|      0.4484771|null           |FALSE      |
|GO:0009311 |oligosaccharide metabolic process                                                                       | 0.4267553|6.93119700998515     |-2.17068520756109  | 1.9822712|  -10.336692|  0.9493902|      0.4499218|null           |FALSE      |
|GO:0050935 |iridophore differentiation                                                                              | 0.0224608|-3.30328932599797    |6.7116882186934    | 0.7781513|   -2.977004|  0.9244586|      0.4542384|null           |FALSE      |
|GO:0060776 |simple leaf morphogenesis                                                                               | 0.0224608|-2.19541515713702    |6.63285484479638   | 0.7781513|  -15.554054|  0.9134994|      0.4581927|null           |FALSE      |
|GO:0000025 |maltose catabolic process                                                                               | 0.0089843|6.3711747039743      |-1.93523656063165  | 0.4771213|   -4.860108|  0.9350785|      0.4597570|null           |FALSE      |
|GO:0051782 |negative regulation of cell division                                                                    | 0.0404295|-6.25576909051909    |2.36558449310613   | 1.0000000|   -4.382314|  0.8559921|      0.4611342|null           |FALSE      |
|GO:0031098 |stress-activated protein kinase signaling cascade                                                       | 0.3252325|-3.1571658981202     |-3.22490044429577  | 1.8633229|  -17.979816|  0.7855473|      0.4634292|null           |FALSE      |
|GO:0006952 |defense response                                                                                        | 7.2503481|-0.730669531356728   |-6.1851449822815   | 3.2081725|  -56.039009|  0.9151060|      0.4644931|null           |FALSE      |
|GO:0071368 |cellular response to cytokinin stimulus                                                                 | 0.3593729|-2.08445141982054    |-6.07899089705686  | 1.9084850|   -9.739302|  0.8807703|      0.4666976|null           |FALSE      |
|GO:0070134 |positive regulation of mitochondrial translational initiation                                           | 0.0134765|-6.61733202747271    |-0.982044797977501 | 0.6020600|   -3.636248|  0.8555298|      0.4670366|null           |FALSE      |
|GO:1904541 |fungal-type cell wall disassembly involved in conjugation with cellular fusion                          | 0.0044922|1.20847370574945     |6.20360954758139   | 0.3010300|   -5.079019|  0.8990940|      0.4680595|null           |FALSE      |
|GO:0061614 |pri-miRNA transcription by RNA polymerase II                                                            | 0.0044922|5.90184581973976     |2.68747701891453   | 0.3010300|   -2.077188|  0.8992878|      0.4728445|null           |FALSE      |
|GO:0042773 |ATP synthesis coupled electron transport                                                                | 0.2830062|2.82146076370032     |-7.3491440678727   | 1.8061800|   -7.751660|  0.9373885|      0.4728635|null           |FALSE      |
|GO:0010258 |NADH dehydrogenase complex (plastoquinone) assembly                                                     | 0.0359373|2.84536706205272     |5.87115241519019   | 0.9542425|   -1.546163|  0.9209507|      0.4782752|null           |FALSE      |
|GO:0018063 |cytochrome c-heme linkage                                                                               | 0.0374347|3.33262731725018     |4.43425627081865   | 0.9542425|   -3.227392|  0.8786739|      0.4795346|null           |FALSE      |
|GO:0009626 |plant-type hypersensitive response                                                                      | 0.3279278|-0.77899636859711    |-6.04234968880074  | 1.8692317|  -16.522254|  0.8504587|      0.4809374|null           |FALSE      |
|GO:0006005 |L-fucose biosynthetic process                                                                           | 0.0044922|5.42419159723074     |4.11073491672216   | 0.3010300|   -5.700801|  0.9254947|      0.4814675|null           |FALSE      |
|GO:0006203 |dGTP catabolic process                                                                                  | 0.0134765|5.34243729805674     |1.10432186524176   | 0.6020600|   -1.983263|  0.8834027|      0.4816793|null           |FALSE      |
|GO:0051607 |defense response to virus                                                                               | 0.2246081|-0.0835554521569006  |-6.60376216912546  | 1.7075702|   -4.801239|  0.9116839|      0.4865194|null           |FALSE      |
|GO:0031333 |negative regulation of protein-containing complex assembly                                              | 0.0853511|-6.12945685657378    |2.00023254487366   | 1.3010300|   -5.068979|  0.8436196|      0.4869655|null           |FALSE      |
|GO:0042351 |&#39;de novo&#39; GDP-L-fucose biosynthetic process                                                             | 0.0179686|5.37998598421506     |2.21092881497792   | 0.6989700|   -3.583883|  0.8784326|      0.4886794|null           |FALSE      |
|GO:0033292 |T-tubule organization                                                                                   | 0.0716381|0.199693859594683    |6.27889141127433   | 1.2041200|  -15.023967|  0.8709968|      0.4928611|null           |FALSE      |
|GO:0036309 |protein localization to M-band                                                                          | 0.2713265|4.17165808399316     |-4.38810446894929  | 1.7853298|  -15.023967|  0.9058106|      0.4950361|null           |FALSE      |
|GO:0043627 |response to estrogen                                                                                    | 1.1430508|-1.96947820041636    |-6.48140015406912  | 2.4065402|   -1.596382|  0.9062300|      0.4956526|null           |FALSE      |
|GO:0034080 |CENP-A containing chromatin assembly                                                                    | 0.0134765|2.5064032996944      |5.53656336850478   | 0.6020600|   -5.066592|  0.9156426|      0.4961586|null           |FALSE      |
|GO:0035967 |cellular response to topologically incorrect protein                                                    | 0.2964826|-1.57748718788306    |-6.0262498207151   | 1.8260748|   -3.743924|  0.8732979|      0.4967938|null           |FALSE      |
|GO:0070121 |Kupffer&#39;s vesicle development                                                                           | 1.4053739|-2.61068145232434    |7.04641351507991   | 2.4927604|   -2.089786|  0.9328491|      0.4968539|null           |FALSE      |
|GO:0032527 |protein exit from endoplasmic reticulum                                                                 | 0.0359373|4.37227239958501     |-4.55460713406244  | 0.9542425|   -8.961732|  0.9129281|      0.4998121|null           |FALSE      |
|GO:0006611 |protein export from nucleus                                                                             | 0.0404295|4.24070241416007     |-4.55284929623066  | 1.0000000|   -2.080898|  0.9122716|      0.5041019|null           |FALSE      |
|GO:0050821 |protein stabilization                                                                                   | 0.1437492|-1.55323772859594    |-0.662439059280755 | 1.5185139|   -3.421658|  0.8764261|      0.5055010|null           |FALSE      |
|GO:0002938 |tRNA guanine ribose methylation                                                                         | 0.0044922|6.58981105103757     |1.28209271566652   | 0.3010300|   -5.700801|  0.9171224|      0.5055471|null           |FALSE      |
|GO:0010324 |membrane invagination                                                                                   | 0.0044922|2.62896878643155     |6.5409171982587    | 0.3010300|   -6.184266|  0.9426523|      0.5057439|null           |FALSE      |
|GO:0009088 |threonine biosynthetic process                                                                          | 0.0449216|5.5235594151898      |3.38265086125343   | 1.0413927|   -8.292342|  0.9012374|      0.5069649|null           |FALSE      |
|GO:0097237 |cellular response to toxic substance                                                                    | 0.8355420|-1.91660087330853    |-5.98207219809208  | 2.2718416|   -3.138372|  0.8812308|      0.5080142|null           |FALSE      |
|GO:0071289 |cellular response to nickel ion                                                                         | 0.0044922|-3.02353226445505    |-6.17862178034418  | 0.3010300|   -1.958210|  0.9111009|      0.5089993|null           |FALSE      |
|GO:0009451 |RNA modification                                                                                        | 1.4105386|6.09013083551678     |1.19055898197162   | 2.4983106| -190.721643|  0.8850567|      0.5091922|null           |FALSE      |
|GO:0006045 |N-acetylglucosamine biosynthetic process                                                                | 0.0359373|5.47894195523842     |4.63273282163786   | 0.9542425|   -3.301968|  0.9270242|      0.5116103|null           |FALSE      |
|GO:0052544 |defense response by callose deposition in cell wall                                                     | 0.0718746|1.59475468429105     |-3.46805702060261  | 1.2304489|   -5.613441|  0.7928344|      0.5168809|null           |FALSE      |
|GO:1900426 |positive regulation of defense response to bacterium                                                    | 0.1167962|-6.8989389612897     |-0.362677336071039 | 1.4313638|  -13.256654|  0.8340098|      0.5185815|null           |FALSE      |
|GO:0006509 |membrane protein ectodomain proteolysis                                                                 | 0.4761691|6.88317608342165     |-0.497203520687196 | 2.0293838|  -10.923143|  0.9415952|      0.5203722|null           |FALSE      |
|GO:0060291 |long-term synaptic potentiation                                                                         | 0.0044922|-6.04855110629916    |-0.686911851283696 | 0.3010300|   -9.880986|  0.8280914|      0.5247898|null           |FALSE      |
|GO:0010618 |aerenchyma formation                                                                                    | 0.0089843|-2.86286066311377    |7.01619316676351   | 0.4771213|  -14.702027|  0.9416374|      0.5264717|null           |FALSE      |
|GO:0042407 |cristae formation                                                                                       | 0.0089843|2.50053511008116     |6.30981426138387   | 0.4771213|   -7.288255|  0.9348110|      0.5269143|null           |FALSE      |
|GO:0032418 |lysosome localization                                                                                   | 0.0089843|4.08320301646875     |-4.16399192634063  | 0.4771213|   -3.744559|  0.9254065|      0.5279778|null           |FALSE      |
|GO:0006885 |regulation of pH                                                                                        | 0.2515610|-1.88574587871377    |-0.594457690911824 | 1.7558749|  -19.750916|  0.8670491|      0.5303141|null           |FALSE      |
|GO:0051070 |galactomannan biosynthetic process                                                                      | 0.0044922|5.72767654809276     |4.34694284903325   | 0.3010300|   -6.568896|  0.9266011|      0.5326031|null           |FALSE      |
|GO:0033007 |negative regulation of mast cell activation involved in immune response                                 | 0.0898432|-6.45479336024951    |0.929120269156447  | 1.3222193|   -6.024302|  0.7946529|      0.5331877|null           |FALSE      |
|GO:0010064 |embryonic shoot morphogenesis                                                                           | 0.0044922|-2.21104248842062    |6.77902024944818   | 0.3010300|   -6.862429|  0.9249109|      0.5336678|null           |FALSE      |
|GO:0010365 |positive regulation of ethylene biosynthetic process                                                    | 0.0089843|-6.90952492868364    |-1.27860985519434  | 0.4771213|   -2.346859|  0.8647769|      0.5384409|null           |FALSE      |
|GO:0009617 |response to bacterium                                                                                   | 2.1113158|-0.480937312147326   |-6.75660743487679  | 2.6730209|  -10.289218|  0.9042746|      0.5453319|null           |FALSE      |
|GO:1901684 |arsenate ion transmembrane transport                                                                    | 0.0224608|3.88361732851002     |-5.3406851263847   | 0.7781513|   -3.269484|  0.9198247|      0.5470253|null           |FALSE      |
|GO:1904837 |beta-catenin-TCF complex assembly                                                                       | 0.5076142|2.54796800451435     |5.71705609704978   | 2.0569049|   -8.582182|  0.9058896|      0.5532566|null           |FALSE      |
|GO:0001667 |ameboidal-type cell migration                                                                           | 0.0044922|5.25702737979661     |-4.33508990024064  | 0.3010300|   -9.728819|  0.9255120|      0.5585812|null           |FALSE      |
|GO:0006468 |protein phosphorylation                                                                                 | 5.0671578|6.29911633285597     |0.396957852233656  | 3.0526939|  -23.861870|  0.8907838|      0.5616227|null           |FALSE      |
|GO:0009867 |jasmonic acid mediated signaling pathway                                                                | 0.3998023|-3.30459233062543    |-3.66051728047254  | 1.9542425|  -14.238823|  0.7625066|      0.5617056|null           |FALSE      |
|GO:0000375 |RNA splicing,via transesterification reactions                                                          | 1.1140560|6.10215724020801     |1.54932367284088   | 2.3961993|   -6.872762|  0.8903925|      0.5628313|null           |FALSE      |
|GO:0048509 |regulation of meristem development                                                                      | 0.2964826|-5.16061222318677    |-1.33259768233374  | 1.8260748|  -21.686326|  0.8672061|      0.5634440|null           |FALSE      |
|GO:0071320 |cellular response to cAMP                                                                               | 0.1613662|-2.29350692971063    |-6.19895768295331  | 1.5563025|  -11.821209|  0.8705757|      0.5656929|null           |FALSE      |
|GO:1902265 |abscisic acid homeostasis                                                                               | 0.0089843|-0.222475308715226   |-0.688495447381255 | 0.4771213|   -7.364876|  0.8933171|      0.5661740|null           |FALSE      |
|GO:0007066 |female meiosis sister chromatid cohesion                                                                | 0.0044922|0.651621814264355    |5.99205156606431   | 0.3010300|   -5.070379|  0.8820383|      0.5767286|null           |FALSE      |
|GO:0080003 |thalianol metabolic process                                                                             | 0.0089843|4.10845697920672     |1.37453223097521   | 0.4771213|  -21.022129|  0.9022956|      0.5772455|null           |FALSE      |
|GO:0046330 |positive regulation of JNK cascade                                                                      | 0.0179686|-7.11406466765452    |-0.192458231676479 | 0.6989700|   -9.722158|  0.8270692|      0.5831457|null           |FALSE      |
|GO:0009852 |auxin catabolic process                                                                                 | 0.0089843|0.575852619819533    |-0.435117531446739 | 0.4771213|   -8.055166|  0.8469552|      0.5840401|null           |FALSE      |
|GO:0071229 |cellular response to acid chemical                                                                      | 0.2740218|-2.17530185861355    |-5.98694798221151  | 1.7923917|   -8.441768|  0.8885371|      0.5845806|null           |FALSE      |
|GO:0048439 |flower morphogenesis                                                                                    | 0.0269530|-1.82453659402957    |6.7179363660989    | 0.8450980|   -2.532840|  0.9047518|      0.5912012|null           |FALSE      |
|GO:1900033 |negative regulation of trichome patterning                                                              | 0.0269530|-6.17448023606405    |1.13001692540716   | 0.8450980|   -9.916671|  0.7941076|      0.5919133|null           |FALSE      |
|GO:0045815 |epigenetic maintenance of chromatin in transcription-competent conformation                             | 0.0044922|-1.54861110511436    |2.70422555178705   | 0.3010300|   -2.677396|  0.8242731|      0.5941185|null           |FALSE      |
|GO:0042308 |negative regulation of protein import into nucleus                                                      | 0.0044922|-6.26047519918527    |3.32788491004211   | 0.3010300|   -2.526602|  0.8405781|      0.5958785|null           |FALSE      |
|GO:0009664 |plant-type cell wall organization                                                                       | 0.6693320|2.13902497998678     |5.61413255522135   | 2.1760913|   -5.970664|  0.9072256|      0.5966817|null           |FALSE      |
|GO:0080163 |regulation of protein serine/threonine phosphatase activity                                             | 0.0853511|-4.00189404612411    |0.800392841568919  | 1.3010300|   -3.131269|  0.8597717|      0.6025765|null           |FALSE      |
|GO:0010842 |retina layer formation                                                                                  | 0.0988275|-2.4005077387882     |6.55132221949652   | 1.3617278|   -2.222439|  0.9051649|      0.6076887|null           |FALSE      |
|GO:0048658 |anther wall tapetum development                                                                         | 0.0853511|-1.63611462219893    |6.67839434780781   | 1.3010300|  -12.553830|  0.8997085|      0.6091195|null           |FALSE      |
|GO:0010423 |negative regulation of brassinosteroid biosynthetic process                                             | 0.0044922|-4.82080154329404    |1.92271932653288   | 0.3010300|   -3.083357|  0.8296222|      0.6098624|null           |FALSE      |
|GO:1901526 |positive regulation of mitophagy                                                                        | 0.0044922|-7.08526991381637    |-1.43823153998601  | 0.3010300|   -3.744559|  0.8655825|      0.6099952|null           |FALSE      |
|GO:0009734 |auxin-activated signaling pathway                                                                       | 0.9972598|-3.36273493289832    |-3.40034311473034  | 2.3483049|   -1.757187|  0.7550982|      0.6116609|null           |FALSE      |
|GO:0098703 |calcium ion import across plasma membrane                                                               | 0.0170702|2.65119406531936     |-3.65056978794137  | 0.6020600|   -9.880986|  0.8063703|      0.6124892|null           |FALSE      |
|GO:0010150 |leaf senescence                                                                                         | 0.5076142|-2.09057726229977    |6.5437739166435    | 2.0569049|   -8.133262|  0.9012909|      0.6130285|null           |FALSE      |
|GO:0000390 |spliceosomal complex disassembly                                                                        | 0.0134765|3.53226687869371     |4.06506747322212   | 0.6020600|   -6.520794|  0.8685268|      0.6146354|null           |FALSE      |
|GO:0080126 |ovary septum development                                                                                | 0.0044922|-1.52420608084124    |6.87797671137175   | 0.3010300|  -16.865539|  0.9150752|      0.6149303|null           |FALSE      |
|GO:0010930 |negative regulation of auxin mediated signaling pathway                                                 | 0.0134765|-6.69065101763482    |1.64362008426335   | 0.6020600|   -3.593907|  0.8283117|      0.6168852|null           |FALSE      |
|GO:0019348 |dolichol metabolic process                                                                              | 0.0269530|3.25430865718879     |1.74460463727765   | 0.8450980|   -2.857557|  0.8925309|      0.6194937|null           |FALSE      |
|GO:0046345 |abscisic acid catabolic process                                                                         | 0.0269530|3.42727728758679     |1.24244356840095   | 0.8450980|  -35.511723|  0.8722247|      0.6194937|null           |FALSE      |
|GO:0000226 |microtubule cytoskeleton organization                                                                   | 0.7232379|2.26867697064053     |5.30809332982271   | 2.2095150|  -11.150007|  0.9067467|      0.6208058|null           |FALSE      |
|GO:1901599 |(-)-pinoresinol biosynthetic process                                                                    | 0.0089843|4.99445189478575     |2.76167945605765   | 0.4771213|   -2.089786|  0.8871899|      0.6243888|null           |FALSE      |
|GO:0009631 |cold acclimation                                                                                        | 0.2560532|-0.142888069770785   |-5.89317809364086  | 1.7634280|   -2.700705|  0.9319428|      0.6252015|null           |FALSE      |
|GO:0010082 |regulation of root meristem growth                                                                      | 0.1527335|-3.54161346323317    |4.23866792290747   | 1.5440680|  -54.195293|  0.7841844|      0.6297772|null           |FALSE      |
|GO:1900865 |chloroplast RNA modification                                                                            | 0.1212884|6.35065217366093     |1.21156762571234   | 1.4471580|   -3.916060|  0.9024080|      0.6305983|null           |FALSE      |
|GO:1905614 |negative regulation of developmental vegetative growth                                                  | 0.0134765|-6.27542615267775    |2.52476552959001   | 0.6020600|  -18.376548|  0.8508444|      0.6315937|null           |FALSE      |
|GO:0045861 |negative regulation of proteolysis                                                                      | 0.2111316|-5.55586825741768    |1.57273057353601   | 1.6812412|   -4.202439|  0.8202516|      0.6323680|null           |FALSE      |
|GO:0001188 |RNA polymerase I preinitiation complex assembly                                                         | 0.0179686|4.19182578975725     |3.74912669015869   | 0.6989700|   -1.484557|  0.8414282|      0.6325950|null           |FALSE      |
|GO:0050728 |negative regulation of inflammatory response                                                            | 0.0346538|-6.69139117308451    |1.83965778943381   | 0.9030900|  -11.106471|  0.8389393|      0.6338016|null           |FALSE      |
|GO:0071731 |response to nitric oxide                                                                                | 0.0269530|-2.67825424532809    |-6.77360507879083  | 0.8450980|   -3.335147|  0.9209260|      0.6341328|null           |FALSE      |
|GO:0007165 |signal transduction                                                                                     | 8.3913571|-3.54806873296914    |-2.56641895455443  | 3.2716093|  -63.045228|  0.7369905|      0.6344952|null           |FALSE      |
|GO:0080037 |negative regulation of cytokinin-activated signaling pathway                                            | 0.0224608|-6.60650852114453    |1.53342436883049   | 0.7781513|   -9.070433|  0.8231612|      0.6358455|null           |FALSE      |
|GO:0015700 |arsenite transport                                                                                      | 0.0224608|3.92722832831669     |-5.42936321410096  | 0.7781513|   -4.650523|  0.9369611|      0.6364426|null           |FALSE      |
|GO:0061734 |parkin-mediated stimulation of mitophagy in response to mitochondrial depolarization                    | 5.4400072|-1.16602817994155    |-5.79380778865562  | 3.0835026|   -3.744559|  0.8665656|      0.6378968|null           |FALSE      |
|GO:0033542 |fatty acid beta-oxidation,unsaturated,even number                                                       | 0.0044922|3.54237349628041     |0.934050540019219  | 0.3010300|   -8.187130|  0.9078885|      0.6436924|null           |FALSE      |
|GO:0035024 |negative regulation of Rho protein signal transduction                                                  | 0.0044922|-6.86221677575412    |1.88931119919524   | 0.3010300|   -3.940235|  0.8354476|      0.6447047|null           |FALSE      |
|GO:1905034 |regulation of antifungal innate immune response                                                         | 0.0089843|-7.83725932971154    |0.467477284037492  | 0.4771213|   -5.043818|  0.8798357|      0.6459136|null           |FALSE      |
|GO:0010500 |transmitting tissue development                                                                         | 0.0179686|-1.52794410837131    |6.79208822886904   | 0.6989700|  -10.077522|  0.9102808|      0.6473838|null           |FALSE      |
|GO:0034307 |regulation of ascospore formation                                                                       | 0.0044922|-5.91503865799493    |-2.71130163931535  | 0.3010300|   -4.046449|  0.8788871|      0.6492064|null           |FALSE      |
|GO:0032260 |response to jasmonic acid stimulus involved in jasmonic acid and ethylene-dependent systemic resistance | 0.0134765|-1.0195819607326     |-6.67807070848535  | 0.6020600|  -13.981089|  0.8951824|      0.6492412|null           |FALSE      |
|GO:0010227 |floral organ abscission                                                                                 | 0.1123040|-1.65492177532267    |6.60820057331424   | 1.4149733|   -6.507642|  0.9004983|      0.6505166|null           |FALSE      |
|GO:0010942 |positive regulation of cell death                                                                       | 0.1527335|-6.55794528460104    |-1.12908506093033  | 1.5440680|   -6.291367|  0.8501657|      0.6549617|null           |FALSE      |
|GO:1902042 |negative regulation of extrinsic apoptotic signaling pathway via death domain receptors                 | 0.0044922|-6.91500504618958    |1.55761426612148   | 0.3010300|   -4.331802|  0.8321359|      0.6549617|null           |FALSE      |
|GO:0009972 |cytidine deamination                                                                                    | 0.0404295|5.29416912623416     |1.51342903303764   | 1.0000000|  -16.574468|  0.8744144|      0.6586003|null           |FALSE      |
|GO:2000026 |regulation of multicellular organismal development                                                      | 1.5677643|-4.98480765290488    |-0.76441350097114  | 2.5440680|  -11.646700|  0.8223296|      0.6593517|null           |FALSE      |
|GO:2000038 |regulation of stomatal complex development                                                              | 0.0539059|-5.28518963713917    |-2.07924087919317  | 1.1139434|   -6.601214|  0.8608544|      0.6593634|null           |FALSE      |
|GO:0030150 |protein import into mitochondrial matrix                                                                | 0.1257805|3.53160902493454     |-2.05067793566718  | 1.4623980|   -9.888893|  0.8541338|      0.6622637|null           |FALSE      |
|GO:1900459 |positive regulation of brassinosteroid mediated signaling pathway                                       | 0.0404295|-6.92086570274957    |-0.181646281241227 | 1.0000000|   -5.248078|  0.8255615|      0.6624068|null           |FALSE      |
|GO:0000028 |ribosomal small subunit assembly                                                                        | 0.1527335|2.57194216499813     |5.63854903639821   | 1.5440680|   -4.099637|  0.9036803|      0.6628457|null           |FALSE      |
|GO:0009961 |response to 1-aminocyclopropane-1-carboxylic acid                                                       | 0.0179686|-2.53776817152387    |-6.82383928876968  | 0.6989700|   -1.744255|  0.9202127|      0.6635002|null           |FALSE      |
|GO:0043086 |negative regulation of catalytic activity                                                               | 1.1410089|-3.1040291393413     |0.173870819586679  | 2.4065402|   -5.846133|  0.8717257|      0.6653174|null           |FALSE      |
|GO:0006813 |potassium ion transport                                                                                 | 0.4222632|3.89394720830787     |-4.93897776858236  | 1.9777236|  -10.222966|  0.9223747|      0.6681870|null           |FALSE      |
|GO:0015886 |heme transport                                                                                          | 0.0718746|4.03426286749224     |-5.05484409516048  | 1.2304489|   -2.509303|  0.9259611|      0.6715812|null           |FALSE      |
|GO:0045338 |farnesyl diphosphate metabolic process                                                                  | 0.0898432|4.0811879060252      |0.0506981617095167 | 1.3222193|  -24.428219|  0.8981716|      0.6735153|null           |FALSE      |
|GO:2000033 |regulation of seed dormancy process                                                                     | 0.0718746|-5.44183254646981    |-2.11223130040738  | 1.2304489|   -2.985810|  0.8504579|      0.6760121|null           |FALSE      |
|GO:0010642 |negative regulation of platelet-derived growth factor receptor signaling pathway                        | 0.0686659|-6.43667820945763    |1.36649287000925   | 1.2041200|   -2.526602|  0.8107486|      0.6816798|null           |FALSE      |
|GO:1900449 |regulation of glutamate receptor signaling pathway                                                      | 0.1735903|-6.61039884950616    |0.309158812445112  | 1.5682017|   -4.331802|  0.8425925|      0.6858401|null           |FALSE      |
|GO:0042073 |intraciliary transport                                                                                  | 0.0242577|3.72584695642038     |-2.10290361366886  | 0.7781513|   -5.067553|  0.8512920|      0.6863235|null           |FALSE      |
|GO:0010727 |negative regulation of hydrogen peroxide metabolic process                                              | 0.0044922|-5.85583218761338    |2.92572565247116   | 0.3010300|   -3.745360|  0.8654100|      0.6872518|null           |FALSE      |
|GO:0030994 |primary cell septum disassembly                                                                         | 0.0817573|1.98003653577218     |6.23982444037705   | 1.2787536|   -6.121054|  0.9274023|      0.6894654|null           |FALSE      |
|GO:0010030 |positive regulation of seed germination                                                                 | 0.0898432|-6.1271390711791     |-1.42044398785891  | 1.3222193|   -9.733717|  0.8215098|      0.6895163|null           |FALSE      |
|GO:0006812 |cation transport                                                                                        | 2.2505728|4.00292649968702     |-4.83574303846473  | 2.7007037|   -7.647909|  0.9162393|      0.6936154|null           |FALSE      |
|GO:0048767 |root hair elongation                                                                                    | 0.3099591|-2.07212675078825    |6.39305784087038   | 1.8450980|  -13.755379|  0.8611765|      0.6947824|null           |FALSE      |
|GO:2000095 |regulation of Wnt signaling pathway,planar cell polarity pathway                                        | 0.2030457|-6.18432586340477    |0.0721622897717001 | 1.6627578|   -1.707227|  0.8137973|      0.6948060|null           |FALSE      |
|GO:2001040 |positive regulation of cellular response to drug                                                        | 0.2554321|-6.50282286568103    |-0.230253351972413 | 1.7558749|   -3.743868|  0.8220447|      0.6964743|null           |FALSE      |
|GO:0010120 |camalexin biosynthetic process                                                                          | 0.0494138|5.87072731499301     |2.48612541910044   | 1.0791812|   -4.492216|  0.8788899|      0.6980352|null           |FALSE      |

[1] &quot;missing_seagrasses_GO.txt&quot;


|TermID     |Name                                                                                         |  Frequency|PlotX              |PlotY                |   LogSize|       Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:--------------------------------------------------------------------------------------------|----------:|:------------------|:--------------------|---------:|-----------:|----------:|--------------:|:--------------|:----------|
|GO:0010082 |regulation of root meristem growth                                                           |  0.1527335|-0.109685317050515 |-2.76600868766711    | 1.5440680|  -85.813720|  0.7545471|      0.0000000|null           |FALSE      |
|GO:0051762 |sesquiterpene biosynthetic process                                                           |  0.1123040|-2.67225569918502  |6.05949581421155     | 1.4149733| -123.263200|  0.8689906|      0.0000000|null           |FALSE      |
|GO:1902025 |nitrate import                                                                               |  0.0808589|-5.44176530860059  |-2.23067578936915    | 1.2787536|  -31.159863|  0.9346240|      0.0000000|null           |FALSE      |
|GO:0032197 |transposition,RNA-mediated                                                                   |  0.0044922|4.13271599152358   |-7.75538418876687    | 0.3010300|   -1.308417|  0.9809233|      0.0198115|null           |FALSE      |
|GO:0006335 |DNA replication-dependent chromatin assembly                                                 |  0.0179686|2.08292821428531   |-7.69296192962809    | 0.6989700|   -2.182306|  0.9442327|      0.0215931|null           |FALSE      |
|GO:0009626 |plant-type hypersensitive response                                                           |  0.3279278|5.40554503721905   |-2.84098381054351    | 1.8692317|  -29.280582|  0.7982384|      0.0266051|null           |FALSE      |
|GO:0051554 |flavonol metabolic process                                                                   |  0.0853511|-5.7938205514577   |0.602661277635636    | 1.3010300|  -32.847642|  0.9553768|      0.0732686|null           |FALSE      |
|GO:0031146 |SCF-dependent proteasomal ubiquitin-dependent protein catabolic process                      |  0.3773415|-4.69301964648176  |2.22596326372768     | 1.9294189|  -71.808392|  0.9124035|      0.0932346|null           |FALSE      |
|GO:0009311 |oligosaccharide metabolic process                                                            |  0.4267553|-6.80364353770996  |0.52832810489312     | 1.9822712|   -5.851302|  0.9628224|      0.1045125|null           |FALSE      |
|GO:0097503 |sialylation                                                                                  |  0.0134765|-6.09920543642178  |2.25240825325758     | 0.6020600|   -1.496617|  0.9619123|      0.1181081|null           |FALSE      |
|GO:0080184 |response to phenylpropanoid                                                                  |  0.0044922|7.78520894764213   |0.622211116284786    | 0.3010300|  -11.208254|  0.8996056|      0.1196905|null           |FALSE      |
|GO:0006885 |regulation of pH                                                                             |  0.2515610|-0.320261868703792 |1.96074542722621     | 1.7558749|  -19.446794|  0.8631261|      0.1360520|null           |FALSE      |
|GO:0009962 |regulation of flavonoid biosynthetic process                                                 |  0.1257805|1.0364898457127    |4.23483464398829     | 1.4623980|  -35.352336|  0.9043729|      0.1471751|null           |FALSE      |
|GO:0009820 |alkaloid metabolic process                                                                   |  0.0224608|-6.24659496820002  |1.300739721765       | 0.7781513|   -9.047597|  0.9648735|      0.1530936|null           |FALSE      |
|GO:0010469 |regulation of signaling receptor activity                                                    |  0.3059115|3.21962785634639   |5.07945787886728     | 1.8260748|  -45.181379|  0.8265039|      0.1578322|null           |FALSE      |
|GO:0090305 |nucleic acid phosphodiester bond hydrolysis                                                  |  1.6171780|-4.58186688278905  |3.87245337427438     | 2.5575072|  -54.837630|  0.9062679|      0.1763754|null           |FALSE      |
|GO:0010942 |positive regulation of cell death                                                            |  0.1527335|4.75948745615231   |4.44648623904895     | 1.5440680|  -18.391164|  0.8698226|      0.1784806|null           |FALSE      |
|GO:0010310 |regulation of hydrogen peroxide metabolic process                                            |  0.0539059|1.51959379754424   |4.13981204618127     | 1.1139434|   -3.310012|  0.9066640|      0.2101184|null           |FALSE      |
|GO:0060294 |cilium movement involved in cell motility                                                    |  0.0059895|-4.50952176778443  |-2.45323069254133    | 0.4771213|  -13.305352|  0.9156667|      0.2109973|null           |FALSE      |
|GO:2000083 |negative regulation of L-ascorbic acid biosynthetic process                                  |  0.0044922|2.26736760651647   |6.324651964588       | 0.3010300|  -16.999532|  0.8934076|      0.2131819|null           |FALSE      |
|GO:0018342 |protein prenylation                                                                          |  0.0494138|-5.66041388268604  |2.4384663879156      | 1.0791812|   -3.422514|  0.9357659|      0.2200164|null           |FALSE      |
|GO:0009819 |drought recovery                                                                             |  0.0494138|7.22517736760047   |-1.63739167382403    | 1.0791812|  -28.692016|  0.8663769|      0.2221654|null           |FALSE      |
|GO:0097167 |circadian regulation of translation                                                          |  0.0044922|-0.230621984239474 |-1.58902587153404    | 0.3010300|   -4.377239|  0.8423061|      0.2266615|null           |FALSE      |
|GO:0009410 |response to xenobiotic stimulus                                                              |  0.0359373|7.86920798884618   |-0.745290396418097   | 0.9542425|   -6.200545|  0.9022146|      0.2284321|null           |FALSE      |
|GO:0036371 |protein localization to T-tubule                                                             |  0.0314451|-5.47173236192814  |-3.2019412959009     | 0.9030900|  -18.160410|  0.9199490|      0.2340388|null           |FALSE      |
|GO:0071466 |cellular response to xenobiotic stimulus                                                     |  0.0224608|6.8491677263583    |-0.00650460744941887 | 0.7781513|   -8.870023|  0.8697970|      0.2373516|null           |FALSE      |
|GO:0009865 |pollen tube adhesion                                                                         |  0.0134765|0.21145504516023   |-4.34002730430884    | 0.6020600|   -6.744444|  0.8559881|      0.2428352|null           |FALSE      |
|GO:0010025 |wax biosynthetic process                                                                     |  0.0988275|-2.00002299579681  |6.50244576321831     | 1.3617278|   -3.394830|  0.9470983|      0.2514105|null           |FALSE      |
|GO:0010233 |phloem transport                                                                             |  0.0943354|-3.31765702310291  |-4.00170204408472    | 1.3424227|   -5.714913|  0.8357407|      0.2779671|null           |FALSE      |
|GO:0045168 |cell-cell signaling involved in cell fate commitment                                         |  0.1662100|0.534827078918881  |-4.6084203880314     | 1.5797836|  -23.213581|  0.8323801|      0.2805344|null           |FALSE      |
|GO:0009611 |response to wounding                                                                         |  1.0062441|6.24961515669884   |-2.94107893694659    | 2.3521825|   -4.682342|  0.8957601|      0.2870502|null           |FALSE      |
|GO:0010375 |stomatal complex patterning                                                                  |  0.0494138|-2.17003154932499  |-6.00489822607152    | 1.0791812|  -20.044409|  0.8927858|      0.2884235|null           |FALSE      |
|GO:0019756 |cyanogenic glycoside biosynthetic process                                                    |  0.1069562|-4.06609889607895  |4.37468848733656     | 1.4313638|  -30.624232|  0.8870206|      0.2895405|null           |FALSE      |
|GO:0008053 |mitochondrial fusion                                                                         |  0.0778641|1.8245985013249    |-7.74187623135665    | 1.2552725|   -1.911618|  0.9414539|      0.2993263|null           |FALSE      |
|GO:0001101 |response to acid chemical                                                                    |  1.7923723|6.82888972882824   |-1.55679308105716    | 2.6020600|   -7.876771|  0.8652911|      0.3051929|null           |FALSE      |
|GO:0033387 |putrescine biosynthetic process from ornithine                                               |  0.0123534|-3.78341188754192  |4.62095399062327     | 0.4771213|  -11.194175|  0.9072157|      0.3196216|null           |FALSE      |
|GO:0031930 |mitochondria-nucleus signaling pathway                                                       |  0.0539059|4.59852621243135   |1.10111666644446     | 1.1139434|   -4.333619|  0.8075250|      0.3228612|null           |FALSE      |
|GO:0010114 |response to red light                                                                        |  0.2785140|7.28322584238529   |-3.53118513756851    | 1.7993405|   -5.748793|  0.9158559|      0.3255957|null           |FALSE      |
|GO:0016567 |protein ubiquitination                                                                       |  3.7105251|-5.30621452859928  |2.80348017389446     | 2.9175055|  -24.814192|  0.9086591|      0.3270943|null           |FALSE      |
|GO:0032504 |multicellular organism reproduction                                                          |  0.8040969|-2.08854027054373  |-4.97604615534729    | 2.2552725|   -1.631797|  0.8926102|      0.3415835|null           |FALSE      |
|GO:0048211 |Golgi vesicle docking                                                                        |  0.0089843|-5.39734981335701  |-3.42745908292038    | 0.4771213|  -10.411023|  0.9281242|      0.3424968|null           |FALSE      |
|GO:1903553 |positive regulation of extracellular exosome assembly                                        |  0.0044922|6.05542744123121   |4.63670044425773     | 0.3010300|   -1.911602|  0.9019680|      0.3498300|null           |FALSE      |
|GO:0031542 |positive regulation of anthocyanin biosynthetic process                                      |  0.0179686|5.20687714094448   |4.81821386748668     | 0.6989700|   -1.676296|  0.8948536|      0.3513679|null           |FALSE      |
|GO:0016045 |detection of bacterium                                                                       |  0.0134765|5.53554438974607   |-4.96538758908962    | 0.6020600|   -6.852833|  0.8894975|      0.3569672|null           |FALSE      |
|GO:0044364 |disruption of cells of other organism                                                        |  0.0269530|5.17263369693389   |-6.23628106075078    | 0.8450980|   -1.911602|  0.9481181|      0.3726472|null           |FALSE      |
|GO:0060305 |regulation of cell diameter                                                                  |  0.0044922|-1.48281863826069  |-0.560562567262344   | 0.3010300|   -3.355775|  0.8571371|      0.3779708|null           |FALSE      |
|GO:0009867 |jasmonic acid mediated signaling pathway                                                     |  0.3998023|5.12908425755338   |0.499811914769806    | 1.9542425|  -25.814933|  0.7247355|      0.3795777|null           |FALSE      |
|GO:0000966 |RNA 5&#39;-end processing                                                                        |  0.0808589|-4.74225019799345  |4.23273145718116     | 1.2787536|  -49.527264|  0.9222748|      0.3797277|null           |FALSE      |
|GO:0045056 |transcytosis                                                                                 |  0.0044922|-3.55009408718876  |-4.1808383081505     | 0.3010300|  -10.411023|  0.8766246|      0.3809345|null           |FALSE      |
|GO:0010555 |response to mannitol                                                                         |  0.0673824|7.3785877158269    |-0.52514192657132    | 1.2041200|   -3.074696|  0.8785807|      0.3887576|null           |FALSE      |
|GO:0090559 |regulation of membrane permeability                                                          |  0.0089843|-1.60999290982953  |1.2138003947998      | 0.4771213|   -4.867058|  0.8937880|      0.3950913|null           |FALSE      |
|GO:0010618 |aerenchyma formation                                                                         |  0.0089843|-1.69181706493828  |-6.98055600677105    | 0.4771213|  -18.139990|  0.9176805|      0.3960626|null           |FALSE      |
|GO:0080156 |mitochondrial mRNA modification                                                              |  0.1347648|-4.99801453740328  |3.54749430215652     | 1.4913617|  -40.112965|  0.9038002|      0.3977989|null           |FALSE      |
|GO:0010438 |cellular response to sulfur starvation                                                       |  0.0314451|4.99622603553296   |-2.48943934806999    | 0.9030900|   -2.224533|  0.8478891|      0.3994265|null           |FALSE      |
|GO:0007267 |cell-cell signaling                                                                          |  0.3414042|7.08661928371544   |2.79236781376193     | 1.8864907|   -9.465565|  0.9203509|      0.4013598|null           |FALSE      |
|GO:0080027 |response to herbivore                                                                        |  0.0808589|5.64913192141135   |-4.4634431798544     | 1.2787536|   -8.477227|  0.8880317|      0.4067789|null           |FALSE      |
|GO:0034394 |protein localization to cell surface                                                         |  0.0134765|-5.62533899949281  |-3.30112473723937    | 0.6020600|   -6.337011|  0.9270314|      0.4074874|null           |FALSE      |
|GO:0006952 |defense response                                                                             |  7.2503481|5.98818705739724   |-2.46546068940657    | 3.2081725|  -92.003340|  0.8719910|      0.4100881|null           |FALSE      |
|GO:0010221 |negative regulation of vernalization response                                                |  0.0044922|3.25497664357868   |6.53272036074585     | 0.3010300|  -13.253502|  0.8819520|      0.4116470|null           |FALSE      |
|GO:0009609 |response to symbiotic bacterium                                                              |  0.0089843|5.45593900354879   |-5.11675691265225    | 0.4771213|   -4.928401|  0.8990011|      0.4143200|null           |FALSE      |
|GO:0009871 |jasmonic acid and ethylene-dependent systemic resistance,ethylene mediated signaling pathway |  0.0089843|5.18674976783802   |-1.10904446751428    | 0.4771213|   -9.345083|  0.7613423|      0.4175401|null           |FALSE      |
|GO:0035864 |response to potassium ion                                                                    |  0.0269530|7.83967755208712   |-0.386904837364733   | 0.8450980|   -7.406889|  0.8977954|      0.4217509|null           |FALSE      |
|GO:0007166 |cell surface receptor signaling pathway                                                      |  0.5794888|4.30826985958598   |0.844292495099563    | 2.1139434|   -9.718756|  0.7705725|      0.4253892|null           |FALSE      |
|GO:0009625 |response to insect                                                                           |  0.1482413|5.75708557392155   |-4.25067464373727    | 1.5314789|  -18.198404|  0.8827903|      0.4269324|null           |FALSE      |
|GO:1905034 |regulation of antifungal innate immune response                                              |  0.0089843|4.50207792718059   |6.20915021295885     | 0.4771213|  -13.646261|  0.8893037|      0.4305344|null           |FALSE      |
|GO:1904526 |regulation of microtubule binding                                                            |  0.0044922|0.0232742946886805 |7.01020548067142     | 0.3010300|   -7.008658|  0.9140743|      0.4321507|null           |FALSE      |
|GO:1905255 |regulation of RNA binding transcription factor activity                                      |  0.0044922|0.157229428325751  |7.10404214991097     | 0.3010300|   -7.822215|  0.9140743|      0.4321507|null           |FALSE      |
|GO:2000117 |negative regulation of cysteine-type endopeptidase activity                                  |  0.0044922|1.81055005416492   |6.13073999979249     | 0.3010300|  -13.861990|  0.8713185|      0.4321507|null           |FALSE      |
|GO:0003283 |atrial septum development                                                                    |  1.6401201|-1.34102184990592  |-5.55850469601831    | 3.2564772|   -4.737842|  0.8398372|      0.4359177|null           |FALSE      |
|GO:1901021 |positive regulation of calcium ion transmembrane transporter activity                        |  0.0065885|5.03741744643543   |5.08418018082881     | 0.3010300|  -18.160410|  0.8497678|      0.4428862|null           |FALSE      |
|GO:0000304 |response to singlet oxygen                                                                   |  0.0583981|7.2486177095831    |-1.33801071449535    | 1.1461280|  -14.098692|  0.8696953|      0.4434063|null           |FALSE      |
|GO:0097185 |cellular response to azide                                                                   |  0.0561520|6.90151641294608   |-0.435738925510425   | 1.1139434|  -12.341169|  0.8528112|      0.4472027|null           |FALSE      |
|GO:0002011 |morphogenesis of an epithelial sheet                                                         |  0.0134765|-1.26214745659859  |-6.99990062091443    | 0.6020600|   -9.046140|  0.9128097|      0.4499787|null           |FALSE      |
|GO:0007492 |endoderm development                                                                         |  0.0134765|-0.936594635690295 |-7.06198971913429    | 0.6020600|   -1.911602|  0.9223529|      0.4499787|null           |FALSE      |
|GO:0002213 |defense response to insect                                                                   |  0.0943354|5.50050339097931   |-4.03046578729106    | 1.3424227|   -4.652790|  0.8765845|      0.4532505|null           |FALSE      |
|GO:0071218 |cellular response to misfolded protein                                                       |  0.1033197|6.01473363092483   |-1.22391220786385    | 1.3802112|   -2.846974|  0.8331602|      0.4588220|null           |FALSE      |
|GO:0019722 |calcium-mediated signaling                                                                   |  0.3279278|4.27935470515041   |0.705252385741705    | 1.8692317|   -2.599013|  0.7748853|      0.4636835|null           |FALSE      |
|GO:0006654 |phosphatidic acid biosynthetic process                                                       |  0.0269530|-2.76499470453125  |6.01304443528722     | 0.8450980|  -10.365896|  0.8965692|      0.4645185|null           |FALSE      |
|GO:0046394 |carboxylic acid biosynthetic process                                                         |  2.1742060|-2.99653893662866  |5.20107021751928     | 2.6857417|   -2.668466|  0.8867768|      0.4686518|null           |FALSE      |
|GO:0071236 |cellular response to antibiotic                                                              |  0.8690604|6.33405887862492   |-1.11421222730063    | 2.2878017|  -19.250722|  0.8278819|      0.4715434|null           |FALSE      |
|GO:0002084 |protein depalmitoylation                                                                     |  0.0134765|-5.06060472628882  |2.21645785465974     | 0.6020600|   -5.504809|  0.9248904|      0.4823511|null           |FALSE      |
|GO:0050821 |protein stabilization                                                                        |  0.1437492|-0.578086355658073 |1.78194038920402     | 1.5185139|   -1.499445|  0.8732697|      0.4825149|null           |FALSE      |
|GO:0051607 |defense response to virus                                                                    |  0.2246081|5.62191440618752   |-3.79512741770091    | 1.7075702|   -3.609544|  0.8680501|      0.4865194|null           |FALSE      |
|GO:0033292 |T-tubule organization                                                                        |  0.0716381|-0.152159494759891 |-6.50783016852228    | 1.2041200|  -18.160410|  0.8520286|      0.4928611|null           |FALSE      |
|GO:0036309 |protein localization to M-band                                                               |  0.2713265|-5.30014152338077  |-3.00598530846985    | 1.7853298|  -18.160410|  0.9097144|      0.4950361|null           |FALSE      |
|GO:0036159 |inner dynein arm assembly                                                                    |  0.0179686|2.22264856285414   |-7.45246857732446    | 0.6989700|   -8.286640|  0.9245271|      0.4957761|null           |FALSE      |
|GO:0032527 |protein exit from endoplasmic reticulum                                                      |  0.0359373|-5.50241095008743  |-3.01064001905342    | 0.9542425|  -15.058593|  0.9201985|      0.4998121|null           |FALSE      |
|GO:0006649 |phospholipid transfer to membrane                                                            |  0.0044922|-4.606427295442    |-3.91596268482205    | 0.3010300|   -3.066668|  0.9117770|      0.5057439|null           |FALSE      |
|GO:0097237 |cellular response to toxic substance                                                         |  0.8355420|6.27275871814218   |-1.02861146243556    | 2.2718416|   -2.921033|  0.8284769|      0.5080142|null           |FALSE      |
|GO:0009451 |RNA modification                                                                             |  1.4105386|-4.79235958757143  |3.44761237456938     | 2.4983106|  -31.353508|  0.8982904|      0.5091922|null           |FALSE      |
|GO:0052544 |defense response by callose deposition in cell wall                                          |  0.0718746|2.44418679468191   |-2.72418970380921    | 1.2304489|  -12.781323|  0.7778892|      0.5168809|null           |FALSE      |
|GO:0042177 |negative regulation of protein catabolic process                                             |  0.0134765|1.9714369818875    |6.30642543336602     | 0.6020600|   -6.581183|  0.8919636|      0.5178942|null           |FALSE      |
|GO:0060776 |simple leaf morphogenesis                                                                    |  0.0224608|-1.79208693300584  |-6.16177903743101    | 0.7781513|   -6.747710|  0.8771229|      0.5240669|null           |FALSE      |
|GO:0009733 |response to auxin                                                                            |  1.8597547|6.78919215880805   |-1.2069336204352     | 2.6180481|   -4.959063|  0.8462783|      0.5265885|null           |FALSE      |
|GO:0035024 |negative regulation of Rho protein signal transduction                                       |  0.0044922|3.3970257370564    |6.20061969122911     | 0.3010300|   -9.928858|  0.8655127|      0.5272151|null           |FALSE      |
|GO:0035600 |tRNA methylthiolation                                                                        |  0.0089843|-5.14926206456979  |3.70692514795869     | 0.4771213|   -2.034953|  0.9230223|      0.5275488|null           |FALSE      |
|GO:0009914 |hormone transport                                                                            |  0.4447240|-2.08159047250868  |-0.0808543301705264  | 2.0000000|  -16.231674|  0.8095167|      0.5303141|null           |FALSE      |
|GO:0009960 |endosperm development                                                                        |  0.1437492|-1.21523516078419  |-6.06158190023576    | 1.5185139|   -5.605621|  0.8662183|      0.5317308|null           |FALSE      |
|GO:2000280 |regulation of root development                                                               |  0.3593729|2.16733339555634   |3.40030660806032     | 1.9084850|  -76.487902|  0.8856727|      0.5325899|null           |FALSE      |
|GO:0071368 |cellular response to cytokinin stimulus                                                      |  0.3593729|6.38545211914592   |-0.755772405552649   | 1.9084850|  -13.837441|  0.8232207|      0.5432496|null           |FALSE      |
|GO:0009617 |response to bacterium                                                                        |  2.1113158|5.81970803892476   |-3.49657190179512    | 2.6730209|  -21.588290|  0.8525457|      0.5453319|null           |FALSE      |
|GO:1901684 |arsenate ion transmembrane transport                                                         |  0.0224608|-5.46178216099305  |-2.02125744710787    | 0.7781513|  -10.319721|  0.9233873|      0.5470253|null           |FALSE      |
|GO:1902290 |positive regulation of defense response to oomycetes                                         |  0.0449216|4.71286104533436   |5.28408516880409     | 1.0413927|  -10.069439|  0.8615933|      0.5510349|null           |FALSE      |
|GO:0060465 |pharynx development                                                                          |  1.4053739|-1.23478981619111  |-5.72791067638091    | 2.4927604|   -1.911602|  0.8561079|      0.5514316|null           |FALSE      |
|GO:0048312 |intracellular distribution of mitochondria                                                   |  0.0044922|-4.49952961711597  |-3.55535368176737    | 0.3010300|   -1.911602|  0.9084221|      0.5580199|null           |FALSE      |
|GO:0006468 |protein phosphorylation                                                                      |  5.0671578|-5.01137688556231  |3.06771792464711     | 3.0526939|  -45.943129|  0.8941858|      0.5616227|null           |FALSE      |
|GO:0071447 |cellular response to hydroperoxide                                                           |  0.1529288|6.20312520369242   |-1.34308939777294    | 1.5440680|   -5.949636|  0.8203493|      0.5631622|null           |FALSE      |
|GO:0048509 |regulation of meristem development                                                           |  0.2964826|2.01731528682391   |3.32254618974147     | 1.8260748|  -33.358186|  0.8872923|      0.5634440|null           |FALSE      |
|GO:1902265 |abscisic acid homeostasis                                                                    |  0.0089843|-1.61119517347583  |1.010713717388       | 0.4771213|   -4.605998|  0.8896166|      0.5661740|null           |FALSE      |
|GO:0010930 |negative regulation of auxin mediated signaling pathway                                      |  0.0134765|3.38982660363781   |5.90718476867603     | 0.6020600|   -6.338787|  0.8582675|      0.5666125|null           |FALSE      |
|GO:0009826 |unidimensional cell growth                                                                   |  1.0736265|-1.31960192793519  |-6.6764330104046     | 2.3802112|   -4.149882|  0.8638742|      0.5680023|null           |FALSE      |
|GO:0007498 |mesoderm development                                                                         |  0.3503886|-1.02997947434765  |-6.75309538379422    | 1.8976271|   -1.651274|  0.9022502|      0.5707614|null           |FALSE      |
|GO:0072498 |embryonic skeletal joint development                                                         |  1.8732312|-1.39459700088761  |-5.68234920251791    | 2.6211763|   -1.911602|  0.8522091|      0.5710239|null           |FALSE      |
|GO:0009644 |response to high light intensity                                                             |  0.2785140|7.36333606198217   |-3.42364781314123    | 1.7993405|   -2.721079|  0.9158559|      0.5759002|null           |FALSE      |
|GO:0009961 |response to 1-aminocyclopropane-1-carboxylic acid                                            |  0.0179686|7.53387057573971   |-0.0470372697315185  | 0.6989700|   -9.695553|  0.8846771|      0.5766178|null           |FALSE      |
|GO:0080003 |thalianol metabolic process                                                                  |  0.0089843|-3.31849884248374  |5.84154852841977     | 0.4771213|  -26.491320|  0.8906941|      0.5772455|null           |FALSE      |
|GO:0048512 |circadian behavior                                                                           |  0.0224608|-6.6984106332665   |-0.253599367427567   | 0.7781513|   -1.869036|  0.9951277|      0.5780757|null           |FALSE      |
|GO:0098660 |inorganic ion transmembrane transport                                                        |  1.9810431|-5.32546090153174  |-2.47508557418628    | 2.6454223|  -11.268271|  0.8988203|      0.5817700|null           |FALSE      |
|GO:0007584 |response to nutrient                                                                         |  0.0583981|6.56864828120139   |-2.42067343961015    | 1.1461280|   -1.943252|  0.8709867|      0.5817840|null           |FALSE      |
|GO:0009852 |auxin catabolic process                                                                      |  0.0089843|-2.09444832853631  |1.53258847440362     | 0.4771213|  -10.033313|  0.8440032|      0.5840401|null           |FALSE      |
|GO:0071229 |cellular response to acid chemical                                                           |  0.2740218|6.61524210303362   |-0.769615236749597   | 1.7923917|  -21.633441|  0.8389718|      0.5845806|null           |FALSE      |
|GO:0010376 |stomatal complex formation                                                                   |  0.0224608|-1.52427102182742  |-6.29924327060846    | 0.7781513|   -6.020245|  0.8675775|      0.5847861|null           |FALSE      |
|GO:1905663 |positive regulation of telomerase RNA reverse transcriptase activity                         |  0.0485153|1.34570662358855   |5.44317764219498     | 1.0413927|   -1.471697|  0.8770473|      0.5862896|null           |FALSE      |
|GO:0009745 |sucrose mediated signaling                                                                   |  0.0089843|5.98493864377556   |0.610047096247365    | 0.4771213|   -2.254390|  0.7900171|      0.5951780|null           |FALSE      |
|GO:0009664 |plant-type cell wall organization                                                            |  0.6693320|1.88775427925691   |-7.40091231641397    | 2.1760913|   -8.936784|  0.9262356|      0.5956427|null           |FALSE      |
|GO:0003341 |cilium movement                                                                              |  0.0059895|3.29673871803096   |-7.80883120919463    | 0.3010300|  -14.626165|  0.9526986|      0.5985002|null           |FALSE      |
|GO:0080141 |regulation of jasmonic acid biosynthetic process                                             |  0.0269530|1.33657162911936   |4.85359962931766     | 0.8450980|   -5.405514|  0.9057182|      0.6034067|null           |FALSE      |
|GO:0035725 |sodium ion transmembrane transport                                                           |  0.0673824|-5.68750920759221  |-2.38187387247182    | 1.2041200|   -3.931946|  0.9197284|      0.6086318|null           |FALSE      |
|GO:0009734 |auxin-activated signaling pathway                                                            |  0.9972598|4.93481819317296   |0.564565786416898    | 2.3483049|  -16.624630|  0.7168559|      0.6116609|null           |FALSE      |
|GO:0010150 |leaf senescence                                                                              |  0.5076142|-1.65231456410754  |-5.86247393440201    | 2.0569049|   -3.522762|  0.8602333|      0.6130285|null           |FALSE      |
|GO:0046345 |abscisic acid catabolic process                                                              |  0.0269530|-2.96543327492442  |5.37576735533737     | 0.8450980|  -46.799108|  0.8550913|      0.6194937|null           |FALSE      |
|GO:1901599 |(-)-pinoresinol biosynthetic process                                                         |  0.0089843|-3.6887320513004   |4.9371874795957      | 0.4771213|   -3.071305|  0.8966972|      0.6243888|null           |FALSE      |
|GO:0045926 |negative regulation of growth                                                                |  0.1751943|2.53093512700136   |5.39597908541204     | 1.6020600|  -13.458879|  0.8791890|      0.6297772|null           |FALSE      |
|GO:1900865 |chloroplast RNA modification                                                                 |  0.1212884|-4.93976066007596  |3.61981970020766     | 1.4471580|   -1.937036|  0.9101989|      0.6305983|null           |FALSE      |
|GO:1905614 |negative regulation of developmental vegetative growth                                       |  0.0134765|2.45339570489287   |5.8507474859347      | 0.6020600|   -1.756960|  0.8834762|      0.6315937|null           |FALSE      |
|GO:0048438 |floral whorl development                                                                     |  0.8085890|-1.52873945294347  |-5.76655965348633    | 2.2576786|   -2.152846|  0.8543795|      0.6324877|null           |FALSE      |
|GO:0031640 |killing of cells of other organism                                                           |  1.3072189|5.27611776252277   |-6.06554257822258    | 2.4653829|   -1.911602|  0.9026632|      0.6338299|null           |FALSE      |
|GO:0007422 |peripheral nervous system development                                                        |  3.6745878|-1.32475215326011  |-5.81760520170601    | 2.9132839|   -1.651274|  0.8337928|      0.6342863|null           |FALSE      |
|GO:0007165 |signal transduction                                                                          |  8.3913571|4.31596836439588   |0.919233831423454    | 3.2716093|  -87.009031|  0.7076707|      0.6344952|null           |FALSE      |
|GO:0048821 |erythrocyte development                                                                      |  0.0179686|-0.826954660379493 |-3.54100794791879    | 0.6989700|   -1.679676|  0.7400395|      0.6349558|null           |FALSE      |
|GO:0015700 |arsenite transport                                                                           |  0.0224608|-5.62958124058799  |-2.05897324481178    | 0.7781513|  -12.058546|  0.9404722|      0.6364426|null           |FALSE      |
|GO:0048658 |anther wall tapetum development                                                              |  0.0853511|-1.75412025673136  |-5.72689587941107    | 1.3010300|   -1.852856|  0.8749979|      0.6449005|null           |FALSE      |
|GO:0010208 |pollen wall assembly                                                                         |  0.2021473|-0.48271007749585  |-6.02075132139762    | 1.6627578|   -2.259532|  0.7957302|      0.6467488|null           |FALSE      |
|GO:0015708 |silicic acid import across plasma membrane                                                   |  0.0331729|-5.64089699952159  |-2.18690801577896    | 0.9030900|   -3.071305|  0.9191437|      0.6528448|null           |FALSE      |
|GO:0048235 |pollen sperm cell differentiation                                                            |  0.1572256|-0.922456576073329 |-5.68091502961974    | 1.5563025|   -1.640496|  0.8451238|      0.6530142|null           |FALSE      |
|GO:0051513 |regulation of monopolar cell growth                                                          |  0.0224608|2.94534387071754   |3.74928256912001     | 0.7781513|   -2.081591|  0.8907853|      0.6532683|null           |FALSE      |
|GO:0042742 |defense response to bacterium                                                                |  1.6216702|5.732720893716     |-3.34543413269915    | 2.5587086|   -9.971460|  0.8382382|      0.6545694|null           |FALSE      |
|GO:0012502 |induction of programmed cell death                                                           |  0.0044922|5.93503397840952   |4.87628507661586     | 0.3010300|   -7.493662|  0.8941082|      0.6549617|null           |FALSE      |
|GO:1902042 |negative regulation of extrinsic apoptotic signaling pathway via death domain receptors      |  0.0044922|3.78005833844715   |5.96588673680689     | 0.3010300|   -5.949636|  0.8567293|      0.6549617|null           |FALSE      |
|GO:2000026 |regulation of multicellular organismal development                                           |  1.5677643|2.36008373443357   |3.56259700675508     | 2.5440680|  -25.760968|  0.8333533|      0.6593517|null           |FALSE      |
|GO:0043086 |negative regulation of catalytic activity                                                    |  1.1410089|0.870510548017941  |5.10988336764104     | 2.4065402|   -2.990442|  0.8755621|      0.6653174|null           |FALSE      |
|GO:0009737 |response to abscisic acid                                                                    |  2.3538925|6.70421956462235   |-1.27988534013835    | 2.7201593|   -4.235759|  0.8304505|      0.6656796|null           |FALSE      |
|GO:0009740 |gibberellic acid mediated signaling pathway                                                  |  0.3503886|5.20191651832519   |0.524823671681727    | 1.8976271|   -4.867794|  0.7296730|      0.6682909|null           |FALSE      |
|GO:0009742 |brassinosteroid mediated signaling pathway                                                   |  0.3638651|5.2684246227962    |0.568493411292098    | 1.9138139|   -3.724605|  0.7228966|      0.6705553|null           |FALSE      |
|GO:0009866 |induced systemic resistance,ethylene mediated signaling pathway                              |  0.0179686|5.19828151022013   |0.273330378254551    | 0.6989700|   -3.234087|  0.6902924|      0.6707259|null           |FALSE      |
|GO:0017148 |negative regulation of translation                                                           |  0.3503886|2.49919639670197   |4.96381414251268     | 1.8976271|   -4.095920|  0.8557567|      0.6727984|null           |FALSE      |
|GO:0045338 |farnesyl diphosphate metabolic process                                                       |  0.0898432|-2.58074822132634  |6.5054238690941      | 1.3222193|  -43.241889|  0.8844653|      0.6735153|null           |FALSE      |
|GO:0061025 |membrane fusion                                                                              |  0.3324199|1.58798407360409   |-7.67392243553499    | 1.8750613|   -3.333018|  0.9348984|      0.6738613|null           |FALSE      |
|GO:0009684 |indoleacetic acid biosynthetic process                                                       |  0.0494138|-2.53185542210302  |3.33168087539631     | 1.0791812|   -8.410423|  0.7768718|      0.6856031|null           |FALSE      |
|GO:1900449 |regulation of glutamate receptor signaling pathway                                           |  0.1735903|3.78956709080436   |5.00404470890982     | 1.5682017|   -5.949636|  0.8558237|      0.6858401|null           |FALSE      |
|GO:0048265 |response to pain                                                                             | 11.9626252|3.20378416500377   |-3.26102746950985    | 3.4255342|   -1.911602|  0.7401901|      0.6889007|null           |FALSE      |
|GO:0010030 |positive regulation of seed germination                                                      |  0.0898432|4.28645963387319   |4.16228985419916     | 1.3222193|  -15.745440|  0.8431154|      0.6895163|null           |FALSE      |
|GO:0071805 |potassium ion transmembrane transport                                                        |  0.2560532|-5.47423316346405  |-2.41164406848132    | 1.7634280|   -2.235091|  0.9120567|      0.6907954|null           |FALSE      |

[1] &quot;missing_zmar_vs_all_GO.txt&quot;


|TermID     |Name                                                                                             |  Frequency|PlotX              |PlotY                |   LogSize|       Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:------------------------------------------------------------------------------------------------|----------:|:------------------|:--------------------|---------:|-----------:|----------:|--------------:|:--------------|:----------|
|GO:0010082 |regulation of root meristem growth                                                               |  0.1527335|1.85403708473292   |2.85635617663625     | 1.5440680|  -65.058597|  0.7962497|      0.0000000|null           |FALSE      |
|GO:0048512 |circadian behavior                                                                               |  0.0224608|2.71353685649827   |1.85282357937546     | 0.7781513|   -8.585635|  0.9936051|      0.0000000|null           |FALSE      |
|GO:0051762 |sesquiterpene biosynthetic process                                                               |  0.1123040|5.52214287821191   |-4.07768076522801    | 1.4149733| -178.473129|  0.8478180|      0.0000000|null           |FALSE      |
|GO:1902025 |nitrate import                                                                                   |  0.0808589|2.37211171983073   |6.08336278690752     | 1.2787536|  -25.565208|  0.9412517|      0.0000000|null           |FALSE      |
|GO:0072089 |stem cell proliferation                                                                          |  0.0044922|7.35594187744464   |-0.920244759077816   | 0.3010300|   -2.235250|  0.9793616|      0.0198115|null           |FALSE      |
|GO:0003341 |cilium movement                                                                                  |  0.0059895|6.82902480635737   |-0.0373438669500855  | 0.3010300|  -12.938618|  0.9561658|      0.0201566|null           |FALSE      |
|GO:0090136 |epithelial cell-cell adhesion                                                                    |  0.0134765|7.45381212068478   |0.384679866624264    | 0.6020600|  -15.423612|  0.9727170|      0.0211975|null           |FALSE      |
|GO:0045218 |zonula adherens maintenance                                                                      |  0.0494138|5.31362195201209   |-0.559971782614379   | 1.0791812|  -15.423612|  0.9564499|      0.0231095|null           |FALSE      |
|GO:0045199 |maintenance of epithelial cell apical/basal polarity                                             |  0.0853511|7.35330404506341   |-1.38646273911676    | 1.3010300|   -3.134638|  0.9750764|      0.0240209|null           |FALSE      |
|GO:0008356 |asymmetric cell division                                                                         |  0.0988275|-3.91368972639839  |-7.92457426698016    | 1.3617278|   -1.634371|  0.9748152|      0.0242778|null           |FALSE      |
|GO:0009626 |plant-type hypersensitive response                                                               |  0.3279278|-5.89524236320756  |-1.34672726581891    | 1.8692317|  -90.965700|  0.8166673|      0.0266051|null           |FALSE      |
|GO:0097502 |mannosylation                                                                                    |  0.1796864|7.45074543017952   |-0.380316058147324   | 1.6127839|  -10.991734|  0.9627293|      0.0640480|null           |FALSE      |
|GO:0009780 |photosynthetic NADP+ reduction                                                                   |  0.0044922|-3.04565921363003  |-8.23834275350206    | 0.3010300|   -9.224979|  0.9471469|      0.0658471|null           |FALSE      |
|GO:0006739 |NADP metabolic process                                                                           |  0.1707021|-4.32357708893095  |-7.74925515677811    | 1.5910646|   -1.889197|  0.9628679|      0.0661605|null           |FALSE      |
|GO:0051554 |flavonol metabolic process                                                                       |  0.0853511|6.98924292401156   |-2.319052082908      | 1.3010300|  -29.467082|  0.9308590|      0.0732686|null           |FALSE      |
|GO:0000023 |maltose metabolic process                                                                        |  0.0179686|-1.75198861808003  |-8.16539947305047    | 0.6989700|   -9.397924|  0.9268904|      0.0748216|null           |FALSE      |
|GO:0031146 |SCF-dependent proteasomal ubiquitin-dependent protein catabolic process                          |  0.3773415|1.20403039733698   |-7.18864152230653    | 1.9294189|  -58.398957|  0.8769966|      0.0932346|null           |FALSE      |
|GO:1904526 |regulation of microtubule binding                                                                |  0.0044922|-6.91667895386103  |3.92654710831226     | 0.3010300|  -44.692162|  0.9297507|      0.1028503|null           |FALSE      |
|GO:1900000 |regulation of anthocyanin catabolic process                                                      |  0.0044922|-5.73157542639681  |4.74763773473703     | 0.3010300|  -58.235448|  0.9270884|      0.1174435|null           |FALSE      |
|GO:0080184 |response to phenylpropanoid                                                                      |  0.0044922|-6.34839352740113  |-4.85606202062558    | 0.3010300|  -12.023888|  0.9132274|      0.1196905|null           |FALSE      |
|GO:0006629 |lipid metabolic process                                                                          |  4.9368851|6.6681243497042    |-2.97736317350312    | 3.0413927|   -5.178395|  0.9357432|      0.1343063|null           |FALSE      |
|GO:0000966 |RNA 5&#39;-end processing                                                                            |  0.0808589|0.854823528289534  |-4.97881447650478    | 1.2787536|  -43.193677|  0.8845923|      0.1347712|null           |FALSE      |
|GO:0010942 |positive regulation of cell death                                                                |  0.1527335|-3.43242255228072  |4.03506332979585     | 1.5440680|   -9.756210|  0.8819069|      0.1493786|null           |FALSE      |
|GO:0042550 |photosystem I stabilization                                                                      |  0.0134765|-5.29752412840758  |4.60294625338595     | 0.6020600|   -1.664732|  0.9251158|      0.1577237|null           |FALSE      |
|GO:0010119 |regulation of stomatal movement                                                                  |  0.4492161|-3.18241985759577  |3.69239517880949     | 2.0043214|   -3.608488|  0.9102617|      0.1842481|null           |FALSE      |
|GO:0035902 |response to immobilization stress                                                                |  0.0089843|-4.48700297652034  |-5.75253195402776    | 0.4771213|  -11.892224|  0.9345050|      0.1969794|null           |FALSE      |
|GO:0009962 |regulation of flavonoid biosynthetic process                                                     |  0.1257805|-4.39027163393011  |4.55947048139588     | 1.4623980|  -26.956820|  0.9162721|      0.1973047|null           |FALSE      |
|GO:2000083 |negative regulation of L-ascorbic acid biosynthetic process                                      |  0.0044922|-4.13217983402666  |5.68543289141352     | 0.3010300|  -15.208309|  0.9081422|      0.2131819|null           |FALSE      |
|GO:0018874 |benzoate metabolic process                                                                       |  0.0089843|3.30894411684607   |-5.65650820072734    | 0.4771213|  -24.132287|  0.8722929|      0.2133626|null           |FALSE      |
|GO:0010731 |protein glutathionylation                                                                        |  0.0359373|0.23179617232949   |-7.89575989060279    | 0.9542425|   -7.897618|  0.9169940|      0.2148306|null           |FALSE      |
|GO:0009819 |drought recovery                                                                                 |  0.0494138|-5.71774640130884  |-3.79357514367811    | 1.0791812|  -49.271654|  0.8781749|      0.2221654|null           |FALSE      |
|GO:0010881 |regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion |  0.0089843|-2.78313353751928  |0.79333505542387     | 0.4771213|  -31.134784|  0.7464492|      0.2228889|null           |FALSE      |
|GO:0048211 |Golgi vesicle docking                                                                            |  0.0089843|3.22454037043428   |5.58416946606005     | 0.4771213|   -7.280960|  0.9311962|      0.2267775|null           |FALSE      |
|GO:0009410 |response to xenobiotic stimulus                                                                  |  0.0359373|-6.90559224951564  |-3.63864651969862    | 0.9542425|   -4.474451|  0.9133429|      0.2284321|null           |FALSE      |
|GO:0015768 |maltose transport                                                                                |  0.0134765|1.82087524410025   |6.3480257219144      | 0.6020600|   -8.225758|  0.9603697|      0.2325128|null           |FALSE      |
|GO:0046900 |tetrahydrofolylpolyglutamate metabolic process                                                   |  0.0314451|0.35450786083172   |-5.47104499243001    | 0.9030900|  -23.148594|  0.8932936|      0.2369653|null           |FALSE      |
|GO:0032194 |ubiquinone biosynthetic process via 3,4-dihydroxy-5-polyprenylbenzoate                           |  0.0044922|4.81833758949831   |-5.50743257961458    | 0.3010300|   -6.158100|  0.8842323|      0.2451575|null           |FALSE      |
|GO:0019756 |cyanogenic glycoside biosynthetic process                                                        |  0.1069562|2.21964427846894   |-4.78653736186863    | 1.4313638|  -27.544023|  0.8500621|      0.2582345|null           |FALSE      |
|GO:0032508 |DNA duplex unwinding                                                                             |  0.2919905|4.89242053853836   |-0.392408169397354   | 1.8195439|   -8.740588|  0.9483469|      0.2718370|null           |FALSE      |
|GO:0010221 |negative regulation of vernalization response                                                    |  0.0044922|-2.09018196689136  |6.39663431116432     | 0.3010300|   -9.913795|  0.8950821|      0.2761077|null           |FALSE      |
|GO:0007613 |memory                                                                                           |  0.0943354|4.85369242542638   |3.14479715517276     | 1.3424227|  -13.659670|  0.8962772|      0.2779671|null           |FALSE      |
|GO:0045168 |cell-cell signaling involved in cell fate commitment                                             |  0.1662100|3.9160177914835    |1.63452856032901     | 1.5797836|  -25.077146|  0.8672699|      0.2860549|null           |FALSE      |
|GO:0009611 |response to wounding                                                                             |  1.0062441|-4.87083991473154  |-1.91767104451796    | 2.3521825|  -13.994126|  0.9070272|      0.2870502|null           |FALSE      |
|GO:0010375 |stomatal complex patterning                                                                      |  0.0494138|5.57392921569354   |2.8316655456304      | 1.0791812|   -9.239498|  0.9146556|      0.2884235|null           |FALSE      |
|GO:0009646 |response to absence of light                                                                     |  0.2021473|-5.20248694516425  |-5.29875393662278    | 1.6627578|   -9.882841|  0.9251993|      0.3180436|null           |FALSE      |
|GO:0071236 |cellular response to antibiotic                                                                  |  0.8690604|-5.35733593375126  |-2.7875374059421     | 2.2878017|  -28.340526|  0.8425737|      0.3202583|null           |FALSE      |
|GO:0090559 |regulation of membrane permeability                                                              |  0.0089843|0.531987793112153  |1.363155027571       | 0.4771213|   -3.990589|  0.9031348|      0.3244429|null           |FALSE      |
|GO:0034394 |protein localization to cell surface                                                             |  0.0134765|3.07509441187718   |5.4951088370683      | 0.6020600|   -4.662473|  0.9314074|      0.3265777|null           |FALSE      |
|GO:0016567 |protein ubiquitination                                                                           |  3.7105251|0.696936718636189  |-6.92920584028848    | 2.9175055|  -37.811969|  0.8805105|      0.3270943|null           |FALSE      |
|GO:1902075 |cellular response to salt                                                                        |  0.0044922|-4.89551722915344  |-4.10496055530993    | 0.3010300|   -8.793402|  0.8911291|      0.3281054|null           |FALSE      |
|GO:0018106 |peptidyl-histidine phosphorylation                                                               |  0.0763667|0.884367290673051  |-7.45967855195643    | 1.2552725|  -11.443885|  0.8974999|      0.3366283|null           |FALSE      |
|GO:1905255 |regulation of RNA binding transcription factor activity                                          |  0.0044922|-6.85383499832262  |4.04037201529893     | 0.3010300|   -6.738528|  0.9297507|      0.3410427|null           |FALSE      |
|GO:0051729 |germline cell cycle switching,mitotic to meiotic cell cycle                                      |  0.0044922|-3.55174126377588  |5.62653010183989     | 0.3010300|   -1.944420|  0.8775938|      0.3498300|null           |FALSE      |
|GO:0000720 |pyrimidine dimer repair by nucleotide-excision repair                                            |  0.0044922|-1.77041258231116  |-4.01823562748931    | 0.3010300|  -18.648794|  0.8228111|      0.3500169|null           |FALSE      |
|GO:0009649 |entrainment of circadian clock                                                                   |  0.0808589|-5.59827145433831  |0.951687270917336    | 1.2787536|   -8.786995|  0.8513633|      0.3565069|null           |FALSE      |
|GO:0006368 |transcription elongation from RNA polymerase II promoter                                         |  0.0763667|1.66012714903375   |-5.24921629505824    | 1.2552725|   -7.027962|  0.8525233|      0.3611562|null           |FALSE      |
|GO:0071475 |cellular hyperosmotic salinity response                                                          |  0.0089843|-4.7049168144457   |-3.32203058197198    | 0.4771213|  -25.518451|  0.8640520|      0.3661508|null           |FALSE      |
|GO:0071466 |cellular response to xenobiotic stimulus                                                         |  0.0224608|-4.97248864949219  |-3.61473766740921    | 0.7781513|  -24.704446|  0.8798301|      0.3682726|null           |FALSE      |
|GO:0007267 |cell-cell signaling                                                                              |  0.3414042|-6.76640968856736  |2.69951518732322     | 1.8864907|  -11.712089|  0.9377389|      0.3721964|null           |FALSE      |
|GO:0044364 |disruption of cells of other organism                                                            |  0.0269530|-7.80194606756014  |1.25324837033824     | 0.8450980|   -1.420387|  0.9565990|      0.3726472|null           |FALSE      |
|GO:1901527 |abscisic acid-activated signaling pathway involved in stomatal movement                          |  0.0314451|-4.51505034044825  |-0.7961745196869     | 0.9030900|  -27.719958|  0.7900026|      0.3779456|null           |FALSE      |
|GO:0090305 |nucleic acid phosphodiester bond hydrolysis                                                      |  1.6171780|1.2313872113748    |-5.36298728511882    | 2.5575072|  -20.943718|  0.8607349|      0.3797277|null           |FALSE      |
|GO:0045056 |transcytosis                                                                                     |  0.0044922|3.9062965623997    |4.22505283624622     | 0.3010300|   -7.280960|  0.8907465|      0.3809345|null           |FALSE      |
|GO:0009871 |jasmonic acid and ethylene-dependent systemic resistance,ethylene mediated signaling pathway     |  0.0089843|-5.55025902063648  |-0.587181797694016   | 0.4771213|  -33.260491|  0.7833883|      0.3823772|null           |FALSE      |
|GO:0009690 |cytokinin metabolic process                                                                      |  0.1751943|0.681706891430514  |-1.93195936935421    | 1.6020600|  -29.269152|  0.7745278|      0.3859662|null           |FALSE      |
|GO:0001680 |tRNA 3&#39;-terminal CCA addition                                                                    |  0.0224608|0.876167307334035  |-4.5289858772925     | 0.7781513|   -9.315195|  0.8895505|      0.3887837|null           |FALSE      |
|GO:0009608 |response to symbiont                                                                             |  0.0539059|-7.22009260532647  |-0.00374689127492698 | 1.1139434|   -4.377533|  0.9032337|      0.3943270|null           |FALSE      |
|GO:0010618 |aerenchyma formation                                                                             |  0.0089843|6.13175365977926   |1.83769766650144     | 0.4771213|  -20.516176|  0.9369195|      0.3960626|null           |FALSE      |
|GO:0002215 |defense response to nematode                                                                     |  0.0179686|-6.82246584673269  |-0.230772732931021   | 0.6989700|   -3.631838|  0.9005053|      0.4008543|null           |FALSE      |
|GO:0009311 |oligosaccharide metabolic process                                                                |  0.4267553|5.9299890408198    |-2.33932589696657    | 1.9822712|   -5.250044|  0.9328493|      0.4018629|null           |FALSE      |
|GO:0019450 |L-cysteine catabolic process to pyruvate                                                         |  0.0044922|3.76788252426548   |-6.37586182378446    | 0.3010300|   -7.764462|  0.8563798|      0.4022447|null           |FALSE      |
|GO:0080027 |response to herbivore                                                                            |  0.0808589|-7.2306748185484   |-0.211665553690911   | 1.2787536|  -35.422927|  0.9004287|      0.4067789|null           |FALSE      |
|GO:0002238 |response to molecule of fungal origin                                                            |  0.0853511|-6.81717811377333  |-1.72635412430574    | 1.3010300|   -5.145763|  0.8665145|      0.4084989|null           |FALSE      |
|GO:0010730 |negative regulation of hydrogen peroxide biosynthetic process                                    |  0.0044922|-4.01581301771407  |5.75426741338803     | 0.3010300|   -5.234106|  0.9081762|      0.4086043|null           |FALSE      |
|GO:0048599 |oocyte development                                                                               |  0.0044922|5.06056377797917   |1.91339653607701     | 0.3010300|   -1.946502|  0.8959250|      0.4096889|null           |FALSE      |
|GO:0006952 |defense response                                                                                 |  7.2503481|-5.50660375906572  |-1.97292827608941    | 3.2081725| -112.399149|  0.8870782|      0.4100881|null           |FALSE      |
|GO:0097366 |response to bronchodilator                                                                       |  2.0370454|-6.25361459039956  |-2.86350711463551    | 2.6570559|  -15.805890|  0.8804006|      0.4110011|null           |FALSE      |
|GO:0071732 |cellular response to nitric oxide                                                                |  0.0179686|-5.52624255700767  |-3.73089650020743    | 0.6989700|  -10.024297|  0.8539062|      0.4112191|null           |FALSE      |
|GO:0090615 |mitochondrial mRNA processing                                                                    |  0.0539059|0.82560010515135   |-4.75851234933306    | 1.1139434|  -27.921063|  0.8849979|      0.4120236|null           |FALSE      |
|GO:0006260 |DNA replication                                                                                  |  0.6423790|1.7570952400045    |-7.20161751562465    | 2.1583625|   -1.831471|  0.8803870|      0.4120353|null           |FALSE      |
|GO:0007166 |cell surface receptor signaling pathway                                                          |  0.5794888|-4.15133490230283  |0.567493594651857    | 2.1139434|   -7.866601|  0.7907883|      0.4151148|null           |FALSE      |
|GO:0032775 |DNA methylation on adenine                                                                       |  0.0808589|0.529673586593329  |-5.93368999700507    | 1.2787536|   -1.562079|  0.8755006|      0.4216009|null           |FALSE      |
|GO:0035864 |response to potassium ion                                                                        |  0.0269530|-6.26904879769833  |-4.27547898538299    | 0.8450980|   -6.154876|  0.9042461|      0.4217509|null           |FALSE      |
|GO:0052544 |defense response by callose deposition in cell wall                                              |  0.0718746|-2.37335753601829  |-0.741578476012768   | 1.2304489|  -45.430189|  0.7980696|      0.4249084|null           |FALSE      |
|GO:0009625 |response to insect                                                                               |  0.1482413|-7.15579119280906  |-0.419568088487055   | 1.5314789|  -46.560252|  0.8959159|      0.4269324|null           |FALSE      |
|GO:0010412 |mannan metabolic process                                                                         |  0.0134765|-1.35850544719206  |-7.63025838963527    | 0.6020600|   -6.629378|  0.9091739|      0.4309542|null           |FALSE      |
|GO:0010469 |regulation of signaling receptor activity                                                        |  0.3059115|-2.27171301492212  |5.086403340552       | 1.8260748|  -29.831756|  0.8449026|      0.4321507|null           |FALSE      |
|GO:0000454 |snoRNA guided rRNA pseudouridine synthesis                                                       |  0.0089843|1.36247086745249   |-4.04039416642493    | 0.4771213|   -1.851063|  0.8742588|      0.4346934|null           |FALSE      |
|GO:0071218 |cellular response to misfolded protein                                                           |  0.1033197|-5.05958006327968  |-2.70312777443612    | 1.3802112|   -1.530161|  0.8450162|      0.4371512|null           |FALSE      |
|GO:2000122 |negative regulation of stomatal complex development                                              |  0.0269530|-2.98786975777609  |5.416333154454       | 0.8450980|   -3.053135|  0.8781078|      0.4392129|null           |FALSE      |
|GO:0036376 |sodium ion export across plasma membrane                                                         |  0.0554033|2.11925536451833   |5.96400853646955     | 1.1139434|  -10.962936|  0.9194143|      0.4394271|null           |FALSE      |
|GO:0002237 |response to molecule of bacterial origin                                                         |  0.2111316|-6.6569004284952   |-1.74968877390379    | 1.6812412|   -8.204322|  0.8523708|      0.4396406|null           |FALSE      |
|GO:0036371 |protein localization to T-tubule                                                                 |  0.0314451|3.00946601385142   |5.64425010486917     | 0.9030900|  -16.269247|  0.9254959|      0.4440885|null           |FALSE      |
|GO:0032527 |protein exit from endoplasmic reticulum                                                          |  0.0359373|2.6551409026226    |5.60789441279035     | 0.9542425|  -10.973980|  0.9237650|      0.4479977|null           |FALSE      |
|GO:0001101 |response to acid chemical                                                                        |  1.7923723|-6.29037513105709  |-2.94478619147836    | 2.6020600|   -2.228327|  0.8818272|      0.4485887|null           |FALSE      |
|GO:0007492 |endoderm development                                                                             |  0.0134765|6.23240827739625   |2.17316991215425     | 0.6020600|   -1.420387|  0.9410100|      0.4499787|null           |FALSE      |
|GO:0008298 |intracellular mRNA localization                                                                  |  0.0404295|2.85849907634051   |5.46527937101967     | 1.0000000|   -3.700549|  0.9268890|      0.4515035|null           |FALSE      |
|GO:0002213 |defense response to insect                                                                       |  0.0943354|-6.61615920379366  |-0.648828668288297   | 1.3424227|   -1.414236|  0.8886396|      0.4532505|null           |FALSE      |
|GO:0002230 |positive regulation of defense response to virus by host                                         |  0.0224608|-1.35082156663637  |6.23865943042795     | 0.7781513|  -23.040381|  0.8969273|      0.4583337|null           |FALSE      |
|GO:0019752 |carboxylic acid metabolic process                                                                |  4.3214591|4.20083090934012   |-6.08925858439222    | 2.9836263|   -7.747054|  0.8493378|      0.4613525|null           |FALSE      |
|GO:0006654 |phosphatidic acid biosynthetic process                                                           |  0.0269530|5.13348119817049   |-4.33783631959327    | 0.8450980|   -1.388060|  0.8614261|      0.4645185|null           |FALSE      |
|GO:0002239 |response to oomycetes                                                                            |  0.4177710|-7.00188554251013  |-0.680937716838015   | 1.9731279|  -58.651888|  0.8871683|      0.4664338|null           |FALSE      |
|GO:0050821 |protein stabilization                                                                            |  0.1437492|-0.25717164431368  |2.20614024110478     | 1.5185139|   -1.545126|  0.8842338|      0.4689739|null           |FALSE      |
|GO:0006898 |receptor-mediated endocytosis                                                                    |  0.1976551|2.94636290484832   |5.94566170769202     | 1.6532125|   -4.215610|  0.9291139|      0.4736740|null           |FALSE      |
|GO:0016074 |sno(s)RNA metabolic process                                                                      |  0.0853511|1.02851055893872   |-4.86274355563181    | 1.3010300|   -1.846556|  0.8824005|      0.4749514|null           |FALSE      |
|GO:0044403 |biological process involved in symbiotic interaction                                             |  0.4312475|-7.81655133624164  |1.05428129191795     | 1.9867717|   -8.969484|  0.9468454|      0.4754856|null           |FALSE      |
|GO:0009727 |detection of ethylene stimulus                                                                   |  0.0134765|-7.10650535925052  |-3.24866776364135    | 0.6020600|  -17.881501|  0.8982612|      0.4765194|null           |FALSE      |
|GO:0010891 |negative regulation of sequestering of triglyceride                                              |  0.0044922|-3.25267838304602  |6.48819933317862     | 0.3010300|   -7.518312|  0.8977293|      0.4769945|null           |FALSE      |
|GO:0051607 |defense response to virus                                                                        |  0.2246081|-6.52868588584436  |-0.90075085041233    | 1.7075702|   -1.773648|  0.8812170|      0.4865194|null           |FALSE      |
|GO:0006885 |regulation of pH                                                                                 |  0.2515610|-0.436382224355416 |2.41703954561543     | 1.7558749|  -21.758538|  0.8743013|      0.4902551|null           |FALSE      |
|GO:0070365 |hepatocyte differentiation                                                                       |  0.0494138|5.09085869734019   |2.1821653734719      | 1.0791812|   -6.381409|  0.8531320|      0.4914523|null           |FALSE      |
|GO:0033292 |T-tubule organization                                                                            |  0.0716381|5.16628245756623   |1.2156416474347      | 1.2041200|  -16.269247|  0.8862545|      0.4928611|null           |FALSE      |
|GO:0010150 |leaf senescence                                                                                  |  0.5076142|5.408956791536     |2.54965439756318     | 2.0569049|  -22.361394|  0.8905230|      0.5023722|null           |FALSE      |
|GO:0097237 |cellular response to toxic substance                                                             |  0.8355420|-5.44644008342422  |-2.75973306769322    | 2.2718416|  -15.715204|  0.8430985|      0.5080142|null           |FALSE      |
|GO:0048312 |intracellular distribution of mitochondria                                                       |  0.0044922|3.13624729483009   |4.58947984134058     | 0.3010300|   -1.420387|  0.9210913|      0.5090434|null           |FALSE      |
|GO:0009451 |RNA modification                                                                                 |  1.4105386|1.09454808071833   |-5.77515753201823    | 2.4983106|  -15.068086|  0.8538397|      0.5091922|null           |FALSE      |
|GO:1901726 |negative regulation of histone deacetylase activity                                              |  0.0494138|-3.87099935136356  |5.23782059083548     | 1.0791812|   -8.437473|  0.8797078|      0.5094696|null           |FALSE      |
|GO:0009992 |cellular water homeostasis                                                                       |  0.0089843|1.44742842137189   |0.86601950791838     | 0.4771213|   -2.939957|  0.8641337|      0.5104375|null           |FALSE      |
|GO:0006183 |GTP biosynthetic process                                                                         |  0.0359373|2.81144840222488   |-5.17545553549896    | 0.9542425|   -3.957863|  0.8250314|      0.5116103|null           |FALSE      |
|GO:0006097 |glyoxylate cycle                                                                                 |  0.0314451|4.09661637568492   |-6.20636610268115    | 0.9030900|   -7.795499|  0.8717126|      0.5129277|null           |FALSE      |
|GO:0036309 |protein localization to M-band                                                                   |  0.2713265|2.80569055112218   |5.61516045560594     | 1.7853298|  -16.269247|  0.9153105|      0.5168809|null           |FALSE      |
|GO:0048316 |seed development                                                                                 |  2.6099456|5.11840704189552   |2.48727578461485     | 2.7649230|   -7.268778|  0.8720233|      0.5172320|null           |FALSE      |
|GO:0043966 |histone H3 acetylation                                                                           |  0.0853511|0.221204306515285  |-7.60961823291865    | 1.3010300|   -5.253700|  0.9065026|      0.5222382|null           |FALSE      |
|GO:0060291 |long-term synaptic potentiation                                                                  |  0.0044922|-1.06273983049681  |3.34685803938291     | 0.3010300|  -10.857594|  0.8508110|      0.5247898|null           |FALSE      |
|GO:0006432 |phenylalanyl-tRNA aminoacylation                                                                 |  0.0134765|2.25682280770735   |-5.68283107249334    | 0.6020600|   -4.788760|  0.8353541|      0.5291698|null           |FALSE      |
|GO:1905034 |regulation of antifungal innate immune response                                                  |  0.0089843|-1.20358180421795  |6.44366692748713     | 0.4771213|   -6.383180|  0.9012601|      0.5294786|null           |FALSE      |
|GO:2000280 |regulation of root development                                                                   |  0.3593729|-2.04244567242534  |3.89590942995165     | 1.9084850|  -50.621305|  0.8988447|      0.5325899|null           |FALSE      |
|GO:0009961 |response to 1-aminocyclopropane-1-carboxylic acid                                                |  0.0179686|-6.58333273445832  |-3.92903227614793    | 0.6989700|   -2.849513|  0.8875445|      0.5385267|null           |FALSE      |
|GO:1901684 |arsenate ion transmembrane transport                                                             |  0.0224608|2.22078760449027   |6.19496291303995     | 0.7781513|   -5.722287|  0.9276661|      0.5470253|null           |FALSE      |
|GO:0080110 |sporopollenin biosynthetic process                                                               |  0.0359373|3.61949253253796   |-1.00397663534149    | 0.9542425|   -8.596184|  0.7724830|      0.5476284|null           |FALSE      |
|GO:0071398 |cellular response to fatty acid                                                                  |  0.4402318|-5.57994582761479  |-2.97384742805575    | 1.9956352|  -23.248938|  0.8270138|      0.5536354|null           |FALSE      |
|GO:0032504 |multicellular organism reproduction                                                              |  0.8040969|5.18003574597184   |3.20107979833905     | 2.2552725|   -1.308625|  0.9115321|      0.5539836|null           |FALSE      |
|GO:1990169 |stress response to copper ion                                                                    |  0.0224608|-5.89998322396396  |-3.91730060104988    | 0.7781513|   -4.599134|  0.8972724|      0.5582896|null           |FALSE      |
|GO:0009617 |response to bacterium                                                                            |  2.1113158|-6.71792043871839  |-1.0210003035457     | 2.6730209|  -10.985497|  0.8701063|      0.5594759|null           |FALSE      |
|GO:0006696 |ergosterol biosynthetic process                                                                  |  0.0479164|3.79616432899382   |-4.75297280722759    | 1.0413927|   -3.317833|  0.8171388|      0.5602955|null           |FALSE      |
|GO:0010114 |response to red light                                                                            |  0.2785140|-5.14612618971936  |-5.13696360026752    | 1.7993405|   -3.646740|  0.9233495|      0.5606314|null           |FALSE      |
|GO:0035518 |histone H2A monoubiquitination                                                                   |  0.0089843|-0.255174224748155 |-8.01011676945895    | 0.4771213|   -1.668789|  0.9187566|      0.5609440|null           |FALSE      |
|GO:0006468 |protein phosphorylation                                                                          |  5.0671578|1.22071250400682   |-6.71077457099569    | 3.0526939|  -40.763992|  0.8598127|      0.5616227|null           |FALSE      |
|GO:0044255 |cellular lipid metabolic process                                                                 |  3.4050582|5.96794359083919   |-4.04806996319238    | 2.8802418|   -2.977720|  0.8700978|      0.5623316|null           |FALSE      |
|GO:0010898 |positive regulation of triglyceride catabolic process                                            |  0.1161545|-3.91925646087433  |4.21625362672607     | 1.4149733|   -7.518312|  0.8827537|      0.5626579|null           |FALSE      |
|GO:0048509 |regulation of meristem development                                                               |  0.2964826|-1.89462293638197  |3.92080896382945     | 1.8260748|  -18.319363|  0.9002900|      0.5634440|null           |FALSE      |
|GO:1902265 |abscisic acid homeostasis                                                                        |  0.0089843|0.536257445328091  |1.54891213440271     | 0.4771213|   -2.721576|  0.8990399|      0.5661740|null           |FALSE      |
|GO:0010930 |negative regulation of auxin mediated signaling pathway                                          |  0.0134765|-2.33539284511316  |5.91756961780614     | 0.6020600|   -4.664218|  0.8708589|      0.5666125|null           |FALSE      |
|GO:0060359 |response to ammonium ion                                                                         |  0.0449216|-6.40033603127563  |-3.94440831055343    | 1.0413927|   -4.895017|  0.8902623|      0.5687396|null           |FALSE      |
|GO:0016099 |monoterpenoid biosynthetic process                                                               |  0.0044922|5.72256841991197   |-3.96359608316876    | 0.3010300|  -44.057848|  0.8811846|      0.5751028|null           |FALSE      |
|GO:0080003 |thalianol metabolic process                                                                      |  0.0089843|4.12163744446057   |-3.99216327243334    | 0.4771213|  -23.185147|  0.8704191|      0.5772455|null           |FALSE      |
|GO:0050829 |defense response to Gram-negative bacterium                                                      |  0.0988275|-6.71195544519404  |-0.738702911338103   | 1.3617278|   -5.789530|  0.8837626|      0.5774326|null           |FALSE      |
|GO:0097167 |circadian regulation of translation                                                              |  0.0044922|1.36771140528326   |3.08994331106631     | 0.3010300|   -3.424748|  0.8692616|      0.5780757|null           |FALSE      |
|GO:0015886 |heme transport                                                                                   |  0.0718746|2.21081479041718   |5.94218835387769     | 1.2304489|   -3.017938|  0.9412379|      0.5790223|null           |FALSE      |
|GO:0106004 |tRNA (guanine-N7)-methylation                                                                    |  0.0134765|0.608776579545044  |-5.89892555230976    | 0.6020600|   -4.538367|  0.8841118|      0.5801978|null           |FALSE      |
|GO:0071669 |plant-type cell wall organization or biogenesis                                                  |  1.1769462|-3.13423274061765  |-7.62912748344021    | 2.4199557|  -22.659946|  0.9522018|      0.5820901|null           |FALSE      |
|GO:1902290 |positive regulation of defense response to oomycetes                                             |  0.0449216|-1.85227238667524  |5.63472044108792     | 1.0413927|  -11.533178|  0.8706707|      0.5823779|null           |FALSE      |
|GO:0071731 |response to nitric oxide                                                                         |  0.0269530|-6.39500320761542  |-4.08390671440053    | 0.8450980|   -3.642485|  0.8881428|      0.5832186|null           |FALSE      |
|GO:0042631 |cellular response to water deprivation                                                           |  0.1707021|-5.58256039245665  |-2.30188040535942    | 1.5910646|  -21.407502|  0.7942538|      0.5832916|null           |FALSE      |
|GO:0070914 |UV-damage excision repair                                                                        |  0.0089843|-2.09665948725329  |-4.12030769376234    | 0.4771213|   -2.353867|  0.8122948|      0.5832916|null           |FALSE      |
|GO:0060465 |pharynx development                                                                              |  1.4053739|5.21639978772319   |2.61152220721345     | 2.4927604|   -1.420387|  0.8839155|      0.5838147|null           |FALSE      |
|GO:0071266 |&#39;de novo&#39; L-methionine biosynthetic process                                                      |  0.0044922|4.19043043927024   |-5.81021622420296    | 0.3010300|   -7.513695|  0.8586559|      0.5860863|null           |FALSE      |
|GO:0007165 |signal transduction                                                                              |  8.3913571|-4.20556732619324  |0.593971465701832    | 3.2716093| -140.613488|  0.7343275|      0.5893442|null           |FALSE      |
|GO:0071446 |cellular response to salicylic acid stimulus                                                     |  0.2605453|-5.58761087801798  |-3.12145984677086    | 1.7708520|  -16.698116|  0.8351211|      0.5943078|null           |FALSE      |
|GO:0009773 |photosynthetic electron transport in photosystem I                                               |  0.0763667|-2.90490874730459  |-8.25535470768478    | 1.2552725|   -3.872559|  0.9374888|      0.5949536|null           |FALSE      |
|GO:0098660 |inorganic ion transmembrane transport                                                            |  1.9810431|2.45360728482874   |5.88576703915754     | 2.6454223|   -8.590303|  0.9031928|      0.5981993|null           |FALSE      |
|GO:0060294 |cilium movement involved in cell motility                                                        |  0.0059895|3.77925340130644   |5.10589883131856     | 0.4771213|  -10.751648|  0.9234940|      0.5985002|null           |FALSE      |
|GO:0007263 |nitric oxide mediated signal transduction                                                        |  0.0089843|-3.81653369337475  |0.431643089606601    | 0.4771213|   -6.950778|  0.8393932|      0.6046800|null           |FALSE      |
|GO:0072498 |embryonic skeletal joint development                                                             |  1.8732312|5.24327152071272   |2.51658772960219     | 2.6211763|   -1.420387|  0.8806162|      0.6058216|null           |FALSE      |
|GO:1901255 |nucleotide-excision repair involved in interstrand cross-link repair                             |  0.0044922|-1.75693651094372  |-3.95299427162733    | 0.3010300|   -1.694668|  0.8228111|      0.6058423|null           |FALSE      |
|GO:0060213 |positive regulation of nuclear-transcribed mRNA poly(A) tail shortening                          |  0.0044922|-4.8825303600124   |4.30587597987053     | 0.3010300|   -6.381409|  0.9026574|      0.6079678|null           |FALSE      |
|GO:0070301 |cellular response to hydrogen peroxide                                                           |  0.0449216|-5.11795984566023  |-3.12000089659845    | 1.0413927|  -22.946515|  0.8373058|      0.6082701|null           |FALSE      |
|GO:0043617 |cellular response to sucrose starvation                                                          |  0.0179686|-5.24082129338113  |-1.15371696116577    | 0.6989700|  -11.048683|  0.8618931|      0.6083709|null           |FALSE      |
|GO:0052324 |plant-type cell wall cellulose biosynthetic process                                              |  0.0583981|3.33404938761739   |-3.05336262179286    | 1.1461280|   -2.751928|  0.8509965|      0.6106041|null           |FALSE      |
|GO:0016045 |detection of bacterium                                                                           |  0.0134765|-7.479422702251    |0.153002203343607    | 0.6020600|  -12.507868|  0.8998251|      0.6121094|null           |FALSE      |
|GO:0098703 |calcium ion import across plasma membrane                                                        |  0.0170702|1.61446003241648   |4.63802731989905     | 0.6020600|  -10.857594|  0.8154986|      0.6124892|null           |FALSE      |
|GO:0060776 |simple leaf morphogenesis                                                                        |  0.0224608|5.63149251974423   |2.41302828105194     | 0.7781513|   -2.621028|  0.9059724|      0.6130285|null           |FALSE      |
|GO:0034721 |histone H3-K4 demethylation,trimethyl-H3-K4-specific                                             |  0.0404295|-0.06953902457476  |-7.74802162409636    | 1.0000000|   -2.012384|  0.9123704|      0.6176463|null           |FALSE      |
|GO:0019348 |dolichol metabolic process                                                                       |  0.0269530|5.17578773341096   |-4.82803272749995    | 0.8450980|   -3.425914|  0.8458432|      0.6194937|null           |FALSE      |
|GO:0046345 |abscisic acid catabolic process                                                                  |  0.0269530|4.72896425525045   |-4.81902876510671    | 0.8450980|  -40.398158|  0.8071561|      0.6194937|null           |FALSE      |
|GO:0000226 |microtubule cytoskeleton organization                                                            |  0.7232379|5.48438321823205   |-0.245894550502096   | 2.2095150|   -7.960852|  0.9320214|      0.6208058|null           |FALSE      |
|GO:0006165 |nucleoside diphosphate phosphorylation                                                           |  0.3593729|2.64186329553339   |-5.27348142100191    | 1.9084850|   -2.837631|  0.8271526|      0.6231203|null           |FALSE      |
|GO:0010016 |shoot system morphogenesis                                                                       |  0.7097615|5.41615233249326   |2.36697720298566     | 2.2013971|   -9.300459|  0.8799230|      0.6244288|null           |FALSE      |
|GO:0005977 |glycogen metabolic process                                                                       |  0.0853511|-1.12705649077983  |-7.67769598395234    | 1.3010300|   -2.640777|  0.9050998|      0.6280829|null           |FALSE      |
|GO:0045926 |negative regulation of growth                                                                    |  0.1751943|-3.0572390969267   |5.55803910151506     | 1.6020600|  -30.727605|  0.8954048|      0.6297772|null           |FALSE      |
|GO:2001294 |malonyl-CoA catabolic process                                                                    |  0.0089843|2.58470846800546   |-5.41449148195213    | 0.4771213|   -5.384890|  0.8254937|      0.6317562|null           |FALSE      |
|GO:0046477 |glycosylceramide catabolic process                                                               |  0.0179686|2.99400554257678   |-4.64483004580199    | 0.6989700|   -2.369658|  0.8418743|      0.6330611|null           |FALSE      |
|GO:1903507 |negative regulation of nucleic acid-templated transcription                                      |  1.0556579|-3.33356469303437  |4.87620741634673     | 2.3729120|   -1.958349|  0.8720632|      0.6336059|null           |FALSE      |
|GO:0031640 |killing of cells of other organism                                                               |  1.3072189|-7.83200645189723  |0.952335232431278    | 2.4653829|   -1.420387|  0.9121394|      0.6338299|null           |FALSE      |
|GO:0000025 |maltose catabolic process                                                                        |  0.0089843|4.58445594231306   |-2.54547228982552    | 0.4771213|   -6.061973|  0.9066224|      0.6340028|null           |FALSE      |
|GO:0009877 |nodulation                                                                                       |  0.0988275|5.98648850092432   |1.92990170500729     | 1.3617278|   -4.476728|  0.9257847|      0.6360522|null           |FALSE      |
|GO:0015700 |arsenite transport                                                                               |  0.0224608|2.1093730807965    |6.16693991189603     | 0.7781513|   -7.248628|  0.9472324|      0.6364426|null           |FALSE      |
|GO:1901599 |(-)-pinoresinol biosynthetic process                                                             |  0.0089843|3.4401472752472    |-4.65425137148356    | 0.4771213|   -2.479543|  0.8431700|      0.6369392|null           |FALSE      |
|GO:0006293 |nucleotide-excision repair,preincision complex stabilization                                     |  0.0119791|-1.95662696029528  |-2.26337140745141    | 0.4771213|   -6.189060|  0.7378240|      0.6370511|null           |FALSE      |
|GO:0009852 |auxin catabolic process                                                                          |  0.0089843|1.13006169965014   |0.069490756962471    | 0.4771213|   -8.841270|  0.8324548|      0.6412626|null           |FALSE      |
|GO:0071229 |cellular response to acid chemical                                                               |  0.2740218|-5.37866516183528  |-3.11474071636372    | 1.7923917|  -15.446531|  0.8519168|      0.6436302|null           |FALSE      |
|GO:0048265 |response to pain                                                                                 | 11.9626252|-3.44715686446661  |-1.43227622745241    | 3.4255342|   -1.420387|  0.7760406|      0.6480493|null           |FALSE      |
|GO:0031347 |regulation of defense response                                                                   |  1.2308522|-1.92010445986362  |5.10584360159096     | 2.4393327|   -5.011113|  0.8670982|      0.6512180|null           |FALSE      |
|GO:0030639 |polyketide biosynthetic process                                                                  |  0.0179686|5.55222951154861   |-2.96144419742833    | 0.6989700|   -9.213466|  0.9204388|      0.6523745|null           |FALSE      |
|GO:0051513 |regulation of monopolar cell growth                                                              |  0.0224608|-2.08006177599226  |3.62176407353108     | 0.7781513|   -1.337060|  0.9022145|      0.6532683|null           |FALSE      |
|GO:0012502 |induction of programmed cell death                                                               |  0.0044922|-4.65577777229914  |3.79450211965137     | 0.3010300|   -6.158100|  0.9045731|      0.6549617|null           |FALSE      |
|GO:1902042 |negative regulation of extrinsic apoptotic signaling pathway via death domain receptors          |  0.0044922|-2.49013770907636  |5.95166223775384     | 0.3010300|   -4.973551|  0.8702445|      0.6549617|null           |FALSE      |
|GO:0000304 |response to singlet oxygen                                                                       |  0.0583981|-5.98985665332477  |-3.49007396183438    | 1.1461280|  -12.794415|  0.8795620|      0.6554547|null           |FALSE      |
|GO:0009083 |branched-chain amino acid catabolic process                                                      |  0.1167962|3.46974844055578   |-6.360813437743      | 1.4313638|  -20.784983|  0.8460379|      0.6580541|null           |FALSE      |
|GO:2000026 |regulation of multicellular organismal development                                               |  1.5677643|-2.33437637305051  |3.80004493699785     | 2.5440680|  -16.454846|  0.8585602|      0.6593517|null           |FALSE      |
|GO:0046482 |para-aminobenzoic acid metabolic process                                                         |  0.0179686|2.72142092781904   |-5.99976993461053    | 0.6989700|  -17.318271|  0.8564305|      0.6615462|null           |FALSE      |
|GO:0032870 |cellular response to hormone stimulus                                                            |  4.2316158|-5.6118179549043   |-2.66420600453923    | 2.9745117|  -11.812818|  0.8057412|      0.6623081|null           |FALSE      |
|GO:0010104 |regulation of ethylene-activated signaling pathway                                               |  0.1257805|-1.69545070490941  |5.44972425179867     | 1.4623980|   -9.485067|  0.8650525|      0.6681187|null           |FALSE      |
|GO:0036159 |inner dynein arm assembly                                                                        |  0.0179686|5.68747980209538   |-0.316612801083686   | 0.6989700|   -6.262662|  0.9384078|      0.6711268|null           |FALSE      |
|GO:0070911 |global genome nucleotide-excision repair                                                         |  0.0119791|-1.88971946046442  |-3.97894128589853    | 0.4771213|   -1.368073|  0.8126285|      0.6716500|null           |FALSE      |
|GO:0045338 |farnesyl diphosphate metabolic process                                                           |  0.0898432|5.75057837562711   |-4.31223576450644    | 1.3222193|  -27.993512|  0.8584719|      0.6735153|null           |FALSE      |
|GO:0009801 |cinnamic acid ester metabolic process                                                            |  0.0089843|2.02376386069319   |-3.28016870110978    | 0.4771213|  -12.547408|  0.8928747|      0.6748672|null           |FALSE      |
|GO:0033387 |putrescine biosynthetic process from ornithine                                                   |  0.0123534|3.09211966606623   |-5.75797204323341    | 0.4771213|   -9.901953|  0.8537844|      0.6792812|null           |FALSE      |
|GO:0080024 |indolebutyric acid metabolic process                                                             |  0.0224608|1.57713029495705   |-3.10052263858256    | 0.7781513|  -11.831338|  0.7692882|      0.6811036|null           |FALSE      |
|GO:0071368 |cellular response to cytokinin stimulus                                                          |  0.3593729|-5.64508425285129  |-2.85986753356041    | 1.9084850|  -11.332566|  0.8432083|      0.6812564|null           |FALSE      |
|GO:0008277 |regulation of G protein-coupled receptor signaling pathway                                       |  0.1735903|-1.75133612314467  |5.29350116251497     | 1.5910646|   -1.694668|  0.8652961|      0.6858401|null           |FALSE      |
|GO:1900449 |regulation of glutamate receptor signaling pathway                                               |  0.1735903|-1.80585068537604  |5.33459104295517     | 1.5682017|   -4.973551|  0.8652961|      0.6858401|null           |FALSE      |
|GO:0010439 |regulation of glucosinolate biosynthetic process                                                 |  0.0449216|-4.72811096596063  |4.8283002595811      | 1.0413927|  -11.890992|  0.9114000|      0.6868343|null           |FALSE      |
|GO:0010310 |regulation of hydrogen peroxide metabolic process                                                |  0.0539059|-4.52121204051442  |4.88453520626293     | 1.1139434|   -1.816874|  0.9154016|      0.6872518|null           |FALSE      |
|GO:0010030 |positive regulation of seed germination                                                          |  0.0898432|-2.57771651182984  |3.70487877845101     | 1.3222193|  -13.607739|  0.8615093|      0.6895163|null           |FALSE      |
|GO:0003283 |atrial septum development                                                                        |  1.6401201|5.1449156567373    |2.40525212269863     | 3.2564772|   -3.074783|  0.8594190|      0.6897469|null           |FALSE      |
|GO:0071805 |potassium ion transmembrane transport                                                            |  0.2560532|2.32106951697353   |5.90703293679039     | 1.7634280|   -1.841910|  0.9141416|      0.6907954|null           |FALSE      |
|GO:0005983 |starch catabolic process                                                                         |  0.0763667|-0.107659417273279 |-7.31590935022743    | 1.2552725|   -5.349583|  0.8846263|      0.6912926|null           |FALSE      |
|GO:1902198 |3-methylbut-2-enoyl-CoA(4-) metabolic process                                                    |  0.0202147|2.6166328324834    |-5.54192035328825    | 0.6989700|   -3.911273|  0.8352944|      0.6947265|null           |FALSE      |
|GO:0048767 |root hair elongation                                                                             |  0.3099591|5.34438166560857   |2.18799891429401     | 1.8450980|   -2.636992|  0.8459609|      0.6947824|null           |FALSE      |

[1] &quot;missing_zmuel_vs_all_GO.txt&quot;


|TermID     |Name                                                                                             | Frequency|PlotX                |PlotY               |   LogSize|       Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:------------------------------------------------------------------------------------------------|---------:|:--------------------|:-------------------|---------:|-----------:|----------:|--------------:|:--------------|:----------|
|GO:0010082 |regulation of root meristem growth                                                               | 0.1527335|3.16869702627119     |4.45246384290096    | 1.5440680|  -68.162621|  0.8004613|      0.0000000|null           |FALSE      |
|GO:0019079 |viral genome replication                                                                         | 0.0269530|-6.40461549342608    |-5.45779854717161   | 0.8450980|   -5.731469|  1.0000000|      0.0000000|null           |FALSE      |
|GO:0048512 |circadian behavior                                                                               | 0.0224608|3.56218444695079     |1.36075000763022    | 0.7781513|   -8.853277|  0.9936608|      0.0000000|null           |FALSE      |
|GO:0051762 |sesquiterpene biosynthetic process                                                               | 0.1123040|3.36141698904477     |-5.73217317400478   | 1.4149733| -181.717133|  0.8593891|      0.0000000|null           |FALSE      |
|GO:1902025 |nitrate import                                                                                   | 0.0808589|5.74097838889235     |-2.09909797970759   | 1.2787536|  -26.329774|  0.9275676|      0.0000000|null           |FALSE      |
|GO:0034090 |maintenance of meiotic sister chromatid cohesion                                                 | 0.0179686|6.21158102699487     |1.11210247108656    | 0.6989700|  -13.377534|  0.9341446|      0.0215931|null           |FALSE      |
|GO:0009626 |plant-type hypersensitive response                                                               | 0.3279278|-5.97818959377893    |-0.197844192579732  | 1.8692317|  -93.136155|  0.8166683|      0.0266051|null           |FALSE      |
|GO:0006739 |NADP metabolic process                                                                           | 0.1707021|2.42124396072051     |-1.09769857392856   | 1.5910646|   -1.769930|  0.9669582|      0.0637985|null           |FALSE      |
|GO:0051554 |flavonol metabolic process                                                                       | 0.0853511|-5.71261326066196    |-5.61968913811348   | 1.3010300|  -30.086084|  0.9408664|      0.0732686|null           |FALSE      |
|GO:0000023 |maltose metabolic process                                                                        | 0.0179686|-5.03154882342965    |-5.88518613300346   | 0.6989700|   -9.773194|  0.9305868|      0.0748216|null           |FALSE      |
|GO:0009854 |oxidative photosynthetic carbon pathway                                                          | 0.0404295|-6.6900214575787     |-5.15085917266825   | 1.0000000|   -4.677642|  0.9498394|      0.0757533|null           |FALSE      |
|GO:0031146 |SCF-dependent proteasomal ubiquitin-dependent protein catabolic process                          | 0.3773415|-2.19818876846121    |-5.64635792787252   | 1.9294189|  -70.924487|  0.8907957|      0.0932346|null           |FALSE      |
|GO:0080184 |response to phenylpropanoid                                                                      | 0.0044922|-6.89141228781458    |3.90470605397759    | 0.3010300|  -12.298286|  0.9106411|      0.1196905|null           |FALSE      |
|GO:0090615 |mitochondrial mRNA processing                                                                    | 0.0539059|-0.869419701829744   |-4.63813020618443   | 1.1139434|  -43.900684|  0.9023025|      0.1306016|null           |FALSE      |
|GO:0006629 |lipid metabolic process                                                                          | 4.9368851|-3.64546742506239    |-6.90877510928273   | 3.0413927|   -4.332034|  0.9424280|      0.1343063|null           |FALSE      |
|GO:0009962 |regulation of flavonoid biosynthetic process                                                     | 0.1257805|-1.26249955287126    |7.21950569367648    | 1.4623980|  -22.227242|  0.9130691|      0.1471751|null           |FALSE      |
|GO:0009820 |alkaloid metabolic process                                                                       | 0.0224608|-1.65878970453779    |-7.64288308747159   | 0.7781513|   -3.710300|  0.9509119|      0.1530936|null           |FALSE      |
|GO:0010469 |regulation of signaling receptor activity                                                        | 0.3059115|-0.536599883652841   |6.30880268691039    | 1.8260748|  -31.275882|  0.8358372|      0.1578322|null           |FALSE      |
|GO:0010119 |regulation of stomatal movement                                                                  | 0.4492161|-0.651814184124113   |4.79177196832018    | 2.0043214|   -3.570823|  0.9070496|      0.1956799|null           |FALSE      |
|GO:0035902 |response to immobilization stress                                                                | 0.0089843|-5.31027273752086    |-2.51322102031514   | 0.4771213|  -12.183306|  0.9323405|      0.1969794|null           |FALSE      |
|GO:0018874 |benzoate metabolic process                                                                       | 0.0089843|0.815537791422452    |-3.99087228082464   | 0.4771213|  -24.723612|  0.8916921|      0.2082263|null           |FALSE      |
|GO:0060294 |cilium movement involved in cell motility                                                        | 0.0059895|5.6464528379681      |-2.95429105034015   | 0.4771213|  -20.759764|  0.9121569|      0.2109973|null           |FALSE      |
|GO:2000083 |negative regulation of L-ascorbic acid biosynthetic process                                      | 0.0044922|1.46846603339721     |6.4306584146359     | 0.3010300|  -15.536707|  0.9019624|      0.2131819|null           |FALSE      |
|GO:0010731 |protein glutathionylation                                                                        | 0.0359373|-2.37902883428036    |-6.80327875259231   | 0.9542425|   -8.376087|  0.9256076|      0.2148306|null           |FALSE      |
|GO:0009819 |drought recovery                                                                                 | 0.0494138|-7.0366025398024     |1.40412676656072    | 1.0791812|  -20.745503|  0.8736617|      0.2221654|null           |FALSE      |
|GO:0034721 |histone H3-K4 demethylation,trimethyl-H3-K4-specific                                             | 0.0404295|-2.59112507758414    |-6.61351517589381   | 1.0000000|   -2.288318|  0.9231806|      0.2238850|null           |FALSE      |
|GO:0048211 |Golgi vesicle docking                                                                            | 0.0089843|5.17069374947415     |-1.44123130574125   | 0.4771213|   -7.655842|  0.9262178|      0.2267775|null           |FALSE      |
|GO:0009410 |response to xenobiotic stimulus                                                                  | 0.0359373|-7.5038518001035     |2.5334129740837     | 0.9542425|   -4.287566|  0.9101026|      0.2284321|null           |FALSE      |
|GO:0046900 |tetrahydrofolylpolyglutamate metabolic process                                                   | 0.0314451|-0.4068599665303     |-5.82088430807219   | 0.9030900|  -23.721833|  0.9127431|      0.2308025|null           |FALSE      |
|GO:0015768 |maltose transport                                                                                | 0.0134765|5.33284522759955     |-2.67050665237557   | 0.6020600|   -8.593896|  0.9507231|      0.2325128|null           |FALSE      |
|GO:0006597 |spermine biosynthetic process                                                                    | 0.0269530|0.400174011544778    |-6.53970525846829   | 0.8450980|  -33.656807|  0.8923135|      0.2345224|null           |FALSE      |
|GO:2000142 |regulation of DNA-templated transcription,initiation                                             | 0.0898432|-1.36563689756168    |6.85721452008261    | 1.3222193|   -3.491777|  0.9098022|      0.2598207|null           |FALSE      |
|GO:0009865 |pollen tube adhesion                                                                             | 0.0134765|3.9300565646782      |3.34710279979918    | 0.6020600|   -2.838595|  0.8848024|      0.2682041|null           |FALSE      |
|GO:1900706 |positive regulation of siderophore biosynthetic process                                          | 0.1379093|-1.42893381057407    |6.37568617241891    | 1.4913617|  -12.168786|  0.8813707|      0.2682144|null           |FALSE      |
|GO:0007613 |memory                                                                                           | 0.0943354|4.80090465456903     |4.92690063543031    | 1.3424227|   -7.987966|  0.9006075|      0.2779671|null           |FALSE      |
|GO:0045168 |cell-cell signaling involved in cell fate commitment                                             | 0.1662100|3.74487296064584     |3.3577574363665     | 1.5797836|  -45.831663|  0.8733870|      0.2805344|null           |FALSE      |
|GO:0044550 |secondary metabolite biosynthetic process                                                        | 0.6244104|4.0347787932528      |-5.35113393812265   | 2.1461280|  -31.883123|  0.9153338|      0.2815131|null           |FALSE      |
|GO:0010881 |regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion | 0.0089843|-2.33433459575431    |2.44393942287482    | 0.4771213|  -31.738087|  0.7413410|      0.2860549|null           |FALSE      |
|GO:0009611 |response to wounding                                                                             | 1.0062441|-5.34633537126051    |-0.281631214499564  | 2.3521825|  -17.276112|  0.9038883|      0.2870502|null           |FALSE      |
|GO:0010305 |leaf vascular tissue pattern formation                                                           | 0.1347648|5.07362185034631     |4.3926726466339     | 1.4913617|  -10.168418|  0.9125047|      0.3105240|null           |FALSE      |
|GO:0071236 |cellular response to antibiotic                                                                  | 0.8690604|-6.17464856722021    |1.69178303257545    | 2.2878017|  -32.390998|  0.8385759|      0.3202583|null           |FALSE      |
|GO:0010114 |response to red light                                                                            | 0.2785140|-7.88709067031068    |-0.0384574656191098 | 1.7993405|  -10.030785|  0.9134787|      0.3255957|null           |FALSE      |
|GO:0034394 |protein localization to cell surface                                                             | 0.0134765|4.82403087225397     |-1.77155455157968   | 0.6020600|   -4.966521|  0.9268630|      0.3265777|null           |FALSE      |
|GO:0016567 |protein ubiquitination                                                                           | 3.7105251|-1.90793814920064    |-6.17223392529393   | 2.9175055|  -54.100002|  0.8919161|      0.3270943|null           |FALSE      |
|GO:0071347 |cellular response to interleukin-1                                                               | 0.0044922|-6.16380345824222    |3.27953142694249    | 0.3010300|   -8.739769|  0.8822764|      0.3281054|null           |FALSE      |
|GO:1902075 |cellular response to salt                                                                        | 0.0044922|-6.05067823915317    |3.43385584985847    | 0.3010300|   -9.029474|  0.8883589|      0.3281054|null           |FALSE      |
|GO:0018106 |peptidyl-histidine phosphorylation                                                               | 0.0763667|-1.00453326899056    |-6.85034529123687   | 1.2552725|  -11.606010|  0.9078545|      0.3366283|null           |FALSE      |
|GO:0009745 |sucrose mediated signaling                                                                       | 0.0089843|-4.85989339692621    |3.36958775063354    | 0.4771213|   -1.408307|  0.8033981|      0.3442773|null           |FALSE      |
|GO:0000720 |pyrimidine dimer repair by nucleotide-excision repair                                            | 0.0044922|-3.22148780341852    |-1.96952337894261   | 0.3010300|  -19.147167|  0.8344001|      0.3500169|null           |FALSE      |
|GO:0009088 |threonine biosynthetic process                                                                   | 0.0449216|1.17310470098281     |-5.81393296638507   | 1.0413927|   -9.669230|  0.8676126|      0.3530967|null           |FALSE      |
|GO:0009649 |entrainment of circadian clock                                                                   | 0.0808589|-3.68035634271225    |1.77311578172994    | 1.2787536|   -9.376970|  0.8409690|      0.3565069|null           |FALSE      |
|GO:0015840 |urea transport                                                                                   | 0.0224608|5.59450621998255     |-2.52793888711056   | 0.7781513|   -2.786399|  0.9473443|      0.3585451|null           |FALSE      |
|GO:0071475 |cellular hyperosmotic salinity response                                                          | 0.0089843|-5.75904106868445    |1.23458411628001    | 0.4771213|  -26.267986|  0.8578631|      0.3661508|null           |FALSE      |
|GO:0090305 |nucleic acid phosphodiester bond hydrolysis                                                      | 1.6171780|-0.861548733901835   |-5.28522603804221   | 2.5575072|  -39.740579|  0.8786989|      0.3665118|null           |FALSE      |
|GO:0006778 |porphyrin-containing compound metabolic process                                                  | 0.3458964|-0.537707210243402   |-5.53379599309764   | 1.8920946|   -7.149977|  0.8968346|      0.3675002|null           |FALSE      |
|GO:0071466 |cellular response to xenobiotic stimulus                                                         | 0.0224608|-5.92625617867512    |2.71836189136775    | 0.7781513|  -26.952461|  0.8767743|      0.3682726|null           |FALSE      |
|GO:0044364 |disruption of cells of other organism                                                            | 0.0269530|-7.09050121279601    |-3.31804175298192   | 0.8450980|   -1.511515|  0.9567279|      0.3726472|null           |FALSE      |
|GO:0009143 |nucleoside triphosphate catabolic process                                                        | 0.0269530|0.339317524539531    |-5.12055791650568   | 0.8450980|  -11.547948|  0.8584364|      0.3764378|null           |FALSE      |
|GO:0045056 |transcytosis                                                                                     | 0.0044922|4.9250904403623      |0.338396110381116   | 0.3010300|   -7.655842|  0.8880990|      0.3809345|null           |FALSE      |
|GO:0009871 |jasmonic acid and ethylene-dependent systemic resistance,ethylene mediated signaling pathway     | 0.0089843|-4.91497274311753    |1.84741425731275    | 0.4771213|  -34.033423|  0.7780342|      0.3823772|null           |FALSE      |
|GO:0007166 |cell surface receptor signaling pathway                                                          | 0.5794888|-3.55788057266949    |3.23850695609229    | 2.1139434|  -13.075849|  0.7828278|      0.3892452|null           |FALSE      |
|GO:0031508 |pericentric heterochromatin assembly                                                             | 0.0044922|2.01902981904031     |5.07549811017996    | 0.3010300|   -5.799869|  0.8606742|      0.3892522|null           |FALSE      |
|GO:0071219 |cellular response to molecule of bacterial origin                                                | 0.0494138|-6.37425661756341    |0.240079690776506   | 1.0791812|   -7.355856|  0.8408387|      0.3917536|null           |FALSE      |
|GO:0006314 |intron homing                                                                                    | 0.0224608|-1.45749769516832    |-4.77877232505935   | 0.7781513|   -2.696105|  0.9045879|      0.3922694|null           |FALSE      |
|GO:0009608 |response to symbiont                                                                             | 0.0539059|-6.96062247572051    |-1.68078535651699   | 1.1139434|   -4.556275|  0.8975579|      0.3943270|null           |FALSE      |
|GO:0006885 |regulation of pH                                                                                 | 0.2515610|0.476990163575186    |2.07081820123557    | 1.7558749|  -15.787702|  0.8723212|      0.3950913|null           |FALSE      |
|GO:0090559 |regulation of membrane permeability                                                              | 0.0089843|0.883816973855998    |0.653420370076529   | 0.4771213|   -5.895177|  0.9015602|      0.3950913|null           |FALSE      |
|GO:0010618 |aerenchyma formation                                                                             | 0.0089843|5.74311738729231     |3.65323230355741    | 0.4771213|  -20.935965|  0.9360794|      0.3960626|null           |FALSE      |
|GO:0070667 |negative regulation of mast cell proliferation                                                   | 0.1886708|0.65095109596041     |5.64737841657619    | 1.6334685|   -6.937809|  0.8833570|      0.3989108|null           |FALSE      |
|GO:0010438 |cellular response to sulfur starvation                                                           | 0.0314451|-5.13006645787612    |0.301676600478449   | 0.9030900|  -11.472929|  0.8498319|      0.3994265|null           |FALSE      |
|GO:0002215 |defense response to nematode                                                                     | 0.0179686|-6.39747554061988    |-1.75480918809749   | 0.6989700|   -3.584755|  0.8964457|      0.4008543|null           |FALSE      |
|GO:0009311 |oligosaccharide metabolic process                                                                | 0.4267553|-4.72986435467478    |-5.99088821278792   | 1.9822712|   -3.206762|  0.9355419|      0.4018629|null           |FALSE      |
|GO:0080027 |response to herbivore                                                                            | 0.0808589|-6.97577955981419    |-1.49997306599779   | 1.2787536|  -36.907775|  0.8945834|      0.4067789|null           |FALSE      |
|GO:0002238 |response to molecule of fungal origin                                                            | 0.0853511|-6.76593989957551    |0.299721732670062   | 1.3010300|   -5.783769|  0.8625929|      0.4084989|null           |FALSE      |
|GO:0010730 |negative regulation of hydrogen peroxide biosynthetic process                                    | 0.0044922|1.31429672384796     |6.44271635184764    | 0.3010300|   -5.445886|  0.8992081|      0.4086043|null           |FALSE      |
|GO:0006952 |defense response                                                                                 | 7.2503481|-5.80300961175794    |0.205838427869862   | 3.2081725|  -77.842261|  0.8832043|      0.4100881|null           |FALSE      |
|GO:0097366 |response to bronchodilator                                                                       | 2.0370454|-6.79990556751747    |1.589641115837      | 2.6570559|  -19.329640|  0.8760806|      0.4110011|null           |FALSE      |
|GO:0010221 |negative regulation of vernalization response                                                    | 0.0044922|1.25476684148901     |6.92406051161221    | 0.3010300|  -10.216725|  0.8855898|      0.4116470|null           |FALSE      |
|GO:0000966 |RNA 5&#39;-end processing                                                                            | 0.0808589|-0.816946768228306   |-4.83703138983795   | 1.2787536|  -42.278057|  0.9022833|      0.4120236|null           |FALSE      |
|GO:0007267 |cell-cell signaling                                                                              | 0.3414042|-5.23712642022033    |5.94719551464269    | 1.8864907|  -12.693990|  0.9374458|      0.4151148|null           |FALSE      |
|GO:0035864 |response to potassium ion                                                                        | 0.0269530|-7.26858644364407    |2.74850073681214    | 0.8450980|   -6.384732|  0.9023575|      0.4217509|null           |FALSE      |
|GO:0052544 |defense response by callose deposition in cell wall                                              | 0.0718746|-2.04843468948667    |0.0714596107887345  | 1.2304489|  -56.608053|  0.7869065|      0.4249084|null           |FALSE      |
|GO:1902616 |acyl carnitine transmembrane transport                                                           | 0.2648333|5.45106735863343     |-2.21477636341461   | 1.7708520|   -4.435575|  0.9118470|      0.4263133|null           |FALSE      |
|GO:0009625 |response to insect                                                                               | 0.1482413|-6.89977979392054    |-1.30005192468353   | 1.5314789|  -48.167708|  0.8897974|      0.4269324|null           |FALSE      |
|GO:0009052 |pentose-phosphate shunt,non-oxidative branch                                                     | 0.0359373|1.73110773483091     |-7.26261446780791   | 0.9542425|   -4.247369|  0.9140941|      0.4273002|null           |FALSE      |
|GO:0043603 |cellular amide metabolic process                                                                 | 3.8452900|-1.65146159738895    |-5.32692438258694   | 2.9329808|   -1.610456|  0.8995416|      0.4300273|null           |FALSE      |
|GO:0010412 |mannan metabolic process                                                                         | 0.0134765|-4.16143682099871    |-5.59911887650618   | 0.6020600|   -6.890912|  0.9162539|      0.4309542|null           |FALSE      |
|GO:1904526 |regulation of microtubule binding                                                                | 0.0044922|-4.78463141793161    |7.08395033124768    | 0.3010300|   -5.193765|  0.9212428|      0.4321507|null           |FALSE      |
|GO:1905255 |regulation of RNA binding transcription factor activity                                          | 0.0044922|-4.67765831479174    |7.15559547456318    | 0.3010300|   -6.937809|  0.9249234|      0.4321507|null           |FALSE      |
|GO:0048658 |anther wall tapetum development                                                                  | 0.0853511|5.11668994213366     |3.42601821209417    | 1.3010300|   -6.492437|  0.9003442|      0.4362705|null           |FALSE      |
|GO:0071218 |cellular response to misfolded protein                                                           | 0.1033197|-5.71644438366167    |1.44161527551445    | 1.3802112|   -1.767598|  0.8422972|      0.4371512|null           |FALSE      |
|GO:0010942 |positive regulation of cell death                                                                | 0.1527335|-1.93092241898537    |6.35229493481476    | 1.5440680|  -10.466450|  0.8803558|      0.4414794|null           |FALSE      |
|GO:0036371 |protein localization to T-tubule                                                                 | 0.0314451|4.96371585077453     |-1.65242430830559   | 0.9030900|  -16.616139|  0.9229566|      0.4440885|null           |FALSE      |
|GO:0051258 |protein polymerization                                                                           | 0.1347648|6.55046536712325     |0.707316458842153   | 1.4913617|   -1.407216|  0.9405531|      0.4473070|null           |FALSE      |
|GO:0032527 |protein exit from endoplasmic reticulum                                                          | 0.0359373|5.13808623144101     |-1.92421920778915   | 0.9542425|  -11.527071|  0.9153755|      0.4479977|null           |FALSE      |
|GO:0001101 |response to acid chemical                                                                        | 1.7923723|-6.87122672508893    |1.66904004109387    | 2.6020600|   -5.560432|  0.8775527|      0.4485887|null           |FALSE      |
|GO:0007492 |endoderm development                                                                             | 0.0134765|5.76980736149969     |3.24608050580759    | 0.6020600|   -1.511509|  0.9443035|      0.4499787|null           |FALSE      |
|GO:0032232 |negative regulation of actin filament bundle assembly                                            | 0.0314451|0.80409035910113     |6.03774783558834    | 0.9030900|   -5.942017|  0.8856321|      0.4530513|null           |FALSE      |
|GO:0002213 |defense response to insect                                                                       | 0.0943354|-6.26379585966241    |-1.27355076207675   | 1.3424227|   -1.890326|  0.8840647|      0.4532505|null           |FALSE      |
|GO:0000914 |phragmoplast assembly                                                                            | 0.0179686|6.59266445902253     |0.905116821533742   | 0.6989700|   -3.761422|  0.9483564|      0.4564333|null           |FALSE      |
|GO:0002230 |positive regulation of defense response to virus by host                                         | 0.0224608|0.284379530770929    |7.51317241999159    | 0.7781513|  -23.982792|  0.8926696|      0.4583337|null           |FALSE      |
|GO:0008202 |steroid metabolic process                                                                        | 0.4582004|2.07136748011286     |-5.17906384937228   | 2.0128372|  -15.171247|  0.8861342|      0.4697108|null           |FALSE      |
|GO:0009314 |response to radiation                                                                            | 3.1400207|-7.30919599289784    |0.389898206977731   | 2.8450980|   -2.104823|  0.8981983|      0.4702593|null           |FALSE      |
|GO:0006898 |receptor-mediated endocytosis                                                                    | 0.1976551|5.14087072107399     |-2.23872227793681   | 1.6532125|   -4.752375|  0.9219814|      0.4736740|null           |FALSE      |
|GO:0010898 |positive regulation of triglyceride catabolic process                                            | 0.1161545|-1.54847742344211    |6.46625451567221    | 1.4149733|   -7.797760|  0.8831707|      0.4740009|null           |FALSE      |
|GO:0071398 |cellular response to fatty acid                                                                  | 0.4402318|-6.18623457735453    |1.94185598568289    | 1.9956352|  -23.828574|  0.8246805|      0.4760094|null           |FALSE      |
|GO:0009727 |detection of ethylene stimulus                                                                   | 0.0134765|-7.07164494176693    |3.32671874824269    | 0.6020600|  -18.408248|  0.8957870|      0.4765194|null           |FALSE      |
|GO:0010891 |negative regulation of sequestering of triglyceride                                              | 0.0044922|2.09886181579533     |6.46816837225529    | 0.3010300|   -7.797760|  0.8870658|      0.4769945|null           |FALSE      |
|GO:0019450 |L-cysteine catabolic process to pyruvate                                                         | 0.0044922|0.48017912818578     |-3.7503870624331    | 0.3010300|   -7.982055|  0.8792809|      0.4780209|null           |FALSE      |
|GO:0071732 |cellular response to nitric oxide                                                                | 0.0179686|-6.37491610875536    |2.70395525505677    | 0.6989700|  -13.088987|  0.8549169|      0.4811921|null           |FALSE      |
|GO:0019323 |pentose catabolic process                                                                        | 0.0224608|-0.738350314330328   |-3.53321077047901   | 0.7781513|   -2.936443|  0.8996926|      0.4826138|null           |FALSE      |
|GO:0048587 |regulation of short-day photoperiodism,flowering                                                 | 0.0494138|0.0968104012020863   |7.32919805452626    | 1.0791812|   -2.668406|  0.8927762|      0.4852974|null           |FALSE      |
|GO:0051607 |defense response to virus                                                                        | 0.2246081|-6.28596211290728    |-0.998703375224674  | 1.7075702|   -2.216937|  0.8763176|      0.4865194|null           |FALSE      |
|GO:0070914 |UV-damage excision repair                                                                        | 0.0089843|-3.62723950298701    |-1.74271593034529   | 0.4771213|   -2.695172|  0.8167975|      0.4911276|null           |FALSE      |
|GO:0033292 |T-tubule organization                                                                            | 0.0716381|5.4824228117652      |2.48043494068836    | 1.2041200|  -16.616139|  0.8893992|      0.4928611|null           |FALSE      |
|GO:0016024 |CDP-diacylglycerol biosynthetic process                                                          | 0.0808589|2.58378545575652     |-6.16039773155476   | 1.2787536|   -2.501440|  0.8653858|      0.5012011|null           |FALSE      |
|GO:0070462 |plus-end specific microtubule depolymerization                                                   | 0.0224608|6.50415626488132     |0.282771163886974   | 0.7781513|  -11.321631|  0.9351941|      0.5018801|null           |FALSE      |
|GO:0097237 |cellular response to toxic substance                                                             | 0.8355420|-6.07399380015953    |1.72582264733466    | 2.2718416|  -16.933808|  0.8372070|      0.5080142|null           |FALSE      |
|GO:0048312 |intracellular distribution of mitochondria                                                       | 0.0044922|5.57838160278155     |-0.708840285094388  | 0.3010300|   -1.511515|  0.9114772|      0.5090434|null           |FALSE      |
|GO:0009451 |RNA modification                                                                                 | 1.4105386|-1.04409768432039    |-5.61958326355288   | 2.4983106|  -24.705821|  0.8733816|      0.5091922|null           |FALSE      |
|GO:1901726 |negative regulation of histone deacetylase activity                                              | 0.0494138|0.341411029665645    |6.07167413384728    | 1.0791812|   -8.720248|  0.8669664|      0.5094696|null           |FALSE      |
|GO:0009992 |cellular water homeostasis                                                                       | 0.0089843|2.00944853152021     |0.795742793715992   | 0.4771213|   -1.388395|  0.8598607|      0.5104375|null           |FALSE      |
|GO:0018107 |peptidyl-threonine phosphorylation                                                               | 0.0628903|-0.898801470362119   |-6.91509663253815   | 1.1760913|   -5.188056|  0.9090496|      0.5112764|null           |FALSE      |
|GO:0051090 |regulation of DNA-binding transcription factor activity                                          | 0.0539059|-1.9537082388933     |7.04350639171783    | 1.1139434|   -1.439312|  0.8947475|      0.5127988|null           |FALSE      |
|GO:0036309 |protein localization to M-band                                                                   | 0.2713265|5.17791363212752     |-1.69345591462304   | 1.7853298|  -16.616139|  0.9088293|      0.5168809|null           |FALSE      |
|GO:0001841 |neural tube formation                                                                            | 0.0134765|5.42104426518251     |3.95527263651604    | 0.6020600|   -6.892856|  0.8946559|      0.5181634|null           |FALSE      |
|GO:0006509 |membrane protein ectodomain proteolysis                                                          | 0.4761691|-1.87591260783136    |-6.74665331217099   | 2.0293838|  -12.551149|  0.9309484|      0.5203722|null           |FALSE      |
|GO:0060776 |simple leaf morphogenesis                                                                        | 0.0224608|5.20768169544924     |3.67193108710772    | 0.7781513|   -2.956538|  0.9063481|      0.5240669|null           |FALSE      |
|GO:0060291 |long-term synaptic potentiation                                                                  | 0.0044922|-0.00922519439248941 |3.50629612262138    | 0.3010300|  -11.130410|  0.8496862|      0.5247898|null           |FALSE      |
|GO:0032212 |positive regulation of telomere maintenance via telomerase                                       | 0.0044922|-2.32638027936424    |7.06938568427949    | 0.3010300|   -1.707599|  0.8989334|      0.5293887|null           |FALSE      |
|GO:0019752 |carboxylic acid metabolic process                                                                | 4.3214591|0.966710131622377    |-4.3510083797751    | 2.9836263|   -8.261766|  0.8686713|      0.5297900|null           |FALSE      |
|GO:0009914 |hormone transport                                                                                | 0.4447240|2.18300769522972     |0.635531993480009   | 2.0000000|  -12.890690|  0.8141414|      0.5303141|null           |FALSE      |
|GO:0005977 |glycogen metabolic process                                                                       | 0.0853511|-3.92220151484726    |-5.90513437631914   | 1.3010300|   -2.633283|  0.9123031|      0.5313878|null           |FALSE      |
|GO:2000280 |regulation of root development                                                                   | 0.3593729|-1.42085700496769    |5.1991572176851     | 1.9084850|  -50.486827|  0.8979095|      0.5325899|null           |FALSE      |
|GO:0033007 |negative regulation of mast cell activation involved in immune response                          | 0.0898432|0.0461918210828196   |6.38281621881756    | 1.3222193|   -6.937809|  0.8386662|      0.5331877|null           |FALSE      |
|GO:0038018 |Wnt receptor catabolic process                                                                   | 0.0044922|0.506094393023209    |4.37945365498043    | 0.3010300|   -4.390571|  0.8062319|      0.5331877|null           |FALSE      |
|GO:0009638 |phototropism                                                                                     | 0.1123040|-7.11292109726562    |-0.653677195768085  | 1.4149733|   -4.027267|  0.8923521|      0.5346332|null           |FALSE      |
|GO:0035518 |histone H2A monoubiquitination                                                                   | 0.0089843|-2.75252881257923    |-6.87955680921461   | 0.4771213|   -1.807329|  0.9287301|      0.5364766|null           |FALSE      |
|GO:0009617 |response to bacterium                                                                            | 2.1113158|-6.60366993229154    |-0.619613212518776  | 2.6730209|  -17.072972|  0.8624122|      0.5453319|null           |FALSE      |
|GO:1901684 |arsenate ion transmembrane transport                                                             | 0.0224608|5.92324792141206     |-1.95801917989971   | 0.7781513|  -13.199874|  0.9160531|      0.5470253|null           |FALSE      |
|GO:0010016 |shoot system morphogenesis                                                                       | 0.7097615|5.18198952868377     |3.84229133575739    | 2.2013971|   -9.930844|  0.8823966|      0.5474395|null           |FALSE      |
|GO:0009961 |response to 1-aminocyclopropane-1-carboxylic acid                                                | 0.0179686|-7.03663597675668    |3.05121350230284    | 0.6989700|   -3.168515|  0.8893669|      0.5514906|null           |FALSE      |
|GO:0071731 |response to nitric oxide                                                                         | 0.0269530|-7.11004518577441    |2.86908994324817    | 0.8450980|   -3.940569|  0.8903598|      0.5514906|null           |FALSE      |
|GO:0044403 |biological process involved in symbiotic interaction                                             | 0.4312475|-7.12708396170035    |-3.15218378961793   | 1.9867717|   -9.564115|  0.9469536|      0.5582824|null           |FALSE      |
|GO:0002239 |response to oomycetes                                                                            | 0.4177710|-6.77559883994916    |-1.00757383316427   | 1.9731279|  -15.875443|  0.8805188|      0.5594759|null           |FALSE      |
|GO:0006696 |ergosterol biosynthetic process                                                                  | 0.0479164|1.62696714126467     |-5.20226527047584   | 1.0413927|   -3.585889|  0.8411812|      0.5602955|null           |FALSE      |
|GO:0006468 |protein phosphorylation                                                                          | 5.0671578|-1.09494534080029    |-6.29466983127573   | 3.0526939|  -60.750918|  0.8722791|      0.5616227|null           |FALSE      |
|GO:0048509 |regulation of meristem development                                                               | 0.2964826|-1.59978256263173    |5.30806378274139    | 1.8260748|  -27.802412|  0.8993770|      0.5634440|null           |FALSE      |
|GO:1902265 |abscisic acid homeostasis                                                                        | 0.0089843|0.971734200595401    |0.776441290772236   | 0.4771213|   -3.059058|  0.8975045|      0.5661740|null           |FALSE      |
|GO:0099402 |plant organ development                                                                          | 4.5236063|5.01987302292313     |3.94023744287203    | 3.0034605|   -7.443147|  0.8751553|      0.5741508|null           |FALSE      |
|GO:0016099 |monoterpenoid biosynthetic process                                                               | 0.0044922|3.60224935282567     |-5.6664488746732    | 0.3010300|  -44.932002|  0.8910278|      0.5751028|null           |FALSE      |
|GO:0009644 |response to high light intensity                                                                 | 0.2785140|-7.90479369380161    |0.058093904260743   | 1.7993405|   -4.900573|  0.9134787|      0.5759002|null           |FALSE      |
|GO:0080003 |thalianol metabolic process                                                                      | 0.0089843|2.63951096245644     |-5.24025178205792   | 0.4771213|  -23.789992|  0.8836863|      0.5772455|null           |FALSE      |
|GO:0097167 |circadian regulation of translation                                                              | 0.0044922|3.0196882557362      |5.1448281752013     | 0.3010300|   -3.599961|  0.8699947|      0.5780757|null           |FALSE      |
|GO:0098660 |inorganic ion transmembrane transport                                                            | 1.9810431|5.54785375368248     |-1.91501195522481   | 2.6454223|  -10.737609|  0.8891466|      0.5817700|null           |FALSE      |
|GO:0007584 |response to nutrient                                                                             | 0.0583981|-6.81704788522583    |0.72787815524019    | 1.1461280|   -1.321440|  0.8768802|      0.5817840|null           |FALSE      |
|GO:0032508 |DNA duplex unwinding                                                                             | 0.2919905|6.31029910202645     |0.809795350117837   | 1.8195439|   -2.916109|  0.9387474|      0.5830189|null           |FALSE      |
|GO:0071229 |cellular response to acid chemical                                                               | 0.2740218|-6.30836104187756    |2.02671530285101    | 1.7923917|  -18.646332|  0.8483261|      0.5845806|null           |FALSE      |
|GO:0007165 |signal transduction                                                                              | 8.3913571|-3.48768549696694    |3.14694780748662    | 3.2716093| -130.964462|  0.7241077|      0.5893442|null           |FALSE      |
|GO:0071446 |cellular response to salicylic acid stimulus                                                     | 0.2605453|-6.18909363065602    |2.14848210463845    | 1.7708520|  -20.847283|  0.8325862|      0.5943078|null           |FALSE      |
|GO:0009664 |plant-type cell wall organization                                                                | 0.6693320|6.02922624683468     |0.68195456579232    | 2.1760913|  -11.167694|  0.9312716|      0.5956427|null           |FALSE      |
|GO:0009877 |nodulation                                                                                       | 0.0988275|5.60101449605812     |3.71421250476112    | 1.3617278|   -4.911798|  0.9248941|      0.5966461|null           |FALSE      |
|GO:0036376 |sodium ion export across plasma membrane                                                         | 0.0554033|5.79257480870064     |-1.8421560509349    | 1.1139434|  -11.218898|  0.9064655|      0.5981993|null           |FALSE      |
|GO:0003341 |cilium movement                                                                                  | 0.0059895|5.52463768180414     |-4.43702056629655   | 0.3010300|  -13.254910|  0.9521397|      0.5985002|null           |FALSE      |
|GO:0046938 |phytochelatin biosynthetic process                                                               | 0.0134765|0.773861772577023    |-6.35750558752569   | 0.6020600|   -5.445886|  0.8924920|      0.6011438|null           |FALSE      |
|GO:0042177 |negative regulation of protein catabolic process                                                 | 0.0134765|1.46961641658569     |6.24775297284435    | 0.6020600|   -2.832164|  0.8985487|      0.6021055|null           |FALSE      |
|GO:0007263 |nitric oxide mediated signal transduction                                                        | 0.0089843|-3.81664830846604    |3.60806776969927    | 0.4771213|   -6.441332|  0.8335000|      0.6046800|null           |FALSE      |
|GO:0006165 |nucleoside diphosphate phosphorylation                                                           | 0.3593729|0.483782744040257    |-5.44204366915297   | 1.9084850|   -3.191198|  0.8484215|      0.6053333|null           |FALSE      |
|GO:1901255 |nucleotide-excision repair involved in interstrand cross-link repair                             | 0.0044922|-3.19351940680774    |-1.91988608818598   | 0.3010300|   -1.955409|  0.8344001|      0.6058423|null           |FALSE      |
|GO:0044262 |cellular carbohydrate metabolic process                                                          | 2.1068236|-4.31991695836957    |-6.00528466099857   | 2.6720979|   -2.920548|  0.9089215|      0.6073647|null           |FALSE      |
|GO:0070301 |cellular response to hydrogen peroxide                                                           | 0.0449216|-5.82385320238816    |1.68034586480937    | 1.0413927|  -24.083938|  0.8375083|      0.6082701|null           |FALSE      |
|GO:0016045 |detection of bacterium                                                                           | 0.0134765|-7.10929147605326    |-1.93730258901782   | 0.6020600|  -13.150286|  0.8940235|      0.6121094|null           |FALSE      |
|GO:0098703 |calcium ion import across plasma membrane                                                        | 0.0170702|3.49355110854603     |-0.686098836041214  | 0.6020600|  -11.130410|  0.8068935|      0.6124892|null           |FALSE      |
|GO:0010376 |stomatal complex formation                                                                       | 0.0224608|5.38907163659156     |3.71293973967371    | 0.7781513|   -1.310240|  0.9002650|      0.6165396|null           |FALSE      |
|GO:0046345 |abscisic acid catabolic process                                                                  | 0.0269530|2.04297175850148     |-4.75755501937756   | 0.8450980|  -40.691415|  0.8282908|      0.6194937|null           |FALSE      |
|GO:0080024 |indolebutyric acid metabolic process                                                             | 0.0224608|0.222410979822831    |-2.72403913078784   | 0.7781513|  -12.490669|  0.7838030|      0.6227725|null           |FALSE      |
|GO:0010150 |leaf senescence                                                                                  | 0.5076142|5.01065340919018     |3.79264330767496    | 2.0569049|   -3.287542|  0.8924637|      0.6244288|null           |FALSE      |
|GO:1903553 |positive regulation of extracellular exosome assembly                                            | 0.0044922|-2.9978354540893     |6.98247498152236    | 0.3010300|   -1.511515|  0.9045147|      0.6254462|null           |FALSE      |
|GO:0045926 |negative regulation of growth                                                                    | 0.1751943|0.984300986473375    |5.66567897041249    | 1.6020600|   -7.549587|  0.8853934|      0.6297772|null           |FALSE      |
|GO:0033473 |indoleacetic acid conjugate metabolic process                                                    | 0.0044922|0.359782248166804    |-2.92493228488283   | 0.3010300|   -6.508933|  0.8005614|      0.6304322|null           |FALSE      |
|GO:1903507 |negative regulation of nucleic acid-templated transcription                                      | 1.0556579|0.150500648416157    |5.69460196998637    | 2.3729120|   -5.889535|  0.8582344|      0.6336059|null           |FALSE      |
|GO:0031640 |killing of cells of other organism                                                               | 1.3072189|-7.16483527702922    |-3.07039601470595   | 2.4653829|   -1.511515|  0.9135847|      0.6338299|null           |FALSE      |
|GO:0000025 |maltose catabolic process                                                                        | 0.0089843|-4.43130460217352    |-5.50904621006544   | 0.4771213|   -6.402236|  0.9145029|      0.6340028|null           |FALSE      |
|GO:0060465 |pharynx development                                                                              | 1.4053739|5.13446812797228     |4.12262717819466    | 2.4927604|   -1.511515|  0.8875713|      0.6351702|null           |FALSE      |
|GO:0015700 |arsenite transport                                                                               | 0.0224608|5.93316713699228     |-2.1293250751625    | 0.7781513|  -15.521384|  0.9357928|      0.6364426|null           |FALSE      |
|GO:0031347 |regulation of defense response                                                                   | 1.2308522|-0.328020473508236   |6.0713048266626     | 2.4393327|  -12.817341|  0.8607194|      0.6384662|null           |FALSE      |
|GO:0003283 |atrial septum development                                                                        | 1.6401201|4.99724017721641     |4.07939756544877    | 3.2564772|   -3.375052|  0.8687517|      0.6408854|null           |FALSE      |
|GO:0042631 |cellular response to water deprivation                                                           | 0.1707021|-5.96916319746578    |1.0088626572304     | 1.5910646|   -8.324906|  0.7858586|      0.6436302|null           |FALSE      |
|GO:0010106 |cellular response to iron ion starvation                                                         | 0.0359373|-3.80005971481052    |0.886210546160771   | 0.9542425|   -2.715632|  0.7544693|      0.6452536|null           |FALSE      |
|GO:1905034 |regulation of antifungal innate immune response                                                  | 0.0089843|0.537271907553654    |7.64943341133016    | 0.4771213|   -6.763737|  0.8950847|      0.6459136|null           |FALSE      |
|GO:0010105 |negative regulation of ethylene-activated signaling pathway                                      | 0.0898432|0.492428082403706    |6.28451105645917    | 1.3222193|  -15.525570|  0.8390935|      0.6505621|null           |FALSE      |
|GO:1902290 |positive regulation of defense response to oomycetes                                             | 0.0449216|-0.619246547322421   |7.0565359529804     | 1.0413927|  -12.360237|  0.8686328|      0.6512180|null           |FALSE      |
|GO:0044255 |cellular lipid metabolic process                                                                 | 3.4050582|3.0224353098249      |-5.86964448547349   | 2.8802418|   -3.248190|  0.8748410|      0.6525364|null           |FALSE      |
|GO:0072498 |embryonic skeletal joint development                                                             | 1.8732312|5.10940448271605     |4.03093728062706    | 2.6211763|   -1.511515|  0.8845939|      0.6529264|null           |FALSE      |
|GO:0051513 |regulation of monopolar cell growth                                                              | 0.0224608|-2.66239782601804    |6.43110970442071    | 0.7781513|   -1.473899|  0.8994634|      0.6532683|null           |FALSE      |
|GO:0042742 |defense response to bacterium                                                                    | 1.6216702|-6.29655717034464    |-0.592277065374495  | 2.5587086|  -14.961942|  0.8483041|      0.6545694|null           |FALSE      |
|GO:0012502 |induction of programmed cell death                                                               | 0.0044922|-2.96249290356003    |7.20299631604374    | 0.3010300|   -5.115042|  0.9034924|      0.6549617|null           |FALSE      |
|GO:1902042 |negative regulation of extrinsic apoptotic signaling pathway via death domain receptors          | 0.0044922|0.680138894362733    |6.69709663942723    | 0.3010300|   -5.153496|  0.8607304|      0.6549617|null           |FALSE      |
|GO:0009852 |auxin catabolic process                                                                          | 0.0089843|0.591816186296342    |-0.286039654689895  | 0.4771213|   -9.060693|  0.8366526|      0.6550946|null           |FALSE      |
|GO:0000304 |response to singlet oxygen                                                                       | 0.0583981|-6.95491294248736    |1.8809019823314     | 1.1461280|  -13.389297|  0.8782887|      0.6554547|null           |FALSE      |
|GO:2000026 |regulation of multicellular organismal development                                               | 1.5677643|-1.11927286562136    |5.07458312826737    | 2.5440680|  -13.816111|  0.8549052|      0.6593517|null           |FALSE      |
|GO:0046482 |para-aminobenzoic acid metabolic process                                                         | 0.0179686|0.32086569309814     |-4.594312719354     | 0.6989700|  -17.998248|  0.8763834|      0.6615462|null           |FALSE      |
|GO:1901599 |(-)-pinoresinol biosynthetic process                                                             | 0.0089843|1.25688166037821     |-4.79428038364358   | 0.4771213|   -2.589659|  0.8653362|      0.6615462|null           |FALSE      |
|GO:0030150 |protein import into mitochondrial matrix                                                         | 0.1257805|5.53504202556525     |-0.943669898837932  | 1.4623980|   -7.293749|  0.8749276|      0.6622637|null           |FALSE      |
|GO:0032870 |cellular response to hormone stimulus                                                            | 4.2316158|-6.17054378563895    |1.49527646327916    | 2.9745117|  -14.968531|  0.8018501|      0.6623081|null           |FALSE      |
|GO:0010930 |negative regulation of auxin mediated signaling pathway                                          | 0.0134765|0.77125352334918     |6.52491456025655    | 0.6020600|   -4.968536|  0.8606039|      0.6682671|null           |FALSE      |
|GO:0006654 |phosphatidic acid biosynthetic process                                                           | 0.0269530|2.71805187284431     |-6.13269912506018   | 0.8450980|   -1.576801|  0.8746047|      0.6687375|null           |FALSE      |
|GO:0045338 |farnesyl diphosphate metabolic process                                                           | 0.0898432|2.98467399857656     |-6.08748802195093   | 1.3222193|  -28.998426|  0.8694725|      0.6735153|null           |FALSE      |
|GO:0009801 |cinnamic acid ester metabolic process                                                            | 0.0089843|1.9240582726004      |-4.06627861991197   | 0.4771213|  -13.089991|  0.9040753|      0.6748672|null           |FALSE      |
|GO:1903711 |spermidine transmembrane transport                                                               | 0.2061403|5.58309323879068     |-2.07777865182741   | 1.6627578|   -6.503290|  0.8993891|      0.6759740|null           |FALSE      |
|GO:0071368 |cellular response to cytokinin stimulus                                                          | 0.3593729|-6.01810259116035    |2.01812433924568    | 1.9084850|  -11.780061|  0.8400378|      0.6812564|null           |FALSE      |
|GO:0050829 |defense response to Gram-negative bacterium                                                      | 0.0988275|-6.37487805885133    |-1.23392365167515   | 1.3617278|   -5.405397|  0.8783485|      0.6840535|null           |FALSE      |
|GO:1900449 |regulation of glutamate receptor signaling pathway                                               | 0.1735903|-0.245791748763254   |6.51564070760175    | 1.5682017|   -5.153496|  0.8598522|      0.6858401|null           |FALSE      |
|GO:0010310 |regulation of hydrogen peroxide metabolic process                                                | 0.0539059|-1.52042479436821    |7.34827297459386    | 1.1139434|   -1.409169|  0.9122973|      0.6872518|null           |FALSE      |
|GO:0030435 |sporulation resulting in formation of a cellular spore                                           | 0.1134757|5.5266447686767      |3.5329073374612     | 1.3802112|   -3.232252|  0.8961892|      0.6894313|null           |FALSE      |
|GO:0010030 |positive regulation of seed germination                                                          | 0.0898432|-2.20958879221526    |6.07820381031407    | 1.3222193|  -13.622275|  0.8600820|      0.6895163|null           |FALSE      |
|GO:0005983 |starch catabolic process                                                                         | 0.0763667|-3.56926145070783    |-5.52671436048728   | 1.2552725|   -5.865772|  0.8948552|      0.6912926|null           |FALSE      |
|GO:0071266 |&#39;de novo&#39; L-methionine biosynthetic process                                                      | 0.0044922|1.47513565240432     |-5.80760803894222   | 0.3010300|   -7.778089|  0.8782424|      0.6914940|null           |FALSE      |
|GO:0071281 |cellular response to iron ion                                                                    | 0.0089843|-6.33146528068404    |2.90571401610628    | 0.4771213|  -11.562767|  0.8728597|      0.6952859|null           |FALSE      |

[1] &quot;only_in_posi_GO.txt&quot;


|TermID     |Name                                                   | Frequency|      PlotX|      PlotY|  LogSize|     Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:------------------------------------------------------|---------:|----------:|----------:|--------:|---------:|----------:|--------------:|:--------------|:----------|
|GO:0005983 |starch catabolic process                               | 0.0763667| -3.8813652| -5.3358250| 1.255273| -2.055820|  0.9140657|      0.0000000|null           |FALSE      |
|GO:0030174 |regulation of DNA-dependent DNA replication initiation | 0.0179686|  0.4265201|  5.5465242| 0.698970| -1.307768|  0.9472392|      0.0000000|null           |FALSE      |
|GO:0046686 |response to cadmium ion                                | 0.3054670|  4.8594911| -3.9230928| 1.838849| -1.754551|  0.8683807|      0.0000000|null           |FALSE      |
|GO:0034721 |histone H3-K4 demethylation,trimethyl-H3-K4-specific   | 0.0404295| -5.4557157| -0.5292386| 1.000000| -1.884385|  0.9145322|      0.1526367|null           |FALSE      |
|GO:0071333 |cellular response to glucose stimulus                  | 0.0718746|  5.2493877| -1.1536548| 1.230449| -1.582738|  0.7989313|      0.2725221|null           |FALSE      |

[1] &quot;only_in_zmar_GO.txt&quot;


|TermID     |Name                                                       | Frequency|PlotX              |PlotY             |   LogSize|      Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:----------------------------------------------------------|---------:|:------------------|:-----------------|---------:|----------:|----------:|--------------:|:--------------|:----------|
|GO:0010157 |response to chlorate                                       | 0.0044922|4.11727482676329   |-4.31307727133405 | 0.3010300|  -5.513515|  0.9765257|      0.0000000|null           |FALSE      |
|GO:0015770 |sucrose transport                                          | 0.0988275|-2.2016111679016   |7.31804822593944  | 1.3617278|  -1.318982|  0.9441763|      0.0000000|null           |FALSE      |
|GO:0048281 |inflorescence morphogenesis                                | 0.0134765|-6.15738581104616  |-2.02338408241478 | 0.6020600|  -4.588824|  0.9216674|      0.0000000|null           |FALSE      |
|GO:0051603 |proteolysis involved in cellular protein catabolic process | 2.6458829|5.07318089917103   |3.548550747514    | 2.7708520| -23.445501|  0.7934847|      0.0000000|null           |FALSE      |
|GO:0032197 |transposition,RNA-mediated                                 | 0.0044922|6.45267081405976   |-2.47326679908214 | 0.3010300|  -2.708915|  0.9734785|      0.0243997|null           |FALSE      |
|GO:0090156 |cellular sphingolipid homeostasis                          | 0.0134765|-0.719385498441574 |0.202855403794292 | 0.6020600|  -4.494330|  0.9214548|      0.0265366|null           |FALSE      |
|GO:0046785 |microtubule polymerization                                 | 0.0673824|-5.4735268155328   |3.54033713403236  | 1.2041200| -13.774183|  0.8741245|      0.0304424|null           |FALSE      |
|GO:1901332 |negative regulation of lateral root development            | 0.0314451|-3.0872975533626   |-5.28366704358069 | 0.9030900|  -2.847257|  0.9454833|      0.0999311|null           |FALSE      |
|GO:0010286 |heat acclimation                                           | 0.2650375|1.54610909278461   |-5.55548969412341 | 1.7781513|  -2.016484|  0.9263497|      0.1180921|null           |FALSE      |
|GO:0080156 |mitochondrial mRNA modification                            | 0.1347648|2.9569750418804    |6.55563757076898  | 1.4913617|  -7.414447|  0.8668707|      0.1671223|null           |FALSE      |
|GO:0006537 |glutamate biosynthetic process                             | 0.0179686|5.02213287863734   |0.883914374352552 | 0.6989700|  -1.818101|  0.9002232|      0.1746337|null           |FALSE      |
|GO:0048194 |Golgi vesicle budding                                      | 0.0179686|-4.15434920378404  |5.15500734302651  | 0.6989700|  -1.775108|  0.8332491|      0.2966091|null           |FALSE      |
|GO:1990019 |protein storage vacuole organization                       | 0.0314451|-5.65716661625443  |4.30570174104844  | 0.9030900|  -8.591400|  0.8788757|      0.3074104|null           |FALSE      |
|GO:0006278 |RNA-dependent DNA biosynthetic process                     | 0.2605453|3.92166854490806   |4.67899894103106  | 1.7708520|  -2.761258|  0.8356628|      0.3399747|null           |FALSE      |
|GO:0090709 |regulation of timing of plant organ formation              | 0.0089843|-2.63520669673627  |-5.76789500570929 | 0.4771213|  -1.857388|  0.9468768|      0.3711372|null           |FALSE      |
|GO:0032989 |cellular component morphogenesis                           | 0.2156237|-5.80507880412366  |1.11186310798387  | 1.6901961|  -4.508650|  0.7980114|      0.4158062|null           |FALSE      |
|GO:0090305 |nucleic acid phosphodiester bond hydrolysis                | 1.6171780|3.61075902694305   |5.74703474854484  | 2.5575072|  -1.818070|  0.8442740|      0.4238293|null           |FALSE      |
|GO:0006624 |vacuolar protein processing                                | 0.0359373|6.00977521179013   |2.96902057026307  | 0.9542425|  -9.895786|  0.8501799|      0.4917335|null           |FALSE      |
|GO:0010090 |trichome morphogenesis                                     | 0.2830062|-6.48251052398473  |-1.51683278420309 | 1.8061800|  -4.097280|  0.8796241|      0.5409970|null           |FALSE      |
|GO:0045332 |phospholipid translocation                                 | 0.0583981|-3.36594051217474  |4.04518049010372  | 1.1461280|  -1.546385|  0.7802830|      0.5429416|null           |FALSE      |
|GO:0009408 |response to heat                                           | 1.1230403|1.87565292947954   |-5.30459277047393 | 2.3996737|  -1.548177|  0.9252721|      0.6700248|null           |FALSE      |

[1] &quot;only_in_zmuel_GO.txt&quot;


|TermID     |Name                                                                              | Frequency|PlotX              |PlotY             |   LogSize|     Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:---------------------------------------------------------------------------------|---------:|:------------------|:-----------------|---------:|---------:|----------:|--------------:|:--------------|:----------|
|GO:0010222 |stem vascular tissue pattern formation                                            | 0.0269530|1.07283805315438   |6.05337856756737  | 0.8450980| -5.047945|  0.8692095|      0.0000000|null           |FALSE      |
|GO:0040007 |growth                                                                            | 1.7025291|-4.20269087624433  |4.45541099264997  | 2.5797836| -4.373448|  1.0000000|      0.0000000|null           |FALSE      |
|GO:0043007 |maintenance of rDNA                                                               | 0.0134765|-4.79194830772234  |-5.2652738185036  | 0.6020600| -3.318770|  0.8595533|      0.0000000|null           |FALSE      |
|GO:0090284 |positive regulation of protein glycosylation in Golgi                             | 0.3503886|5.52434946077459   |-1.66001622766217 | 1.8976271| -5.506616|  0.7602859|      0.0000000|null           |FALSE      |
|GO:0046459 |short-chain fatty acid metabolic process                                          | 0.0134765|-6.43997106107652  |2.14154081682534  | 0.6020600| -1.740640|  0.9820174|      0.0647135|null           |FALSE      |
|GO:2001240 |negative regulation of extrinsic apoptotic signaling pathway in absence of ligand | 0.0044922|3.2825539658983    |-5.19493425610034 | 0.3010300| -2.928272|  0.8010540|      0.1398481|null           |FALSE      |
|GO:1900049 |regulation of histone exchange                                                    | 0.0089843|6.00399214585632   |1.96681723424834  | 0.4771213| -4.140205|  0.8866402|      0.1463228|null           |FALSE      |
|GO:2000114 |regulation of establishment of cell polarity                                      | 0.0224608|3.13346841502438   |1.22873162742566  | 0.7781513| -3.856587|  0.8804434|      0.1558620|null           |FALSE      |
|GO:0008347 |glial cell migration                                                              | 0.0044922|-0.468516669030391 |5.81781912407268  | 0.3010300| -4.650340|  0.8597621|      0.2230397|null           |FALSE      |
|GO:0080119 |ER body organization                                                              | 0.0269530|-5.4646802946657   |-3.23577417983695 | 0.8450980| -2.146026|  0.8761290|      0.2757630|null           |FALSE      |
|GO:0007030 |Golgi organization                                                                | 0.1931629|-5.01799431464012  |-4.09426436776298 | 1.6434527| -1.799156|  0.8646678|      0.3265117|null           |FALSE      |
|GO:0034260 |negative regulation of GTPase activity                                            | 0.5039777|2.69066877112455   |-2.79123080946892 | 2.0530784| -1.793183|  0.7678298|      0.3612841|null           |FALSE      |
|GO:0006336 |DNA replication-independent chromatin assembly                                    | 0.0404295|-4.18163079321898  |-5.4166316965584  | 1.0000000| -1.395115|  0.8581688|      0.4336933|null           |FALSE      |
|GO:0090303 |positive regulation of wound healing                                              | 1.1320246|4.71115835003647   |-3.62259419254947 | 2.4031205| -4.387055|  0.7462134|      0.5237917|null           |FALSE      |
|GO:0046328 |regulation of JNK cascade                                                         | 0.0185302|4.25596051700435   |-5.380562569243   | 0.6989700| -2.672143|  0.8018145|      0.6029437|null           |FALSE      |

[1] &quot;only_seagrasses_GO.txt&quot;


|TermID     |Name                                                                              | Frequency|PlotX               |PlotY              |   LogSize|      Value| Uniqueness| Dispensability|Representative |Eliminated |
|:----------|:---------------------------------------------------------------------------------|---------:|:-------------------|:------------------|---------:|----------:|----------:|--------------:|:--------------|:----------|
|GO:0010022 |meristem determinacy                                                              | 0.1167962|5.10538257972174    |3.93058055852156   | 1.4313638|  -2.675579|  0.8822107|      0.0000000|null           |FALSE      |
|GO:0050982 |detection of mechanical stimulus                                                  | 0.0134765|5.48954453449006    |-3.45455976546847  | 0.6020600|  -5.431629|  0.9623703|      0.0000000|null           |FALSE      |
|GO:0051603 |proteolysis involved in cellular protein catabolic process                        | 2.6458829|-4.97011082355978   |4.87461685653077   | 2.7708520| -11.456833|  0.8231088|      0.0000000|null           |FALSE      |
|GO:1900049 |regulation of histone exchange                                                    | 0.0089843|1.27510476399194    |-6.37230497955094  | 0.4771213|  -5.042366|  0.9156042|      0.0000000|null           |FALSE      |
|GO:0032197 |transposition,RNA-mediated                                                        | 0.0044922|-0.0363631710054332 |8.38289031877423   | 0.3010300|  -9.168511|  0.9797437|      0.0243997|null           |FALSE      |
|GO:0033298 |contractile vacuole organization                                                  | 0.0898432|0.0685623466913723  |5.02901889676936   | 1.3222193|  -5.583611|  0.8741731|      0.0312650|null           |FALSE      |
|GO:0033275 |actin-myosin filament sliding                                                     | 0.1033197|7.17629834241572    |-0.889225282234449 | 1.3802112|  -6.682353|  0.8518190|      0.0316809|null           |FALSE      |
|GO:1902466 |positive regulation of histone H3-K27 trimethylation                              | 0.0089843|-2.02245480726285   |-4.95361923973496  | 0.4771213|  -4.579819|  0.8596224|      0.1175562|null           |FALSE      |
|GO:0051591 |response to cAMP                                                                  | 0.3255457|4.56091905069599    |-5.08373939084912  | 1.8633229|  -4.509703|  0.9729961|      0.1286131|null           |FALSE      |
|GO:0097193 |intrinsic apoptotic signaling pathway                                             | 0.2467993|1.48626964906999    |-3.87601791751878  | 1.7403627|  -3.003236|  0.8667449|      0.1604888|null           |FALSE      |
|GO:1904872 |regulation of telomerase RNA localization to Cajal body                           | 0.3049731|-4.1637977415722    |-5.03197066647001  | 1.6434527|  -1.978326|  0.9173185|      0.1642114|null           |FALSE      |
|GO:0043086 |negative regulation of catalytic activity                                         | 1.1410089|-4.60630467444152   |-3.96142120939216  | 2.4065402|  -1.351735|  0.9001337|      0.1652546|null           |FALSE      |
|GO:0006278 |RNA-dependent DNA biosynthetic process                                            | 0.2605453|-5.94377417689447   |2.44285837123937   | 1.7708520|  -9.577934|  0.8559526|      0.2387877|null           |FALSE      |
|GO:0006971 |hypotonic response                                                                | 0.0134765|5.70927467820515    |-2.69679457233377  | 0.6020600|  -2.372108|  0.9720130|      0.2465384|null           |FALSE      |
|GO:0031154 |culmination involved in sorocarp development                                      | 0.0404295|4.14443147510805    |5.41765266435799   | 1.0000000|  -1.309504|  0.9274572|      0.2565030|null           |FALSE      |
|GO:0016131 |brassinosteroid metabolic process                                                 | 0.1482413|-5.85824682220161   |-0.555217674438774 | 1.5314789|  -4.909247|  0.8279495|      0.2739357|null           |FALSE      |
|GO:0032060 |bleb assembly                                                                     | 0.2650375|0.161221356584404   |5.90768424345302   | 0.6989700|  -5.583611|  0.7438491|      0.2823551|null           |FALSE      |
|GO:0032989 |cellular component morphogenesis                                                  | 0.2156237|2.62795159365186    |4.32029914928754   | 1.6901961|  -3.724531|  0.7843404|      0.3028309|null           |FALSE      |
|GO:0048578 |positive regulation of long-day photoperiodism,flowering                          | 0.0359373|-1.49540471545987   |-5.70411228631739  | 0.9542425|  -3.385409|  0.8703649|      0.3077731|null           |FALSE      |
|GO:0097242 |amyloid-beta clearance                                                            | 0.9159091|6.10168926008816    |1.90603126115971   | 2.3117539|  -1.718349|  0.9372783|      0.3266559|null           |FALSE      |
|GO:2001240 |negative regulation of extrinsic apoptotic signaling pathway in absence of ligand | 0.0044922|-0.508028228805618  |-3.76118044687232  | 0.3010300|  -1.641177|  0.9048595|      0.3625130|null           |FALSE      |
|GO:0010992 |ubiquitin recycling                                                               | 0.0044922|-7.11473742497255   |-3.00141215254591  | 0.3010300|  -1.718349|  0.9144109|      0.3658745|null           |FALSE      |
|GO:0051707 |response to other organism                                                        | 6.1812138|4.81880060161709    |-3.96329998526569  | 3.1389339|  -2.284036|  0.9584055|      0.3914002|null           |FALSE      |
|GO:0090501 |RNA phosphodiester bond hydrolysis                                                | 0.8804636|-5.52201100536925   |2.76829528954853   | 2.2944662|  -6.648430|  0.8623412|      0.3997076|null           |FALSE      |
|GO:0080156 |mitochondrial mRNA modification                                                   | 0.1347648|-5.06402324699177   |3.12601556079868   | 1.4913617|  -3.221374|  0.8625111|      0.4346267|null           |FALSE      |
|GO:0006468 |protein phosphorylation                                                           | 5.0671578|-4.5728608342374    |4.83048351650044   | 3.0526939|  -3.226314|  0.8657436|      0.4388141|null           |FALSE      |
|GO:0090701 |specification of plant organ identity                                             | 0.0583981|4.67949010626168    |3.19668875244973   | 1.1461280|  -2.713400|  0.8814304|      0.4606874|null           |FALSE      |
|GO:0051127 |positive regulation of actin nucleation                                           | 0.0224608|-0.416659710728561  |-6.15089275310248  | 0.7781513|  -1.905381|  0.8789310|      0.4727066|null           |FALSE      |
|GO:0043007 |maintenance of rDNA                                                               | 0.0134765|-2.7647009059521    |3.42928408966334   | 0.6020600|  -6.455202|  0.7931148|      0.4859577|null           |FALSE      |
|GO:0016926 |protein desumoylation                                                             | 0.0359373|-4.15454837743016   |6.08804431336123   | 0.9542425|  -1.543029|  0.8979943|      0.4917335|null           |FALSE      |
|GO:0009048 |dosage compensation by inactivation of X chromosome                               | 0.4941377|-1.01950627788254   |-0.478820408432556 | 2.0453230|  -2.792735|  0.7614649|      0.5100284|null           |FALSE      |
|GO:0010815 |bradykinin catabolic process                                                      | 0.0359373|-5.74902565955968   |4.26330179904675   | 0.9542425|  -1.718349|  0.8737350|      0.5423309|null           |FALSE      |
|GO:0071048 |nuclear retention of unspliced pre-mRNA at the site of transcription              | 0.0224608|-3.9940913239265    |0.945559603995522  | 0.7781513|  -1.532452|  0.7576435|      0.5428672|null           |FALSE      |
|GO:0010090 |trichome morphogenesis                                                            | 0.2830062|4.32651906519084    |4.48361807671413   | 1.8061800|  -2.336224|  0.8509741|      0.5489694|null           |FALSE      |
|GO:0051555 |flavonol biosynthetic process                                                     | 0.0808589|-7.77784729498117   |-0.543426939272562 | 1.2787536|  -2.102840|  0.9341821|      0.6044851|null           |FALSE      |
|GO:0001932 |regulation of protein phosphorylation                                             | 0.4716769|-2.43827215980733   |-3.50411995794489  | 2.0253059|  -3.934885|  0.8569219|      0.6309617|null           |FALSE      |
|GO:0010222 |stem vascular tissue pattern formation                                            | 0.0269530|5.10585264823939    |2.89924850527089   | 0.8450980|  -1.487551|  0.9003958|      0.6358398|null           |FALSE      |
|GO:0048576 |positive regulation of short-day photoperiodism,flowering                         | 0.0044922|-1.52900243057358   |-6.31737604388969  | 0.3010300|  -1.364131|  0.8820965|      0.6596159|null           |FALSE      |
|GO:0010492 |maintenance of shoot apical meristem identity                                     | 0.0898432|4.94484297515035    |4.09268906330811   | 1.3222193|  -1.433920|  0.8838636|      0.6967210|null           |FALSE      |</code></pre>
</div>
<div id="venn-diagram-go-overlaps" class="section level1">
<h1>Venn diagram GO overlaps</h1>
<p>Let’s also check which GO terms overlap between the 4 seagrasses!</p>
<p>Let’s have a look at all of these plots. Manually zooming in leads to ggrepel reloading labels, so on the small scale a lot of these plots don’t have labels.</p>
<pre class="r"><code>for(i in names(plot_list)) {
  file_name = paste(&#39;output/&#39;,  i, &#39;.png&#39;, sep=&#39;&#39;)
  
  cowplot::save_plot(file_name, plot_list[[i]], base_height=8)
  
}</code></pre>
<p>How many shared lost GO-terms are there? Hopefully, all four species will have lost the most GO-terms.</p>
<pre class="r"><code>a &lt;- list(`P. australis` = results_list$missing_posi_vs_all_GO.txt$Name,
          `A. antarctica` = results_list$missing_amphi_vs_all_GO.txt$Name,
          `Z. marina` = results_list$missing_zmar_vs_all_GO.txt$Name,
          `Z. muelleri` = results_list$missing_zmuel_vs_all_GO.txt$Name,
          `A. thaliana` = results_list$missing_arabidopsis_vs_all_GO.txt$Name)

a_go_ids &lt;- list(`P. australis` = results_list$missing_posi_vs_all_GO.txt$TermID,
          `A. antarctica` = results_list$missing_amphi_vs_all_GO.txt$TermID,
          `Z. marina` = results_list$missing_zmar_vs_all_GO.txt$TermID,
          `Z. muelleri` = results_list$missing_zmuel_vs_all_GO.txt$TermID,
          `A. thaliana` = results_list$missing_arabidopsis_vs_all_GO.txt$TermID)</code></pre>
<pre class="r"><code>plot(euler(a), 
     quantities = TRUE,
     fill = rev(wes_palette(&quot;Zissou1&quot;, 15, type = &#39;continuous&#39;)),
    alpha = 0.5,
     labels = list(font = 4))</code></pre>
<p><img src="figure/GOenrichment.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>upset(fromList(a), order.by=&#39;freq&#39;, )</code></pre>
<p><img src="figure/GOenrichment.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>a_no_ara &lt;- list(`P. australis` = results_list$missing_posi_vs_all_GO.txt$Name,
          `A. antarctica` = results_list$missing_amphi_vs_all_GO.txt$Name,
          `Z. marina` = results_list$missing_zmar_vs_all_GO.txt$Name,
          `Z. muelleri` = results_list$missing_zmuel_vs_all_GO.txt$Name)</code></pre>
<p>What are the shared GO-terms in seagrasses, WITHOUT the Ara loss??</p>
<pre class="r"><code>setdiff(Reduce(intersect, a_no_ara), Reduce(intersect, a)) %&gt;% enframe() %&gt;% writexl::write_xlsx(&#39;./data/shared_lost_genes.xlsx&#39;)</code></pre>
<p>What if we remove Posidonia?</p>
<pre class="r"><code>b &lt;- list(`A. antarctica` = results_list$missing_amphi_vs_all_GO.txt$Name,
          `Z. marina` = results_list$missing_zmar_vs_all_GO.txt$Name,
          `Z. muelleri` = results_list$missing_zmuel_vs_all_GO.txt$Name)
intersections &lt;- Reduce(intersect, b)
intersections[grepl(&#39;ethylene&#39;, intersections)]</code></pre>
<pre><code>[1] &quot;jasmonic acid and ethylene-dependent systemic resistance,ethylene mediated signaling pathway&quot;
[2] &quot;regulation of ethylene-activated signaling pathway&quot;                                          </code></pre>
<p>OK we need a big list of all GO-terms here - which GO-term is lost in which species. That will be a supplementary table.</p>
<pre class="r"><code>all_species &lt;- c(&quot;P. australis&quot;,&quot;A. antarctica&quot;,&quot;Z. marina&quot;,&quot;Z. muelleri&quot;, &#39;A. thaliana&#39;)  
all_go_terms &lt;- Reduce(union, a)
all_go_ids &lt;- Reduce(union, a_go_ids)
results_d &lt;- data.frame(&#39;GOID&#39; = character(),
                        &#39;GO&#39; = character(),
                        &#39;P. australis&#39; = character(),
                        &#39;A. antarctica&#39; = character(),
                        &#39;Z. marina&#39; = character(),
                        &#39;Z. muelleri&#39; = character(),
                        &#39;A. thaliana&#39; = character())


for (index in seq_along(all_go_terms)){
  go &lt;- all_go_terms[index]
  go_id &lt;- all_go_ids[index]
  specs &lt;- c()
  for (species in names(a)) {
    if ( length(a[[species]][grep(paste(&#39;^&#39;, go, &#39;$&#39;, sep=&#39;&#39;), a[[species]])]) &gt; 0 ) {
      specs &lt;- c(specs, species)
    }
  }
  
  results_d[index,] &lt;- c(go_id, go, gsub(&#39;FALSE&#39;, &#39;Present&#39;, gsub(&#39;TRUE&#39;, &#39;Lost&#39;, all_species %in% specs)))
}</code></pre>
<pre class="r"><code>writexl::write_xlsx(results_d, &#39;data/Lost_GO_terms_in_five_species.xlsx&#39;)</code></pre>
</div>
<div id="after-filtering-for-plant-specific-go-terms" class="section level1">
<h1>after filtering for plant-specific GO-terms</h1>
<p>We will use the GO-terms that are plant-specific as identified by the GOMAP paper. See <a href="https://github.com/wkpalan/GOMAP-maize-analysis/blob/main/6.plantSpecific/1.getSppSpecific.R" class="uri">https://github.com/wkpalan/GOMAP-maize-analysis/blob/main/6.plantSpecific/1.getSppSpecific.R</a> or <a href="https://plantmethods.biomedcentral.com/articles/10.1186/s13007-021-00754-1" class="uri">https://plantmethods.biomedcentral.com/articles/10.1186/s13007-021-00754-1</a></p>
<pre class="r"><code>go_plant &lt;- read_tsv(&#39;https://raw.githubusercontent.com/wkpalan/GOMAP-maize-analysis/main/data/go/speciesSpecificGOTerms.txt&#39;)</code></pre>
<pre><code>Rows: 45031 Columns: 5</code></pre>
<pre><code>-- Column specification --------------------------------------------------------
Delimiter: &quot;\t&quot;
chr (1): GOterm
dbl (4): NCBITaxon:10090, NCBITaxon:33090, NCBITaxon:3702, NCBITaxon:40674</code></pre>
<pre><code>
i Use `spec()` to retrieve the full column specification for this data.
i Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
<pre class="r"><code># taxon 33090 is Viridiplantae
plantSpecificGO &lt;- go_plant %&gt;% dplyr::filter(`NCBITaxon:33090`==1) %&gt;% pull(GOterm)
plantSpecificGO &lt;- c(plantSpecificGO,c(&quot;GO:0005575&quot;,&quot;GO:0008150&quot;,&quot;GO:0003674&quot;))</code></pre>
<pre class="r"><code>results_d %&gt;% filter(GOID %in% plantSpecificGO) %&gt;% writexl::write_xlsx(&#39;data/Lost_GO_terms_in_five_species.PlantSpecific.xlsx&#39;)</code></pre>
<p>Now let’s redo the Venn diagram with those filtered GO-terms</p>
<pre class="r"><code>filters &lt;- lapply(a_go_ids, function(ch) ch %in% plantSpecificGO)

newa &lt;- list()
for (species in names(filters)) {
  before &lt;- a[[species]]
  after &lt;- before[filters[[species]]]
  newa[[species]] &lt;- after
}</code></pre>
<pre class="r"><code>plot(euler(newa), 
     quantities = TRUE,
     fill = rev(wes_palette(&quot;Zissou1&quot;, 15, type = &#39;continuous&#39;)),
    alpha = 0.5,
     labels = list(font = 4))</code></pre>
<p><img src="figure/GOenrichment.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Not much difference?</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] rrvgo_1.4.4          UpSetR_1.4.0         eulerr_6.1.1        
 [4] ggrepel_0.9.1        stringi_1.7.5        httr_1.4.2          
 [7] wesanderson_0.3.6    rvest_1.0.2          forcats_0.5.1       
[10] stringr_1.4.0        dplyr_1.0.7          purrr_0.3.4         
[13] readr_2.0.2          tidyr_1.1.4          tibble_3.1.5        
[16] ggplot2_3.3.5        tidyverse_1.3.1      topGO_2.44.0        
[19] SparseM_1.81         GO.db_3.13.0         AnnotationDbi_1.54.1
[22] IRanges_2.26.0       S4Vectors_0.30.2     Biobase_2.52.0      
[25] graph_1.70.0         BiocGenerics_0.38.0  workflowr_1.6.2     

loaded via a namespace (and not attached):
  [1] colorspace_2.0-2       ellipsis_0.3.2         rprojroot_2.0.2       
  [4] XVector_0.32.0         fs_1.5.0               rstudioapi_0.13       
  [7] farver_2.1.0           bit64_4.0.5            fansi_0.5.0           
 [10] lubridate_1.8.0        xml2_1.3.2             cachem_1.0.6          
 [13] GOSemSim_2.18.1        knitr_1.36             polyclip_1.10-0       
 [16] jsonlite_1.7.2         broom_0.7.9            gridBase_0.4-7        
 [19] dbplyr_2.1.1           png_0.1-7              pheatmap_1.0.12       
 [22] shiny_1.7.1            compiler_4.1.0         backports_1.2.1       
 [25] assertthat_0.2.1       fastmap_1.1.0          cli_3.0.1             
 [28] later_1.3.0            htmltools_0.5.2        tools_4.1.0           
 [31] igraph_1.2.7           NLP_0.2-1              gtable_0.3.0          
 [34] glue_1.4.2             GenomeInfoDbData_1.2.6 Rcpp_1.0.7            
 [37] slam_0.1-48            cellranger_1.1.0       jquerylib_0.1.4       
 [40] vctrs_0.3.8            Biostrings_2.60.2      writexl_1.4.0         
 [43] polylabelr_0.2.0       xfun_0.27              mime_0.12             
 [46] lifecycle_1.0.1        zlibbioc_1.38.0        scales_1.1.1          
 [49] treemap_2.4-3          vroom_1.5.5            hms_1.1.1             
 [52] promises_1.2.0.1       RColorBrewer_1.1-2     yaml_2.2.1            
 [55] curl_4.3.2             memoise_2.0.0          gridExtra_2.3         
 [58] sass_0.4.0             RSQLite_2.2.8          highr_0.9             
 [61] GenomeInfoDb_1.28.4    rlang_0.4.12           pkgconfig_2.0.3       
 [64] matrixStats_0.61.0     bitops_1.0-7           evaluate_0.14         
 [67] lattice_0.20-44        labeling_0.4.2         cowplot_1.1.1         
 [70] bit_4.0.4              tidyselect_1.1.1       plyr_1.8.6            
 [73] magrittr_2.0.1         R6_2.5.1               generics_0.1.1        
 [76] DBI_1.1.1              pillar_1.6.4           haven_2.4.3           
 [79] whisker_0.4            withr_2.4.2            KEGGREST_1.32.0       
 [82] RCurl_1.98-1.5         modelr_0.1.8           crayon_1.4.1          
 [85] wordcloud_2.6          utf8_1.2.2             tzdb_0.1.2            
 [88] rmarkdown_2.11         grid_4.1.0             readxl_1.3.1          
 [91] data.table_1.14.2      blob_1.2.2             git2r_0.28.0          
 [94] reprex_2.0.1           digest_0.6.28          xtable_1.8-4          
 [97] tm_0.7-8               httpuv_1.6.3           munsell_0.5.0         
[100] bslib_0.3.1           </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
