<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Philipp Bayer" />

<meta name="date" content="2022-03-10" />

<title>Visualising MSAs</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Amphibolis Posidonia Comparison</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Visualising MSAs</h1>
<h4 class="author">Philipp Bayer</h4>
<h4 class="date">10 March 2022</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-03-23
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Amphibolis_Posidonia_Comparison/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210414code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210414)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210414code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210414)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree9093bee0576708746a5d589fb4a211a5c2e422c3targetblank9093beea"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/9093bee0576708746a5d589fb4a211a5c2e422c3" target="_blank">9093bee</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree9093bee0576708746a5d589fb4a211a5c2e422c3targetblank9093beea" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/9093bee0576708746a5d589fb4a211a5c2e422c3" target="_blank">9093bee</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/OTT.nb.html
    Ignored:    analysis/figure/
    Ignored:    analysis/plotRgenes.nb.html

Untracked files:
    Untracked:  data/EIN3_OG0000629/MSA.png
    Untracked:  data/EIN3_OG0000629/OG0000629.RiceAraSeagrasses.NamesFixed.fa
    Untracked:  data/EIN3_OG0000629/OG0000629.RiceAraSeagrasses.aln.NamesFixed.fa
    Untracked:  data/EIN3_OG0000629/OG0000629.RiceAraSeagrasses.aln.fa
    Untracked:  data/EIN3_OG0000629/OG0000629.RiceAraSeagrasses.fa_pfam.tsv
    Untracked:  data/EIN3_OG0000629/alignment.svg
    Untracked:  data/SOS3_OG0000189/MSA.png
    Untracked:  data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.FixedAmphi.fa_pfam.tsv
    Untracked:  data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.NamesFixed.fasta
    Untracked:  data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.fa_pfam.tsv

Unstaged changes:
    Modified:   analysis/ThreeDStructure.Rmd
    Modified:   output/SOS3_phylogeny.png

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/MSA.Rmd</code>) and HTML (<code>docs/MSA.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/9093bee0576708746a5d589fb4a211a5c2e422c3/analysis/MSA.Rmd" target="_blank">9093bee</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-23
</td>
<td>
workflowr::wflow_publish(files = “analysis/MSA.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/c1761040ed409dc624390b1e02033d8960b3c5f1/docs/MSA.html" target="_blank">c176104</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/a02475f1f812ed36772cdc0e32fc5cb908e9d288/analysis/MSA.Rmd" target="_blank">a02475f</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-16
</td>
<td>
fix MSA code to be nicer
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/61333e05bd0c6af1532c7b6b37035d6b57ba7d45/docs/MSA.html" target="_blank">61333e0</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/e9128f4b398d11ba3e7f9e73501d4c00991a4e43/analysis/MSA.Rmd" target="_blank">e9128f4</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-11
</td>
<td>
fix MSA
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/56d032ac636f2ebf920a54a697f24d8f3237ccc3/docs/MSA.html" target="_blank">56d032a</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/76a65cd561c5fc2fbf7ca549a5a68b86243be3cc/analysis/MSA.Rmd" target="_blank">76a65cd</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-11
</td>
<td>
finalised MSA!
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/a99417457d803c370ca5e748e6f02f6a2e3b081d/docs/MSA.html" target="_blank">a994174</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/e57f6d85b6a3cf876167d0723563ba3dcf4188b1/analysis/MSA.Rmd" target="_blank">e57f6d8</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-10
</td>
<td>
Fix one label!
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/ebd08f5d59b994e94c64805201b6325c506a0a40/analysis/MSA.Rmd" target="_blank">ebd08f5</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-10
</td>
<td>
Add missing files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/ebd08f5d59b994e94c64805201b6325c506a0a40/docs/MSA.html" target="_blank">ebd08f5</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-10
</td>
<td>
Add missing files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/8886b0e42ae473b0aa6a8ba5b72eb8e166010f98/docs/MSA.html" target="_blank">8886b0e</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/02708fd56aaa43dd30ead64b0fbde43166372dca/analysis/MSA.Rmd" target="_blank">02708fd</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-10
</td>
<td>
New: fancy MSAs!!
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>I found some interesting gene clusters. Let’s look at them</p>
<pre class="r"><code>library(tidyverse)
library(ggmsa)
library(Biostrings)
library(ape)
library(ggtree)
library(treeio)
knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())</code></pre>
<pre class="r"><code>ggmsa(&#39;data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.fasta&#39;, start = 221, end = 260, char_width = 0.5, seq_name = T) + geom_seqlogo() + geom_msaBar()</code></pre>
<pre><code>Coordinate system already present. Adding new coordinate system, which will replace the existing one.</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-1-1.png" width="1152" style="display: block; margin: auto;" /></p>
<pre class="r"><code>x &lt;- readAAStringSet(&#39;data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.fasta&#39;)
d &lt;- as.dist(stringDist(x, method = &quot;hamming&quot;)/width(x)[1])
tree &lt;- bionj(d)
p &lt;- ggtree(tree) + geom_tiplab()
ggtree(tree) + geom_tiplab()</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data &lt;- tidy_msa(&#39;data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.fasta&#39;, 221, 300)
p + geom_facet(geom = geom_msa, data = data,  panel = &#39;Multiple Sequence Alignment (221-300AA)&#39;,
               font = NULL, color = &quot;Clustal&quot;, by_conversation= TRUE) +
    xlim_tree(1)</code></pre>
<pre><code>Warning: Unknown or uninitialised column: `name`.</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-3-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>Let’s use a RAXML made tree</p>
<p>Commands run, after I shortened protein IDs manually. I pulled out the rice/Arabidopsis/seagrass proteins manually from the OG0000189.fa Orthofinder produced, and shortened their names so they fit into Phylip format.</p>
<p><code>muscle -in OG0000189.RiceAraSeagrasses.fa -out OG0000189.RiceAraSeagrasses.phy -phyi</code></p>
<p><code>raxmlHPC -p12345 -m PROTGAMMAAUTO -s OG0000189.RiceAraSeagrasses.phy -n AUTO</code></p>
<pre class="r"><code>treestring &lt;- &#39;((3O.sativa:0.11386470583955717040,4O.sativa:0.03615896391829190315):0.07379737996482453599,(5O.sativa:0.33751444720146173140,((7A.thalian:0.45696796911334131019,((8Z.mueller:0.33996861614583556710,15Z.marina:0.16088168380872580610):0.31923212677234219514,10P.austra:0.57486248434538900209):0.22324965676023655892):0.10039129162924786964,((13Z.marina:0.11346816437532522559,(12Z.muelle:0.00973855109814669891,10Z.muelle:0.02645894329604922546):0.09840311662577182206):0.35332170475064506032,((14Z.marina:0.12821969530885793387,(9Z.mueller:0.06883167458475099310,11Z.muelle:0.02355162378347611801):0.15839770535961147924):0.63099939044074537797,(1A.amphibo:0.53760770644490452064,6P.austral:0.07001018412366488697):0.08839123570898912985):0.05074368132159561007):0.09141392275039983417):0.12967268161775510893):0.11320488597472283532,2O.sativa:1.70657194075952300949):0.0;&#39;
plot(ape::read.tree(text=treestring))</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>p2 &lt;- ggtree(ape::read.tree(text=treestring)) + geom_tiplab()
p2</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-4-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>That’s very different from the above dendrogram!</p>
<pre class="r"><code>data2 &lt;- data %&gt;% mutate(name = str_sub(name, 1, 10), 
                         name = str_trim(name))
p2 + geom_facet(geom = geom_msa, data = data2,  panel = &#39;Multiple Sequence Alignment (221-260AA)&#39;,
               font = NULL, color = &quot;Clustal&quot;) +
    xlim_tree(2)</code></pre>
<pre><code>Warning: Unknown or uninitialised column: `name`.</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-5-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>Good! I used blastp with Swissprot to see whether I could get ‘official’ gene names for some of these, especially the <em>O. sativa</em>. I uploaded the fasta in data/OG000189.fa to blastp/swissprot, and pulled out new names where available.</p>
<p>Hits are to: A. thaliana Calcineurin B-like protein 4 (CBL4, Alternative name: SOS3) O. sativa Calcineurin B-like protein 8 (CBL8) O. sativa Calcineurin B-like protein 4 (CBL4)</p>
<p>Let’s rename the tree and the MSA data table using <code>rename_taxa</code></p>
<pre class="r"><code>trees &lt;- ape::read.tree(text=treestring)

rename_df &lt;- tibble::tribble(
                       ~old,        ~new,
                &quot;3O.sativa&quot;,    &quot;OsCBL8&quot;,
                &quot;4O.sativa&quot;,  &quot;OsCBL7.1&quot;,
                &quot;5O.sativa&quot;,    &quot;OsCBL4&quot;,
               &quot;7A.thalian&quot;,    &quot;AtSOS3&quot;,
               &quot;8Z.mueller&quot;,   &quot;ZmuSOS3&quot;,
               &quot;15Z.marina&quot;,   &quot;ZmaSOS3&quot;,
               &quot;10P.austra&quot;,    &quot;PaSOS3&quot;,
               &quot;13Z.marina&quot;, &quot;ZmaSOS3.2&quot;,
               &quot;12Z.muelle&quot;, &quot;ZmuSOS3.2&quot;,
               &quot;10Z.muelle&quot;, &quot;ZmuSOS3.3&quot;,
               &quot;14Z.marina&quot;, &quot;ZmaSOS3.2&quot;,
               &quot;9Z.mueller&quot;, &quot;ZmuSOS3.4&quot;,
               &quot;11Z.muelle&quot;, &quot;ZmuSOS3.5&quot;,
               &quot;1A.amphibo&quot;,    &quot;AaSOS3&quot;,
               &quot;6P.austral&quot;,  &quot;PaSOS3.2&quot;,
                &quot;2O.sativa&quot;,  &quot;OsCBL7.2&quot;
               )


trees &lt;- rename_taxa(trees, rename_df, old, new)
p3 &lt;- ggtree(trees) + geom_tiplab(fontface=&#39;italic&#39;)

p3</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#str_replace_all takes a named vector
replace_vector &lt;- rename_df$new
names(replace_vector) &lt;- rename_df$old
data2 &lt;- data2 %&gt;% mutate(name = str_replace_all(name, replace_vector))</code></pre>
<pre class="r"><code>final_p &lt;- p3 + geom_facet(geom = geom_msa, data = data2,  panel = &#39;Multiple Sequence Alignment (221-260AA)&#39;,
               font = NULL, color = &quot;Chemistry_AA&quot;) +
    xlim_tree(2)</code></pre>
<pre><code>Warning: Unknown or uninitialised column: `name`.</code></pre>
<pre class="r"><code>final_p</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-8-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>cowplot::save_plot(final_p, filename = &#39;output/SOS3_phylogeny.png&#39;, base_width=10)</code></pre>
<p>“However, apparent photosynthesis is still maintained at a salinity 15% that of normal seawater and at temperatures of 3 and 30°C, consistent with the ecological role of Z. muelleri as an intertidal species.” <a href="https://www.sciencedirect.com/science/article/abs/pii/0304377085900634" class="uri">https://www.sciencedirect.com/science/article/abs/pii/0304377085900634</a></p>
<p>Now let’s also add support values.</p>
<p>I ran this based on <code>Using RAxML-NG in Practice</code></p>
<div id="phylogeny-for-the-sos3-cluster" class="section level2">
<h2>Phylogeny for the SOS3 cluster</h2>
<pre><code>
 # to get best model
modeltest-ng -i OG0000189.RiceAraSeagrasses.aln.fasta -t ml -d aa -p 8

 # to get fixed fasta
raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta --model JTT-DCMUT+G4 --check

# to make a regular tree
raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta.raxml.reduced.phy --model JTT-DCMUT+G4 --prefix T3 --threads 2 --seed 2 

# make 200 bootstrap trees, does not converge
raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta.raxml.reduced.phy --model JTT-DCMUT+G4 --prefix T8 --threads 8 --seed 2 --bootstrap --bs-trees 200 

# make another 400  with different seed
raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta.raxml.reduced.phy --model JTT-DCMUT+G4 --prefix T11 --threads 8 --seed 333 --bootstrap --bs-trees 400 

# check whether they converge with &lt;3% WRF cutoff
raxml-ng --bsconverge --bs-trees allbootstraps --prefix T12 --seed 2 --threads 1 --bs-cutoff 0.03


# to make the final trees with bootstrap values

cat T8.raxml.bootstraps T11.raxml.bootstraps &gt; allbootstraps

raxml-ng --support --tree T3.raxml.bestTree --bs-trees allbootstraps --prefix T13 

#yes, after 550 - close one!

# let&#39;s make another 400 so we can have a nice 1000 bootstraps - different seed again!

raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta.raxml.reduced.phy --model JTT-DCMUT+G4 --prefix T14 --threads 8 --seed 1234 --bs-trees 400  --bootstrap

cat T8.raxml.bootstraps T11.raxml.bootstraps T14.raxml.bootstraps &gt; allbootstraps
raxml-ng --support --tree T3.raxml.bestTree --bs-trees allbootstraps --prefix T15 

</code></pre>
<pre class="r"><code>newtree &lt;- &#39;((((((12Z.muelleri_maker-6084_47488_1_44441--0.5-mRNA-1:0.009685,10Z.muelleri_maker-8016_47756--0.9-mRNA-1:0.026505)99:0.098293,13Z.marina_Zosma01g02330:0.113268)99:0.354433,(((11Z.muelleri_maker-8016_47756--0.8-mRNA-1:0.023273,9Z.muelleri_maker-2123_50249_30746_38748--0.4-mRNA-1:0.068317)99:0.153634,14Z.marina_Zosma01g02340:0.129762)100:0.616637,(1A.amphibolis_maker-scf7180000542070-augustus-gene-0.0-mRNA-1:0.518908,6P.australis_maker-scf7180003728898-augustus-gene-0.1-mRNA-1:0.074916)72:0.082830)22:0.047398)31:0.080205,((2O.sativa_LOC_Os03g33570.1:1.647537,(3O.sativa_LOC_Os02g18930.1:0.113637,4O.sativa_LOC_Os02g18880.1:0.035299)53:0.061341)66:0.111396,5O.sativa_LOC_Os05g45810.1:0.344545)75:0.137407)42:0.100482,7A.thaliana_AT5G24270.1:0.448151)83:0.209634,(15Z.marina_Zosma06g25260:0.152988,8Z.muelleri_snap_masked-14205_19045--0.3-mRNA-1:0.336420)100:0.314204,10P.australis_maker-scf7180004080271-augustus-gene-0.8-mRNA-1:0.580458)83:0.0;&#39;

tree3 &lt;- ape::read.tree(text=newtree, branch.label=&#39;support&#39;)

p3 &lt;- ggtree(tree3) + geom_tiplab()
p3</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>OK the nodes are now named differently due to raxml-ng, time to fix again</p>
<pre class="r"><code>old_names &lt;- tree3$tip.label

# [1] &quot;3O.sativa&quot;  &quot;4O.sativa&quot;  &quot;5O.sativa&quot;  &quot;7A.thalian&quot; &quot;8Z.mueller&quot; &quot;15Z.marina&quot; &quot;10P.austra&quot; &quot;13Z.marina&quot;
# [9] &quot;12Z.muelle&quot; &quot;10Z.muelle&quot; &quot;14Z.marina&quot; &quot;9Z.mueller&quot; &quot;11Z.muelle&quot; &quot;1A.amphibo&quot; &quot;6P.austral&quot; &quot;2O.sativa&quot;
#new_names  &lt;- c( &#39;OsCBL8&#39;, &#39;OsCBL7.1&#39;, &#39;OsCBL4&#39;, &#39;AtSOS3&#39;, &#39;ZmuSOS3&#39;, &#39;ZmaSOS3&#39;, &#39;PaSOS3&#39;, &#39;ZmaSOS3.2&#39;,
#                 &#39;ZmuSOS3.2&#39;, &#39;ZmuSOS3.3&#39;, &#39;ZmaSOS3.3&#39;, &#39;ZmuSOS3.2&#39;, &#39;ZmuSOS3.4&#39;, &#39;AaSOS3&#39;, &#39;PaSOS3.2&#39;, &#39;OsCBL7.2&#39;)
new_names  &lt;- c(&#39;ZmuSOS3.2&#39;, &#39;ZmuSOS3.3&#39;, &#39;ZmaSOS3.2&#39;, &#39;ZmuSOS3.4&#39;, &#39;ZmuSOS3.5&#39;, &#39;ZmaSOS3.3&#39;,&#39;AaSOS3&#39;,  
                &#39;PaSOS3.2&#39;, &#39;OsCBL7.2&#39;, &#39;OsCBL8&#39;, &#39;OsCBL7.1&#39;, &#39;OsCBL4&#39;, &#39;AtSOS3&#39;, &#39;ZmaSOS3&#39;, &#39;ZmuSOS3&#39;, &#39;PaSOS3&#39;)
rename_df &lt;- data.frame(old = old_names, new = new_names)


tree3 &lt;- rename_taxa(tree3, rename_df, old, new)
p3 &lt;- ggtree(tree3) + geom_tiplab(fontface=&#39;italic&#39;)
p3 &lt;- p3 + geom_nodelab(aes(subset=label&gt;80))
p3</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>replace_vector &lt;- new_names
names(replace_vector) &lt;- old_names

data &lt;- tidy_msa(&#39;data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.fasta&#39;, 221, 260)
data3 &lt;- data %&gt;% mutate(name = str_replace(name, &#39; &#39;, &#39;_&#39;),
                          name = str_replace_all(name, replace_vector))
p3 + geom_facet(geom = geom_msa, data = data3,  panel = &#39;Multiple Sequence Alignment (221-260AA)&#39;,
               font = NULL, color = &quot;Chemistry_AA&quot;) +
    xlim_tree(3)</code></pre>
<pre><code>Warning: Unknown or uninitialised column: `name`.</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-12-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>msaplot(p3, &#39;data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.NamesFixed.fasta&#39;,
        offset = 0.5) + theme(legend.position=&#39;none&#39;)</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-13-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="phylogeny-for-the-ein3-cluster" class="section level2">
<h2>Phylogeny for the EIN3 cluster</h2>
<p>First, to align the sequences again:</p>
<pre><code>modeltest-ng reports JTT+I+G4+F

raxml-ng --msa OG0000629.RiceAraSeagrasses.aln.fa --model JTT+I+G4+F


 # to get fixed fasta - all good!!
raxml-ng --msa OG0000629.RiceAraSeagrasses.aln.fa --model JTT+I+G4+F --check

# to make a regular tree
raxml-ng --msa OG0000629.RiceAraSeagrasses.aln.fa --model JTT+I+G4+F --prefix T3 --threads 2 --seed 2 

# make 1000 bootstrap trees
raxml-ng --msa OG0000629.RiceAraSeagrasses.aln.fa --model JTT+I+G4+F --prefix T8 --threads 16 --seed 2 --bootstrap --bs-trees 1000 

# check whether they converge with &lt;3% WRF cutoff
raxml-ng --bsconverge --bs-trees T8.raxml.bootstraps --prefix T12 --seed 2 --threads 1 --bs-cutoff 0.03

# it actually converged with 50 trees. haha.

# to make the final trees with bootstrap values

raxml-ng --support --tree T3.raxml.bestTree --bs-trees allbootstraps --prefix T13 </code></pre>
<p>There we go:</p>
<pre class="r"><code>tree3 &lt;- ape::read.tree(text=&#39;(((LOC_Os07g48630.1:0.641123,(LOC_Os03g20780.1:0.007239,LOC_Os03g20790.1:0.001048)100:0.214718)100:0.418001,(((Zosma02g24210:0.119273,(maker-1834_92013--0.52-mRNA-1:0.074816,augustus_masked-5412_21708--0.0-mRNA-1:0.082668)97:0.073791)100:0.640955,augustus_masked-scf7180003860583-processed-gene-0.3-mRNA-1:0.182720)89:0.114788,(maker-3356_51779--0.18-mRNA-1:0.661393,Zosma01g42670:0.320772)100:0.468870)67:0.098845)100:0.660268,AT3G20770.1:0.149857,AT2G27050.1:0.250165)100:0.0;&#39;, branch.label=&#39;support&#39;)

rename_df &lt;- tibble::tribble(
                                                       ~old,        ~new,
                                         &quot;LOC_Os07g48630.1&quot;,    &quot;OsEIL2&quot;,
                                         &quot;LOC_Os03g20780.1&quot;,    &quot;OsEIL1&quot;,
                                         &quot;LOC_Os03g20790.1&quot;,  &quot;OsEIL1.2&quot;,
                                            &quot;Zosma02g24210&quot;, &quot;ZmaEIN3.1&quot;,
                            &quot;maker-1834_92013--0.52-mRNA-1&quot;, &quot;ZmuEIN3.1&quot;,
                   &quot;augustus_masked-5412_21708--0.0-mRNA-1&quot;, &quot;ZmuEIN3.2&quot;,
&quot;augustus_masked-scf7180003860583-processed-gene-0.3-mRNA-1&quot;,   &quot;PaEIN3&quot;,
                            &quot;maker-3356_51779--0.18-mRNA-1&quot;, &quot;ZmuEIN3.3&quot;,
                                            &quot;Zosma01g42670&quot;, &quot;ZmaEIN3.2&quot;,
                                              &quot;AT3G20770.1&quot;,    &quot;AtEIN3&quot;,
                                              &quot;AT2G27050.1&quot;,    &quot;AtEIL1&quot;
               )

tree3 &lt;- rename_taxa(tree3, rename_df, old, new)

p3 &lt;-  ggtree(tree3) + geom_tiplab(fontface=&#39;italic&#39;)
p3 &lt;- p3 + geom_nodelab(aes(subset=label&gt;80))
p3 + xlim_tree(2.1)</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>msaplot(p3, &#39;./data/EIN3_OG0000629/OG0000629.RiceAraSeagrasses.aln.NamesFixed.fa&#39;,
        offset = 0.5)  + theme(legend.position=&#39;none&#39;)</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>domains &lt;- tibble::tribble(
                                                                    ~Gene,     ~Pfam, ~Start, ~End,   ~Evalue,
                                                          &quot;Zosma01g42670&quot;, &quot;PF04873&quot;,     3L, 246L,  2.9e-117,
                                                            &quot;AT2G27050.1&quot;, &quot;PF04873&quot;,    49L, 298L,  2.6e-130,
                                                       &quot;LOC_Os03g20780.1&quot;, &quot;PF04873&quot;,    78L, 332L,  1.4e-132,
             &quot;augustus_masked-scf7180003860583-processed-gene-0.3-mRNA-1&quot;, &quot;PF04873&quot;,    76L, 325L,  1.9e-131,
                                          &quot;maker-1834_92013--0.52-mRNA-1&quot;, &quot;PF04873&quot;,   136L, 408L,    2e-123,
                                                       &quot;LOC_Os07g48630.1&quot;, &quot;PF04873&quot;,    81L, 338L,  3.7e-129,
                                                            &quot;AT3G20770.1&quot;, &quot;PF04873&quot;,    49L, 296L,  4.8e-129,
                                 &quot;augustus_masked-5412_21708--0.0-mRNA-1&quot;, &quot;PF04873&quot;,    16L, 285L,  2.2e-123,
                                          &quot;maker-3356_51779--0.18-mRNA-1&quot;, &quot;PF04873&quot;,    85L, 125L,     2e-12,
                                                       &quot;LOC_Os03g20790.1&quot;, &quot;PF04873&quot;,    78L, 332L,  1.4e-132,
                                                          &quot;Zosma02g24210&quot;, &quot;PF04873&quot;,     1L, 206L,  1.9e-113
             )
domains$length &lt;- domains$End - domains$Start
replace_vector &lt;- rename_df$new
names(replace_vector) &lt;- rename_df$old
domains &lt;- domains %&gt;% mutate(Gene = str_replace_all(Gene, replace_vector))
domains</code></pre>
<pre><code># A tibble: 11 x 6
   Gene      Pfam    Start   End   Evalue length
   &lt;chr&gt;     &lt;chr&gt;   &lt;int&gt; &lt;int&gt;    &lt;dbl&gt;  &lt;int&gt;
 1 ZmaEIN3.2 PF04873     3   246 2.9e-117    243
 2 AtEIL1    PF04873    49   298 2.6e-130    249
 3 OsEIL1    PF04873    78   332 1.4e-132    254
 4 PaEIN3    PF04873    76   325 1.9e-131    249
 5 ZmuEIN3.1 PF04873   136   408 2  e-123    272
 6 OsEIL2    PF04873    81   338 3.7e-129    257
 7 AtEIN3    PF04873    49   296 4.8e-129    247
 8 ZmuEIN3.2 PF04873    16   285 2.2e-123    269
 9 ZmuEIN3.3 PF04873    85   125 2  e- 12     40
10 OsEIL1.2  PF04873    78   332 1.4e-132    254
11 ZmaEIN3.1 PF04873     1   206 1.9e-113    205</code></pre>
<pre class="r"><code>g &lt;- domains %&gt;% left_join(rename_df, by=c(&#39;Gene&#39;=&#39;new&#39;)) %&gt;%  ggplot(aes(y = Gene, yend=Gene, x=Start,  xend=End,  color=Evalue)) + geom_segment(size=2) + theme_minimal() + xlab(expression(paste(&quot;Position of &quot;, italic(&quot;EIN3&quot;))))
g</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-17-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>library(aplot)</code></pre>
<pre><code>Warning: package &#39;aplot&#39; was built under R version 4.1.2</code></pre>
<pre class="r"><code>g &lt;- g + theme(axis.title.y=element_blank(),
               axis.text.y = element_blank(),
                legend.position=&#39;none&#39;)

g %&gt;% insert_left(p3, width=2.5)</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-18-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>library(patchwork)</code></pre>
<pre><code>Warning: package &#39;patchwork&#39; was built under R version 4.1.1</code></pre>
<pre class="r"><code># let&#39;s make a patchwork
# PROBLEM: patchwork does not automatically sort, so we&#39;ll have to sort the plot `g` manually 

plot_order &lt;- p3[[&#39;data&#39;]] %&gt;% filter(isTip == TRUE) %&gt;% select(label, y) %&gt;% arrange(desc(y))
g &lt;- domains %&gt;% left_join(rename_df, by=c(&#39;Gene&#39;=&#39;new&#39;)) %&gt;%  ggplot(aes(y = factor(Gene, levels = plot_order$label), yend=Gene, x=Start,  xend=End,  color=Evalue)) + geom_segment(size=2) + theme_minimal() + xlab(expression(paste(&quot;Position of &quot;, italic(&quot;EIN3&quot;))))+ theme(axis.title.y=element_blank(),
               axis.text.y = element_blank(),
                legend.position=&#39;none&#39;)
g</code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-19-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>msaplot(p3, &#39;./data/EIN3_OG0000629/OG0000629.RiceAraSeagrasses.aln.NamesFixed.fa&#39;,
        offset = 0.5)  + theme(legend.position=&#39;none&#39;) + g  + 
  plot_layout(widths = c(3.5, 1)) </code></pre>
<p><img src="figure/MSA.Rmd/unnamed-chunk-19-2.png" width="960" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] patchwork_1.1.1     aplot_0.1.2         treeio_1.16.2      
 [4] ggtree_3.0.4        ape_5.5             Biostrings_2.60.2  
 [7] GenomeInfoDb_1.28.4 XVector_0.32.0      IRanges_2.26.0     
[10] S4Vectors_0.30.2    BiocGenerics_0.38.0 ggmsa_1.1.5        
[13] forcats_0.5.1       stringr_1.4.0       dplyr_1.0.7        
[16] purrr_0.3.4         readr_2.1.2         tidyr_1.1.4        
[19] tibble_3.1.5        ggplot2_3.3.5       tidyverse_1.3.1    
[22] workflowr_1.6.2    

loaded via a namespace (and not attached):
 [1] colorspace_2.0-2       ellipsis_0.3.2         rprojroot_2.0.2       
 [4] fs_1.5.0               rstudioapi_0.13        farver_2.1.0          
 [7] fansi_0.5.0            lubridate_1.8.0        xml2_1.3.2            
[10] extrafont_0.17         knitr_1.36             polyclip_1.10-0       
[13] jsonlite_1.7.2         broom_0.7.9            Rttf2pt1_1.3.8        
[16] dbplyr_2.1.1           ggforce_0.3.3          compiler_4.1.0        
[19] httr_1.4.2             backports_1.2.1        assertthat_0.2.1      
[22] fastmap_1.1.0          lazyeval_0.2.2         cli_3.2.0             
[25] later_1.3.0            tweenr_1.0.2           htmltools_0.5.2       
[28] tools_4.1.0            gtable_0.3.0           glue_1.6.2            
[31] GenomeInfoDbData_1.2.6 maps_3.4.0             Rcpp_1.0.7            
[34] cellranger_1.1.0       jquerylib_0.1.4        vctrs_0.3.8           
[37] ggalt_0.4.0            nlme_3.1-152           extrafontdb_1.0       
[40] xfun_0.27              rvest_1.0.2            lifecycle_1.0.1       
[43] zlibbioc_1.38.0        MASS_7.3-54            scales_1.1.1          
[46] hms_1.1.1              promises_1.2.0.1       proj4_1.0-11          
[49] RColorBrewer_1.1-2     yaml_2.2.1             R4RNA_1.22.0          
[52] ggfun_0.0.5            seqmagick_0.1.5        yulab.utils_0.0.4     
[55] sass_0.4.0             stringi_1.7.5          highr_0.9             
[58] tidytree_0.3.7         rlang_0.4.12           pkgconfig_2.0.3       
[61] bitops_1.0-7           evaluate_0.14          lattice_0.20-44       
[64] labeling_0.4.2         cowplot_1.1.1          tidyselect_1.1.1      
[67] magrittr_2.0.1         R6_2.5.1               generics_0.1.1        
[70] DBI_1.1.1              pillar_1.6.4           haven_2.4.3           
[73] whisker_0.4            withr_2.5.0            RCurl_1.98-1.5        
[76] ash_1.0-15             modelr_0.1.8           crayon_1.4.1          
[79] KernSmooth_2.23-20     utf8_1.2.2             tzdb_0.1.2            
[82] rmarkdown_2.11         grid_4.1.0             readxl_1.3.1          
[85] git2r_0.28.0           reprex_2.0.1           digest_0.6.28         
[88] httpuv_1.6.3           gridGraphics_0.5-1     munsell_0.5.0         
[91] ggplotify_0.1.0        bslib_0.3.1           </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
