<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Philipp Bayer" />

<meta name="date" content="2021-04-15" />

<title>Clustering/Orthofinder analysis</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Amphibolis Posidonia Comparison</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Clustering/Orthofinder analysis</h1>
<h4 class="author">Philipp Bayer</h4>
<h4 class="date">15 April 2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-03-08
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Amphibolis_Posidonia_Comparison/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210414code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210414)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210414code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210414)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree309567ac01c00d11b657cb12e0827fee71b4b241targetblank309567aa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/309567ac01c00d11b657cb12e0827fee71b4b241" target="_blank">309567a</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree309567ac01c00d11b657cb12e0827fee71b4b241targetblank309567aa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/309567ac01c00d11b657cb12e0827fee71b4b241" target="_blank">309567a</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/OTT.nb.html
    Ignored:    analysis/figure/
    Ignored:    analysis/plotRgenes.nb.html

Untracked files:
    Untracked:  data/Lost_present_gene_lists/Genes_only_in_Algae.txt
    Untracked:  data/Lost_present_gene_lists/Genes_only_in_Duckweeds.txt
    Untracked:  output/group_venn_image.Rdata
    Untracked:  output/seagrass_venn_image.Rdata

Unstaged changes:
    Modified:   data/Lost_present_gene_lists/Genes_only_in_Aquatics_and_Seagrasses.txt
    Modified:   data/Lost_present_gene_lists/Genes_only_in_Aquatics_and_Seagrasses_and_Terrestrials.txt
    Modified:   data/arabidopsis_gene_level_comparison.xlsx
    Modified:   data/arabidopsis_gene_level_comparison_only_losts.xlsx
    Modified:   data/arabidopsis_gene_level_counts.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Orthofinder.Rmd</code>) and HTML (<code>docs/Orthofinder.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/309567ac01c00d11b657cb12e0827fee71b4b241/analysis/Orthofinder.Rmd" target="_blank">309567a</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-03-08
</td>
<td>
split aquatics up
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/679f242b0ea240b5785bd2ce773fb0d664231395/docs/Orthofinder.html" target="_blank">679f242</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/5fa37b7aa8ee4ae02b145e616f82bebedefea69e/analysis/Orthofinder.Rmd" target="_blank">5fa37b7</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-25
</td>
<td>
Fix bug in the very last lines
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/6ab5b93cf239cb43186fb64ca253e753e0b7507a/analysis/Orthofinder.Rmd" target="_blank">6ab5b93</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-25
</td>
<td>
Fix typo
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/d366ade33481bb34ed4546aa6bc395e7a1c1bba7/docs/Orthofinder.html" target="_blank">d366ade</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/b7bce950478f4dac79ba039a1d53a5d2addd1869/analysis/Orthofinder.Rmd" target="_blank">b7bce95</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-25
</td>
<td>
Fix super huge string lists
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/0d1bf58d1db23e69b42a67218c7a908f352f7d3b/docs/Orthofinder.html" target="_blank">0d1bf58</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/9be89cfbd6f2d4dd85ca5650729db622ceeb416f/analysis/Orthofinder.Rmd" target="_blank">9be89cf</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-23
</td>
<td>
Rebuild Orthofinder analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/6592f5305d20df6ac45ea485b0c34aa97566f0a7/analysis/Orthofinder.Rmd" target="_blank">6592f53</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-22
</td>
<td>
Add changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/36b882caad120fe07c574f91bffba50ac1b8aa78/docs/Orthofinder.html" target="_blank">36b882c</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/1ae7e946486ccd6e8c467acda4fc72cd484c7669/analysis/Orthofinder.Rmd" target="_blank">1ae7e94</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-21
</td>
<td>
venn diagram
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/ab9a1f6eba9f91725081729b83cf2467438a2133/analysis/Orthofinder.Rmd" target="_blank">ab9a1f6</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-02-21
</td>
<td>
add some more changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/37bc0d6fdbd50869cc51f6e94f34337385e1d6ee/docs/Orthofinder.html" target="_blank">37bc0d6</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/19df596756466f7bde20d005d69647b438612740/analysis/Orthofinder.Rmd" target="_blank">19df596</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2022-01-16
</td>
<td>
better analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/d36f0a77b5bba3e9cc85a40594b6190639aea22b/analysis/Orthofinder.Rmd" target="_blank">d36f0a7</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-12-20
</td>
<td>
Add missing files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/02d3e09dfdd71b4818b5d99df05378d82a223bc9/analysis/Orthofinder.Rmd" target="_blank">02d3e09</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-27
</td>
<td>
Add missing files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/02d3e09dfdd71b4818b5d99df05378d82a223bc9/docs/Orthofinder.html" target="_blank">02d3e09</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-27
</td>
<td>
Add missing files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/c0db4a55cee717b3c1b0b2b02c5e9d1bfddac065/docs/Orthofinder.html" target="_blank">c0db4a5</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/08b28eacde8985a9d65405b6bc555c4c9d934f92/analysis/Orthofinder.Rmd" target="_blank">08b28ea</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
wflow_publish(files = c("analysis/*"))
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/f67a752e4029d05a1011dee99b6d5327d12c5031/analysis/Orthofinder.Rmd" target="_blank">f67a752</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-15
</td>
<td>
Missing commits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/f67a752e4029d05a1011dee99b6d5327d12c5031/docs/Orthofinder.html" target="_blank">f67a752</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-15
</td>
<td>
Missing commits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/76c8918c38098e7cd5dc2b843a040ee9a67e3238/docs/Orthofinder.html" target="_blank">76c8918</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/49dcbb89ac6f0a14c0066977c3ac23fdc393b3dd/analysis/Orthofinder.Rmd" target="_blank">49dcbb8</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-15
</td>
<td>
wflow_publish(files = list.files(“analysis/”, pattern = "*Rmd",
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Here I use Orthofinder results to pull out genes unique to seagrasses, unique to duckweeds, and make some nice summary tables</p>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>Warning: package &#39;tidyverse&#39; was built under R version 4.1.1</code></pre>
<pre><code>-- Attaching packages --------------------------------------- tidyverse 1.3.1 --</code></pre>
<pre><code>v ggplot2 3.3.5     v purrr   0.3.4
v tibble  3.1.5     v dplyr   1.0.7
v tidyr   1.1.4     v stringr 1.4.0
v readr   2.0.2     v forcats 0.5.1</code></pre>
<pre><code>Warning: package &#39;ggplot2&#39; was built under R version 4.1.1</code></pre>
<pre><code>Warning: package &#39;tibble&#39; was built under R version 4.1.1</code></pre>
<pre><code>Warning: package &#39;tidyr&#39; was built under R version 4.1.1</code></pre>
<pre><code>Warning: package &#39;readr&#39; was built under R version 4.1.1</code></pre>
<pre><code>Warning: package &#39;purrr&#39; was built under R version 4.1.1</code></pre>
<pre><code>Warning: package &#39;dplyr&#39; was built under R version 4.1.1</code></pre>
<pre><code>Warning: package &#39;stringr&#39; was built under R version 4.1.1</code></pre>
<pre><code>Warning: package &#39;forcats&#39; was built under R version 4.1.1</code></pre>
<pre><code>-- Conflicts ------------------------------------------ tidyverse_conflicts() --
x dplyr::filter() masks stats::filter()
x dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(cowplot)</code></pre>
<pre><code>Warning: package &#39;cowplot&#39; was built under R version 4.1.1</code></pre>
<pre class="r"><code>theme_set(theme_cowplot())
library(RColorBrewer)</code></pre>
<pre><code>Warning: package &#39;RColorBrewer&#39; was built under R version 4.1.1</code></pre>
<pre class="r"><code>library(wesanderson)</code></pre>
<pre><code>Warning: package &#39;wesanderson&#39; was built under R version 4.1.1</code></pre>
<pre class="r"><code>library(patchwork)</code></pre>
<pre><code>Warning: package &#39;patchwork&#39; was built under R version 4.1.1</code></pre>
<pre><code>
Attaching package: &#39;patchwork&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:cowplot&#39;:

    align_plots</code></pre>
<pre class="r"><code>library(UpSetR)</code></pre>
<pre><code>Warning: package &#39;UpSetR&#39; was built under R version 4.1.1</code></pre>
<pre class="r"><code>library(kableExtra)</code></pre>
<pre><code>Warning: package &#39;kableExtra&#39; was built under R version 4.1.2</code></pre>
<pre><code>
Attaching package: &#39;kableExtra&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    group_rows</code></pre>
<pre class="r"><code>library(ggVennDiagram)</code></pre>
<pre><code>Warning: package &#39;ggVennDiagram&#39; was built under R version 4.1.2</code></pre>
<pre class="r"><code>knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())</code></pre>
<pre class="r"><code>groups &lt;- read_tsv(&#39;./data/Orthogroups.tsv.gz&#39;) </code></pre>
<pre><code>Rows: 31136 Columns: 20</code></pre>
<pre><code>-- Column specification --------------------------------------------------------
Delimiter: &quot;\t&quot;
chr (20): Orthogroup, Amphibolis_final.genome.scf.bigger1kbp.all.maker.prote...</code></pre>
<pre><code>
i Use `spec()` to retrieve the full column specification for this data.
i Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
<pre class="r"><code>names(groups) &lt;- c(&#39;Orthogroup&#39;, &#39;A. antarctica&#39;, &#39;A. trichopada&#39;, &#39;B. distachyon&#39;, &#39;C. reinhardtii&#39;, &#39;L. gibba&#39;, &#39;O. sativa&#39;, &#39;P. australis&#39;, &#39;P. patens&#39;, &#39;P. trichocarpa&#39;, &#39;S. moellendorffii&#39;, &#39;S. polyrhiza&#39;, &#39;A. thaliana&#39;, &#39;T. parvula&#39;, &#39;V. vinifera&#39;, &#39;Z. mays&#39;, &#39;Z. muelleri&#39;, &#39;Z. marina&#39;, &#39;O. lucimarinus&#39;, &#39;W. australis&#39;)</code></pre>
<pre class="r"><code># for upsetr, we need to know only which OG-groups are shared between species, the actual genes don&#39;t matter
per_spec &lt;- groups %&gt;% pivot_longer(-Orthogroup) %&gt;% 
  filter(!is.na(value)) %&gt;% # species not in an orthogroup are still listed, they just have NA genes for this group
  select(-value) # don&#39;t need all gene names, speed things up</code></pre>
<pre class="r"><code># now I want the data in this format:
# listInput &lt;- list(one = c(1, 2, 3, 5, 7, 8, 11, 12, 13), two = c(1, 2, 4, 5, 
#   10), three = c(1, 5, 6, 7, 8, 9, 10, 12, 13))
x &lt;- per_spec %&gt;% 
  select(name, Orthogroup) %&gt;% # turn the table around
  deframe() # convert to named vector</code></pre>
<pre class="r"><code>mylist &lt;- lapply(split(x, names(x)), unname) # yuck - ugly code to convert the named vector to a list</code></pre>
<pre class="r"><code>x &lt;- upset(fromList(mylist), order.by=&#39;freq&#39;, nsets = length(groups) - 1)</code></pre>
<pre class="r"><code>x</code></pre>
<p><img src="figure/Orthofinder.Rmd/unnamed-chunk-6-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>Let’s get the species-only cluster numbers</p>
<pre class="r"><code>species_specific_orthos &lt;- per_spec %&gt;% 
  group_by(Orthogroup) %&gt;% 
  summarise(counts = length(name)) %&gt;%
  filter(counts == 1)</code></pre>
<pre class="r"><code>per_spec %&gt;% 
  filter(Orthogroup %in% species_specific_orthos$Orthogroup) %&gt;% 
  group_by(name) %&gt;% 
  count() %&gt;% 
  arrange(n) %&gt;% 
  kbl() %&gt;%
  kable_styling()</code></pre>
<table class="table" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
name
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
A. antarctica
</td>
<td style="text-align:right;">
74
</td>
</tr>
<tr>
<td style="text-align:left;">
S. polyrhiza
</td>
<td style="text-align:right;">
155
</td>
</tr>
<tr>
<td style="text-align:left;">
O. lucimarinus
</td>
<td style="text-align:right;">
198
</td>
</tr>
<tr>
<td style="text-align:left;">
L. gibba
</td>
<td style="text-align:right;">
267
</td>
</tr>
<tr>
<td style="text-align:left;">
P. australis
</td>
<td style="text-align:right;">
267
</td>
</tr>
<tr>
<td style="text-align:left;">
T. parvula
</td>
<td style="text-align:right;">
273
</td>
</tr>
<tr>
<td style="text-align:left;">
Z. marina
</td>
<td style="text-align:right;">
322
</td>
</tr>
<tr>
<td style="text-align:left;">
P. trichocarpa
</td>
<td style="text-align:right;">
485
</td>
</tr>
<tr>
<td style="text-align:left;">
A. thaliana
</td>
<td style="text-align:right;">
489
</td>
</tr>
<tr>
<td style="text-align:left;">
B. distachyon
</td>
<td style="text-align:right;">
507
</td>
</tr>
<tr>
<td style="text-align:left;">
Z. muelleri
</td>
<td style="text-align:right;">
619
</td>
</tr>
<tr>
<td style="text-align:left;">
V. vinifera
</td>
<td style="text-align:right;">
682
</td>
</tr>
<tr>
<td style="text-align:left;">
A. trichopada
</td>
<td style="text-align:right;">
848
</td>
</tr>
<tr>
<td style="text-align:left;">
W. australis
</td>
<td style="text-align:right;">
1006
</td>
</tr>
<tr>
<td style="text-align:left;">
C. reinhardtii
</td>
<td style="text-align:right;">
1105
</td>
</tr>
<tr>
<td style="text-align:left;">
Z. mays
</td>
<td style="text-align:right;">
1108
</td>
</tr>
<tr>
<td style="text-align:left;">
S. moellendorffii
</td>
<td style="text-align:right;">
1334
</td>
</tr>
<tr>
<td style="text-align:left;">
O. sativa
</td>
<td style="text-align:right;">
1419
</td>
</tr>
<tr>
<td style="text-align:left;">
P. patens
</td>
<td style="text-align:right;">
1568
</td>
</tr>
</tbody>
</table>
<p>How many orthogroups are shared between the four seagrasses?</p>
<pre class="r"><code>newlist &lt;- mylist[c(&#39;A. antarctica&#39;, &#39;Z. marina&#39;, &#39;P. australis&#39;, &#39;Z. muelleri&#39;)]

x &lt;- upset(fromList(newlist), order.by=&#39;freq&#39;, nsets = 4)</code></pre>
<pre class="r"><code>x</code></pre>
<p><img src="figure/Orthofinder.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<div id="connect-orthofinder-results-with-functional-table-of-arabidopsis-genes" class="section level1">
<h1>Connect Orthofinder results with functional table of Arabidopsis genes</h1>
<p>File comes from <a href="https://www.arabidopsis.org/download_files/Genes/TAIR10_genome_release/TAIR10_functional_descriptions" class="uri">https://www.arabidopsis.org/download_files/Genes/TAIR10_genome_release/TAIR10_functional_descriptions</a></p>
<pre class="r"><code>download.file(&#39;https://www.arabidopsis.org/download_files/Genes/TAIR10_genome_release/TAIR10_functional_descriptions&#39;, &#39;data/TAIR10_functional_descriptions&#39;)</code></pre>
<pre class="r"><code>functions &lt;- read_tsv(&#39;data/TAIR10_functional_descriptions&#39;)</code></pre>
<pre><code>Rows: 41671 Columns: 5</code></pre>
<pre><code>-- Column specification --------------------------------------------------------
Delimiter: &quot;\t&quot;
chr (5): Model_name, Type, Short_description, Curator_summary, Computational...</code></pre>
<pre><code>
i Use `spec()` to retrieve the full column specification for this data.
i Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
<p>I also downloaded the gene symbols:</p>
<pre class="r"><code>esearch -db gene -query &quot;Arabidopsis thaliana [ORGN]&quot; | esummary | xtract -pattern DocumentSummary -element Name,OtherAliases |  awk -F &quot;\t|,&quot; &#39;{OFS=&quot;\t&quot;}{print $2,$1}&#39; &gt; arabidopsis_gene_symbols.txt</code></pre>
<pre class="r"><code>symbols &lt;- read_tsv(&#39;./data/arabidopsis_gene_symbols.txt&#39;, col_names = c(&#39;Gene&#39;,&#39;Symbol&#39;))</code></pre>
<pre><code>Rows: 44111 Columns: 2</code></pre>
<pre><code>-- Column specification --------------------------------------------------------
Delimiter: &quot;\t&quot;
chr (2): Gene, Symbol</code></pre>
<pre><code>
i Use `spec()` to retrieve the full column specification for this data.
i Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
<p>OK now I need to flip the Orthofinder table to then join the functional table</p>
<pre class="r"><code>ara_groups &lt;- groups %&gt;% select(Orthogroup, `A. thaliana`) %&gt;% filter(!is.na(`A. thaliana`)) %&gt;% separate_rows(Orthogroup, `A. thaliana`, convert=T)
ara_groups &lt;- ara_groups %&gt;% separate(`A. thaliana`, c(&#39;Gene&#39;,&#39;Number&#39;), remove=FALSE)
ara_groups &lt;- left_join(ara_groups, symbols)</code></pre>
<pre><code>Joining, by = &quot;Gene&quot;</code></pre>
<pre class="r"><code>ara_groups$Number &lt;- NULL
head(ara_groups)</code></pre>
<pre><code># A tibble: 6 x 4
  Orthogroup `A. thaliana` Gene      Symbol   
  &lt;chr&gt;      &lt;chr&gt;         &lt;chr&gt;     &lt;chr&gt;    
1 OG0000000  AT1G01250.1   AT1G01250 AT1G01250
2 OG0000000  AT1G04370.1   AT1G04370 ERF14    
3 OG0000000  AT1G06160.1   AT1G06160 ORA59    
4 OG0000000  AT1G12630.1   AT1G12630 AT1G12630
5 OG0000000  AT1G12890.1   AT1G12890 AT1G12890
6 OG0000000  AT1G19210.1   AT1G19210 AT1G19210</code></pre>
<pre class="r"><code>ara_joined &lt;- left_join(ara_groups, functions, by=c(`A. thaliana`=&#39;Model_name&#39;))</code></pre>
<p>You know what? we shouldn’t directly compare a dicot (A. thaliana) with monocots (seagrasses), should also add another regular monocot (rice)</p>
<pre class="r"><code>posi_groups &lt;- groups %&gt;% select(Orthogroup, `P. australis`) %&gt;% filter(!is.na(`P. australis`)) %&gt;% separate_rows(Orthogroup, `P. australis`, convert=T, sep = &#39;, &#39;)
amphi_groups &lt;- groups %&gt;% select(Orthogroup, `A. antarctica`) %&gt;% filter(!is.na(`A. antarctica`)) %&gt;% separate_rows(Orthogroup, `A. antarctica`, convert=T, sep = &#39;, &#39;)
zmar_groups &lt;- groups %&gt;%  select(Orthogroup, `Z. marina`) %&gt;% filter(!is.na(`Z. marina`)) %&gt;% separate_rows(Orthogroup, `Z. marina`, convert=T, sep = &#39;, &#39;)
zmuel_groups &lt;- groups %&gt;%  select(Orthogroup, `Z. muelleri`) %&gt;% filter(!is.na(`Z. muelleri`)) %&gt;% separate_rows(Orthogroup, `Z. muelleri`, convert=T, sep = &#39;, &#39;)
w_australis_groups &lt;- groups %&gt;%  select(Orthogroup, `W. australis`) %&gt;% filter(!is.na(`W. australis`)) %&gt;% separate_rows(Orthogroup, `W. australis`, convert=T, sep = &#39;, &#39;)
s_polyrhiza_groups &lt;- groups %&gt;%  select(Orthogroup, `S. polyrhiza`) %&gt;% filter(!is.na(`S. polyrhiza`)) %&gt;% separate_rows(Orthogroup, `S. polyrhiza`, convert=T, sep = &#39;, &#39;)
l_gibba_groups &lt;- groups %&gt;%  select(Orthogroup, `L. gibba`) %&gt;% filter(!is.na(`L. gibba`)) %&gt;% separate_rows(Orthogroup, `L. gibba`, convert=T, sep = &#39;, &#39;)
rice_groups&lt;- groups %&gt;%  select(Orthogroup, `O. sativa`) %&gt;% filter(!is.na(`O. sativa`)) %&gt;% separate_rows(Orthogroup, `O. sativa`, convert=T, sep = &#39;, &#39;)</code></pre>
<pre class="r"><code>big_joined &lt;- ara_joined %&gt;% mutate(
  present_in_zmuel = case_when(Orthogroup %in% unique(zmuel_groups$Orthogroup) ~ T,
                               TRUE ~ F),
  present_in_zmar = case_when(Orthogroup %in% unique(zmar_groups$Orthogroup) ~ T,
                              TRUE ~ F),
  present_in_amphi = case_when(Orthogroup %in% unique(amphi_groups$Orthogroup) ~ T,
                               TRUE ~ F),
  present_in_posi = case_when(Orthogroup %in% unique(posi_groups$Orthogroup) ~ T,
                              TRUE ~ F),
  present_in_waustralis = case_when(Orthogroup %in% unique(w_australis_groups$Orthogroup) ~ T,
                              TRUE ~ F),
  present_in_spolyrhiza = case_when(Orthogroup %in% unique(s_polyrhiza_groups$Orthogroup) ~ T,
                              TRUE ~ F),
  present_in_lgibba = case_when(Orthogroup %in% unique(l_gibba_groups$Orthogroup) ~ T,
                              TRUE ~ F),
  present_in_rice = case_when(Orthogroup %in% unique(rice_groups$Orthogroup) ~ T,
                              TRUE ~ F)
  )</code></pre>
<p>Done :) Now we have a big table with all <em>A. thaliana</em> gene functions and whether these genes are present in clusters of the four seagrasses and three duckweeds.</p>
<p>Let’s also add counts - how often is this gene lost?</p>
<pre class="r"><code>big_joined &lt;- big_joined %&gt;% mutate(Lost_in_Seagrasses = 4 - (present_in_posi + present_in_zmuel + present_in_zmar + present_in_amphi),
                      Lost_in_duckweeds=3 - (present_in_lgibba + present_in_spolyrhiza + present_in_waustralis),
                      Lost_in_both = (Lost_in_Seagrasses + Lost_in_duckweeds))</code></pre>
<pre class="r"><code>big_joined %&gt;% writexl::write_xlsx(&#39;data/arabidopsis_gene_level_comparison.xlsx&#39;)
big_joined %&gt;% filter_at(vars(starts_with(&#39;present&#39;)), any_vars(. == FALSE)) %&gt;% writexl::write_xlsx(&#39;data/arabidopsis_gene_level_comparison_only_losts.xlsx&#39;)</code></pre>
<p>That was about loss - can we also check for gene family extension using the <em>A. thaliana</em> genes?</p>
</div>
<div id="gene-family-extension" class="section level1">
<h1>Gene family extension</h1>
<p>First, let’s count how many members each of these orthogroups has per species.</p>
<pre class="r"><code>ara_count &lt;- ara_joined %&gt;% count(Orthogroup)
posi_count &lt;- posi_groups %&gt;% count(Orthogroup)
amphi_count &lt;- amphi_groups %&gt;% count(Orthogroup)
zmar_count &lt;- zmar_groups %&gt;% count(Orthogroup)
zmuel_count &lt;- zmuel_groups %&gt;% count(Orthogroup)
lgibba_count &lt;- l_gibba_groups %&gt;% count(Orthogroup)
s_polyrhi_count &lt;- s_polyrhiza_groups %&gt;% count(Orthogroup)
w_aus_count &lt;- w_australis_groups %&gt;% count(Orthogroup)
rice_count &lt;- rice_groups %&gt;% count(Orthogroup)</code></pre>
<pre class="r"><code>counts &lt;- plyr::join_all(list(ara_count, posi_count, amphi_count, zmar_count, zmuel_count, lgibba_count, s_polyrhi_count, w_aus_count, rice_count), by=&#39;Orthogroup&#39;, type=&#39;left&#39;) 
names(counts) &lt;- c(&#39;Orthogroup&#39;, &#39;A. thaliana&#39;, &#39;P. australis&#39;, &#39;A. antarctica&#39;, &#39;Z. marina&#39;, &#39;Z. muelleri&#39;, &#39;L. gibba&#39;, &#39;S. polyrhiza&#39;, &#39;W. australis&#39;, &#39;O. sativa&#39;)
head(counts)</code></pre>
<pre><code>  Orthogroup A. thaliana P. australis A. antarctica Z. marina Z. muelleri
1  OG0000000         112           19            23        76          76
2  OG0000001          81           49            41        85         116
3  OG0000002          97           60            61        16          23
4  OG0000003          88           12            13        51          58
5  OG0000005          64           31            33        31          50
6  OG0000006          53           21            17        28          30
  L. gibba S. polyrhiza W. australis O. sativa
1       69           48           84       113
2       96           59           63        86
3       56           20           35        84
4       55           35           35       131
5       36           35           53        59
6       60           21           28        53</code></pre>
<pre class="r"><code>joined_counts &lt;- left_join(big_joined, counts, by=&#39;Orthogroup&#39;) %&gt;% select(-starts_with(&#39;present&#39;)) %&gt;% 
  select(-starts_with(&#39;Lost_i&#39;))
joined_counts %&gt;% writexl::write_xlsx(&#39;data/arabidopsis_gene_level_counts.xlsx&#39;)</code></pre>
</div>
<div id="final-version-redoing---terrestrial-vs-aquatics-vs-seagrasses" class="section level1">
<h1>FINAL version redoing - terrestrial vs aquatics vs seagrasses</h1>
<p>Plotting all species separately is messy as I have so many species. I’m joining the species into three groups: aquatics, terrestrials, and seagrasses!</p>
<pre class="r"><code>per_group_spec &lt;-
  per_spec %&gt;% mutate(group = case_when(
    name %in% c(
      &#39;A. trichopada&#39;,
      &#39;B. distachyon&#39;,
      &#39;O. sativa&#39;,
      &#39;P. patens&#39;,
      &#39;P. trichocarpa&#39;,
      &#39;S. moellendorffii&#39;,
      &#39;A. thaliana&#39;,
      &#39;T. parvula&#39;,
      &#39;V. vinifera&#39;,
      &#39;Z. mays&#39;
    ) ~ &#39;Terrestrials&#39;,
    name %in% c(
      &#39;L. gibba&#39;,
      &#39;S. polyrhiza&#39;,
      &#39;W. australis&#39;
    ) ~ &#39;Duckweeds&#39;,
    name %in% c(
      &#39;C. reinhardtii&#39;,
      &#39;O. lucimarinus&#39;
    ) ~ &#39;Algae&#39;,
    name %in% c(
      &#39;P. australis&#39;,
      &#39;Z. muelleri&#39;,
      &#39;Z. marina&#39;,
      &#39;A. antarctica&#39;
      ) ~ &#39;Seagrasses&#39;
  )) </code></pre>
<pre class="r"><code># now I want the data in this format:
# listInput &lt;- list(one = c(1, 2, 3, 5, 7, 8, 11, 12, 13), two = c(1, 2, 4, 5, 
#   10), three = c(1, 5, 6, 7, 8, 9, 10, 12, 13))
groupx &lt;- per_group_spec %&gt;% 
  select(group, Orthogroup) %&gt;% # turn the table around
  deframe() # convert to named vector</code></pre>
<pre class="r"><code>mylistgroup &lt;- lapply(split(groupx, names(groupx)), unname) # yuck - ugly code to convert the named vector to a list
mylistgroup &lt;- lapply(mylistgroup, unique)</code></pre>
<pre class="r"><code>xgroup &lt;- upset(fromList(mylistgroup), order.by=&#39;freq&#39;, nsets = length(groups) - 1)</code></pre>
<pre class="r"><code>xgroup</code></pre>
<p><img src="figure/Orthofinder.Rmd/unnamed-chunk-28-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>MUCH better. The biggest chunk of orthogroups is not in aquatics or seagrasses with 13562 orthogroups. 3845 orthogroups are in duckweeds, seagrasses, terrestrials, but not algae - 2217 orthogroups are unique in seagrasses.</p>
<p>OK now we need to do the GOenrichment for those four groups.</p>
<p>Let’s also make an Venn plot, we have only four groups:</p>
<pre class="r"><code># code from https://github.com/gaospecial/ggVennDiagram/blob/4cb2aa13c7beae469f9b9836c5d4f94610bd872e/R/ggVennDiagram.R to customise
venn &lt;- Venn(mylistgroup)
data &lt;- process_data(venn)
region_label &lt;- data@region %&gt;%
      dplyr::filter(.data$component == &quot;region&quot;) %&gt;%
      dplyr::mutate(percent = paste(round(.data$count*100/sum(.data$count),
                                          digits = 0),&quot;%&quot;, sep=&quot;&quot;)) %&gt;%
      dplyr::mutate(both = paste(.data$count,paste0(&quot;(&quot;,.data$percent,&quot;)&quot;),sep = &quot;\n&quot;))

group_venn &lt;- ggplot() +
    geom_sf(aes_string(fill=&quot;count&quot;), data = data@region) +
    geom_sf(aes_string(color = &quot;id&quot;), data = data@setEdge, show.legend = F,
            lty = &#39;solid&#39;, size = 1, color=&#39;gray&#39;) +
    geom_sf_text(aes_string(label = &quot;name&quot;), data = data@setLabel,
                 size = NA, 
                 color = &#39;black&#39;) +
    theme_void() +
     geom_sf_label(aes_string(label=&#39;both&#39;),
                             data = region_label,
                             alpha= 0.0,
                             color = &#39;black&#39;,
                             size = NA,
                             lineheight = 0.85,
                             label.size = NA) +
  scale_fill_gradientn(colours=wes_palette(&quot;Zissou1&quot;, 100, type = &quot;continuous&quot;)) + 
  theme(legend.position = &quot;none&quot;) + 
  scale_x_continuous(expand = expansion(mult = .2)) #trick from https://github.com/gaospecial/ggVennDiagram/blob/9435aa0ab4abb470c670ecb938c71576461ccedc/vignettes/using-ggVennDiagram.Rmd#L65

group_venn</code></pre>
<p><img src="figure/Orthofinder.Rmd/unnamed-chunk-29-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>Nice!</p>
</div>
<div id="write-out-the-per-group-go-terms" class="section level1">
<h1>Write out the per-group GO-terms</h1>
<p>Now we pull out all gene-IDs that are only in terrestrials, only in seagrasses, only in aquatics, and unique for each of these three groups, AND we pull out the IDs only present in the four seagrasses each compared with the others.</p>
<pre class="r"><code>gene_per_spec &lt;- groups %&gt;% pivot_longer(-Orthogroup) %&gt;% 
  filter(!is.na(value)) %&gt;% 
  mutate(group = case_when(
    name %in% c(
      &#39;A. trichopada&#39;,
      &#39;B. distachyon&#39;,
      &#39;O. sativa&#39;,
      &#39;P. patens&#39;,
      &#39;P. trichocarpa&#39;,
      &#39;S. moellendorffii&#39;,
      &#39;A. thaliana&#39;,
      &#39;T. parvula&#39;,
      &#39;V. vinifera&#39;,
      &#39;Z. mays&#39;
    ) ~ &#39;Terrestrials&#39;,
    name %in% c(
      &#39;L. gibba&#39;,
      &#39;S. polyrhiza&#39;,
      &#39;W. australis&#39;
    ) ~ &#39;Duckweeds&#39;,
    name %in% c(
      &#39;C. reinhardtii&#39;,
      &#39;O. lucimarinus&#39;
    ) ~ &#39;Algae&#39;,
    name %in% c(
      &#39;P. australis&#39;,
      &#39;Z. muelleri&#39;, &#39;Z. marina&#39;, &#39;A. antarctica&#39;) ~ &#39;Seagrasses&#39;
  )) </code></pre>
<p>mylistgroup now looks like this: key is Aquatics, Seagrasses, Terrestrials, values are OG00000, OG00001, OG00002 etc.</p>
<pre class="r"><code>present_algae_only &lt;- setdiff(setdiff(setdiff(mylistgroup$Algae, mylistgroup$Duckweeds), 
                         mylistgroup$Seagrasses), mylistgroup$Terrestrials)
present_seagrasses_only &lt;- setdiff(setdiff(setdiff(mylistgroup$Seagrasses, mylistgroup$Duckweeds),                            mylistgroup$Terrestrials), mylistgroup$Algae)
present_terrestrials_only &lt;- setdiff(setdiff(setdiff(mylistgroup$Terrestrials, mylistgroup$Duckweeds), mylistgroup$Seagrasses), mylistgroup$Algae)
present_duckweeds_only &lt;- setdiff(setdiff(setdiff(mylistgroup$Duckweeds, mylistgroup$Terrestrials), mylistgroup$Seagrasses), mylistgroup$Algae)

print(cbind(length(present_algae_only), length(present_seagrasses_only), length(present_terrestrials_only), length(present_duckweeds_only)))</code></pre>
<pre><code>     [,1] [,2]  [,3] [,4]
[1,] 1668 2217 13562 1765</code></pre>
<p>OK those numbers fit with the above Venn diagram. I now have four lists of orthogroup names.</p>
<p>I also want the orthogroups present in aquatics + seagrasses, NOT terrestrials</p>
<pre class="r"><code>present_seagrass_and_aquatics_only &lt;- setdiff(intersect(intersect(mylistgroup$Duckweeds, mylistgroup$Seagrasses), mylistgroup$Algae),
        mylistgroup$Terrestrials)
print(length(present_seagrass_and_aquatics_only))</code></pre>
<pre><code>[1] 2</code></pre>
<p>That’s the tiny intersection of genes shared between aquatics and seagrasses, let’s also get the union of seagrass genes, aquatic genes, and their intersection</p>
<pre class="r"><code>present_seagrass_and_aquatics_union &lt;- union(mylistgroup$Algae, union(mylistgroup$Duckweeds, mylistgroup$Seagrasses))
print(length(present_seagrass_and_aquatics_union))</code></pre>
<pre><code>[1] 17574</code></pre>
<p>OK let’s also get the intersection in the middle</p>
<pre class="r"><code>present_all &lt;- intersect(intersect(mylistgroup$Algae, intersect(mylistgroup$Terrestrials, mylistgroup$Duckweeds)), mylistgroup$Seagrasses)
length(present_all)</code></pre>
<pre><code>[1] 5136</code></pre>
<p>Now we pull out these gene IDs and write them out for files to continue in <a href="GOenrichment.html">GOenrichment</a>.</p>
<pre class="r"><code># gene_per_spec has the genes per orthogroup, we need to join with the various set-results we&#39;ve made
genes_in_all &lt;- gene_per_spec %&gt;% filter(Orthogroup %in% present_all) %&gt;% separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()
genes_only_in_aquatics_and_seagrasses &lt;- gene_per_spec %&gt;% filter(Orthogroup %in% present_seagrass_and_aquatics_only) %&gt;% separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

genes_present_seagrass_and_aquatics_union &lt;- gene_per_spec %&gt;% filter(Orthogroup %in% present_seagrass_and_aquatics_union) %&gt;% separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

genes_only_in_duckweeds &lt;- gene_per_spec %&gt;% filter(Orthogroup %in% present_duckweeds_only) %&gt;% separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()
genes_only_in_algae &lt;- gene_per_spec %&gt;% filter(Orthogroup %in% present_algae_only) %&gt;% separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()
genes_only_in_seagrasses &lt;- gene_per_spec %&gt;% filter(Orthogroup %in% present_seagrasses_only) %&gt;% separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()
genes_only_in_terrestrials &lt;- gene_per_spec %&gt;% filter(Orthogroup %in% present_terrestrials_only) %&gt;% separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()</code></pre>
<p>Now we have a bunch of lists, we can write them into files for the <a href="GOenrichment.html">GO enrichment part</a></p>
<pre class="r"><code>fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_Terrestrials.txt&#39;)
writeLines(genes_only_in_terrestrials, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_Seagrasses.txt&#39;)
writeLines(genes_only_in_seagrasses, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_Duckweeds.txt&#39;)
writeLines(genes_only_in_duckweeds, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_Algae.txt&#39;)
writeLines(genes_only_in_algae, fileConn)
close(fileConn)


fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_Aquatics_and_Seagrasses.txt&#39;)
writeLines(genes_only_in_aquatics_and_seagrasses, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_union_of_Seagrass_and_Aquatics_union.txt&#39;)
writeLines(genes_present_seagrass_and_aquatics_union, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_Aquatics_and_Seagrasses_and_Terrestrials.txt&#39;)
writeLines(genes_in_all, fileConn)
close(fileConn)</code></pre>
<div id="dig-into-seagrass-specific-differences" class="section level2">
<h2>Dig into seagrass specific differences</h2>
<p>In this section, I compare the four seagrasses with each other only, so we can ignore the previous section and go back to the original orthogroups.</p>
<pre class="r"><code>seagrass_groups &lt;- groups %&gt;% select(Orthogroup, `A. antarctica`, `P. australis`, `Z. marina`, `Z. muelleri`) %&gt;%
  rowwise() %&gt;% 
  mutate(na_count = sum(is.na(c(`A. antarctica`, `P. australis`, `Z. marina`, `Z. muelleri`)))) %&gt;% 
  filter(na_count != 4) %&gt;% 
  select(-na_count)</code></pre>
<p>Let’s make a Venn diagram like above. We need a list, with the species as keys, and the c(rownumber)</p>
<pre class="r"><code>big_list &lt;- list()
big_list[[&#39;A. antarctica&#39;]] &lt;- seagrass_groups %&gt;% select(Orthogroup, `A. antarctica`) %&gt;% filter(!is.na(`A. antarctica`)) %&gt;% pull(Orthogroup)
big_list[[&#39;P. australis&#39;]] &lt;- seagrass_groups %&gt;% select(Orthogroup, `P. australis`) %&gt;% filter(!is.na(`P. australis`)) %&gt;% pull(Orthogroup)
big_list[[&#39;Z. marina&#39;]] &lt;- seagrass_groups %&gt;% select(Orthogroup, `Z. marina`) %&gt;% filter(!is.na(`Z. marina`)) %&gt;% pull(Orthogroup)
big_list[[&#39;Z. muelleri&#39;]] &lt;- seagrass_groups %&gt;% select(Orthogroup, `Z. muelleri`) %&gt;% filter(!is.na(`Z. muelleri`)) %&gt;% pull(Orthogroup)</code></pre>
<pre class="r"><code>venn &lt;- Venn(big_list)
data &lt;- process_data(venn)
# code from https://github.com/gaospecial/ggVennDiagram/blob/4cb2aa13c7beae469f9b9836c5d4f94610bd872e/R/ggVennDiagram.R to customise
region_label &lt;- data@region %&gt;%
      dplyr::filter(.data$component == &quot;region&quot;) %&gt;%
      dplyr::mutate(percent = paste(round(.data$count*100/sum(.data$count),
                                          digits = 0),&quot;%&quot;, sep=&quot;&quot;)) %&gt;%
      dplyr::mutate(both = paste(.data$count,paste0(&quot;(&quot;,.data$percent,&quot;)&quot;),sep = &quot;\n&quot;))

seagrass_venn &lt;- ggplot() +
    geom_sf(aes_string(fill=&quot;count&quot;), data = data@region) +
    geom_sf(aes_string(color = &quot;id&quot;), data = data@setEdge, show.legend = F,
            lty = &#39;solid&#39;, size = 1, color=&#39;gray&#39;) +
    geom_sf_text(aes_string(label = &quot;name&quot;), data = data@setLabel,
                 size = NA, fontface=&#39;italic&#39;,
                 color = &#39;black&#39;) +
    theme_void() +
     geom_sf_label(aes_string(label=&#39;both&#39;),
                             data = region_label,
                             alpha= 0,
                             color = &#39;black&#39;,
                             size = NA,
                             lineheight = 0.85,
                             label.size = NA) +
  scale_fill_gradientn(colours=wes_palette(&quot;Zissou1&quot;, 100, type = &quot;continuous&quot;)) + 
  theme(legend.position = &quot;none&quot;) + 
  scale_x_continuous(expand = expansion(mult = .2)) #trick from https://github.com/gaospecial/ggVennDiagram/blob/9435aa0ab4abb470c670ecb938c71576461ccedc/vignettes/using-ggVennDiagram.Rmd#L65
seagrass_venn</code></pre>
<p><img src="figure/Orthofinder.Rmd/unnamed-chunk-39-1.png" width="768" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="both-venns-together" class="section level1">
<h1>Both Venns together</h1>
<p>Let’s save both Venns so I can combine them with other plots in other Rmds.</p>
<pre class="r"><code>save(group_venn, file=&#39;output/group_venn_image.Rdata&#39;)
save(seagrass_venn, file=&#39;output/seagrass_venn_image.Rdata&#39;)</code></pre>
<pre class="r"><code>group_venn / seagrass_venn +  plot_annotation(tag_levels = &#39;A&#39;)</code></pre>
<p><img src="figure/Orthofinder.Rmd/unnamed-chunk-41-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>OK now as above, we dig out the gene names.</p>
<pre class="r"><code># gene_per_spec has the genes per orthogroup, we need to join with the various set-results we&#39;ve made
groups_only_in_A_antarctica &lt;- setdiff(setdiff(setdiff(big_list[[&#39;A. antarctica&#39;]], big_list[[&#39;P. australis&#39;]]), big_list[[&#39;Z. marina&#39;]]), big_list[[&#39;Z. muelleri&#39;]])
genes_only_in_A_antarctica &lt;- gene_per_spec %&gt;% 
  filter(name == &#39;A. antarctica&#39;, Orthogroup %in% groups_only_in_A_antarctica) %&gt;% 
  separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

groups_only_in_P_australis &lt;- setdiff(setdiff(setdiff(big_list[[&#39;P. australis&#39;]], big_list[[&#39;A. antarctica&#39;]]), big_list[[&#39;Z. marina&#39;]]), big_list[[&#39;Z. muelleri&#39;]])
genes_only_in_P_australis &lt;- gene_per_spec %&gt;% 
  filter(name == &#39;P. australis&#39;, Orthogroup %in% groups_only_in_P_australis) %&gt;% 
  separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

groups_only_in_Z_marina &lt;- setdiff(setdiff(setdiff(big_list[[&#39;Z. marina&#39;]], big_list[[&#39;A. antarctica&#39;]]), big_list[[&#39;P. australis&#39;]]), big_list[[&#39;Z. muelleri&#39;]])
genes_only_in_Z_marina &lt;- gene_per_spec %&gt;% 
  filter(name == &#39;Z. marina&#39;, Orthogroup %in% groups_only_in_Z_marina) %&gt;% 
  separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

groups_only_in_Z_muelleri &lt;- setdiff(setdiff(setdiff(big_list[[&#39;Z. muelleri&#39;]], big_list[[&#39;A. antarctica&#39;]]), big_list[[&#39;P. australis&#39;]]), big_list[[&#39;Z. marina&#39;]])
genes_only_in_Z_muelleri &lt;- gene_per_spec %&gt;% 
  filter(name == &#39;Z. muelleri&#39;, Orthogroup %in% groups_only_in_Z_muelleri) %&gt;% 
  separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

groups_in_all_seagrasses &lt;- union(union(union(big_list[[&#39;Z. muelleri&#39;]], big_list[[&#39;A. antarctica&#39;]]), big_list[[&#39;P. australis&#39;]]), big_list[[&#39;Z. marina&#39;]])

groups_lost_in_A_antarctica &lt;- setdiff(groups_in_all_seagrasses, big_list[[&#39;A. antarctica&#39;]])
genes_lost_in_A_antarctica &lt;- gene_per_spec %&gt;% 
  filter(name != &#39;A. antarctica&#39;, Orthogroup %in% groups_lost_in_A_antarctica) %&gt;% 
  separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

groups_lost_in_P_australis &lt;- setdiff(groups_in_all_seagrasses, big_list[[&#39;P. australis&#39;]])
genes_lost_in_P_australis &lt;- gene_per_spec %&gt;% 
  filter(name != &#39;P. australis&#39;, Orthogroup %in% groups_lost_in_P_australis) %&gt;% 
  separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

groups_lost_in_Z_marina &lt;- setdiff(groups_in_all_seagrasses, big_list[[&#39;Z. marina&#39;]])
genes_lost_in_Z_marina &lt;- gene_per_spec %&gt;% 
  filter(name != &#39;Z. marina&#39;, Orthogroup %in% groups_lost_in_Z_marina) %&gt;% 
  separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()

groups_lost_in_Z_muelleri &lt;- setdiff(groups_in_all_seagrasses, big_list[[&#39;Z. muelleri&#39;]])
genes_lost_in_Z_muelleri &lt;- gene_per_spec %&gt;% 
  filter(name != &#39;Z. muelleri&#39;, Orthogroup %in% groups_lost_in_Z_muelleri) %&gt;% 
  separate_rows(value, sep=&#39;,&#39;) %&gt;% pull(value) %&gt;% str_trim()</code></pre>
<p>Now we have a bunch of lists, we can write them into files for the <a href="GOenrichment.html">GO enrichment part</a></p>
<pre class="r"><code>fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_A_antarctica_not_other_seagrasses.txt&#39;)
writeLines(genes_only_in_A_antarctica, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_P_australis_not_other_seagrasses.txt&#39;)
writeLines(genes_only_in_P_australis, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_Z_marina_not_other_seagrasses.txt&#39;)
writeLines(genes_only_in_Z_marina, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_only_in_Z_muelleri_not_other_seagrasses.txt&#39;)
writeLines(genes_only_in_Z_muelleri, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_lost_in_A_antarctica_not_other_seagrasses.txt&#39;)
writeLines(genes_lost_in_A_antarctica, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_lost_in_P_australis_not_other_seagrasses.txt&#39;)
writeLines(genes_lost_in_P_australis, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_lost_in_Z_marina_not_other_seagrasses.txt&#39;)
writeLines(genes_lost_in_Z_marina, fileConn)
close(fileConn)

fileConn&lt;-file(&#39;data/Lost_present_gene_lists/Genes_lost_in_Z_muelleri_not_other_seagrasses.txt&#39;)
writeLines(genes_lost_in_Z_muelleri, fileConn)
close(fileConn)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggVennDiagram_1.2.0 kableExtra_1.3.4    UpSetR_1.4.0       
 [4] patchwork_1.1.1     wesanderson_0.3.6   RColorBrewer_1.1-2 
 [7] cowplot_1.1.1       forcats_0.5.1       stringr_1.4.0      
[10] dplyr_1.0.7         purrr_0.3.4         readr_2.0.2        
[13] tidyr_1.1.4         tibble_3.1.5        ggplot2_3.3.5      
[16] tidyverse_1.3.1     workflowr_1.6.2    

loaded via a namespace (and not attached):
 [1] fs_1.5.0           sf_1.0-5           lubridate_1.8.0    bit64_4.0.5       
 [5] webshot_0.5.2      httr_1.4.2         rprojroot_2.0.2    tools_4.1.0       
 [9] backports_1.2.1    bslib_0.3.1        utf8_1.2.2         R6_2.5.1          
[13] KernSmooth_2.23-20 DBI_1.1.1          colorspace_2.0-2   withr_2.4.2       
[17] tidyselect_1.1.1   gridExtra_2.3      bit_4.0.4          compiler_4.1.0    
[21] git2r_0.28.0       cli_3.0.1          rvest_1.0.2        xml2_1.3.2        
[25] labeling_0.4.2     sass_0.4.0         scales_1.1.1       classInt_0.4-3    
[29] proxy_0.4-26       systemfonts_1.0.4  digest_0.6.28      rmarkdown_2.11    
[33] svglite_2.1.0      pkgconfig_2.0.3    htmltools_0.5.2    dbplyr_2.1.1      
[37] fastmap_1.1.0      highr_0.9          rlang_0.4.12       readxl_1.3.1      
[41] rstudioapi_0.13    jquerylib_0.1.4    generics_0.1.1     farver_2.1.0      
[45] jsonlite_1.7.2     vroom_1.5.5        magrittr_2.0.1     Rcpp_1.0.7        
[49] munsell_0.5.0      fansi_0.5.0        lifecycle_1.0.1    stringi_1.7.5     
[53] whisker_0.4        yaml_2.2.1         plyr_1.8.6         grid_4.1.0        
[57] parallel_4.1.0     promises_1.2.0.1   crayon_1.4.1       haven_2.4.3       
[61] hms_1.1.1          knitr_1.36         pillar_1.6.4       reprex_2.0.1      
[65] glue_1.4.2         evaluate_0.14      modelr_0.1.8       vctrs_0.3.8       
[69] tzdb_0.1.2         httpuv_1.6.3       cellranger_1.1.0   gtable_0.3.0      
[73] assertthat_0.2.1   xfun_0.27          broom_0.7.9        e1071_1.7-9       
[77] later_1.3.0        class_7.3-19       RVenn_1.1.0        viridisLite_0.4.0 
[81] units_0.7-2        writexl_1.4.0      ellipsis_0.3.2    </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
