<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Phylogenetics</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Amphibolis Posidonia Comparison</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Phylogenetics</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-10-07
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Amphibolis_Posidonia_Comparison/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210414code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210414)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210414code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210414)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree08b28eacde8985a9d65405b6bc555c4c9d934f92targetblank08b28eaa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/08b28eacde8985a9d65405b6bc555c4c9d934f92" target="_blank">08b28ea</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomphilippbayerAmphibolisPosidoniaComparisontree08b28eacde8985a9d65405b6bc555c4c9d934f92targetblank08b28eaa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/tree/08b28eacde8985a9d65405b6bc555c4c9d934f92" target="_blank">08b28ea</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/OTT.nb.html
    Ignored:    analysis/plotGenes.nb.html
    Ignored:    analysis/plotRgenes.nb.html

Untracked files:
    Untracked:  Lost_GO_terms_in_five_species.PlantSpecific.xlsx
    Untracked:  Lost_GO_terms_in_five_species.xlsx
    Untracked:  data/BACKGROUND.txt
    Untracked:  data/Lost_GO_terms_in_five_species.PlantSpecific.xlsx
    Untracked:  data/Lost_GO_terms_in_five_species.xlsx
    Untracked:  data/R_genes.xlsx
    Untracked:  data/lost_in_amphi_GO.txt
    Untracked:  data/lost_in_posi_GO.txt
    Untracked:  data/lost_in_zmar_GO.txt
    Untracked:  data/lost_in_zmuel_GO.txt
    Untracked:  data/missing_amphi_vs_all_GO.txt
    Untracked:  data/missing_aquatics_GO.txt
    Untracked:  data/missing_arabidopsis_vs_all_GO.txt
    Untracked:  data/missing_posi_vs_all_GO.txt
    Untracked:  data/missing_seagrasses_GO.txt
    Untracked:  data/missing_zmar_vs_all_GO.txt
    Untracked:  data/missing_zmuel_vs_all_GO.txt
    Untracked:  data/only_in_amphi_GO.txt
    Untracked:  data/only_in_posi_GO.txt
    Untracked:  data/only_in_zmar_GO.txt
    Untracked:  data/only_in_zmuel_GO.txt
    Untracked:  data/only_seagrasses_GO.txt
    Untracked:  data/shared_lost_genes.xlsx
    Untracked:  data/species.txt
    Untracked:  species.csv
    Untracked:  whatever/

Unstaged changes:
    Modified:   data/Orthogroups.tsv
    Deleted:    data/Orthogroups_SpeciesOverlaps.tsv
    Modified:   data/Species_timetree.svg
    Modified:   data/species.csv

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/OTT.Rmd</code>) and HTML (<code>docs/OTT.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/08b28eacde8985a9d65405b6bc555c4c9d934f92/analysis/OTT.Rmd" target="_blank">08b28ea</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-10-07
</td>
<td>
wflow_publish(files = c("analysis/*"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/76c8918c38098e7cd5dc2b843a040ee9a67e3238/docs/OTT.html" target="_blank">76c8918</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/49dcbb89ac6f0a14c0066977c3ac23fdc393b3dd/analysis/OTT.Rmd" target="_blank">49dcbb8</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-15
</td>
<td>
wflow_publish(files = list.files("analysis/", pattern = "*Rmd",
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/9e449bb146de2fec9bf24d0811114ef7b3408cec/analysis/OTT.Rmd" target="_blank">9e449bb</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-14
</td>
<td>
Add missing files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/9e449bb146de2fec9bf24d0811114ef7b3408cec/docs/OTT.html" target="_blank">9e449bb</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-14
</td>
<td>
Add missing files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/9e91425d59e7e1a19bb8d36bdeeab38a06675469/docs/OTT.html" target="_blank">9e91425</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/philippbayer/Amphibolis_Posidonia_Comparison/1d42715fa5c6ab1bb2dcbded57076b3a86aa63cc/docs/OTT.html" target="_blank">1d42715</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/philippbayer/Amphibolis_Posidonia_Comparison/blob/f3d40146f815c965feb7e08cd16577557e67e3d6/analysis/OTT.Rmd" target="_blank">f3d4014</a>
</td>
<td>
Philipp Bayer
</td>
<td>
2021-04-14
</td>
<td>
wflow_publish("analysis/OTT.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Here I make a subtree from the Open Tree Of Life API for the species we have.</p>
<pre class="r"><code>library(rotl)
library(ggtree)</code></pre>
<pre><code>ggtree v3.0.4  For help: https://yulab-smu.top/treedata-book/

If you use ggtree in published research, please cite the most appropriate paper(s):

1. Guangchuang Yu. Using ggtree to visualize data on tree-like structures. Current Protocols in Bioinformatics, 2020, 69:e96. doi:10.1002/cpbi.96
2. Guangchuang Yu, Tommy Tsan-Yuk Lam, Huachen Zhu, Yi Guan. Two methods for mapping and visualizing associated data on phylogeny using ggtree. Molecular Biology and Evolution 2018, 35(12):3041-3043. doi:10.1093/molbev/msy194
3. Guangchuang Yu, David Smith, Huachen Zhu, Yi Guan, Tommy Tsan-Yuk Lam. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution 2017, 8(1):28-36. doi:10.1111/2041-210X.12628</code></pre>
<pre class="r"><code>library(cowplot)
library(ggplot2)
library(tidyverse)</code></pre>
<pre><code>-- Attaching packages --------------------------------------- tidyverse 1.3.1 --</code></pre>
<pre><code>v tibble  3.1.5     v dplyr   1.0.7
v tidyr   1.1.4     v stringr 1.4.0
v readr   2.0.2     v forcats 0.5.1
v purrr   0.3.4     </code></pre>
<pre><code>-- Conflicts ------------------------------------------ tidyverse_conflicts() --
x tidyr::expand() masks ggtree::expand()
x dplyr::filter() masks stats::filter()
x dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>species &lt;- c(&#39;Zostera marina&#39;, &#39;Zostera muelleri&#39;, &#39;Arabidopsis thaliana&#39;, &#39;Thellungiella parvula&#39;, &#39;Populus trichocarpa&#39;, &#39;Vitis vinifera&#39;, &#39;Amborella trichopoda&#39;, &#39;Oryza sativa&#39;, &#39;Zea mays&#39;, &#39;Brachypodium distachyon&#39;, &#39;Spirodela polyrhiza&#39;, &#39;Selaginella moellendorffi&#39;, &#39;Physcomitrella patens&#39;, &#39;Chlamydomonoas reinhardtii&#39;, &#39;Ostreococcus lucimarinus&#39;, &#39;Lemna gibba&#39;, &#39;Posidonia australis&#39; ,&#39;Amphibolis antarctica&#39;, &#39;Wolffia australiana&#39;)</code></pre>
<p>Let’s call the API to get the OTT IDs for these species, and to double-check we got the right names :)</p>
<p>We should get a tree for 19 species.</p>
<pre class="r"><code>taxon_search &lt;- tnrs_match_names(names = species, context_name = &quot;All life&quot;)</code></pre>
<pre class="r"><code>knitr::kable(taxon_search)</code></pre>
<table>
<colgroup>
<col width="20%" />
<col width="22%" />
<col width="13%" />
<col width="5%" />
<col width="8%" />
<col width="18%" />
<col width="11%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">search_string</th>
<th align="left">unique_name</th>
<th align="left">approximate_match</th>
<th align="right">ott_id</th>
<th align="left">is_synonym</th>
<th align="left">flags</th>
<th align="right">number_matches</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">zostera marina</td>
<td align="left">Zostera marina</td>
<td align="left">FALSE</td>
<td align="right">814202</td>
<td align="left">FALSE</td>
<td align="left">sibling_higher</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">zostera muelleri</td>
<td align="left">Zostera muelleri</td>
<td align="left">FALSE</td>
<td align="right">766348</td>
<td align="left">FALSE</td>
<td align="left">sibling_higher</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">arabidopsis thaliana</td>
<td align="left">Arabidopsis thaliana</td>
<td align="left">FALSE</td>
<td align="right">309263</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">thellungiella parvula</td>
<td align="left">Schrenkiella parvula</td>
<td align="left">FALSE</td>
<td align="right">991614</td>
<td align="left">TRUE</td>
<td align="left">incertae_sedis_inherited</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">populus trichocarpa</td>
<td align="left">Populus trichocarpa</td>
<td align="left">FALSE</td>
<td align="right">8861</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">vitis vinifera</td>
<td align="left">Vitis vinifera</td>
<td align="left">FALSE</td>
<td align="right">756728</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">amborella trichopoda</td>
<td align="left">Amborella trichopoda</td>
<td align="left">FALSE</td>
<td align="right">303950</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">oryza sativa</td>
<td align="left">Oryza sativa</td>
<td align="left">FALSE</td>
<td align="right">709894</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">zea mays</td>
<td align="left">Zea mays</td>
<td align="left">FALSE</td>
<td align="right">605194</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">brachypodium distachyon</td>
<td align="left">Brachypodium distachyon</td>
<td align="left">FALSE</td>
<td align="right">413237</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">spirodela polyrhiza</td>
<td align="left">Spirodela polyrhiza</td>
<td align="left">FALSE</td>
<td align="right">814207</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">selaginella moellendorffi</td>
<td align="left">Selaginella moellendorffii</td>
<td align="left">TRUE</td>
<td align="right">799880</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">physcomitrella patens</td>
<td align="left">Physcomitrella patens</td>
<td align="left">FALSE</td>
<td align="right">821359</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">chlamydomonoas reinhardtii</td>
<td align="left">Chlamydomonas reinhardtii</td>
<td align="left">TRUE</td>
<td align="right">33153</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">ostreococcus lucimarinus</td>
<td align="left">Ostreococcus sp. ‘lucimarinus’</td>
<td align="left">FALSE</td>
<td align="right">851102</td>
<td align="left">TRUE</td>
<td align="left">incertae_sedis</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">lemna gibba</td>
<td align="left">Lemna gibba</td>
<td align="left">FALSE</td>
<td align="right">431383</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">posidonia australis</td>
<td align="left">Posidonia australis</td>
<td align="left">FALSE</td>
<td align="right">91976</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">amphibolis antarctica</td>
<td align="left">Amphibolis antarctica</td>
<td align="left">FALSE</td>
<td align="right">460583</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">wolffia australiana</td>
<td align="left">Wolffia australiana</td>
<td align="left">FALSE</td>
<td align="right">1059895</td>
<td align="left">FALSE</td>
<td align="left"></td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<p>Let’s write the species names out again for timetree.org</p>
<pre class="r"><code>cat(capture.output(cat(taxon_search$unique_name, sep=&#39;\n&#39;), file=&quot;data/species.csv&quot;))</code></pre>
<p>That looks good to me!</p>
<pre class="r"><code>ott_in_tree &lt;- ott_id(taxon_search)[is_in_tree(ott_id(taxon_search))]</code></pre>
<pre class="r"><code>tr &lt;- tol_induced_subtree(ott_ids = ott_in_tree)</code></pre>
<pre><code>Warning in collapse_single_cpp(ances = tree$edge[, 1], desc = tree$edge[, :
printing of extremely long output is truncated</code></pre>
<pre><code>Warning in collapse_single_cpp(ances = tree$edge[, 1], desc = tree$edge[, :
printing of extremely long output is truncated</code></pre>
<pre><code>Warning in collapse_singles(tr, show_progress): Dropping singleton nodes
with labels: Streptophyta ott916750, mrcaott2ott50189, mrcaott2ott108668,
mrcaott2ott59852, mrcaott2ott8171, mrcaott2ott70394, Euphyllophyta ott1007992,
Spermatophyta ott10218, mrcaott2ott2645, mrcaott2ott35778, Mesangiospermae
ott5298374, mrcaott2ott10930, mrcaott2ott2441, mrcaott2ott969, mrcaott2ott62529,
mrcaott2ott8379, eudicotyledons ott431495, Gunneridae ott853757, Pentapetalae
ott5316182, mrcaott2ott2464, fabids ott565281, mrcaott2ott371, mrcaott2ott1479,
mrcaott2ott345, Malpighiales ott429482, mrcaott345ott3853, mrcaott3853ott8858,
mrcaott8858ott33097, mrcaott8858ott12186, mrcaott8858ott98085, Salicaceae
ott530183, mrcaott8858ott270454, mrcaott8858ott703490, mrcaott8858ott33085,
mrcaott8858ott102531, mrcaott8858ott502530, mrcaott8858ott737360,
mrcaott8858ott474976, Saliceae ott509390, Populus ott530178, mrcaott8858ott8861,
mrcaott8861ott815941, mrcaott8861ott320722, mrcaott8861ott8867, malvids
ott565277, mrcaott96ott14140, mrcaott96ott50744, mrcaott96ott378,
mrcaott378ott29446, mrcaott378ott1697, Brassicales ott8844, mrcaott378ott307071,
mrcaott378ott32461, mrcaott378ott509555, mrcaott378ott318175,
mrcaott378ott9635, mrcaott378ott125843, mrcaott378ott509568, mrcaott378ott28763,
mrcaott378ott83547, Brassicaceae ott309271, mrcaott378ott299734,
mrcaott378ott4671, mrcaott4671ott58909, mrcaott4671ott6278, mrcaott6278ott15318,
mrcaott6278ott158438, mrcaott6278ott10585, mrcaott6278ott34460,
mrcaott6278ott9083, mrcaott9083ott19798, mrcaott19798ott31487,
mrcaott31487ott88883, mrcaott31487ott152275, mrcaott11023ott24850,
mrcaott11023ott56298, mrcaott11023ott95692, mrcaott11023ott25468,
mrcaott11023ott25476, Schrenkiella (genus in kingdom Archaeplastida)
ott5740975, Vitales ott1069308, mrcaott8384ott10050, mrcaott10050ott337881,
mrcaott10050ott194302, mrcaott10050ott302228, mrcaott10050ott24244,
mrcaott24244ott240260, mrcaott24244ott39768, mrcaott39768ott211705,
mrcaott39768ott175159, mrcaott175159ott602495, mrcaott175159ott246514,
mrcaott175159ott903683, mrcaott175159ott568892, mrcaott175159ott235835,
mrcaott175159ott941927, mrcaott175159ott568880, mrcaott175159ott175164,
mrcaott175159ott509603, mrcaott509603ott805071, Liliopsida ott1058517,
Petrosaviidae ott5308424, mrcaott121ott4474, mrcaott121ott1439,
mrcaott121ott334, commelinids ott225270, mrcaott252ott213153, Poales ott921871,
mrcaott252ott128594, mrcaott252ott1477, mrcaott252ott7120, mrcaott252ott285512,
mrcaott252ott3717, mrcaott252ott272812, mrcaott252ott334529,
mrcaott252ott427739, Poaceae ott508090, mrcaott252ott196505, mrcaott252ott43427,
mrcaott252ott751372, mrcaott252ott11561, Oryzoideae ott641467, Oryzeae
ott415723, mrcaott36487ott890607, Oryzinae ott5744622, Oryza ott135764,
mrcaott67120ott709900, mrcaott67120ott272949, mrcaott67120ott709898,
mrcaott67120ott135756, mrcaott135756ott709892, mrcaott135756ott709887,
mrcaott656ott1473, mrcaott1473ott33975, Brachypodieae ott693951, Brachypodium
ott413242, mrcaott1787ott2051, mrcaott1787ott1985, Panicoideae ott641461,
mrcaott6065ott9925, Andropogonodae ott5737327, Andropogoneae ott475213,
Tripsacinae ott5759522, Zea ott605186, Araceae ott481972, mrcaott290ott3983,
Lemna ott1008186, mrcaott338992ott338994, mrcaott338994ott431388,
mrcaott338994ott431383, mrcaott66494ott407568, mrcaott407568ott432842,
mrcaott407568ott1059896, mrcaott407568ott1059895, mrcaott1059895ott1059900,
Spirodela ott431386, mrcaott5202ott159280, mrcaott5202ott30666,
mrcaott30666ott106181, mrcaott30666ott208913, mrcaott30666ott814198,
mrcaott30666ott40117, Zosteraceae ott637476, mrcaott40117ott91951,
mrcaott40117ott197961, mrcaott197961ott817762, mrcaott197961ott529863,
mrcaott197961ott766348, mrcaott766348ott766350, mrcaott436201ott436206,
mrcaott436201ott436203, mrcaott436201ott814202, mrcaott87589ott256106,
mrcaott256106ott460583, mrcaott460583ott514367, mrcaott460583ott613178,
Amphibolis ott460577, mrcaott91974ott351467, Posidoniaceae ott460581, Posidonia
(genus in kingdom Archaeplastida) ott91949, mrcaott91976ott928713, Amborellales
ott927960, Amborellaceae ott295910, Amborella ott303952, Lycopodiopsida
ott144795, mrcaott3661ott18736, Selaginellales ott144821, Selaginellaceae
ott144818, Selaginella ott144816, mrcaott10602ott10604, mrcaott10604ott10608,
mrcaott10604ott78166, mrcaott10604ott10618, mrcaott101177ott144813,
mrcaott532022ott583202, mrcaott532022ott799880, mrcaott541ott1066,
Bryophyta ott246594, mrcaott1066ott204205, Bryophytina ott471195, Bryopsida
ott821346, mrcaott1066ott86152, mrcaott1066ott1769, mrcaott1769ott7813,
mrcaott7813ott148635, mrcaott7813ott86148, mrcaott7813ott454749,
mrcaott7813ott86154, mrcaott7813ott31044, mrcaott31044ott844044,
mrcaott31044ott1060862, Physcomitrella ott821349, Chlorophyta ott979501,
mrcaott185ott42071, mrcaott185ott1426, mrcaott1426ott1544, mrcaott1544ott8659,
mrcaott1544ott15345, mrcaott1544ott9282, mrcaott9389ott818260,
mrcaott9389ott23557, mrcaott23557ott527099</code></pre>
<pre class="r"><code>plot(tr)</code></pre>
<p><img src="figure/OTT.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Let’s get rid of those OTT IDs from the tree, I don’t like that. There’s a helper function for that</p>
<pre class="r"><code>tr$tip.label &lt;- strip_ott_ids(tr$tip.label, remove_underscores = TRUE)</code></pre>
<pre class="r"><code>ggtree(tr) + 
  geom_tree() + 
  geom_tiplab(size=3,
              fontface=&#39;italic&#39;) +
  theme_tree() +
  xlim(0, 15)</code></pre>
<p><img src="figure/OTT.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Cool! I took the above species list and uploaded to timetree (see Species_timetree.svg)</p>
<pre class="r"><code>plot(ggdraw()+cowplot::draw_image(&#39;./data/species_timetree.svg&#39;))</code></pre>
<pre><code>Warning: Package `magick` is required to draw images. Image not drawn.</code></pre>
<p><img src="figure/OTT.Rmd/unnamed-chunk-10-1.png" width="1344" style="display: block; margin: auto;" /></p>
<p>Timetree also lets you export the tree timed with MYA</p>
<pre class="r"><code>tree &lt;- read.tree(&#39;./data/timetree_species.nwk&#39;)</code></pre>
<pre class="r"><code>tree$tip.label &lt;- str_replace_all(tree$tip.label, &#39;_&#39;, &#39; &#39;)
p &lt;- ggplot(tree) + 
  geom_tree() + 
  theme_tree2() +
  scale_x_continuous(labels = abs) + 
  geom_tiplab(size=3,  fontface=&#39;italic&#39;)

revts(p) + xlim(-1200, 300)</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/OTT.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.1 (2021-08-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17134)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] forcats_0.5.1   stringr_1.4.0   dplyr_1.0.7     purrr_0.3.4    
 [5] readr_2.0.2     tidyr_1.1.4     tibble_3.1.5    tidyverse_1.3.1
 [9] ggplot2_3.3.5   cowplot_1.1.1   ggtree_3.0.4    rotl_3.0.11    
[13] workflowr_1.6.2

loaded via a namespace (and not attached):
 [1] nlme_3.1-152       fs_1.5.0           lubridate_1.7.10   progress_1.2.2    
 [5] httr_1.4.2         rprojroot_2.0.2    tools_4.1.1        backports_1.2.1   
 [9] utf8_1.2.2         R6_2.5.1           DBI_1.1.1          lazyeval_0.2.2    
[13] colorspace_2.0-2   withr_2.4.2        tidyselect_1.1.1   prettyunits_1.1.1 
[17] curl_4.3.2         compiler_4.1.1     git2r_0.28.0       cli_3.0.1         
[21] rvest_1.0.1        xml2_1.3.2         labeling_0.4.2     scales_1.1.1      
[25] digest_0.6.28      yulab.utils_0.0.2  rmarkdown_2.11     rentrez_1.2.3     
[29] pkgconfig_2.0.3    htmltools_0.5.2    dbplyr_2.1.1       fastmap_1.1.0     
[33] highr_0.9          rlang_0.4.11       readxl_1.3.1       rstudioapi_0.13   
[37] farver_2.1.0       gridGraphics_0.5-1 jquerylib_0.1.4    generics_0.1.0    
[41] jsonlite_1.7.2     magrittr_2.0.1     ggplotify_0.1.0    patchwork_1.1.1   
[45] Rcpp_1.0.7         munsell_0.5.0      fansi_0.5.0        ape_5.5           
[49] lifecycle_1.0.1    stringi_1.7.5      whisker_0.4        yaml_2.2.1        
[53] grid_4.1.1         parallel_4.1.1     promises_1.2.0.1   crayon_1.4.1      
[57] rncl_0.8.4         lattice_0.20-44    haven_2.4.3        hms_1.1.1         
[61] knitr_1.36         pillar_1.6.3       reprex_2.0.1       XML_3.99-0.8      
[65] glue_1.4.2         evaluate_0.14      ggfun_0.0.4        modelr_0.1.8      
[69] vctrs_0.3.8        treeio_1.16.2      tzdb_0.1.2         httpuv_1.6.3      
[73] cellranger_1.1.0   gtable_0.3.0       assertthat_0.2.1   xfun_0.26         
[77] broom_0.7.9        tidytree_0.3.5     later_1.3.0        aplot_0.1.1       
[81] ellipsis_0.3.2    </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
