---
title: "Visualising metagenome differences"
author: "Philipp Bayer"
date: "20th Nov 2023"
output: html_document
---

```{r setup, message=FALSE}
library(tidyverse)
library(DT)
library(microshades)
library(ggtext)
library(scales)
knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())

```

```{r}
trunc_str <- function(latin_name) {
  tmp <- str_split(latin_name, pattern = ' ')[[1]]
  return(paste0(substring(tmp[1], 1, 1), '. ', tmp[2]))
}

```


Here we compare metagenomes (taxonomy and genes) across Amphibolis vs the others.

```{r}
metadata <- read_tsv('./data/metagenome/samples.tsv')
metadata <- metadata |> rename(Sample_species = Species,
                               Sample_replicate = Replicate) |> 
  mutate(Tissue = str_to_title(Tissue))
```

# Taxonomies

We have two taxonomies: one based on CAT, one based on GTDB. GTDB is less complete but more accurate. Let's see what they look like.

# GTDB
```{r}
tax <- read_tsv('./data/metagenome/all_gtdb.summary.tsv.gz')
```

Need to put some extra work in; MAGs with no classification are not in the output file.

```{r}
all_mag_counts <- read_tsv('./data/metagenome/MAG_counts.tsv') |> mutate(Sample = str_replace(Sample, '_L2', '')) |> 
  rename(TotalMAGs = MAGs)
```

```{r}
clean_tax <- tax |> 
  separate(user_genome, into = c('sample', 'MAG'), sep='\\.') |> 
  separate(classification, into = c('domain', 'phylum', 
                                    'class', 'order', 
                                    'family', 'genus', 
                                    'species'), sep=';') |> 
  mutate(sample = str_replace_all(sample, 'MEGAHIT-', ''), 
         sample = str_replace_all(sample, '_L2', ''))
         
clean_tax <- clean_tax |> left_join(metadata, by = c('sample'='Sample'))
```

Next, we keep only Posidonia australis and Amphibolis antarctica samples

```{r}
amph_pos_tax <- clean_tax |> filter(Sample_species %in% c('Amphibolis antarctica', 'Posidonia australis')) |> 
  left_join(all_mag_counts, by = c('sample'='Sample'))
```

## GTDB by order

```{r}
colors <-c(microshades_palette("micro_blue", 4, lightest = FALSE), 
           microshades_palette("micro_purple", 4, lightest = FALSE),
           microshades_palette("micro_green", 4, lightest = FALSE),
           microshades_palette("micro_orange", 4, lightest = FALSE))
names(colors) <- c("Rhizobiales", "Clostridiales", "Desulfobacterales", "Chitinophagales", 
"Chromatiales", "Solirubrobacterales", "Xanthomonadales", "Acidimicrobiales", 
"Rhodobacterales", "Corallinales", "Ceramiales", "Alismatales", 
"Thiotrichales", "Flavobacteriales", "Remainder", "Unclassified")
```

```{r fig.height=6}
plot_orders_gtdb <- function(amph_pos_tax, group_remainder = TRUE) {
  temp <- amph_pos_tax |> 
    group_by(sample, Tissue, Sample_species,TotalMAGs) |> 
    mutate(Sample_species = trunc_str(Sample_species)) |> 
    mutate(Sample_species = paste0('*', Sample_species, '*')) |> 
    mutate(order = str_replace_all(order, 'o__','')) |> 
    count(order, .drop = FALSE) 
  if(group_remainder) {
    temp <- temp |> 
    mutate(counted_orders = case_when(n == 1 ~ 'Remainder',
                                      TRUE ~ order))
  } else {
      temp <- temp |> 
        mutate(counted_orders = order)
  }
  temp2 <- temp |> 
    select(-order) |> 
    group_by(sample, Tissue, Sample_species, n, counted_orders) |> 
    summarise(n = sum(n)) |> 
    filter(counted_orders != '' )
  # we need to count the unclassified MAGs that are not in the output
  group_presents <- temp2 |> group_by(sample) |> summarise(classifiedMAGs = sum(n))
  missings <- group_presents |> 
    left_join(temp |> 
                select(sample, TotalMAGs)) |> 
    unique() |> 
    mutate(unclassifiedMAGs = TotalMAGs - classifiedMAGs) |> 
    select(-c(classifiedMAGs,TotalMAGs)) |>
    rename(Unclassified = unclassifiedMAGs) |>
    pivot_longer(-c(sample, Tissue, Sample_species)) |> 
    rename(counted_orders=name, n=value)

  temp2 <-rbind(temp2, missings)

  uniques <- c(setdiff(unique(temp2$counted_orders), c('Unclassified', 'Remainder')), 'Remainder', 'Unclassified')
  temp2 |> 
    mutate(counted_orders = factor(counted_orders, levels=uniques)) |> 
    mutate(sample = str_remove(sample, "[A-Z]+")) |> 
    mutate(sample = str_replace(sample, '31', '3')) |> 
    ggplot(aes(x = factor(sample, levels=rev(unique(sample))), y = n, fill=counted_orders)) +
    geom_col() + 
    theme_minimal() + 
    facet_wrap(~Tissue+Sample_species, scales = 'free_y', ncol=2) + 
    coord_flip() + 
    ylab('Number of contigs') + xlab('Replicate') +
    labs(fill = 'Order') +
    scale_y_continuous(breaks=pretty_breaks())  +
    # move legend position to bottom
    theme(legend.position = "bottom",
          strip.text.x = element_markdown()) +
    scale_fill_manual(values = colors)
}
plot_orders_gtdb(amph_pos_tax)
```

## GTDB by family


```{r}
amph_pos_tax |> 
  group_by(sample, Tissue) |> 
  mutate(family = str_replace_all(family, 'f__','')) |> 
  count(family) |> 
  filter ( n > 1) |> 
  filter(family != '' ) |> 
  ggplot(aes(x = sample, y = n, fill=family)) +
  geom_col() + 
  theme_minimal() + 
  #theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
  facet_wrap(~Tissue) + coord_flip() + 
  ylab('Number of contigs') + xlab('Sample')
```
```{r}
amph_pos_tax |> group_by(sample, species) |> filter(species != 's__') |> 
  count(species)
```

# CAT
Let's do the same using CAT


```{r}
tax_cat <- read_tsv('./data/metagenome/all_cat.summary.reformatted.tsv.gz')
```

```{r}
clean_tax_cat <- tax_cat |> 
  separate(`# bin`, into = c('sample', 'MAG', 'fa'), sep='\\.') |> 
  mutate(sample = str_replace_all(sample, 'MEGAHIT-', ''), 
         sample = str_replace_all(sample, '_L2', '')) |> 
  select(-fa)
         
clean_tax_cat <- clean_tax_cat |> left_join(metadata, by = c('sample'='Sample')) |> left_join(all_mag_counts, by =c('sample'='Sample'))
```

Next, we keep only Posidonia australis and Amphibolis antarctica samples

```{r}
amph_pos_tax_cat <- clean_tax_cat |> filter(Sample_species %in% c('Amphibolis antarctica', 'Posidonia australis'))
```

## CAT by order

```{r fig.height=6}
plot_orders_cat <-
  function(amph_pos_tax_cat, group_remainder = TRUE) {
    temp <- amph_pos_tax_cat |>
      separate(Order, into = c('Order', 'Order_score')) |>
      rename(order = Order) |>
      group_by(sample, Tissue, Sample_species, TotalMAGs) |>
      mutate(Sample_species = trunc_str(Sample_species)) |>
      mutate(Sample_species = paste0('*', Sample_species, '*')) |>
      count(order)
    if (group_remainder) {
      temp <- temp |>
        mutate(counted_orders = case_when(n == 1 ~ 'Remainder',
                                          TRUE ~ order))
    } else {
      temp <- temp |>
        mutate(counted_orders = order)
    }
    temp2 <- temp |>
      select(-order) |>
      group_by(sample, Tissue, Sample_species, n, counted_orders,TotalMAGs) |>
      summarise(n = sum(n)) |>
      filter(counted_orders != '')
      # we need to count the unclassified MAGs that are not in the output
    group_presents <- temp2 |> group_by(sample) |> summarise(classifiedMAGs = sum(n))
    missings <- group_presents |> 
      left_join(temp2) |> 
      select(classifiedMAGs, Tissue, Sample_species, sample, TotalMAGs)|> 
      unique() |> 
      mutate(unclassifiedMAGs = TotalMAGs - classifiedMAGs) |> 
      select(-c(classifiedMAGs,TotalMAGs)) |>
      rename(Unclassified = unclassifiedMAGs) |>
      pivot_longer(-c(sample, Tissue, Sample_species)) |> 
      rename(counted_orders=name, n=value)

    temp2 <-rbind(temp2, missings)

   uniques <- c(setdiff(unique(temp2$counted_orders), c('Unclassified', 'Remainder')), 'Remainder', 'Unclassified')
 
    temp2 |>
      mutate(counted_orders = factor(counted_orders, levels = uniques)) |>
      mutate(sample = str_remove(sample, "[A-Z]+")) |>
      mutate(sample = str_replace(sample, '31', '3')) |>
      ggplot(aes(
        x = factor(sample, levels = rev(unique(sample))),
        y = n,
        fill = counted_orders
      )) +
      geom_col() +
      theme_minimal() +
      facet_wrap( ~ Sample_species + Tissue, ncol = 2, scales = 'free_y') +
      coord_flip() +
      ylab('Number of contigs') + xlab('Replicate') +
      labs(fill = 'Order') +
      scale_y_continuous(breaks = pretty_breaks())  +
      # move legend position to bottom
      theme(legend.position = "bottom",
            strip.text.x = element_markdown()) +
      scale_fill_manual(values = colors)
  }
plot_orders_cat(amph_pos_tax_cat)
```

## CAT by family

```{r}
amph_pos_tax_cat |> 
  separate(Family, into = c('Family', 'Family_score')) |> 
  group_by(sample, Tissue) |> 
  count(Family) |> 
  filter ( n > 1) |> 
  filter(Family != '' ) |> 
  ggplot(aes(x = sample, y = n, fill=Family)) +
  geom_col() + 
  theme_minimal() + 
  #theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
  facet_wrap(~Tissue) + coord_flip() + 
  ylab('Number of contigs') + xlab('Sample')
```

```{r}
amph_pos_tax_cat |> separate(Species, into = c('Species', 'Species_score'), sep =':') |> filter(!is.na(Species)) |>  group_by(sample, Species) |> 
  count(Species)
```
```{r}
amph_pos_tax_cat |> separate(Species, into = c('Species', 'Species_score'), sep =':') |> filter(!is.na(Species)) |>  group_by(sample, Species) |> 
  count(Species) |> filter(str_detect(Species, 'Acidihalobacter'))
```

There we go :) Which MAGs are those?
```{r}
amph_pos_tax_cat |> filter(str_detect(Species, 'Acidihalobacter')) |> datatable()

```

```{r}
amph_pos_tax_cat |> filter(str_detect(Family, 'Ectothiorhodospiraceae'))|> datatable()
```
Are these in the GTDB classification?
```{r}
amph_pos_tax |> filter(str_detect(family, 'Ectothiorhodospiraceae'))|> datatable()
```
Nope!

# Gene comparison

Let's compare gene presence/absence by names first

```{r}
genes <- read_tsv('././data/metagenome/no_hypothetical.all_genes.tsv.gz')

genes <- genes |> 
  separate(filename, into = c('sample', 'MAG'), sep='\\.', extra = 'drop') |>  # get rid of extra filenaming stuff
  mutate(sample = str_replace(sample, 'MEGAHIT-', '')) |> 
  separate(MAG, into = c('MAG', 'rest'), sep ='/') |> 
  select(-rest)
```

```{r}
genessplit <- genes |> separate(gene, into = c('gene', 'gene_copy_number'), sep='_')
```

```{r}
genessplit <- genessplit |> mutate(sample = str_replace(sample, '_L2', '')) |> 
  left_join(metadata, by = c('sample'='Sample')) |> 
  filter(Sample_species %in% c('Amphibolis antarctica', 'Posidonia australis'))

```

```{r}
count_diffs <- genessplit |> 
  group_by(Sample_species, gene) |> 
  count(gene) |> 
  pivot_wider(names_from = Sample_species, values_from = n) |> 
  mutate(difference = `Amphibolis antarctica` - `Posidonia australis`) |> 
  arrange(desc(difference))
```

```{r}
count_diffs |> filter(`Posidonia australis` <100) |> datatable()
```

We hypothesise that the ACC precursor-existing genes in Amphibolis lead to different microbiomes. ACC is broken down by ACC deaminases, which are present in Amphibolis but not Posidonia. Let's see if we can find ACC deaminases in the metagenomes (*acdS*).

```{r}
genessplit |> filter(gene == 'acdS') |> 
  group_by(Sample_species, Tissue, Sample_replicate) |> 
  count(gene) |> 
  mutate(Sample_species = trunc_str(Sample_species)) |> 
  mutate(Sample_species = paste0('*', Sample_species, '*')) |> 
  ggplot(aes(x= interaction(factor(Sample_replicate, levels=4:1), Tissue), y = n, fill = Sample_species)) + 
  geom_col() + 
  facet_wrap(~Sample_species) +
  coord_flip()+ theme_minimal() + 
  scale_y_continuous(breaks=pretty_breaks()) +
  ylab('Total number of *acdS*-containing MAGs') +
  xlab('Sample species') + 
  theme(legend.position = 'none',
        strip.text.x =  element_markdown(),
        axis.title.x = element_markdown())
```

What are those acdS-containing MAGs?


```{r}
genessplit |> filter(gene == 'acdS') |> left_join(amph_pos_tax, by=c('sample', 'MAG'), multiple = 'any') |> datatable()
```
Lots of unknown taxonomies :( 

```{r}
genessplit |> filter(gene == 'acdS') |> left_join(amph_pos_tax, by=c('sample', 'MAG'), multiple = 'any') |> 
  select(-c(Tissue.y, Sample_species.y)) |>
  rename(Tissue = Tissue.x, Sample_species = Sample_species.x) |> 
  plot_orders_gtdb(group_remainder = FALSE)
  #select(sample, MAG, domain:species)
```


```{r}
genessplit |> filter(gene == 'acdS') |> 
  left_join(amph_pos_tax_cat, by=c('sample', 'MAG')) |> 
    select(-c(Tissue.y, Sample_species.y)) |>
  rename(Tissue = Tissue.x, Sample_species = Sample_species.x) |> 
  plot_orders_cat(group_remainder = FALSE) #
  #select(sample, MAG, Superkingdom:Species) |> 
  
```

Let's chekc for nitrogen fixation genes (*nif*) - hypothesis is that Amphibolis roots contain more of these

```{r}
genessplit |> filter(str_detect(gene, 'nifH') | str_detect(gene, 'acdS') | str_detect(gene, 'sctC')) |> 
  mutate(gene_class = case_when(str_detect(gene, 'nifH') ~ '*nifH*',
                                str_detect(gene, 'acdS') ~ '*acdS*',
                                str_detect(gene, 'sctC') ~ '*sctC*')) |>
  #group_by(Sample_species, Tissue, Sample_replicate, MAG) |> 
  #summarise(gene_class = paste0(sort(unique(gene_class)), collapse = ' & ')) |>
  group_by(Sample_species, Tissue, Sample_replicate) |> 
  count(gene_class) |> 
  mutate(Sample_species = trunc_str(Sample_species)) |> 
  mutate(Sample_species = paste0('*', Sample_species, '*')) |> 
  ggplot(aes(x=factor(Sample_species, levels=rev(c(unique(Sample_species)))), y=n, fill=gene_class)) + 
  geom_boxplot(outlier.shape = NA) +
  facet_wrap(~factor(Tissue, levels=rev(c(unique(Tissue))))) + 
  geom_point(position = position_jitterdodge(), aes(color=gene_class)) +
  coord_flip() +
  theme_minimal() +
  labs(color='MAG gene content', fill='MAG gene content') +
  theme(axis.text.y =  element_markdown(),legend.text = element_markdown()) +
  ylab('Count of MAGs') + xlab('Species')

```

```{r}
genessplit |> filter(str_detect(gene, 'nifH') | str_detect(gene, 'acdS') | str_detect(gene, 'sctC')) |> 
  mutate(gene_class = case_when(str_detect(gene, 'nifH') ~ '*nifH*',
                                str_detect(gene, 'acdS') ~ '*acdS*',
                                str_detect(gene, 'sctC') ~ '*sctC*')) |>
  #group_by(Sample_species, Tissue, Sample_replicate, MAG) |> 
  #summarise(gene_class = paste0(sort(unique(gene_class)), collapse = ' & ')) |>
  group_by(Sample_species, Tissue, Sample_replicate) |> 
  count(gene_class) |> 
  mutate(Sample_species = trunc_str(Sample_species)) |> 
  mutate(Sample_species = paste0('*', Sample_species, '*')) |> 
  ggplot(aes(x=factor(Tissue, levels=rev(c('Shoot/Leaf', 'Shoot/Leaf Biofilm', 'Rhizome', 'Root'))), y=n, fill=Sample_species)) + 
  geom_boxplot(outlier.shape = NA) +
  facet_wrap(~factor(gene_class, levels=c('*acdS*', '*nifH*', '*sctC*')), scales = 'free_x') + 
  geom_point(position = position_jitterdodge(), aes(color=Sample_species)) +
  coord_flip() +
  theme_minimal() +
  labs(color='Species', fill='Species') +
  theme(axis.text.y =  element_markdown(),legend.text = element_markdown(),
        strip.text.x = element_markdown()) +
  ylab('Count of MAGs') + xlab('Species')
```
I like this one more. The differences are stronger.


# Statistic test

Can we make a simple association test?

First, a general one:

```{r}
tmp <- genessplit |> filter(str_detect(gene, 'nifH') | str_detect(gene, 'acdS') | str_detect(gene, 'sctC')) |> 
  mutate(gene_class = case_when(str_detect(gene, 'nifH') ~ '*nifH*',
                                str_detect(gene, 'acdS') ~ '*acdS*',
                                str_detect(gene, 'sctC') ~ '*sctC*')) |> 
  group_by(Sample_species, Sample_replicate, Tissue) |> 
  count(gene_class) |> 
  mutate(Sample_species = factor(Sample_species, levels = c('Posidonia australis', 'Amphibolis antarctica'))) 
summary(lm(n ~ gene_class + Sample_species + Sample_replicate + Tissue, data=tmp))
```

OK! So we have an association with gene_class, which is obvious, but we also have an association with species (great!) and root-tissue (also great!)

## Subtests
Now let's make the specific ones; nifh, acdS, and sctC counts within Tissue=root across both species

### nifH
```{r}
tmp <- genessplit |> filter(str_detect(gene, 'nifH') & str_detect(Tissue, 'Root')) |> 
  mutate(gene_class = case_when(str_detect(gene, 'nifH') ~ 'nifH')) |> 
  group_by(Sample_species, Sample_replicate) |> count(gene_class) 
tmp  |> datatable()
```

```{r}
t.test(n ~ Sample_species, data=tmp)
```

P < 0.0005

### sctC

```{r}
tmp <- genessplit |> filter(str_detect(gene, 'sctC') & str_detect(Tissue, 'Root')) |> 
  mutate(gene_class = case_when(str_detect(gene, 'sctC') ~ 'sctC')) |> 
  group_by(Sample_species, Sample_replicate) |> count(gene_class) 
tmp  |> datatable()
```

```{r}
t.test(n ~ Sample_species, data=tmp)
```

p < 0.005

### acdS

```{r}
tmp <- genessplit |> filter(str_detect(gene, 'acdS') & str_detect(Tissue, 'Root')) |> 
  mutate(gene_class = case_when(str_detect(gene, 'acdS') ~ 'acdS')) |> 
  group_by(Sample_species, Sample_replicate) |> count(gene_class) 
tmp |> datatable()
```

can't run t-test, only one group present!

```{r}
# t.test(n ~ Sample_species, data=tmp)
```


