---
title: "Visualising MSAs"
author: "Philipp Bayer"
date: "10 March 2022"
output: html_document
---

I found some interesting gene clusters. Let's look at them

```{r setup, message=FALSE, warning=FALSE}
library(tidyverse)
library(ggmsa)
library(Biostrings)
library(ape)
library(ggtree)
library(treeio)
knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())
```

```{r fig.width=12}
ggmsa('data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.fasta', start = 221, end = 260, char_width = 0.5, seq_name = T) + geom_seqlogo() + geom_msaBar()
```

```{r}

x <- readAAStringSet('data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.fasta')
d <- as.dist(stringDist(x, method = "hamming")/width(x)[1])
tree <- bionj(d)
p <- ggtree(tree) + geom_tiplab()
ggtree(tree) + geom_tiplab()
```

```{r fig.width=10}
data <- tidy_msa('data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.fasta', 221, 300)
p + geom_facet(geom = geom_msa, data = data,  panel = 'Multiple Sequence Alignment (221-300AA)',
               font = NULL, color = "Clustal", by_conversation= TRUE) +
    xlim_tree(1)
```


Let's use a RAXML made tree

Commands run, after I shortened protein IDs manually. I pulled out the rice/Arabidopsis/seagrass proteins manually from the OG0000189.fa Orthofinder produced, and shortened their names so they fit into Phylip format.

```muscle -in OG0000189.RiceAraSeagrasses.fa -out OG0000189.RiceAraSeagrasses.phy -phyi ```

```raxmlHPC -p12345 -m PROTGAMMAAUTO -s OG0000189.RiceAraSeagrasses.phy -n AUTO```


```{r}
treestring <- '((3O.sativa:0.11386470583955717040,4O.sativa:0.03615896391829190315):0.07379737996482453599,(5O.sativa:0.33751444720146173140,((7A.thalian:0.45696796911334131019,((8Z.mueller:0.33996861614583556710,15Z.marina:0.16088168380872580610):0.31923212677234219514,10P.austra:0.57486248434538900209):0.22324965676023655892):0.10039129162924786964,((13Z.marina:0.11346816437532522559,(12Z.muelle:0.00973855109814669891,10Z.muelle:0.02645894329604922546):0.09840311662577182206):0.35332170475064506032,((14Z.marina:0.12821969530885793387,(9Z.mueller:0.06883167458475099310,11Z.muelle:0.02355162378347611801):0.15839770535961147924):0.63099939044074537797,(1A.amphibo:0.53760770644490452064,6P.austral:0.07001018412366488697):0.08839123570898912985):0.05074368132159561007):0.09141392275039983417):0.12967268161775510893):0.11320488597472283532,2O.sativa:1.70657194075952300949):0.0;'
plot(ape::read.tree(text=treestring))
p2 <- ggtree(ape::read.tree(text=treestring)) + geom_tiplab()
p2
```

That's very different from the above dendrogram!

```{r fig.width=10}
data2 <- data %>% mutate(name = str_sub(name, 1, 10), 
                         name = str_trim(name))
p2 + geom_facet(geom = geom_msa, data = data2,  panel = 'Multiple Sequence Alignment (221-260AA)',
               font = NULL, color = "Clustal") +
    xlim_tree(2)
```

Good! I used blastp with Swissprot to see whether I could get 'official' gene names for some of these, especially the *O. sativa*. I uploaded the fasta in data/OG000189.fa to blastp/swissprot, and pulled out new names where available.

Hits are to: A. thaliana Calcineurin B-like protein 4 (CBL4, Alternative name: SOS3)
O. sativa Calcineurin B-like protein 8 (CBL8)
O. sativa Calcineurin B-like protein 4 (CBL4)

Let's rename the tree and the MSA data table using `rename_taxa`
```{r}
trees <- ape::read.tree(text=treestring)

rename_df <- tibble::tribble(
                       ~old,        ~new,
                "3O.sativa",    "OsCBL8",
                "4O.sativa",  "OsCBL7.1",
                "5O.sativa",    "OsCBL4",
               "7A.thalian",    "AtSOS3",
               "8Z.mueller",   "ZmuSOS3",
               "15Z.marina",   "ZmaSOS3",
               "10P.austra",    "PaSOS3",
               "13Z.marina", "ZmaSOS3.2",
               "12Z.muelle", "ZmuSOS3.2",
               "10Z.muelle", "ZmuSOS3.3",
               "14Z.marina", "ZmaSOS3.2",
               "9Z.mueller", "ZmuSOS3.4",
               "11Z.muelle", "ZmuSOS3.5",
               "1A.amphibo",    "AaSOS3",
               "6P.austral",  "PaSOS3.2",
                "2O.sativa",  "OsCBL7.2"
               )


trees <- rename_taxa(trees, rename_df, old, new)
p3 <- ggtree(trees) + geom_tiplab(fontface='italic')

p3
```

```{r}
#str_replace_all takes a named vector
replace_vector <- rename_df$new
names(replace_vector) <- rename_df$old
data2 <- data2 %>% mutate(name = str_replace_all(name, replace_vector))
```


```{r fig.width=10}
final_p <- p3 + geom_facet(geom = geom_msa, data = data2,  panel = 'Multiple Sequence Alignment (221-260AA)',
               font = NULL, color = "Chemistry_AA") +
    xlim_tree(2)
final_p
```

```{r}
cowplot::save_plot(final_p, filename = 'output/SOS3_phylogeny.png', base_width=10)
```



"However, apparent photosynthesis is still maintained at a salinity 15% that of normal seawater and at temperatures of 3 and 30Â°C, consistent with the ecological role of Z. muelleri as an intertidal species." https://www.sciencedirect.com/science/article/abs/pii/0304377085900634


Now let's also add support values.

I ran this based on `Using RAxML-NG in Practice`

## Phylogeny for the SOS3 cluster
```

 # to get best model
modeltest-ng -i OG0000189.RiceAraSeagrasses.aln.fasta -t ml -d aa -p 8

 # to get fixed fasta
raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta --model JTT-DCMUT+G4 --check

# to make a regular tree
raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta.raxml.reduced.phy --model JTT-DCMUT+G4 --prefix T3 --threads 2 --seed 2 

# make 200 bootstrap trees, does not converge
raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta.raxml.reduced.phy --model JTT-DCMUT+G4 --prefix T8 --threads 8 --seed 2 --bootstrap --bs-trees 200 

# make another 400  with different seed
raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta.raxml.reduced.phy --model JTT-DCMUT+G4 --prefix T11 --threads 8 --seed 333 --bootstrap --bs-trees 400 

# check whether they converge with <3% WRF cutoff
raxml-ng --bsconverge --bs-trees allbootstraps --prefix T12 --seed 2 --threads 1 --bs-cutoff 0.03


# to make the final trees with bootstrap values

cat T8.raxml.bootstraps T11.raxml.bootstraps > allbootstraps

raxml-ng --support --tree T3.raxml.bestTree --bs-trees allbootstraps --prefix T13 

#yes, after 550 - close one!

# let's make another 400 so we can have a nice 1000 bootstraps - different seed again!

raxml-ng --msa OG0000189.RiceAraSeagrasses.aln.fasta.raxml.reduced.phy --model JTT-DCMUT+G4 --prefix T14 --threads 8 --seed 1234 --bs-trees 400  --bootstrap

cat T8.raxml.bootstraps T11.raxml.bootstraps T14.raxml.bootstraps > allbootstraps
raxml-ng --support --tree T3.raxml.bestTree --bs-trees allbootstraps --prefix T15 


```




```{r}
newtree <- '((((((12Z.muelleri_maker-6084_47488_1_44441--0.5-mRNA-1:0.009685,10Z.muelleri_maker-8016_47756--0.9-mRNA-1:0.026505)99:0.098293,13Z.marina_Zosma01g02330:0.113268)99:0.354433,(((11Z.muelleri_maker-8016_47756--0.8-mRNA-1:0.023273,9Z.muelleri_maker-2123_50249_30746_38748--0.4-mRNA-1:0.068317)99:0.153634,14Z.marina_Zosma01g02340:0.129762)100:0.616637,(1A.amphibolis_maker-scf7180000542070-augustus-gene-0.0-mRNA-1:0.518908,6P.australis_maker-scf7180003728898-augustus-gene-0.1-mRNA-1:0.074916)72:0.082830)22:0.047398)31:0.080205,((2O.sativa_LOC_Os03g33570.1:1.647537,(3O.sativa_LOC_Os02g18930.1:0.113637,4O.sativa_LOC_Os02g18880.1:0.035299)53:0.061341)66:0.111396,5O.sativa_LOC_Os05g45810.1:0.344545)75:0.137407)42:0.100482,7A.thaliana_AT5G24270.1:0.448151)83:0.209634,(15Z.marina_Zosma06g25260:0.152988,8Z.muelleri_snap_masked-14205_19045--0.3-mRNA-1:0.336420)100:0.314204,10P.australis_maker-scf7180004080271-augustus-gene-0.8-mRNA-1:0.580458)83:0.0;'

tree3 <- ape::read.tree(text=newtree, branch.label='support')

p3 <- ggtree(tree3) + geom_tiplab()
p3
```

OK the nodes are now named differently due to raxml-ng, time to fix again

```{r}
old_names <- tree3$tip.label

# [1] "3O.sativa"  "4O.sativa"  "5O.sativa"  "7A.thalian" "8Z.mueller" "15Z.marina" "10P.austra" "13Z.marina"
# [9] "12Z.muelle" "10Z.muelle" "14Z.marina" "9Z.mueller" "11Z.muelle" "1A.amphibo" "6P.austral" "2O.sativa"
#new_names  <- c( 'OsCBL8', 'OsCBL7.1', 'OsCBL4', 'AtSOS3', 'ZmuSOS3', 'ZmaSOS3', 'PaSOS3', 'ZmaSOS3.2',
#                 'ZmuSOS3.2', 'ZmuSOS3.3', 'ZmaSOS3.3', 'ZmuSOS3.2', 'ZmuSOS3.4', 'AaSOS3', 'PaSOS3.2', 'OsCBL7.2')
new_names  <- c('ZmuSOS3.2', 'ZmuSOS3.3', 'ZmaSOS3.2', 'ZmuSOS3.4', 'ZmuSOS3.5', 'ZmaSOS3.3','AaSOS3',  
                'PaSOS3.2', 'OsCBL7.2', 'OsCBL8', 'OsCBL7.1', 'OsCBL4', 'AtSOS3', 'ZmaSOS3', 'ZmuSOS3', 'PaSOS3')
rename_df <- data.frame(old = old_names, new = new_names)


tree3 <- rename_taxa(tree3, rename_df, old, new)
p3 <- ggtree(tree3) + geom_tiplab(fontface='italic')
p3 <- p3 + geom_nodelab(aes(subset=label>80), label='*')
p3+ ggplot2::xlim(0,2.5)


```

```{r}
muhtree <- ggtree(tree3)
my_data <- muhtree$data
my_data <- my_data %>% mutate(support = replace_na(as.numeric(label), 0))
root <- rootnode(tree3)  


good_p <- ggtree(tree3)  %<+% my_data  +
  geom_tiplab(fontface = 'italic') +
  geom_point2(aes(
    subset = !isTip & node != root,
    fill = cut(support, c(-1, 70, 90, 100))
  ),
  shape = 21,
  size = 4) +
  scale_fill_manual(
    values = c('#d95f02', '#7570b3', '#1b9e77'),
    labels = expression(BP < 70, 70 <= BP * " < 90", BP >= 90),
    name = 'Bootstrap Percentage',
    breaks = c('(-1,70]', '(70,90]', '(90,100]')
  ) +
  ggplot2::xlim(0, 2.5) +
  theme_tree(legend.position = 'bottom')
good_p
```

```{r}
cowplot::save_plot(good_p, filename = 'output/SOS3_better_phylogeny.png')
```


```{r fig.width = 10}
replace_vector <- new_names
names(replace_vector) <- old_names

data <- tidy_msa('data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.fasta', 221, 260)
data3 <- data %>% mutate(name = str_replace(name, ' ', '_'),
                          name = str_replace_all(name, replace_vector))
p3 + geom_facet(geom = geom_msa, data = data3,  panel = 'Multiple Sequence Alignment (221-260AA)',
               font = NULL, color = "Chemistry_AA") +
    xlim_tree(3)
```

```{r fig.width=10}
msaplot(p3, 'data/SOS3_OG0000189/OG0000189.RiceAraSeagrasses.aln.NamesFixed.fasta',
        offset = 0.5) + theme(legend.position='none')
```

## Phylogeny for the EIN3 cluster

First, to align the sequences again:

```
modeltest-ng reports JTT+I+G4+F

raxml-ng --msa OG0000629.RiceAraSeagrasses.aln.fa --model JTT+I+G4+F


 # to get fixed fasta - all good!!
raxml-ng --msa OG0000629.RiceAraSeagrasses.aln.fa --model JTT+I+G4+F --check

# to make a regular tree
raxml-ng --msa OG0000629.RiceAraSeagrasses.aln.fa --model JTT+I+G4+F --prefix T3 --threads 2 --seed 2 

# make 1000 bootstrap trees
raxml-ng --msa OG0000629.RiceAraSeagrasses.aln.fa --model JTT+I+G4+F --prefix T8 --threads 16 --seed 2 --bootstrap --bs-trees 1000 

# check whether they converge with <3% WRF cutoff
raxml-ng --bsconverge --bs-trees T8.raxml.bootstraps --prefix T12 --seed 2 --threads 1 --bs-cutoff 0.03

# it actually converged with 50 trees. haha.

# to make the final trees with bootstrap values

raxml-ng --support --tree T3.raxml.bestTree --bs-trees allbootstraps --prefix T13 
```

There we go:

```{r}
tree3 <- ape::read.tree(text='(((LOC_Os07g48630.1:0.641123,(LOC_Os03g20780.1:0.007239,LOC_Os03g20790.1:0.001048)100:0.214718)100:0.418001,(((Zosma02g24210:0.119273,(maker-1834_92013--0.52-mRNA-1:0.074816,augustus_masked-5412_21708--0.0-mRNA-1:0.082668)97:0.073791)100:0.640955,augustus_masked-scf7180003860583-processed-gene-0.3-mRNA-1:0.182720)89:0.114788,(maker-3356_51779--0.18-mRNA-1:0.661393,Zosma01g42670:0.320772)100:0.468870)67:0.098845)100:0.660268,AT3G20770.1:0.149857,AT2G27050.1:0.250165)100:0.0;', branch.label='support')

rename_df <- tibble::tribble(
                                                       ~old,        ~new,
                                         "LOC_Os07g48630.1",    "OsEIL2",
                                         "LOC_Os03g20780.1",    "OsEIL1",
                                         "LOC_Os03g20790.1",  "OsEIL1.2",
                                            "Zosma02g24210", "ZmaEIN3.1",
                            "maker-1834_92013--0.52-mRNA-1", "ZmuEIN3.1",
                   "augustus_masked-5412_21708--0.0-mRNA-1", "ZmuEIN3.2",
"augustus_masked-scf7180003860583-processed-gene-0.3-mRNA-1",   "PaEIN3",
                            "maker-3356_51779--0.18-mRNA-1", "ZmuEIN3.3",
                                            "Zosma01g42670", "ZmaEIN3.2",
                                              "AT3G20770.1",    "AtEIN3",
                                              "AT2G27050.1",    "AtEIL1"
               )

tree3 <- rename_taxa(tree3, rename_df, old, new)

p3 <-  ggtree(tree3) + geom_tiplab(fontface='italic')
p3 <- p3 + geom_nodelab()
p3 + xlim_tree(2.1)
```

```{r}
muhtree <- ggtree(tree3)
my_data <- muhtree$data
my_data <- my_data %>% mutate(support = replace_na(as.numeric(label), 0))
root <- rootnode(tree3)  


good_p <- ggtree(tree3)  %<+% my_data  +
  geom_tiplab(fontface = 'italic') +
  geom_point2(aes(
    subset = !isTip & node != root,
    fill = cut(support, c(-1, 70, 90, 100))
  ),
  shape = 21,
  size = 4) +
  scale_fill_manual(
    values = c('#d95f02', '#7570b3', '#1b9e77'),
    labels = expression(BP < 70, 70 <= BP * " < 90", BP >= 90),
    name = 'Bootstrap Percentage',
    breaks = c('(-1,70]', '(70,90]', '(90,100]')
  ) +
  ggplot2::xlim(0, 2.5) +
  theme_tree(legend.position = 'bottom')
good_p
```
```{r}
cowplot::save_plot(good_p, filename = 'output/EIN3_better_phylogeny.png')
```

```{r}
msaplot(p3, './data/EIN3_OG0000629/OG0000629.RiceAraSeagrasses.aln.NamesFixed.fa',
        offset = 0.5)  + theme(legend.position='none')

```

```{r}
domains <- tibble::tribble(
                                                                    ~Gene,     ~Pfam, ~Start, ~End,   ~Evalue,
                                                          "Zosma01g42670", "PF04873",     3L, 246L,  2.9e-117,
                                                            "AT2G27050.1", "PF04873",    49L, 298L,  2.6e-130,
                                                       "LOC_Os03g20780.1", "PF04873",    78L, 332L,  1.4e-132,
             "augustus_masked-scf7180003860583-processed-gene-0.3-mRNA-1", "PF04873",    76L, 325L,  1.9e-131,
                                          "maker-1834_92013--0.52-mRNA-1", "PF04873",   136L, 408L,    2e-123,
                                                       "LOC_Os07g48630.1", "PF04873",    81L, 338L,  3.7e-129,
                                                            "AT3G20770.1", "PF04873",    49L, 296L,  4.8e-129,
                                 "augustus_masked-5412_21708--0.0-mRNA-1", "PF04873",    16L, 285L,  2.2e-123,
                                          "maker-3356_51779--0.18-mRNA-1", "PF04873",    85L, 125L,     2e-12,
                                                       "LOC_Os03g20790.1", "PF04873",    78L, 332L,  1.4e-132,
                                                          "Zosma02g24210", "PF04873",     1L, 206L,  1.9e-113
             )
domains$length <- domains$End - domains$Start
replace_vector <- rename_df$new
names(replace_vector) <- rename_df$old
domains <- domains %>% mutate(Gene = str_replace_all(Gene, replace_vector))
domains
```

```{r}
g <- domains %>% left_join(rename_df, by=c('Gene'='new')) %>%  ggplot(aes(y = Gene, yend=Gene, x=Start,  xend=End,  color=Evalue)) + geom_segment(size=2) + theme_minimal() + xlab(expression(paste("Position of ", italic("EIN3"))))
g
```

```{r fig.width=10}
library(aplot)
g <- g + theme(axis.title.y=element_blank(),
               axis.text.y = element_blank(),
                legend.position='none')

g %>% insert_left(p3, width=2.5)
```

```{r fig.width=10}
library(patchwork)
# let's make a patchwork
# PROBLEM: patchwork does not automatically sort, so we'll have to sort the plot `g` manually 

plot_order <- p3[['data']] %>% filter(isTip == TRUE) %>% select(label, y) %>% arrange(desc(y))
g <- domains %>% left_join(rename_df, by=c('Gene'='new')) %>%  ggplot(aes(y = factor(Gene, levels = plot_order$label), yend=Gene, x=Start,  xend=End,  color=Evalue)) + geom_segment(size=2) + theme_minimal() + xlab(expression(paste("Position of ", italic("EIN3"))))+ theme(axis.title.y=element_blank(),
               axis.text.y = element_blank(),
                legend.position='none')
g

msaplot(p3, './data/EIN3_OG0000629/OG0000629.RiceAraSeagrasses.aln.NamesFixed.fa',
        offset = 0.5)  + theme(legend.position='none') + g  + 
  plot_layout(widths = c(3.5, 1)) 
```

